<html><head><meta content="text/html; charset=UTF-8" http-equiv="content-type"><style type="text/css">ol.lst-kix_kfepckt4l6hg-1.start{counter-reset:lst-ctn-kix_kfepckt4l6hg-1 0}.lst-kix_lkbbmj5sol47-5>li:before{content:"\0025a0   "}.lst-kix_lkbbmj5sol47-7>li:before{content:"\0025cb   "}.lst-kix_lcass77ih41v-0>li:before{content:"\0025cf   "}.lst-kix_uilbxigghq1p-7>li:before{content:"\0025cb   "}.lst-kix_rggdlfkt6mfq-5>li:before{content:"" counter(lst-ctn-kix_rggdlfkt6mfq-5,lower-roman) ". "}.lst-kix_rggdlfkt6mfq-3>li:before{content:"" counter(lst-ctn-kix_rggdlfkt6mfq-3,decimal) ". "}.lst-kix_lcass77ih41v-2>li:before{content:"\0025a0   "}.lst-kix_lcass77ih41v-4>li:before{content:"\0025cb   "}.lst-kix_uilbxigghq1p-5>li:before{content:"\0025a0   "}.lst-kix_lcass77ih41v-6>li:before{content:"\0025cf   "}.lst-kix_lcass77ih41v-8>li:before{content:"\0025a0   "}.lst-kix_uilbxigghq1p-1>li:before{content:"\0025cb   "}ul.lst-kix_8r3w2rcyu87a-8{list-style-type:none}ol.lst-kix_rggdlfkt6mfq-0.start{counter-reset:lst-ctn-kix_rggdlfkt6mfq-0 0}ul.lst-kix_8r3w2rcyu87a-6{list-style-type:none}ol.lst-kix_dgmlm9mxaa4-2.start{counter-reset:lst-ctn-kix_dgmlm9mxaa4-2 0}ul.lst-kix_8r3w2rcyu87a-7{list-style-type:none}ul.lst-kix_8r3w2rcyu87a-4{list-style-type:none}ul.lst-kix_8r3w2rcyu87a-5{list-style-type:none}.lst-kix_uilbxigghq1p-3>li:before{content:"\0025cf   "}ul.lst-kix_8r3w2rcyu87a-2{list-style-type:none}.lst-kix_dgmlm9mxaa4-8>li{counter-increment:lst-ctn-kix_dgmlm9mxaa4-8}ul.lst-kix_8r3w2rcyu87a-3{list-style-type:none}ul.lst-kix_8r3w2rcyu87a-0{list-style-type:none}.lst-kix_rggdlfkt6mfq-7>li:before{content:"" counter(lst-ctn-kix_rggdlfkt6mfq-7,lower-latin) ". "}ul.lst-kix_8r3w2rcyu87a-1{list-style-type:none}ol.lst-kix_dgmlm9mxaa4-7.start{counter-reset:lst-ctn-kix_dgmlm9mxaa4-7 0}.lst-kix_rggdlfkt6mfq-1>li:before{content:"" counter(lst-ctn-kix_rggdlfkt6mfq-1,lower-latin) ". "}.lst-kix_3qoh0wbylc0j-8>li:before{content:"" counter(lst-ctn-kix_3qoh0wbylc0j-8,lower-roman) ". "}.lst-kix_xmm4jxv5hwlq-8>li:before{content:"\0025a0   "}.lst-kix_3qoh0wbylc0j-6>li:before{content:"" counter(lst-ctn-kix_3qoh0wbylc0j-6,decimal) ". "}.lst-kix_3qoh0wbylc0j-4>li:before{content:"" counter(lst-ctn-kix_3qoh0wbylc0j-4,lower-latin) ". "}.lst-kix_lkbbmj5sol47-3>li:before{content:"\0025cf   "}.lst-kix_3qoh0wbylc0j-0>li:before{content:"" counter(lst-ctn-kix_3qoh0wbylc0j-0,decimal) ". "}.lst-kix_xmm4jxv5hwlq-4>li:before{content:"\0025cb   "}.lst-kix_3qoh0wbylc0j-2>li:before{content:"" counter(lst-ctn-kix_3qoh0wbylc0j-2,lower-roman) ". "}.lst-kix_xmm4jxv5hwlq-6>li:before{content:"\0025cf   "}ol.lst-kix_rggdlfkt6mfq-5.start{counter-reset:lst-ctn-kix_rggdlfkt6mfq-5 0}.lst-kix_lkbbmj5sol47-1>li:before{content:"\0025cb   "}.lst-kix_mktumxqlukrw-3>li:before{content:"\0025cf   "}.lst-kix_gp2qqn5s6so6-7>li:before{content:"\0025cb   "}ul.lst-kix_kruy5m9jmo50-8{list-style-type:none}.lst-kix_tkg3l6cy3bnc-0>li:before{content:"" counter(lst-ctn-kix_tkg3l6cy3bnc-0,decimal) ". "}.lst-kix_tkg3l6cy3bnc-2>li:before{content:"" counter(lst-ctn-kix_tkg3l6cy3bnc-2,lower-roman) ". "}.lst-kix_gp2qqn5s6so6-5>li:before{content:"\0025a0   "}ul.lst-kix_kruy5m9jmo50-2{list-style-type:none}ul.lst-kix_kruy5m9jmo50-3{list-style-type:none}.lst-kix_mktumxqlukrw-7>li:before{content:"\0025cb   "}.lst-kix_gp2qqn5s6so6-3>li:before{content:"\0025cf   "}.lst-kix_ytu3x5e7mxt6-4>li{counter-increment:lst-ctn-kix_ytu3x5e7mxt6-4}ul.lst-kix_kruy5m9jmo50-0{list-style-type:none}ul.lst-kix_kruy5m9jmo50-1{list-style-type:none}ul.lst-kix_kruy5m9jmo50-6{list-style-type:none}ul.lst-kix_kruy5m9jmo50-7{list-style-type:none}.lst-kix_mktumxqlukrw-5>li:before{content:"\0025a0   "}ul.lst-kix_kruy5m9jmo50-4{list-style-type:none}ul.lst-kix_kruy5m9jmo50-5{list-style-type:none}.lst-kix_xmm4jxv5hwlq-2>li:before{content:"\0025a0   "}ol.lst-kix_zqq05ofg66-6.start{counter-reset:lst-ctn-kix_zqq05ofg66-6 0}.lst-kix_xmm4jxv5hwlq-0>li:before{content:"\0025cf   "}.lst-kix_gp2qqn5s6so6-1>li:before{content:"\0025cb   "}.lst-kix_mktumxqlukrw-1>li:before{content:"\0025cb   "}ol.lst-kix_8joc3jsoa7de-2.start{counter-reset:lst-ctn-kix_8joc3jsoa7de-2 0}.lst-kix_h8tfxvbo4u2a-8>li:before{content:"\0025a0   "}.lst-kix_h8tfxvbo4u2a-2>li:before{content:"\0025a0   "}.lst-kix_esoh3w8sgdnn-8>li:before{content:"" counter(lst-ctn-kix_esoh3w8sgdnn-8,lower-roman) ". "}.lst-kix_tkg3l6cy3bnc-8>li:before{content:"" counter(lst-ctn-kix_tkg3l6cy3bnc-8,lower-roman) ". "}.lst-kix_h8tfxvbo4u2a-4>li:before{content:"\0025cb   "}.lst-kix_tkg3l6cy3bnc-4>li:before{content:"" counter(lst-ctn-kix_tkg3l6cy3bnc-4,lower-latin) ". "}.lst-kix_tkg3l6cy3bnc-6>li:before{content:"" counter(lst-ctn-kix_tkg3l6cy3bnc-6,decimal) ". "}.lst-kix_esoh3w8sgdnn-6>li:before{content:"" counter(lst-ctn-kix_esoh3w8sgdnn-6,decimal) ". "}.lst-kix_h8tfxvbo4u2a-6>li:before{content:"\0025cf   "}.lst-kix_fd9jvhtv6teb-5>li:before{content:"\0025a0   "}ul.lst-kix_m75akhxjyj9r-3{list-style-type:none}ul.lst-kix_m75akhxjyj9r-2{list-style-type:none}.lst-kix_esoh3w8sgdnn-0>li:before{content:"" counter(lst-ctn-kix_esoh3w8sgdnn-0,decimal) ". "}ul.lst-kix_m75akhxjyj9r-5{list-style-type:none}ul.lst-kix_m75akhxjyj9r-4{list-style-type:none}ul.lst-kix_m75akhxjyj9r-7{list-style-type:none}.lst-kix_blb26b155eyp-1>li:before{content:"\0025cb   "}ul.lst-kix_m75akhxjyj9r-6{list-style-type:none}.lst-kix_tkg3l6cy3bnc-6>li{counter-increment:lst-ctn-kix_tkg3l6cy3bnc-6}ul.lst-kix_m75akhxjyj9r-8{list-style-type:none}.lst-kix_fd9jvhtv6teb-1>li:before{content:"\0025cb   "}ol.lst-kix_kfepckt4l6hg-6.start{counter-reset:lst-ctn-kix_kfepckt4l6hg-6 0}.lst-kix_esoh3w8sgdnn-4>li:before{content:"" counter(lst-ctn-kix_esoh3w8sgdnn-4,lower-latin) ". "}.lst-kix_blb26b155eyp-3>li:before{content:"\0025cf   "}.lst-kix_fd9jvhtv6teb-7>li:before{content:"\0025cb   "}.lst-kix_esoh3w8sgdnn-2>li:before{content:"" counter(lst-ctn-kix_esoh3w8sgdnn-2,lower-roman) ". "}.lst-kix_blb26b155eyp-5>li:before{content:"\0025a0   "}.lst-kix_blb26b155eyp-7>li:before{content:"\0025cb   "}.lst-kix_fd9jvhtv6teb-3>li:before{content:"\0025cf   "}ul.lst-kix_m75akhxjyj9r-1{list-style-type:none}ul.lst-kix_m75akhxjyj9r-0{list-style-type:none}.lst-kix_90vpss294crz-1>li:before{content:"\0025cb   "}.lst-kix_90vpss294crz-3>li:before{content:"\0025cf   "}.lst-kix_90vpss294crz-5>li:before{content:"\0025a0   "}.lst-kix_90vpss294crz-7>li:before{content:"\0025cb   "}.lst-kix_lsygr38j1y0-2>li:before{content:"\0025a0   "}ul.lst-kix_ie2cjnul3ua-2{list-style-type:none}ul.lst-kix_ie2cjnul3ua-3{list-style-type:none}ul.lst-kix_ie2cjnul3ua-4{list-style-type:none}.lst-kix_4864lfd9zdwy-7>li:before{content:"\0025cb   "}ul.lst-kix_ie2cjnul3ua-5{list-style-type:none}ul.lst-kix_ie2cjnul3ua-6{list-style-type:none}ul.lst-kix_ie2cjnul3ua-7{list-style-type:none}ul.lst-kix_ie2cjnul3ua-8{list-style-type:none}.lst-kix_n23m8p7c7hw6-1>li:before{content:"\0025cb   "}.lst-kix_lsygr38j1y0-6>li:before{content:"\0025cf   "}ul.lst-kix_pb595s4qvcov-2{list-style-type:none}ul.lst-kix_pb595s4qvcov-3{list-style-type:none}ul.lst-kix_pb595s4qvcov-0{list-style-type:none}ul.lst-kix_pb595s4qvcov-1{list-style-type:none}.lst-kix_n23m8p7c7hw6-5>li:before{content:"\0025a0   "}.lst-kix_sbk9fqxb69uy-2>li{counter-increment:lst-ctn-kix_sbk9fqxb69uy-2}ul.lst-kix_pb595s4qvcov-8{list-style-type:none}.lst-kix_4864lfd9zdwy-3>li:before{content:"\0025cf   "}ul.lst-kix_pb595s4qvcov-6{list-style-type:none}ul.lst-kix_pb595s4qvcov-7{list-style-type:none}ul.lst-kix_pb595s4qvcov-4{list-style-type:none}ul.lst-kix_pb595s4qvcov-5{list-style-type:none}ol.lst-kix_3qoh0wbylc0j-1.start{counter-reset:lst-ctn-kix_3qoh0wbylc0j-1 0}.lst-kix_kruy5m9jmo50-1>li:before{content:"\0025cb   "}.lst-kix_3qoh0wbylc0j-1>li{counter-increment:lst-ctn-kix_3qoh0wbylc0j-1}.lst-kix_rggdlfkt6mfq-1>li{counter-increment:lst-ctn-kix_rggdlfkt6mfq-1}.lst-kix_kruy5m9jmo50-5>li:before{content:"\0025a0   "}.lst-kix_z4fu53gdnrww-7>li:before{content:"\0025cb   "}.lst-kix_q432akg4led-1>li:before{content:"\0025cb   "}.lst-kix_sbk9fqxb69uy-0>li{counter-increment:lst-ctn-kix_sbk9fqxb69uy-0}.lst-kix_z4fu53gdnrww-3>li:before{content:"\0025cf   "}.lst-kix_tj1ix0gi2ui9-2>li:before{content:"\0025a0   "}ul.lst-kix_7g6nqlfa7rn7-8{list-style-type:none}ul.lst-kix_7g6nqlfa7rn7-7{list-style-type:none}ul.lst-kix_7g6nqlfa7rn7-4{list-style-type:none}.lst-kix_tj1ix0gi2ui9-6>li:before{content:"\0025cf   "}ul.lst-kix_7g6nqlfa7rn7-3{list-style-type:none}.lst-kix_8luyzq1gqi6-3>li:before{content:"\0025cf   "}ul.lst-kix_7g6nqlfa7rn7-6{list-style-type:none}ul.lst-kix_7g6nqlfa7rn7-5{list-style-type:none}ul.lst-kix_7g6nqlfa7rn7-0{list-style-type:none}.lst-kix_t27vnxuzkttq-7>li:before{content:"\0025cb   "}.lst-kix_auto7t5cf4tl-1>li:before{content:"\0025cb   "}ul.lst-kix_7g6nqlfa7rn7-2{list-style-type:none}ul.lst-kix_7g6nqlfa7rn7-1{list-style-type:none}.lst-kix_3qoh0wbylc0j-3>li{counter-increment:lst-ctn-kix_3qoh0wbylc0j-3}.lst-kix_8luyzq1gqi6-7>li:before{content:"\0025cb   "}ol.lst-kix_esoh3w8sgdnn-5.start{counter-reset:lst-ctn-kix_esoh3w8sgdnn-5 0}ol.lst-kix_awa32z9rxqor-0.start{counter-reset:lst-ctn-kix_awa32z9rxqor-0 0}.lst-kix_x3csall36p7p-2>li:before{content:"\0025a0   "}.lst-kix_q432akg4led-5>li:before{content:"\0025a0   "}.lst-kix_v0sksjnof0cj-6>li:before{content:"\0025cf   "}.lst-kix_p73ghpm47z3w-2>li:before{content:"\0025a0   "}.lst-kix_an05xfuo7mar-6>li:before{content:"\0025cf   "}.lst-kix_p73ghpm47z3w-6>li:before{content:"\0025cf   "}ol.lst-kix_ytu3x5e7mxt6-6.start{counter-reset:lst-ctn-kix_ytu3x5e7mxt6-6 0}.lst-kix_zqq05ofg66-3>li{counter-increment:lst-ctn-kix_zqq05ofg66-3}.lst-kix_sbk9fqxb69uy-7>li:before{content:"" counter(lst-ctn-kix_sbk9fqxb69uy-7,lower-latin) ". "}.lst-kix_haoc7b4kfhm3-1>li:before{content:"\0025cb   "}.lst-kix_an05xfuo7mar-2>li:before{content:"\0025a0   "}.lst-kix_ie2cjnul3ua-2>li:before{content:"\0025a0   "}.lst-kix_ht2xpfpspp5-6>li:before{content:"\0025cf   "}.lst-kix_sbk9fqxb69uy-3>li:before{content:"" counter(lst-ctn-kix_sbk9fqxb69uy-3,decimal) ". "}.lst-kix_t27vnxuzkttq-3>li:before{content:"\0025cf   "}.lst-kix_auto7t5cf4tl-5>li:before{content:"\0025a0   "}.lst-kix_haoc7b4kfhm3-5>li:before{content:"\0025a0   "}ul.lst-kix_ie2cjnul3ua-0{list-style-type:none}ul.lst-kix_ie2cjnul3ua-1{list-style-type:none}ol.lst-kix_eag5a795mtbr-1.start{counter-reset:lst-ctn-kix_eag5a795mtbr-1 0}.lst-kix_rggdlfkt6mfq-8>li{counter-increment:lst-ctn-kix_rggdlfkt6mfq-8}.lst-kix_ie2cjnul3ua-6>li:before{content:"\0025cf   "}.lst-kix_ht2xpfpspp5-2>li:before{content:"\0025a0   "}.lst-kix_zqq05ofg66-4>li:before{content:"" counter(lst-ctn-kix_zqq05ofg66-4,lower-latin) ". "}.lst-kix_zqq05ofg66-8>li:before{content:"" counter(lst-ctn-kix_zqq05ofg66-8,lower-roman) ". "}.lst-kix_b01r1xbffk2a-4>li:before{content:"\0025cb   "}.lst-kix_z1b7uf8mlquv-1>li:before{content:"\0025cb   "}.lst-kix_tkg3l6cy3bnc-8>li{counter-increment:lst-ctn-kix_tkg3l6cy3bnc-8}.lst-kix_z1b7uf8mlquv-5>li:before{content:"\0025a0   "}.lst-kix_b01r1xbffk2a-8>li:before{content:"\0025a0   "}ol.lst-kix_8joc3jsoa7de-7.start{counter-reset:lst-ctn-kix_8joc3jsoa7de-7 0}.lst-kix_kfepckt4l6hg-6>li{counter-increment:lst-ctn-kix_kfepckt4l6hg-6}.lst-kix_6z3d30mc5xph-1>li:before{content:"\0025cb   "}.lst-kix_6z3d30mc5xph-5>li:before{content:"\0025a0   "}ul.lst-kix_565pscofcl0-3{list-style-type:none}ul.lst-kix_565pscofcl0-4{list-style-type:none}ul.lst-kix_565pscofcl0-1{list-style-type:none}ul.lst-kix_565pscofcl0-2{list-style-type:none}.lst-kix_wje7sh1pcpc1-2>li:before{content:"\0025a0   "}.lst-kix_iprbjac0k1x3-0>li:before{content:"\0025cf   "}.lst-kix_iprbjac0k1x3-8>li:before{content:"\0025a0   "}ul.lst-kix_565pscofcl0-0{list-style-type:none}ol.lst-kix_ytu3x5e7mxt6-1.start{counter-reset:lst-ctn-kix_ytu3x5e7mxt6-1 0}.lst-kix_v0sksjnof0cj-2>li:before{content:"\0025a0   "}.lst-kix_wje7sh1pcpc1-6>li:before{content:"\0025cf   "}ul.lst-kix_565pscofcl0-7{list-style-type:none}ul.lst-kix_565pscofcl0-8{list-style-type:none}.lst-kix_zqq05ofg66-0>li:before{content:"" counter(lst-ctn-kix_zqq05ofg66-0,decimal) ". "}ul.lst-kix_565pscofcl0-5{list-style-type:none}ul.lst-kix_565pscofcl0-6{list-style-type:none}.lst-kix_iprbjac0k1x3-4>li:before{content:"\0025cb   "}.lst-kix_s4u6wv582l9n-1>li:before{content:"\0025cb   "}ul.lst-kix_5mkgtms2qzgk-8{list-style-type:none}.lst-kix_3qoh0wbylc0j-8>li{counter-increment:lst-ctn-kix_3qoh0wbylc0j-8}.lst-kix_kfepckt4l6hg-0>li:before{content:"" counter(lst-ctn-kix_kfepckt4l6hg-0,decimal) ". "}.lst-kix_kfepckt4l6hg-4>li:before{content:"" counter(lst-ctn-kix_kfepckt4l6hg-4,lower-latin) ". "}.lst-kix_b0v72ufkqzg3-2>li:before{content:"\0025a0   "}.lst-kix_b0v72ufkqzg3-6>li:before{content:"\0025cf   "}.lst-kix_s4u6wv582l9n-5>li:before{content:"\0025a0   "}.lst-kix_zqq05ofg66-5>li{counter-increment:lst-ctn-kix_zqq05ofg66-5}.lst-kix_ftoogxlvmqxr-5>li:before{content:"\0025a0   "}.lst-kix_kfepckt4l6hg-8>li:before{content:"" counter(lst-ctn-kix_kfepckt4l6hg-8,lower-roman) ". "}.lst-kix_8r3w2rcyu87a-6>li:before{content:"\0025cf   "}.lst-kix_umwvyipzf505-2>li:before{content:"\0025a0   "}.lst-kix_umwvyipzf505-6>li:before{content:"\0025cf   "}.lst-kix_vams35lnoe4-8>li:before{content:"\0025a0   "}.lst-kix_ftoogxlvmqxr-1>li:before{content:"\0025cb   "}.lst-kix_j5h2oeedi37x-0>li:before{content:"\0025cf   "}.lst-kix_8r3w2rcyu87a-2>li:before{content:"\0025a0   "}.lst-kix_6ay9chlfxidr-5>li:before{content:"\0025a0   "}.lst-kix_esoh3w8sgdnn-6>li{counter-increment:lst-ctn-kix_esoh3w8sgdnn-6}.lst-kix_rggdlfkt6mfq-3>li{counter-increment:lst-ctn-kix_rggdlfkt6mfq-3}ol.lst-kix_3qoh0wbylc0j-6.start{counter-reset:lst-ctn-kix_3qoh0wbylc0j-6 0}.lst-kix_vams35lnoe4-4>li:before{content:"\0025cb   "}.lst-kix_5mkgtms2qzgk-3>li:before{content:"\0025cf   "}ul.lst-kix_eb0mkv6he1ge-8{list-style-type:none}.lst-kix_j5h2oeedi37x-8>li:before{content:"\0025a0   "}.lst-kix_6ay9chlfxidr-1>li:before{content:"\0025cb   "}ul.lst-kix_eb0mkv6he1ge-1{list-style-type:none}ul.lst-kix_eb0mkv6he1ge-0{list-style-type:none}ul.lst-kix_eb0mkv6he1ge-3{list-style-type:none}.lst-kix_j5h2oeedi37x-4>li:before{content:"\0025cb   "}ul.lst-kix_eb0mkv6he1ge-2{list-style-type:none}ul.lst-kix_eb0mkv6he1ge-5{list-style-type:none}.lst-kix_b01r1xbffk2a-0>li:before{content:"\0025cf   "}ul.lst-kix_eb0mkv6he1ge-4{list-style-type:none}.lst-kix_vams35lnoe4-0>li:before{content:"\0025cf   "}ul.lst-kix_eb0mkv6he1ge-7{list-style-type:none}.lst-kix_5mkgtms2qzgk-7>li:before{content:"\0025cb   "}ul.lst-kix_eb0mkv6he1ge-6{list-style-type:none}.lst-kix_2najoyx65arn-7>li:before{content:"\0025cb   "}.lst-kix_jyqknjqk01cc-3>li:before{content:"\0025cf   "}.lst-kix_xq7wzd2lr8u5-5>li:before{content:"\0025a0   "}.lst-kix_i5y1isusbf26-4>li:before{content:"\0025cb   "}.lst-kix_2fszuplxt2n9-5>li:before{content:"\0025a0   "}ul.lst-kix_q432akg4led-8{list-style-type:none}ul.lst-kix_q432akg4led-7{list-style-type:none}.lst-kix_xq7wzd2lr8u5-3>li:before{content:"\0025cf   "}ol.lst-kix_sbk9fqxb69uy-6.start{counter-reset:lst-ctn-kix_sbk9fqxb69uy-6 0}.lst-kix_jyqknjqk01cc-1>li:before{content:"\0025cb   "}.lst-kix_cwtqd0lozkzu-3>li:before{content:"\0025cf   "}.lst-kix_6ay9chlfxidr-7>li:before{content:"\0025cb   "}.lst-kix_nqcns3qrnch2-6>li:before{content:"\0025cf   "}.lst-kix_73hytacj4fgx-2>li:before{content:"\0025a0   "}ol.lst-kix_tkg3l6cy3bnc-7.start{counter-reset:lst-ctn-kix_tkg3l6cy3bnc-7 0}.lst-kix_yxgmy3rkq7bq-6>li:before{content:"\0025cf   "}.lst-kix_yxgmy3rkq7bq-8>li:before{content:"\0025a0   "}.lst-kix_73hytacj4fgx-0>li:before{content:"\0025cf   "}.lst-kix_6k838qjxgg72-2>li:before{content:"\0025a0   "}ul.lst-kix_k38n94udk0el-0{list-style-type:none}ul.lst-kix_k38n94udk0el-3{list-style-type:none}ul.lst-kix_3aierr3stws2-7{list-style-type:none}ul.lst-kix_ss89m2334v3w-7{list-style-type:none}.lst-kix_85agg0xrrcik-4>li:before{content:"\0025cb   "}ul.lst-kix_k38n94udk0el-4{list-style-type:none}ul.lst-kix_3aierr3stws2-8{list-style-type:none}ul.lst-kix_ss89m2334v3w-8{list-style-type:none}ul.lst-kix_k38n94udk0el-1{list-style-type:none}ul.lst-kix_3aierr3stws2-5{list-style-type:none}ul.lst-kix_ss89m2334v3w-5{list-style-type:none}ul.lst-kix_k38n94udk0el-2{list-style-type:none}ul.lst-kix_3aierr3stws2-6{list-style-type:none}ul.lst-kix_ss89m2334v3w-6{list-style-type:none}ul.lst-kix_k38n94udk0el-7{list-style-type:none}.lst-kix_85agg0xrrcik-2>li:before{content:"\0025a0   "}ul.lst-kix_k38n94udk0el-8{list-style-type:none}ul.lst-kix_k38n94udk0el-5{list-style-type:none}ul.lst-kix_k38n94udk0el-6{list-style-type:none}.lst-kix_19nen5usjyzs-6>li:before{content:"\0025cf   "}.lst-kix_19nen5usjyzs-8>li:before{content:"\0025a0   "}.lst-kix_d7lufd20rbm5-0>li:before{content:"\0025cf   "}.lst-kix_kfepckt4l6hg-1>li{counter-increment:lst-ctn-kix_kfepckt4l6hg-1}ul.lst-kix_3aierr3stws2-3{list-style-type:none}ul.lst-kix_ss89m2334v3w-3{list-style-type:none}ul.lst-kix_3aierr3stws2-4{list-style-type:none}ul.lst-kix_ss89m2334v3w-4{list-style-type:none}ul.lst-kix_3aierr3stws2-1{list-style-type:none}ul.lst-kix_ss89m2334v3w-1{list-style-type:none}ul.lst-kix_3aierr3stws2-2{list-style-type:none}ul.lst-kix_ss89m2334v3w-2{list-style-type:none}.lst-kix_6k838qjxgg72-8>li:before{content:"\0025a0   "}ul.lst-kix_3aierr3stws2-0{list-style-type:none}.lst-kix_nqcns3qrnch2-4>li:before{content:"\0025cb   "}ul.lst-kix_ss89m2334v3w-0{list-style-type:none}.lst-kix_yxgmy3rkq7bq-0>li:before{content:"\0025cf   "}.lst-kix_73hytacj4fgx-8>li:before{content:"\0025a0   "}.lst-kix_on63e6od9ck4-0>li:before{content:"\0025cf   "}.lst-kix_74bj55g8eq2b-0>li:before{content:"\0025cf   "}.lst-kix_2ws5k2m4lsl8-7>li:before{content:"\0025cb   "}.lst-kix_3aierr3stws2-6>li:before{content:"\0025cf   "}.lst-kix_cwtqd0lozkzu-1>li:before{content:"\0025cb   "}.lst-kix_ytu3x5e7mxt6-1>li:before{content:"" counter(lst-ctn-kix_ytu3x5e7mxt6-1,lower-latin) ". "}.lst-kix_2ws5k2m4lsl8-5>li:before{content:"\0025a0   "}.lst-kix_on63e6od9ck4-2>li:before{content:"\0025a0   "}.lst-kix_ytu3x5e7mxt6-3>li:before{content:"" counter(lst-ctn-kix_ytu3x5e7mxt6-3,decimal) ". "}.lst-kix_9ei06drf1sg0-4>li:before{content:"\0025cb   "}.lst-kix_on63e6od9ck4-8>li:before{content:"\0025a0   "}.lst-kix_74bj55g8eq2b-8>li:before{content:"\0025a0   "}ul.lst-kix_2tnhjv96pndg-0{list-style-type:none}.lst-kix_2fszuplxt2n9-3>li:before{content:"\0025cf   "}ul.lst-kix_2tnhjv96pndg-2{list-style-type:none}ul.lst-kix_2tnhjv96pndg-1{list-style-type:none}ul.lst-kix_2tnhjv96pndg-4{list-style-type:none}.lst-kix_wk2xuqsimxh-4>li:before{content:"\0025cb   "}ul.lst-kix_2tnhjv96pndg-3{list-style-type:none}ul.lst-kix_2tnhjv96pndg-6{list-style-type:none}ul.lst-kix_2tnhjv96pndg-5{list-style-type:none}.lst-kix_9ei06drf1sg0-6>li:before{content:"\0025cf   "}.lst-kix_74bj55g8eq2b-6>li:before{content:"\0025cf   "}ul.lst-kix_2tnhjv96pndg-8{list-style-type:none}.lst-kix_6k838qjxgg72-0>li:before{content:"\0025cf   "}.lst-kix_wk2xuqsimxh-2>li:before{content:"\0025a0   "}ul.lst-kix_2tnhjv96pndg-7{list-style-type:none}ul.lst-kix_lcass77ih41v-7{list-style-type:none}ul.lst-kix_lcass77ih41v-8{list-style-type:none}ul.lst-kix_lcass77ih41v-5{list-style-type:none}ul.lst-kix_lcass77ih41v-6{list-style-type:none}ul.lst-kix_lcass77ih41v-3{list-style-type:none}ul.lst-kix_lcass77ih41v-4{list-style-type:none}ul.lst-kix_lcass77ih41v-1{list-style-type:none}ul.lst-kix_lcass77ih41v-2{list-style-type:none}.lst-kix_etn6l1m36dy-0>li:before{content:"\0025cf   "}.lst-kix_upyk6it0f1tq-5>li:before{content:"\0025a0   "}ol.lst-kix_awa32z9rxqor-5.start{counter-reset:lst-ctn-kix_awa32z9rxqor-5 0}.lst-kix_m75akhxjyj9r-2>li:before{content:"\0025a0   "}.lst-kix_upyk6it0f1tq-3>li:before{content:"\0025cf   "}.lst-kix_vqsmxaeqhsi-4>li:before{content:"\0025cb   "}ul.lst-kix_lcass77ih41v-0{list-style-type:none}.lst-kix_3aierr3stws2-4>li:before{content:"\0025cb   "}ol.lst-kix_esoh3w8sgdnn-0{list-style-type:none}ol.lst-kix_esoh3w8sgdnn-3{list-style-type:none}.lst-kix_vqsmxaeqhsi-6>li:before{content:"\0025cf   "}ol.lst-kix_esoh3w8sgdnn-2{list-style-type:none}.lst-kix_sp4bx5if0jbk-8>li:before{content:"\0025a0   "}ol.lst-kix_esoh3w8sgdnn-5{list-style-type:none}.lst-kix_x3csall36p7p-6>li:before{content:"\0025cf   "}ol.lst-kix_esoh3w8sgdnn-4{list-style-type:none}ol.lst-kix_esoh3w8sgdnn-7{list-style-type:none}.lst-kix_d7lufd20rbm5-6>li:before{content:"\0025cf   "}.lst-kix_d7lufd20rbm5-8>li:before{content:"\0025a0   "}ol.lst-kix_esoh3w8sgdnn-6{list-style-type:none}.lst-kix_sr3soljrf7bf-7>li:before{content:"\0025cb   "}.lst-kix_x3csall36p7p-8>li:before{content:"\0025a0   "}ol.lst-kix_esoh3w8sgdnn-8{list-style-type:none}.lst-kix_xq8kn6xuala3-2>li:before{content:"\0025a0   "}.lst-kix_sr3soljrf7bf-1>li:before{content:"\0025cb   "}ol.lst-kix_8joc3jsoa7de-4.start{counter-reset:lst-ctn-kix_8joc3jsoa7de-4 0}.lst-kix_etn6l1m36dy-8>li:before{content:"\0025a0   "}.lst-kix_xq8kn6xuala3-4>li:before{content:"\0025cb   "}.lst-kix_m75akhxjyj9r-0>li:before{content:"\0025cf   "}.lst-kix_5s9vwy7hbq8r-7>li:before{content:"\0025cb   "}.lst-kix_d8i9v1tvihft-3>li:before{content:"\0025cf   "}.lst-kix_etn6l1m36dy-2>li:before{content:"\0025a0   "}ul.lst-kix_1qqoijhe7jmx-4{list-style-type:none}ol.lst-kix_awa32z9rxqor-8{list-style-type:none}ul.lst-kix_1qqoijhe7jmx-3{list-style-type:none}.lst-kix_5s9vwy7hbq8r-5>li:before{content:"\0025a0   "}ol.lst-kix_awa32z9rxqor-7{list-style-type:none}ul.lst-kix_1qqoijhe7jmx-6{list-style-type:none}ol.lst-kix_awa32z9rxqor-6{list-style-type:none}ul.lst-kix_1qqoijhe7jmx-5{list-style-type:none}ol.lst-kix_awa32z9rxqor-5{list-style-type:none}ul.lst-kix_1qqoijhe7jmx-0{list-style-type:none}ol.lst-kix_awa32z9rxqor-4{list-style-type:none}ol.lst-kix_awa32z9rxqor-3{list-style-type:none}ul.lst-kix_1qqoijhe7jmx-2{list-style-type:none}ol.lst-kix_awa32z9rxqor-2{list-style-type:none}.lst-kix_d8i9v1tvihft-5>li:before{content:"\0025a0   "}ul.lst-kix_1qqoijhe7jmx-1{list-style-type:none}.lst-kix_19nen5usjyzs-0>li:before{content:"\0025cf   "}ol.lst-kix_awa32z9rxqor-0{list-style-type:none}ul.lst-kix_1qqoijhe7jmx-8{list-style-type:none}.lst-kix_e2h0cgtfdf3g-0>li:before{content:"\0025cf   "}ul.lst-kix_1qqoijhe7jmx-7{list-style-type:none}ol.lst-kix_esoh3w8sgdnn-2.start{counter-reset:lst-ctn-kix_esoh3w8sgdnn-2 0}ul.lst-kix_t27vnxuzkttq-8{list-style-type:none}ul.lst-kix_t27vnxuzkttq-7{list-style-type:none}ul.lst-kix_t27vnxuzkttq-6{list-style-type:none}ul.lst-kix_t27vnxuzkttq-5{list-style-type:none}ul.lst-kix_t27vnxuzkttq-4{list-style-type:none}ul.lst-kix_t27vnxuzkttq-3{list-style-type:none}ul.lst-kix_t27vnxuzkttq-2{list-style-type:none}ul.lst-kix_t27vnxuzkttq-1{list-style-type:none}ul.lst-kix_t27vnxuzkttq-0{list-style-type:none}ul.lst-kix_6kne4t6ii93d-8{list-style-type:none}ul.lst-kix_6kne4t6ii93d-7{list-style-type:none}ul.lst-kix_6kne4t6ii93d-6{list-style-type:none}ul.lst-kix_6kne4t6ii93d-5{list-style-type:none}ul.lst-kix_xmm4jxv5hwlq-1{list-style-type:none}ul.lst-kix_6kne4t6ii93d-4{list-style-type:none}.lst-kix_kfepckt4l6hg-8>li{counter-increment:lst-ctn-kix_kfepckt4l6hg-8}ul.lst-kix_xmm4jxv5hwlq-0{list-style-type:none}ul.lst-kix_6kne4t6ii93d-3{list-style-type:none}ul.lst-kix_xmm4jxv5hwlq-3{list-style-type:none}ul.lst-kix_6kne4t6ii93d-2{list-style-type:none}ul.lst-kix_xmm4jxv5hwlq-2{list-style-type:none}ul.lst-kix_6kne4t6ii93d-1{list-style-type:none}ul.lst-kix_xmm4jxv5hwlq-5{list-style-type:none}.lst-kix_m75akhxjyj9r-8>li:before{content:"\0025a0   "}ul.lst-kix_xmm4jxv5hwlq-4{list-style-type:none}ul.lst-kix_xmm4jxv5hwlq-7{list-style-type:none}ol.lst-kix_awa32z9rxqor-3.start{counter-reset:lst-ctn-kix_awa32z9rxqor-3 0}ul.lst-kix_xmm4jxv5hwlq-6{list-style-type:none}.lst-kix_sp4bx5if0jbk-6>li:before{content:"\0025cf   "}.lst-kix_eag5a795mtbr-0>li:before{content:"" counter(lst-ctn-kix_eag5a795mtbr-0,decimal) ". "}ul.lst-kix_xmm4jxv5hwlq-8{list-style-type:none}.lst-kix_eag5a795mtbr-2>li:before{content:"" counter(lst-ctn-kix_eag5a795mtbr-2,lower-roman) ". "}.lst-kix_2fh1qudxwat7-7>li:before{content:"\0025cb   "}.lst-kix_sp4bx5if0jbk-0>li:before{content:"\0025cf   "}.lst-kix_2najoyx65arn-1>li:before{content:"\0025cb   "}ul.lst-kix_ij4fg4tarntm-1{list-style-type:none}.lst-kix_e2h0cgtfdf3g-6>li:before{content:"\0025cf   "}ul.lst-kix_ij4fg4tarntm-2{list-style-type:none}.lst-kix_2fh1qudxwat7-1>li:before{content:"\0025cb   "}ul.lst-kix_ij4fg4tarntm-0{list-style-type:none}ul.lst-kix_ij4fg4tarntm-5{list-style-type:none}.lst-kix_e2h0cgtfdf3g-8>li:before{content:"\0025a0   "}.lst-kix_eag5a795mtbr-8>li:before{content:"" counter(lst-ctn-kix_eag5a795mtbr-8,lower-roman) ". "}ul.lst-kix_ij4fg4tarntm-6{list-style-type:none}ul.lst-kix_ij4fg4tarntm-3{list-style-type:none}ul.lst-kix_ij4fg4tarntm-4{list-style-type:none}.lst-kix_i5y1isusbf26-2>li:before{content:"\0025a0   "}ul.lst-kix_ij4fg4tarntm-7{list-style-type:none}ul.lst-kix_ij4fg4tarntm-8{list-style-type:none}ol.lst-kix_tkg3l6cy3bnc-2.start{counter-reset:lst-ctn-kix_tkg3l6cy3bnc-2 0}.lst-kix_4864lfd9zdwy-5>li:before{content:"\0025a0   "}.lst-kix_8joc3jsoa7de-8>li{counter-increment:lst-ctn-kix_8joc3jsoa7de-8}.lst-kix_3hobsm4w6nyk-3>li:before{content:"\0025cf   "}.lst-kix_lsygr38j1y0-0>li:before{content:"\0025cf   "}.lst-kix_lsygr38j1y0-8>li:before{content:"\0025a0   "}.lst-kix_tibxkvatc3h0-8>li:before{content:"\0025a0   "}.lst-kix_mhfweobdea1s-3>li:before{content:"\0025cf   "}ol.lst-kix_esoh3w8sgdnn-0.start{counter-reset:lst-ctn-kix_esoh3w8sgdnn-0 0}.lst-kix_84yjavmdod6-3>li:before{content:"\0025cf   "}.lst-kix_n23m8p7c7hw6-7>li:before{content:"\0025cb   "}.lst-kix_hlp788b4qti1-3>li:before{content:"\0025cf   "}.lst-kix_8joc3jsoa7de-1>li:before{content:"\0025cb   "}.lst-kix_awa32z9rxqor-8>li{counter-increment:lst-ctn-kix_awa32z9rxqor-8}.lst-kix_70tvr3ungtsg-3>li:before{content:"\0025cf   "}.lst-kix_tkg3l6cy3bnc-4>li{counter-increment:lst-ctn-kix_tkg3l6cy3bnc-4}ol.lst-kix_tkg3l6cy3bnc-4.start{counter-reset:lst-ctn-kix_tkg3l6cy3bnc-4 0}.lst-kix_z4fu53gdnrww-1>li:before{content:"\0025cb   "}.lst-kix_kruy5m9jmo50-7>li:before{content:"\0025cb   "}ol.lst-kix_dgmlm9mxaa4-5.start{counter-reset:lst-ctn-kix_dgmlm9mxaa4-5 0}.lst-kix_8wqgtnyy1nox-7>li:before{content:"\0025cb   "}.lst-kix_tut3ual308bf-0>li:before{content:"\0025cf   "}ul.lst-kix_2ws5k2m4lsl8-0{list-style-type:none}ul.lst-kix_b0v72ufkqzg3-0{list-style-type:none}.lst-kix_arstwhb1c9gy-3>li:before{content:"\0025cf   "}ul.lst-kix_b0v72ufkqzg3-2{list-style-type:none}ul.lst-kix_b0v72ufkqzg3-1{list-style-type:none}ul.lst-kix_b0v72ufkqzg3-4{list-style-type:none}.lst-kix_awa32z9rxqor-4>li{counter-increment:lst-ctn-kix_awa32z9rxqor-4}ul.lst-kix_b0v72ufkqzg3-3{list-style-type:none}ul.lst-kix_2ws5k2m4lsl8-7{list-style-type:none}.lst-kix_b4xlo74k6gd0-4>li:before{content:"\0025cb   "}ul.lst-kix_2ws5k2m4lsl8-8{list-style-type:none}ul.lst-kix_2ws5k2m4lsl8-5{list-style-type:none}ul.lst-kix_2ws5k2m4lsl8-6{list-style-type:none}ul.lst-kix_2ws5k2m4lsl8-3{list-style-type:none}ul.lst-kix_2ws5k2m4lsl8-4{list-style-type:none}.lst-kix_v0sksjnof0cj-4>li:before{content:"\0025cb   "}ul.lst-kix_2ws5k2m4lsl8-1{list-style-type:none}ol.lst-kix_sbk9fqxb69uy-4.start{counter-reset:lst-ctn-kix_sbk9fqxb69uy-4 0}ul.lst-kix_2ws5k2m4lsl8-2{list-style-type:none}.lst-kix_8i9khg2xgulr-4>li:before{content:"\0025cb   "}.lst-kix_ss89m2334v3w-0>li:before{content:"\0025cf   "}.lst-kix_p73ghpm47z3w-0>li:before{content:"\0025cf   "}.lst-kix_x3csall36p7p-0>li:before{content:"\0025cf   "}.lst-kix_k2lnp2ex00id-7>li:before{content:"\0025cb   "}.lst-kix_8joc3jsoa7de-4>li{counter-increment:lst-ctn-kix_8joc3jsoa7de-4}.lst-kix_dmy2ekb87fbb-8>li:before{content:"\0025a0   "}ul.lst-kix_b0v72ufkqzg3-6{list-style-type:none}.lst-kix_473lniz6s057-8>li:before{content:"\0025a0   "}ul.lst-kix_b0v72ufkqzg3-5{list-style-type:none}ul.lst-kix_b0v72ufkqzg3-8{list-style-type:none}ul.lst-kix_b0v72ufkqzg3-7{list-style-type:none}.lst-kix_tj1ix0gi2ui9-4>li:before{content:"\0025cb   "}.lst-kix_q432akg4led-7>li:before{content:"\0025cb   "}.lst-kix_8luyzq1gqi6-5>li:before{content:"\0025a0   "}.lst-kix_p73ghpm47z3w-8>li:before{content:"\0025a0   "}.lst-kix_an05xfuo7mar-8>li:before{content:"\0025a0   "}.lst-kix_haoc7b4kfhm3-3>li:before{content:"\0025cf   "}ul.lst-kix_x3csall36p7p-3{list-style-type:none}ul.lst-kix_x3csall36p7p-4{list-style-type:none}ul.lst-kix_x3csall36p7p-1{list-style-type:none}.lst-kix_tx9us7pz8zze-2>li:before{content:"\0025a0   "}ul.lst-kix_x3csall36p7p-2{list-style-type:none}ul.lst-kix_x3csall36p7p-7{list-style-type:none}.lst-kix_vx5oc7tbpucq-5>li:before{content:"\0025a0   "}ul.lst-kix_x3csall36p7p-8{list-style-type:none}ul.lst-kix_x3csall36p7p-5{list-style-type:none}ul.lst-kix_x3csall36p7p-6{list-style-type:none}ul.lst-kix_upyk6it0f1tq-5{list-style-type:none}.lst-kix_8yxwyie669z2-2>li:before{content:"\0025a0   "}ul.lst-kix_8wqgtnyy1nox-5{list-style-type:none}.lst-kix_t27vnxuzkttq-1>li:before{content:"\0025cb   "}ul.lst-kix_upyk6it0f1tq-6{list-style-type:none}ul.lst-kix_8wqgtnyy1nox-4{list-style-type:none}.lst-kix_auto7t5cf4tl-3>li:before{content:"\0025cf   "}ul.lst-kix_upyk6it0f1tq-7{list-style-type:none}ul.lst-kix_8wqgtnyy1nox-7{list-style-type:none}ul.lst-kix_xq7wzd2lr8u5-0{list-style-type:none}ul.lst-kix_upyk6it0f1tq-8{list-style-type:none}ul.lst-kix_8wqgtnyy1nox-6{list-style-type:none}ul.lst-kix_xq7wzd2lr8u5-1{list-style-type:none}ul.lst-kix_xq7wzd2lr8u5-2{list-style-type:none}ul.lst-kix_x3csall36p7p-0{list-style-type:none}.lst-kix_an05xfuo7mar-0>li:before{content:"\0025cf   "}ul.lst-kix_8wqgtnyy1nox-8{list-style-type:none}ul.lst-kix_xq7wzd2lr8u5-3{list-style-type:none}.lst-kix_ie2cjnul3ua-0>li:before{content:"\0025cf   "}ul.lst-kix_xq7wzd2lr8u5-4{list-style-type:none}ul.lst-kix_xq7wzd2lr8u5-5{list-style-type:none}.lst-kix_sbk9fqxb69uy-5>li:before{content:"" counter(lst-ctn-kix_sbk9fqxb69uy-5,lower-roman) ". "}.lst-kix_dmy2ekb87fbb-0>li:before{content:"\0025cf   "}ul.lst-kix_xq7wzd2lr8u5-6{list-style-type:none}ul.lst-kix_xq7wzd2lr8u5-7{list-style-type:none}ul.lst-kix_xq7wzd2lr8u5-8{list-style-type:none}ul.lst-kix_upyk6it0f1tq-0{list-style-type:none}.lst-kix_6pozql3s9ehi-2>li:before{content:"\0025a0   "}ul.lst-kix_upyk6it0f1tq-1{list-style-type:none}ul.lst-kix_8wqgtnyy1nox-1{list-style-type:none}ul.lst-kix_upyk6it0f1tq-2{list-style-type:none}ul.lst-kix_8wqgtnyy1nox-0{list-style-type:none}ul.lst-kix_upyk6it0f1tq-3{list-style-type:none}ul.lst-kix_8wqgtnyy1nox-3{list-style-type:none}ul.lst-kix_upyk6it0f1tq-4{list-style-type:none}ul.lst-kix_8wqgtnyy1nox-2{list-style-type:none}.lst-kix_7g6nqlfa7rn7-5>li:before{content:"\0025a0   "}.lst-kix_qxeaj9fcw57j-0>li:before{content:"\0025cf   "}.lst-kix_ht2xpfpspp5-4>li:before{content:"\0025cb   "}.lst-kix_ie2cjnul3ua-8>li:before{content:"\0025a0   "}.lst-kix_eag5a795mtbr-2>li{counter-increment:lst-ctn-kix_eag5a795mtbr-2}.lst-kix_b01r1xbffk2a-6>li:before{content:"\0025cf   "}.lst-kix_awa32z9rxqor-5>li:before{content:"" counter(lst-ctn-kix_awa32z9rxqor-5,lower-roman) ". "}ul.lst-kix_ehpc0o1sivs1-8{list-style-type:none}.lst-kix_z1b7uf8mlquv-7>li:before{content:"\0025cb   "}.lst-kix_4y6tqwnm2166-1>li:before{content:"\0025cb   "}ul.lst-kix_ehpc0o1sivs1-7{list-style-type:none}ul.lst-kix_ehpc0o1sivs1-4{list-style-type:none}ul.lst-kix_ehpc0o1sivs1-3{list-style-type:none}ul.lst-kix_ehpc0o1sivs1-6{list-style-type:none}ul.lst-kix_ehpc0o1sivs1-5{list-style-type:none}.lst-kix_4hyg4ehfafxi-0>li:before{content:"\0025cf   "}ul.lst-kix_ehpc0o1sivs1-0{list-style-type:none}ul.lst-kix_ehpc0o1sivs1-2{list-style-type:none}ul.lst-kix_ehpc0o1sivs1-1{list-style-type:none}.lst-kix_ij4fg4tarntm-0>li:before{content:"\0025cf   "}.lst-kix_iprbjac0k1x3-2>li:before{content:"\0025a0   "}.lst-kix_5mbqwckh3266-3>li:before{content:"\0025cf   "}ol.lst-kix_awa32z9rxqor-8.start{counter-reset:lst-ctn-kix_awa32z9rxqor-8 0}.lst-kix_umwvyipzf505-0>li:before{content:"\0025cf   "}.lst-kix_6vg7rekv2ht5-8>li:before{content:"\0025a0   "}.lst-kix_9fa76bs6q7z7-3>li:before{content:"\0025cf   "}.lst-kix_eb0mkv6he1ge-5>li:before{content:"\0025a0   "}.lst-kix_565pscofcl0-1>li:before{content:"\0025cb   "}.lst-kix_473lniz6s057-0>li:before{content:"\0025cf   "}.lst-kix_zqq05ofg66-2>li:before{content:"" counter(lst-ctn-kix_zqq05ofg66-2,lower-roman) ". "}.lst-kix_tc68vt7q79wj-1>li:before{content:"\0025cb   "}.lst-kix_wje7sh1pcpc1-4>li:before{content:"\0025cb   "}.lst-kix_pb595s4qvcov-4>li:before{content:"\0025cb   "}.lst-kix_kfepckt4l6hg-2>li:before{content:"" counter(lst-ctn-kix_kfepckt4l6hg-2,lower-roman) ". "}ul.lst-kix_2fh1qudxwat7-6{list-style-type:none}.lst-kix_s4u6wv582l9n-7>li:before{content:"\0025cb   "}ul.lst-kix_2fh1qudxwat7-5{list-style-type:none}ul.lst-kix_2fh1qudxwat7-4{list-style-type:none}.lst-kix_ss89m2334v3w-8>li:before{content:"\0025a0   "}ul.lst-kix_2fh1qudxwat7-3{list-style-type:none}ul.lst-kix_2fh1qudxwat7-2{list-style-type:none}ul.lst-kix_2fh1qudxwat7-1{list-style-type:none}ul.lst-kix_2fh1qudxwat7-0{list-style-type:none}.lst-kix_ftoogxlvmqxr-7>li:before{content:"\0025cb   "}.lst-kix_b0v72ufkqzg3-0>li:before{content:"\0025cf   "}.lst-kix_b0v72ufkqzg3-8>li:before{content:"\0025a0   "}ul.lst-kix_2fh1qudxwat7-8{list-style-type:none}.lst-kix_umwvyipzf505-8>li:before{content:"\0025a0   "}ul.lst-kix_2fh1qudxwat7-7{list-style-type:none}.lst-kix_6vg7rekv2ht5-0>li:before{content:"\0025cf   "}.lst-kix_ij4fg4tarntm-8>li:before{content:"\0025a0   "}ol.lst-kix_sbk9fqxb69uy-1.start{counter-reset:lst-ctn-kix_sbk9fqxb69uy-1 0}.lst-kix_6z3d30mc5xph-7>li:before{content:"\0025cb   "}.lst-kix_tibxkvatc3h0-0>li:before{content:"\0025cf   "}.lst-kix_vams35lnoe4-6>li:before{content:"\0025cf   "}.lst-kix_5mkgtms2qzgk-5>li:before{content:"\0025a0   "}.lst-kix_129uykneptsf-3>li:before{content:"\0025cf   "}ul.lst-kix_q432akg4led-6{list-style-type:none}ul.lst-kix_q432akg4led-5{list-style-type:none}.lst-kix_8r3w2rcyu87a-4>li:before{content:"\0025cb   "}ul.lst-kix_q432akg4led-4{list-style-type:none}ul.lst-kix_q432akg4led-3{list-style-type:none}ul.lst-kix_q432akg4led-2{list-style-type:none}.lst-kix_j5h2oeedi37x-2>li:before{content:"\0025a0   "}ul.lst-kix_q432akg4led-1{list-style-type:none}ul.lst-kix_q432akg4led-0{list-style-type:none}.lst-kix_qxeaj9fcw57j-8>li:before{content:"\0025a0   "}.lst-kix_skp7x04r24av-5>li:before{content:"\0025a0   "}.lst-kix_dgmlm9mxaa4-5>li:before{content:"" counter(lst-ctn-kix_dgmlm9mxaa4-5,lower-roman) ". "}ul.lst-kix_tkg3l6cy3bnc-1{list-style-type:none}.lst-kix_4hyg4ehfafxi-8>li:before{content:"\0025a0   "}.lst-kix_k38n94udk0el-5>li:before{content:"\0025a0   "}.lst-kix_tkg3l6cy3bnc-3>li{counter-increment:lst-ctn-kix_tkg3l6cy3bnc-3}ul.lst-kix_cwtqd0lozkzu-0{list-style-type:none}.lst-kix_cb1r7mwbu1gi-1>li:before{content:"\0025cb   "}ul.lst-kix_cwtqd0lozkzu-4{list-style-type:none}ul.lst-kix_cwtqd0lozkzu-3{list-style-type:none}.lst-kix_skp7x04r24av-3>li:before{content:"\0025cf   "}ul.lst-kix_cwtqd0lozkzu-2{list-style-type:none}ul.lst-kix_cwtqd0lozkzu-1{list-style-type:none}.lst-kix_skp7x04r24av-2>li:before{content:"\0025a0   "}ul.lst-kix_cwtqd0lozkzu-8{list-style-type:none}ul.lst-kix_cwtqd0lozkzu-7{list-style-type:none}ul.lst-kix_cwtqd0lozkzu-6{list-style-type:none}ul.lst-kix_cwtqd0lozkzu-5{list-style-type:none}.lst-kix_skp7x04r24av-0>li:before{content:"\0025cf   "}ul.lst-kix_qxeaj9fcw57j-0{list-style-type:none}.lst-kix_ehpc0o1sivs1-8>li:before{content:"\0025a0   "}.lst-kix_6kne4t6ii93d-8>li:before{content:"\0025a0   "}.lst-kix_1ip3hsr9v1h5-2>li:before{content:"\0025a0   "}.lst-kix_eunro7k2bn6e-7>li:before{content:"\0025cb   "}ol.lst-kix_esoh3w8sgdnn-8.start{counter-reset:lst-ctn-kix_esoh3w8sgdnn-8 0}.lst-kix_ytu3x5e7mxt6-8>li{counter-increment:lst-ctn-kix_ytu3x5e7mxt6-8}.lst-kix_ehpc0o1sivs1-4>li:before{content:"\0025cb   "}ul.lst-kix_hlp788b4qti1-0{list-style-type:none}.lst-kix_eunro7k2bn6e-6>li:before{content:"\0025cf   "}.lst-kix_ehpc0o1sivs1-7>li:before{content:"\0025cb   "}ul.lst-kix_1lds0wnd5zeb-8{list-style-type:none}ul.lst-kix_hlp788b4qti1-3{list-style-type:none}ul.lst-kix_qxeaj9fcw57j-4{list-style-type:none}ul.lst-kix_hlp788b4qti1-4{list-style-type:none}ul.lst-kix_qxeaj9fcw57j-3{list-style-type:none}ul.lst-kix_hlp788b4qti1-1{list-style-type:none}ul.lst-kix_qxeaj9fcw57j-2{list-style-type:none}ul.lst-kix_hlp788b4qti1-2{list-style-type:none}ul.lst-kix_qxeaj9fcw57j-1{list-style-type:none}ul.lst-kix_hlp788b4qti1-7{list-style-type:none}ul.lst-kix_qxeaj9fcw57j-8{list-style-type:none}ul.lst-kix_hlp788b4qti1-8{list-style-type:none}ul.lst-kix_qxeaj9fcw57j-7{list-style-type:none}.lst-kix_eunro7k2bn6e-0>li:before{content:"\0025cf   "}ul.lst-kix_hlp788b4qti1-5{list-style-type:none}ul.lst-kix_qxeaj9fcw57j-6{list-style-type:none}ul.lst-kix_hlp788b4qti1-6{list-style-type:none}ul.lst-kix_qxeaj9fcw57j-5{list-style-type:none}ul.lst-kix_1lds0wnd5zeb-0{list-style-type:none}.lst-kix_cb1r7mwbu1gi-4>li:before{content:"\0025cb   "}ul.lst-kix_1lds0wnd5zeb-1{list-style-type:none}ul.lst-kix_1lds0wnd5zeb-2{list-style-type:none}.lst-kix_e134675mgdj3-5>li:before{content:"\0025a0   "}ul.lst-kix_1lds0wnd5zeb-3{list-style-type:none}ul.lst-kix_1lds0wnd5zeb-4{list-style-type:none}.lst-kix_e134675mgdj3-6>li:before{content:"\0025cf   "}ul.lst-kix_1lds0wnd5zeb-5{list-style-type:none}ul.lst-kix_fsz4dbursko6-8{list-style-type:none}ul.lst-kix_1lds0wnd5zeb-6{list-style-type:none}ul.lst-kix_1lds0wnd5zeb-7{list-style-type:none}.lst-kix_eunro7k2bn6e-3>li:before{content:"\0025cf   "}ul.lst-kix_fsz4dbursko6-6{list-style-type:none}.lst-kix_e134675mgdj3-8>li:before{content:"\0025a0   "}.lst-kix_cb1r7mwbu1gi-8>li:before{content:"\0025a0   "}.lst-kix_ehpc0o1sivs1-1>li:before{content:"\0025cb   "}ul.lst-kix_fsz4dbursko6-7{list-style-type:none}ul.lst-kix_fsz4dbursko6-4{list-style-type:none}.lst-kix_cb1r7mwbu1gi-7>li:before{content:"\0025cb   "}ul.lst-kix_fsz4dbursko6-5{list-style-type:none}ul.lst-kix_fsz4dbursko6-2{list-style-type:none}ul.lst-kix_fsz4dbursko6-3{list-style-type:none}ul.lst-kix_fsz4dbursko6-0{list-style-type:none}ul.lst-kix_fsz4dbursko6-1{list-style-type:none}.lst-kix_fsz4dbursko6-4>li:before{content:"\0025cb   "}.lst-kix_ehxsaivn684z-1>li:before{content:"\0025cb   "}.lst-kix_81hzypied9ul-6>li:before{content:"\0025cf   "}.lst-kix_81hzypied9ul-3>li:before{content:"\0025cf   "}.lst-kix_w9ink2jq27yv-2>li:before{content:"\0025a0   "}.lst-kix_fsz4dbursko6-7>li:before{content:"\0025cb   "}.lst-kix_fsz4dbursko6-5>li:before{content:"\0025a0   "}.lst-kix_w9ink2jq27yv-8>li:before{content:"\0025a0   "}.lst-kix_565pscofcl0-3>li:before{content:"\0025cf   "}.lst-kix_565pscofcl0-4>li:before{content:"\0025cb   "}.lst-kix_e134675mgdj3-3>li:before{content:"\0025cf   "}.lst-kix_e134675mgdj3-2>li:before{content:"\0025a0   "}.lst-kix_81hzypied9ul-0>li:before{content:"\0025cf   "}.lst-kix_w9ink2jq27yv-6>li:before{content:"\0025cf   "}.lst-kix_w9ink2jq27yv-5>li:before{content:"\0025a0   "}.lst-kix_565pscofcl0-7>li:before{content:"\0025cb   "}.lst-kix_fsz4dbursko6-2>li:before{content:"\0025a0   "}.lst-kix_565pscofcl0-6>li:before{content:"\0025cf   "}.lst-kix_fsz4dbursko6-1>li:before{content:"\0025cb   "}ul.lst-kix_bo4w5l7q0tq9-3{list-style-type:none}ul.lst-kix_bo4w5l7q0tq9-2{list-style-type:none}.lst-kix_1qqoijhe7jmx-3>li:before{content:"\0025cf   "}ul.lst-kix_bo4w5l7q0tq9-1{list-style-type:none}ul.lst-kix_bo4w5l7q0tq9-0{list-style-type:none}.lst-kix_1qqoijhe7jmx-6>li:before{content:"\0025cf   "}.lst-kix_1ip3hsr9v1h5-8>li:before{content:"\0025a0   "}.lst-kix_1qqoijhe7jmx-0>li:before{content:"\0025cf   "}.lst-kix_1ip3hsr9v1h5-5>li:before{content:"\0025a0   "}ul.lst-kix_wk2xuqsimxh-8{list-style-type:none}ul.lst-kix_bo4w5l7q0tq9-8{list-style-type:none}.lst-kix_fsz4dbursko6-8>li:before{content:"\0025a0   "}ul.lst-kix_bo4w5l7q0tq9-7{list-style-type:none}ul.lst-kix_bo4w5l7q0tq9-6{list-style-type:none}ul.lst-kix_bo4w5l7q0tq9-5{list-style-type:none}ul.lst-kix_bo4w5l7q0tq9-4{list-style-type:none}ol.lst-kix_kfepckt4l6hg-8.start{counter-reset:lst-ctn-kix_kfepckt4l6hg-8 0}ol.lst-kix_awa32z9rxqor-2.start{counter-reset:lst-ctn-kix_awa32z9rxqor-2 0}.lst-kix_4y6tqwnm2166-2>li:before{content:"\0025a0   "}.lst-kix_zqq05ofg66-8>li{counter-increment:lst-ctn-kix_zqq05ofg66-8}ul.lst-kix_an05xfuo7mar-5{list-style-type:none}ul.lst-kix_an05xfuo7mar-4{list-style-type:none}ul.lst-kix_an05xfuo7mar-7{list-style-type:none}ul.lst-kix_an05xfuo7mar-6{list-style-type:none}ul.lst-kix_an05xfuo7mar-1{list-style-type:none}ul.lst-kix_an05xfuo7mar-0{list-style-type:none}ul.lst-kix_an05xfuo7mar-3{list-style-type:none}ul.lst-kix_an05xfuo7mar-2{list-style-type:none}.lst-kix_zqq05ofg66-7>li{counter-increment:lst-ctn-kix_zqq05ofg66-7}.lst-kix_ihv36zxyu5h9-5>li:before{content:"\0025a0   "}ul.lst-kix_an05xfuo7mar-8{list-style-type:none}.lst-kix_5mbqwckh3266-0>li:before{content:"\0025cf   "}.lst-kix_w12y2pivfx64-2>li:before{content:"\0025a0   "}ol.lst-kix_eag5a795mtbr-7.start{counter-reset:lst-ctn-kix_eag5a795mtbr-7 0}.lst-kix_5mbqwckh3266-4>li:before{content:"\0025cb   "}.lst-kix_ehxsaivn684z-8>li:before{content:"\0025a0   "}.lst-kix_pb595s4qvcov-7>li:before{content:"\0025cb   "}.lst-kix_awa32z9rxqor-3>li{counter-increment:lst-ctn-kix_awa32z9rxqor-3}.lst-kix_w12y2pivfx64-8>li:before{content:"\0025a0   "}ul.lst-kix_4864lfd9zdwy-0{list-style-type:none}ul.lst-kix_4864lfd9zdwy-1{list-style-type:none}.lst-kix_565pscofcl0-0>li:before{content:"\0025cf   "}ul.lst-kix_4864lfd9zdwy-2{list-style-type:none}.lst-kix_ehxsaivn684z-4>li:before{content:"\0025cb   "}.lst-kix_pb595s4qvcov-3>li:before{content:"\0025cf   "}ul.lst-kix_4864lfd9zdwy-3{list-style-type:none}.lst-kix_473lniz6s057-1>li:before{content:"\0025cb   "}.lst-kix_4y6tqwnm2166-8>li:before{content:"\0025a0   "}ul.lst-kix_4864lfd9zdwy-8{list-style-type:none}ul.lst-kix_4864lfd9zdwy-4{list-style-type:none}.lst-kix_2tnhjv96pndg-1>li:before{content:"\0025cb   "}ul.lst-kix_4864lfd9zdwy-5{list-style-type:none}ul.lst-kix_4864lfd9zdwy-6{list-style-type:none}.lst-kix_1lds0wnd5zeb-3>li:before{content:"\0025cf   "}ul.lst-kix_4864lfd9zdwy-7{list-style-type:none}.lst-kix_kfepckt4l6hg-2>li{counter-increment:lst-ctn-kix_kfepckt4l6hg-2}.lst-kix_tut3ual308bf-6>li:before{content:"\0025cf   "}ul.lst-kix_z1b7uf8mlquv-0{list-style-type:none}.lst-kix_bo4w5l7q0tq9-2>li:before{content:"\0025a0   "}.lst-kix_ls0ws55dysw8-6>li:before{content:"\0025cf   "}ul.lst-kix_z1b7uf8mlquv-4{list-style-type:none}ul.lst-kix_z1b7uf8mlquv-3{list-style-type:none}ul.lst-kix_z1b7uf8mlquv-2{list-style-type:none}ul.lst-kix_z1b7uf8mlquv-1{list-style-type:none}ul.lst-kix_z1b7uf8mlquv-8{list-style-type:none}.lst-kix_tut3ual308bf-8>li:before{content:"\0025a0   "}ul.lst-kix_z1b7uf8mlquv-7{list-style-type:none}ul.lst-kix_z1b7uf8mlquv-6{list-style-type:none}ul.lst-kix_z1b7uf8mlquv-5{list-style-type:none}.lst-kix_ls0ws55dysw8-8>li:before{content:"\0025a0   "}.lst-kix_dgmlm9mxaa4-0>li:before{content:"" counter(lst-ctn-kix_dgmlm9mxaa4-0,decimal) ". "}.lst-kix_6kne4t6ii93d-1>li:before{content:"\0025cb   "}ol.lst-kix_zqq05ofg66-8.start{counter-reset:lst-ctn-kix_zqq05ofg66-8 0}ol.lst-kix_esoh3w8sgdnn-3.start{counter-reset:lst-ctn-kix_esoh3w8sgdnn-3 0}.lst-kix_6kne4t6ii93d-5>li:before{content:"\0025a0   "}.lst-kix_dgmlm9mxaa4-2>li:before{content:"" counter(lst-ctn-kix_dgmlm9mxaa4-2,lower-roman) ". "}.lst-kix_bo4w5l7q0tq9-4>li:before{content:"\0025cb   "}.lst-kix_ls0ws55dysw8-2>li:before{content:"\0025a0   "}.lst-kix_skp7x04r24av-6>li:before{content:"\0025cf   "}.lst-kix_2tnhjv96pndg-7>li:before{content:"\0025cb   "}.lst-kix_dgmlm9mxaa4-6>li:before{content:"" counter(lst-ctn-kix_dgmlm9mxaa4-6,decimal) ". "}.lst-kix_bo4w5l7q0tq9-8>li:before{content:"\0025a0   "}ol.lst-kix_3qoh0wbylc0j-7.start{counter-reset:lst-ctn-kix_3qoh0wbylc0j-7 0}.lst-kix_84yjavmdod6-5>li:before{content:"\0025a0   "}.lst-kix_84yjavmdod6-8>li:before{content:"\0025a0   "}ul.lst-kix_jyqknjqk01cc-3{list-style-type:none}ul.lst-kix_jyqknjqk01cc-4{list-style-type:none}ul.lst-kix_jyqknjqk01cc-1{list-style-type:none}ul.lst-kix_jyqknjqk01cc-2{list-style-type:none}ul.lst-kix_jyqknjqk01cc-7{list-style-type:none}ul.lst-kix_jyqknjqk01cc-8{list-style-type:none}.lst-kix_3hobsm4w6nyk-5>li:before{content:"\0025a0   "}ul.lst-kix_jyqknjqk01cc-5{list-style-type:none}ul.lst-kix_jyqknjqk01cc-6{list-style-type:none}.lst-kix_3hobsm4w6nyk-8>li:before{content:"\0025a0   "}.lst-kix_mhfweobdea1s-2>li:before{content:"\0025a0   "}.lst-kix_8wqgtnyy1nox-5>li:before{content:"\0025a0   "}.lst-kix_hlp788b4qti1-1>li:before{content:"\0025cb   "}.lst-kix_8joc3jsoa7de-2>li:before{content:"" counter(lst-ctn-kix_8joc3jsoa7de-2,lower-roman) ". "}.lst-kix_8wqgtnyy1nox-2>li:before{content:"\0025a0   "}.lst-kix_tut3ual308bf-2>li:before{content:"\0025a0   "}.lst-kix_mhfweobdea1s-5>li:before{content:"\0025a0   "}.lst-kix_hlp788b4qti1-0>li:before{content:"\0025cf   "}.lst-kix_mhfweobdea1s-6>li:before{content:"\0025cf   "}.lst-kix_dgmlm9mxaa4-5>li{counter-increment:lst-ctn-kix_dgmlm9mxaa4-5}.lst-kix_6pozql3s9ehi-0>li:before{content:"\0025cf   "}.lst-kix_arstwhb1c9gy-1>li:before{content:"\0025cb   "}.lst-kix_8yxwyie669z2-8>li:before{content:"\0025a0   "}.lst-kix_b4xlo74k6gd0-1>li:before{content:"\0025cb   "}.lst-kix_ss89m2334v3w-3>li:before{content:"\0025cf   "}ol.lst-kix_zqq05ofg66-4.start{counter-reset:lst-ctn-kix_zqq05ofg66-4 0}.lst-kix_ss89m2334v3w-2>li:before{content:"\0025a0   "}.lst-kix_b4xlo74k6gd0-2>li:before{content:"\0025a0   "}.lst-kix_k2lnp2ex00id-5>li:before{content:"\0025a0   "}.lst-kix_dmy2ekb87fbb-3>li:before{content:"\0025cf   "}ol.lst-kix_3qoh0wbylc0j-2.start{counter-reset:lst-ctn-kix_3qoh0wbylc0j-2 0}ol.lst-kix_eag5a795mtbr-3.start{counter-reset:lst-ctn-kix_eag5a795mtbr-3 0}.lst-kix_arstwhb1c9gy-0>li:before{content:"\0025cf   "}.lst-kix_dmy2ekb87fbb-6>li:before{content:"\0025cf   "}.lst-kix_8yxwyie669z2-4>li:before{content:"\0025cb   "}ol.lst-kix_kfepckt4l6hg-3.start{counter-reset:lst-ctn-kix_kfepckt4l6hg-3 0}.lst-kix_8wqgtnyy1nox-1>li:before{content:"\0025cb   "}ul.lst-kix_tc68vt7q79wj-8{list-style-type:none}ul.lst-kix_tc68vt7q79wj-7{list-style-type:none}.lst-kix_70tvr3ungtsg-1>li:before{content:"\0025cb   "}ol.lst-kix_3qoh0wbylc0j-8.start{counter-reset:lst-ctn-kix_3qoh0wbylc0j-8 0}.lst-kix_8i9khg2xgulr-6>li:before{content:"\0025cf   "}ul.lst-kix_tc68vt7q79wj-4{list-style-type:none}.lst-kix_hlp788b4qti1-4>li:before{content:"\0025cb   "}ul.lst-kix_tc68vt7q79wj-3{list-style-type:none}.lst-kix_8yxwyie669z2-7>li:before{content:"\0025cb   "}ul.lst-kix_tc68vt7q79wj-6{list-style-type:none}.lst-kix_8i9khg2xgulr-7>li:before{content:"\0025cb   "}ul.lst-kix_tc68vt7q79wj-5{list-style-type:none}.lst-kix_arstwhb1c9gy-4>li:before{content:"\0025cb   "}ul.lst-kix_tc68vt7q79wj-0{list-style-type:none}.lst-kix_vx5oc7tbpucq-2>li:before{content:"\0025a0   "}.lst-kix_vx5oc7tbpucq-3>li:before{content:"\0025cf   "}ul.lst-kix_tc68vt7q79wj-2{list-style-type:none}.lst-kix_tx9us7pz8zze-4>li:before{content:"\0025cb   "}ul.lst-kix_tc68vt7q79wj-1{list-style-type:none}ol.lst-kix_eag5a795mtbr-2.start{counter-reset:lst-ctn-kix_eag5a795mtbr-2 0}.lst-kix_tx9us7pz8zze-7>li:before{content:"\0025cb   "}.lst-kix_esoh3w8sgdnn-8>li{counter-increment:lst-ctn-kix_esoh3w8sgdnn-8}.lst-kix_vx5oc7tbpucq-6>li:before{content:"\0025cf   "}.lst-kix_tx9us7pz8zze-8>li:before{content:"\0025a0   "}.lst-kix_8i9khg2xgulr-3>li:before{content:"\0025cf   "}.lst-kix_dmy2ekb87fbb-2>li:before{content:"\0025a0   "}.lst-kix_b4xlo74k6gd0-5>li:before{content:"\0025a0   "}.lst-kix_k2lnp2ex00id-2>li:before{content:"\0025a0   "}.lst-kix_k2lnp2ex00id-1>li:before{content:"\0025cb   "}ol.lst-kix_kfepckt4l6hg-4.start{counter-reset:lst-ctn-kix_kfepckt4l6hg-4 0}.lst-kix_4hyg4ehfafxi-6>li:before{content:"\0025cf   "}.lst-kix_7g6nqlfa7rn7-7>li:before{content:"\0025cb   "}ol.lst-kix_esoh3w8sgdnn-7.start{counter-reset:lst-ctn-kix_esoh3w8sgdnn-7 0}ul.lst-kix_etn6l1m36dy-8{list-style-type:none}.lst-kix_4hyg4ehfafxi-2>li:before{content:"\0025a0   "}ul.lst-kix_etn6l1m36dy-4{list-style-type:none}ul.lst-kix_tj1ix0gi2ui9-8{list-style-type:none}ul.lst-kix_etn6l1m36dy-5{list-style-type:none}ul.lst-kix_tj1ix0gi2ui9-7{list-style-type:none}ul.lst-kix_etn6l1m36dy-6{list-style-type:none}ul.lst-kix_tj1ix0gi2ui9-6{list-style-type:none}ul.lst-kix_etn6l1m36dy-7{list-style-type:none}ul.lst-kix_etn6l1m36dy-0{list-style-type:none}.lst-kix_kfepckt4l6hg-4>li{counter-increment:lst-ctn-kix_kfepckt4l6hg-4}ul.lst-kix_etn6l1m36dy-1{list-style-type:none}ul.lst-kix_etn6l1m36dy-2{list-style-type:none}.lst-kix_5s9vwy7hbq8r-8>li:before{content:"\0025a0   "}ul.lst-kix_etn6l1m36dy-3{list-style-type:none}.lst-kix_awa32z9rxqor-3>li:before{content:"" counter(lst-ctn-kix_awa32z9rxqor-3,decimal) ". "}ul.lst-kix_4hyg4ehfafxi-8{list-style-type:none}ul.lst-kix_ihv36zxyu5h9-0{list-style-type:none}ul.lst-kix_4hyg4ehfafxi-7{list-style-type:none}ul.lst-kix_ihv36zxyu5h9-1{list-style-type:none}.lst-kix_tc68vt7q79wj-7>li:before{content:"\0025cb   "}ul.lst-kix_4hyg4ehfafxi-6{list-style-type:none}ul.lst-kix_ihv36zxyu5h9-2{list-style-type:none}ul.lst-kix_4hyg4ehfafxi-5{list-style-type:none}ol.lst-kix_3qoh0wbylc0j-3.start{counter-reset:lst-ctn-kix_3qoh0wbylc0j-3 0}ul.lst-kix_ihv36zxyu5h9-3{list-style-type:none}ul.lst-kix_4hyg4ehfafxi-4{list-style-type:none}ul.lst-kix_ihv36zxyu5h9-4{list-style-type:none}ul.lst-kix_lsygr38j1y0-0{list-style-type:none}ul.lst-kix_4hyg4ehfafxi-3{list-style-type:none}ul.lst-kix_ihv36zxyu5h9-5{list-style-type:none}ul.lst-kix_4hyg4ehfafxi-2{list-style-type:none}ul.lst-kix_ihv36zxyu5h9-6{list-style-type:none}ul.lst-kix_lsygr38j1y0-2{list-style-type:none}ul.lst-kix_4hyg4ehfafxi-1{list-style-type:none}ul.lst-kix_ihv36zxyu5h9-7{list-style-type:none}ul.lst-kix_lsygr38j1y0-1{list-style-type:none}ul.lst-kix_4hyg4ehfafxi-0{list-style-type:none}ul.lst-kix_ihv36zxyu5h9-8{list-style-type:none}ul.lst-kix_lsygr38j1y0-4{list-style-type:none}.lst-kix_m75akhxjyj9r-3>li:before{content:"\0025cf   "}ul.lst-kix_lsygr38j1y0-3{list-style-type:none}ul.lst-kix_lsygr38j1y0-6{list-style-type:none}ul.lst-kix_lsygr38j1y0-5{list-style-type:none}ul.lst-kix_lsygr38j1y0-8{list-style-type:none}ul.lst-kix_lsygr38j1y0-7{list-style-type:none}.lst-kix_ij4fg4tarntm-2>li:before{content:"\0025a0   "}ol.lst-kix_zqq05ofg66-3.start{counter-reset:lst-ctn-kix_zqq05ofg66-3 0}ul.lst-kix_mhfweobdea1s-0{list-style-type:none}.lst-kix_5mbqwckh3266-1>li:before{content:"\0025cb   "}.lst-kix_d7lufd20rbm5-5>li:before{content:"\0025a0   "}ul.lst-kix_mhfweobdea1s-2{list-style-type:none}ul.lst-kix_mhfweobdea1s-1{list-style-type:none}.lst-kix_3aierr3stws2-3>li:before{content:"\0025cf   "}ul.lst-kix_mhfweobdea1s-4{list-style-type:none}ul.lst-kix_mhfweobdea1s-3{list-style-type:none}ul.lst-kix_mhfweobdea1s-6{list-style-type:none}.lst-kix_tc68vt7q79wj-3>li:before{content:"\0025cf   "}ul.lst-kix_mhfweobdea1s-5{list-style-type:none}ul.lst-kix_mhfweobdea1s-8{list-style-type:none}ul.lst-kix_mhfweobdea1s-7{list-style-type:none}.lst-kix_ihv36zxyu5h9-8>li:before{content:"\0025a0   "}.lst-kix_pb595s4qvcov-6>li:before{content:"\0025cf   "}.lst-kix_9fa76bs6q7z7-1>li:before{content:"\0025cb   "}.lst-kix_d8i9v1tvihft-2>li:before{content:"\0025a0   "}.lst-kix_eb0mkv6he1ge-7>li:before{content:"\0025cb   "}.lst-kix_ehxsaivn684z-7>li:before{content:"\0025cb   "}.lst-kix_8joc3jsoa7de-0>li{counter-increment:lst-ctn-kix_8joc3jsoa7de-0}.lst-kix_5s9vwy7hbq8r-4>li:before{content:"\0025cb   "}.lst-kix_d8i9v1tvihft-6>li:before{content:"\0025cf   "}ul.lst-kix_w12y2pivfx64-7{list-style-type:none}ul.lst-kix_w12y2pivfx64-8{list-style-type:none}ul.lst-kix_w12y2pivfx64-5{list-style-type:none}ul.lst-kix_w12y2pivfx64-6{list-style-type:none}ul.lst-kix_w12y2pivfx64-3{list-style-type:none}ul.lst-kix_w12y2pivfx64-4{list-style-type:none}ul.lst-kix_w12y2pivfx64-1{list-style-type:none}.lst-kix_ss89m2334v3w-6>li:before{content:"\0025cf   "}ul.lst-kix_w12y2pivfx64-2{list-style-type:none}ul.lst-kix_w12y2pivfx64-0{list-style-type:none}ul.lst-kix_8i9khg2xgulr-0{list-style-type:none}.lst-kix_1lds0wnd5zeb-0>li:before{content:"\0025cf   "}.lst-kix_ls0ws55dysw8-5>li:before{content:"\0025a0   "}ul.lst-kix_8i9khg2xgulr-3{list-style-type:none}ul.lst-kix_8i9khg2xgulr-4{list-style-type:none}ul.lst-kix_8i9khg2xgulr-1{list-style-type:none}ul.lst-kix_8i9khg2xgulr-2{list-style-type:none}.lst-kix_tut3ual308bf-5>li:before{content:"\0025a0   "}.lst-kix_bo4w5l7q0tq9-1>li:before{content:"\0025cb   "}.lst-kix_ij4fg4tarntm-6>li:before{content:"\0025cf   "}.lst-kix_6kne4t6ii93d-2>li:before{content:"\0025a0   "}.lst-kix_dgmlm9mxaa4-3>li:before{content:"" counter(lst-ctn-kix_dgmlm9mxaa4-3,decimal) ". "}ol.lst-kix_ytu3x5e7mxt6-0{list-style-type:none}ol.lst-kix_ytu3x5e7mxt6-1{list-style-type:none}ul.lst-kix_jyqknjqk01cc-0{list-style-type:none}.lst-kix_129uykneptsf-5>li:before{content:"\0025a0   "}ul.lst-kix_tj1ix0gi2ui9-1{list-style-type:none}ul.lst-kix_8i9khg2xgulr-7{list-style-type:none}ul.lst-kix_70tvr3ungtsg-5{list-style-type:none}ul.lst-kix_tj1ix0gi2ui9-0{list-style-type:none}ul.lst-kix_8i9khg2xgulr-8{list-style-type:none}ul.lst-kix_70tvr3ungtsg-6{list-style-type:none}ul.lst-kix_8i9khg2xgulr-5{list-style-type:none}ul.lst-kix_70tvr3ungtsg-3{list-style-type:none}.lst-kix_e2h0cgtfdf3g-5>li:before{content:"\0025a0   "}.lst-kix_bo4w5l7q0tq9-5>li:before{content:"\0025a0   "}ul.lst-kix_8i9khg2xgulr-6{list-style-type:none}ul.lst-kix_70tvr3ungtsg-4{list-style-type:none}ul.lst-kix_tj1ix0gi2ui9-5{list-style-type:none}ul.lst-kix_tj1ix0gi2ui9-4{list-style-type:none}ul.lst-kix_tj1ix0gi2ui9-3{list-style-type:none}ul.lst-kix_70tvr3ungtsg-7{list-style-type:none}.lst-kix_eag5a795mtbr-7>li:before{content:"" counter(lst-ctn-kix_eag5a795mtbr-7,lower-latin) ". "}ul.lst-kix_tj1ix0gi2ui9-2{list-style-type:none}ul.lst-kix_70tvr3ungtsg-8{list-style-type:none}.lst-kix_k38n94udk0el-3>li:before{content:"\0025cf   "}ol.lst-kix_ytu3x5e7mxt6-2{list-style-type:none}ol.lst-kix_ytu3x5e7mxt6-3{list-style-type:none}ol.lst-kix_ytu3x5e7mxt6-4{list-style-type:none}ol.lst-kix_ytu3x5e7mxt6-5{list-style-type:none}ul.lst-kix_70tvr3ungtsg-1{list-style-type:none}ol.lst-kix_ytu3x5e7mxt6-6{list-style-type:none}ul.lst-kix_70tvr3ungtsg-2{list-style-type:none}.lst-kix_2najoyx65arn-2>li:before{content:"\0025a0   "}ol.lst-kix_ytu3x5e7mxt6-7{list-style-type:none}ol.lst-kix_ytu3x5e7mxt6-8{list-style-type:none}ul.lst-kix_70tvr3ungtsg-0{list-style-type:none}.lst-kix_jyqknjqk01cc-2>li:before{content:"\0025a0   "}.lst-kix_2najoyx65arn-8>li:before{content:"\0025a0   "}.lst-kix_wk2xuqsimxh-1>li:before{content:"\0025cb   "}.lst-kix_zqq05ofg66-2>li{counter-increment:lst-ctn-kix_zqq05ofg66-2}.lst-kix_2najoyx65arn-5>li:before{content:"\0025a0   "}.lst-kix_xq7wzd2lr8u5-7>li:before{content:"\0025cb   "}.lst-kix_jyqknjqk01cc-5>li:before{content:"\0025a0   "}.lst-kix_xq7wzd2lr8u5-1>li:before{content:"\0025cb   "}.lst-kix_wk2xuqsimxh-8>li:before{content:"\0025a0   "}.lst-kix_2fszuplxt2n9-7>li:before{content:"\0025cb   "}.lst-kix_xq7wzd2lr8u5-4>li:before{content:"\0025cb   "}.lst-kix_i5y1isusbf26-8>li:before{content:"\0025a0   "}.lst-kix_sbk9fqxb69uy-3>li{counter-increment:lst-ctn-kix_sbk9fqxb69uy-3}.lst-kix_cwtqd0lozkzu-5>li:before{content:"\0025a0   "}.lst-kix_73hytacj4fgx-4>li:before{content:"\0025cb   "}ul.lst-kix_arstwhb1c9gy-0{list-style-type:none}ul.lst-kix_ftoogxlvmqxr-7{list-style-type:none}ul.lst-kix_ftoogxlvmqxr-6{list-style-type:none}ul.lst-kix_arstwhb1c9gy-2{list-style-type:none}ul.lst-kix_arstwhb1c9gy-1{list-style-type:none}ul.lst-kix_ftoogxlvmqxr-8{list-style-type:none}.lst-kix_cwtqd0lozkzu-8>li:before{content:"\0025a0   "}ul.lst-kix_arstwhb1c9gy-4{list-style-type:none}ul.lst-kix_ftoogxlvmqxr-3{list-style-type:none}.lst-kix_85agg0xrrcik-0>li:before{content:"\0025cf   "}ul.lst-kix_arstwhb1c9gy-3{list-style-type:none}ul.lst-kix_ftoogxlvmqxr-2{list-style-type:none}ul.lst-kix_arstwhb1c9gy-6{list-style-type:none}.lst-kix_rggdlfkt6mfq-4>li{counter-increment:lst-ctn-kix_rggdlfkt6mfq-4}ul.lst-kix_ftoogxlvmqxr-5{list-style-type:none}ul.lst-kix_arstwhb1c9gy-5{list-style-type:none}ul.lst-kix_ftoogxlvmqxr-4{list-style-type:none}ul.lst-kix_129uykneptsf-8{list-style-type:none}ul.lst-kix_ftoogxlvmqxr-1{list-style-type:none}.lst-kix_73hytacj4fgx-1>li:before{content:"\0025cb   "}ul.lst-kix_ftoogxlvmqxr-0{list-style-type:none}ul.lst-kix_lkbbmj5sol47-8{list-style-type:none}ul.lst-kix_129uykneptsf-5{list-style-type:none}ul.lst-kix_lkbbmj5sol47-7{list-style-type:none}ul.lst-kix_129uykneptsf-4{list-style-type:none}ul.lst-kix_129uykneptsf-7{list-style-type:none}ul.lst-kix_129uykneptsf-6{list-style-type:none}ol.lst-kix_3qoh0wbylc0j-4.start{counter-reset:lst-ctn-kix_3qoh0wbylc0j-4 0}ul.lst-kix_lkbbmj5sol47-4{list-style-type:none}ul.lst-kix_lkbbmj5sol47-3{list-style-type:none}.lst-kix_85agg0xrrcik-6>li:before{content:"\0025cf   "}ul.lst-kix_lkbbmj5sol47-6{list-style-type:none}ul.lst-kix_lkbbmj5sol47-5{list-style-type:none}ul.lst-kix_lkbbmj5sol47-0{list-style-type:none}ul.lst-kix_lkbbmj5sol47-2{list-style-type:none}ul.lst-kix_lkbbmj5sol47-1{list-style-type:none}.lst-kix_85agg0xrrcik-3>li:before{content:"\0025cf   "}.lst-kix_19nen5usjyzs-5>li:before{content:"\0025a0   "}.lst-kix_eb0mkv6he1ge-1>li:before{content:"\0025cb   "}.lst-kix_9ei06drf1sg0-2>li:before{content:"\0025a0   "}ul.lst-kix_eunro7k2bn6e-2{list-style-type:none}ul.lst-kix_eunro7k2bn6e-1{list-style-type:none}ul.lst-kix_eunro7k2bn6e-0{list-style-type:none}.lst-kix_yxgmy3rkq7bq-2>li:before{content:"\0025a0   "}.lst-kix_3qoh0wbylc0j-7>li{counter-increment:lst-ctn-kix_3qoh0wbylc0j-7}.lst-kix_6k838qjxgg72-6>li:before{content:"\0025cf   "}.lst-kix_d7lufd20rbm5-2>li:before{content:"\0025a0   "}.lst-kix_73hytacj4fgx-7>li:before{content:"\0025cb   "}.lst-kix_nqcns3qrnch2-3>li:before{content:"\0025cf   "}.lst-kix_cwtqd0lozkzu-2>li:before{content:"\0025a0   "}.lst-kix_ytu3x5e7mxt6-4>li:before{content:"" counter(lst-ctn-kix_ytu3x5e7mxt6-4,lower-latin) ". "}.lst-kix_on63e6od9ck4-3>li:before{content:"\0025cf   "}.lst-kix_9ei06drf1sg0-8>li:before{content:"\0025a0   "}.lst-kix_74bj55g8eq2b-2>li:before{content:"\0025a0   "}.lst-kix_nqcns3qrnch2-0>li:before{content:"\0025cf   "}.lst-kix_2ws5k2m4lsl8-4>li:before{content:"\0025cb   "}ul.lst-kix_eunro7k2bn6e-6{list-style-type:none}ul.lst-kix_eunro7k2bn6e-5{list-style-type:none}.lst-kix_2fszuplxt2n9-4>li:before{content:"\0025cb   "}.lst-kix_9ei06drf1sg0-5>li:before{content:"\0025a0   "}ul.lst-kix_eunro7k2bn6e-4{list-style-type:none}.lst-kix_2ws5k2m4lsl8-1>li:before{content:"\0025cb   "}ul.lst-kix_eunro7k2bn6e-3{list-style-type:none}ul.lst-kix_eunro7k2bn6e-8{list-style-type:none}.lst-kix_vqsmxaeqhsi-0>li:before{content:"\0025cf   "}ul.lst-kix_eunro7k2bn6e-7{list-style-type:none}.lst-kix_2fszuplxt2n9-1>li:before{content:"\0025cb   "}.lst-kix_on63e6od9ck4-6>li:before{content:"\0025cf   "}.lst-kix_ytu3x5e7mxt6-7>li:before{content:"" counter(lst-ctn-kix_ytu3x5e7mxt6-7,lower-latin) ". "}.lst-kix_7g6nqlfa7rn7-4>li:before{content:"\0025cb   "}.lst-kix_qxeaj9fcw57j-1>li:before{content:"\0025cb   "}.lst-kix_ht2xpfpspp5-5>li:before{content:"\0025a0   "}.lst-kix_awa32z9rxqor-6>li:before{content:"" counter(lst-ctn-kix_awa32z9rxqor-6,decimal) ". "}.lst-kix_z1b7uf8mlquv-8>li:before{content:"\0025a0   "}ol.lst-kix_kfepckt4l6hg-5.start{counter-reset:lst-ctn-kix_kfepckt4l6hg-5 0}.lst-kix_upyk6it0f1tq-7>li:before{content:"\0025cb   "}.lst-kix_3qoh0wbylc0j-4>li{counter-increment:lst-ctn-kix_3qoh0wbylc0j-4}.lst-kix_b01r1xbffk2a-7>li:before{content:"\0025cb   "}ol.lst-kix_dgmlm9mxaa4-3{list-style-type:none}ol.lst-kix_dgmlm9mxaa4-2{list-style-type:none}.lst-kix_5s9vwy7hbq8r-1>li:before{content:"\0025cb   "}ol.lst-kix_dgmlm9mxaa4-0{list-style-type:none}.lst-kix_sr3soljrf7bf-3>li:before{content:"\0025cf   "}ol.lst-kix_dgmlm9mxaa4-7{list-style-type:none}.lst-kix_x3csall36p7p-4>li:before{content:"\0025cb   "}ol.lst-kix_dgmlm9mxaa4-6{list-style-type:none}ol.lst-kix_dgmlm9mxaa4-5{list-style-type:none}ol.lst-kix_dgmlm9mxaa4-4{list-style-type:none}.lst-kix_sbk9fqxb69uy-6>li{counter-increment:lst-ctn-kix_sbk9fqxb69uy-6}.lst-kix_v0sksjnof0cj-3>li:before{content:"\0025cf   "}.lst-kix_3aierr3stws2-0>li:before{content:"\0025cf   "}.lst-kix_6vg7rekv2ht5-7>li:before{content:"\0025cb   "}.lst-kix_9fa76bs6q7z7-4>li:before{content:"\0025cb   "}.lst-kix_eb0mkv6he1ge-4>li:before{content:"\0025cb   "}.lst-kix_zqq05ofg66-1>li:before{content:"\0025cb   "}ol.lst-kix_eag5a795mtbr-4.start{counter-reset:lst-ctn-kix_eag5a795mtbr-4 0}ol.lst-kix_dgmlm9mxaa4-8{list-style-type:none}.lst-kix_8joc3jsoa7de-3>li{counter-increment:lst-ctn-kix_8joc3jsoa7de-3}.lst-kix_19nen5usjyzs-2>li:before{content:"\0025a0   "}.lst-kix_tc68vt7q79wj-0>li:before{content:"\0025cf   "}.lst-kix_rggdlfkt6mfq-7>li{counter-increment:lst-ctn-kix_rggdlfkt6mfq-7}.lst-kix_etn6l1m36dy-4>li:before{content:"\0025cb   "}.lst-kix_xq8kn6xuala3-8>li:before{content:"\0025a0   "}.lst-kix_b0v72ufkqzg3-1>li:before{content:"\0025cb   "}ul.lst-kix_yxgmy3rkq7bq-7{list-style-type:none}.lst-kix_e2h0cgtfdf3g-2>li:before{content:"\0025a0   "}.lst-kix_b0v72ufkqzg3-7>li:before{content:"\0025cb   "}ul.lst-kix_yxgmy3rkq7bq-6{list-style-type:none}ul.lst-kix_yxgmy3rkq7bq-5{list-style-type:none}ul.lst-kix_yxgmy3rkq7bq-4{list-style-type:none}.lst-kix_6vg7rekv2ht5-1>li:before{content:"\0025cb   "}.lst-kix_kfepckt4l6hg-1>li:before{content:"" counter(lst-ctn-kix_kfepckt4l6hg-1,lower-latin) ". "}ul.lst-kix_yxgmy3rkq7bq-8{list-style-type:none}ul.lst-kix_yxgmy3rkq7bq-3{list-style-type:none}ul.lst-kix_yxgmy3rkq7bq-2{list-style-type:none}ul.lst-kix_yxgmy3rkq7bq-1{list-style-type:none}ul.lst-kix_yxgmy3rkq7bq-0{list-style-type:none}ol.lst-kix_3qoh0wbylc0j-0{list-style-type:none}ol.lst-kix_3qoh0wbylc0j-3{list-style-type:none}ol.lst-kix_3qoh0wbylc0j-4{list-style-type:none}.lst-kix_6ay9chlfxidr-6>li:before{content:"\0025cf   "}ol.lst-kix_kfepckt4l6hg-7.start{counter-reset:lst-ctn-kix_kfepckt4l6hg-7 0}ol.lst-kix_3qoh0wbylc0j-1{list-style-type:none}ol.lst-kix_3qoh0wbylc0j-2{list-style-type:none}ol.lst-kix_3qoh0wbylc0j-7{list-style-type:none}.lst-kix_tibxkvatc3h0-1>li:before{content:"\0025cb   "}.lst-kix_m75akhxjyj9r-6>li:before{content:"\0025cf   "}ol.lst-kix_3qoh0wbylc0j-8{list-style-type:none}ol.lst-kix_3qoh0wbylc0j-5{list-style-type:none}ol.lst-kix_3qoh0wbylc0j-6{list-style-type:none}.lst-kix_5mkgtms2qzgk-4>li:before{content:"\0025cb   "}.lst-kix_8r3w2rcyu87a-3>li:before{content:"\0025cf   "}.lst-kix_esoh3w8sgdnn-3>li{counter-increment:lst-ctn-kix_esoh3w8sgdnn-3}.lst-kix_j5h2oeedi37x-1>li:before{content:"\0025cb   "}.lst-kix_eag5a795mtbr-4>li{counter-increment:lst-ctn-kix_eag5a795mtbr-4}.lst-kix_k38n94udk0el-6>li:before{content:"\0025cf   "}.lst-kix_qxeaj9fcw57j-7>li:before{content:"\0025cb   "}.lst-kix_sp4bx5if0jbk-2>li:before{content:"\0025a0   "}.lst-kix_eag5a795mtbr-4>li:before{content:"" counter(lst-ctn-kix_eag5a795mtbr-4,lower-latin) ". "}.lst-kix_6ay9chlfxidr-0>li:before{content:"\0025cf   "}ol.lst-kix_eag5a795mtbr-6.start{counter-reset:lst-ctn-kix_eag5a795mtbr-6 0}.lst-kix_129uykneptsf-2>li:before{content:"\0025a0   "}.lst-kix_2fh1qudxwat7-5>li:before{content:"\0025a0   "}ol.lst-kix_eag5a795mtbr-8.start{counter-reset:lst-ctn-kix_eag5a795mtbr-8 0}.lst-kix_z4fu53gdnrww-0>li:before{content:"\0025cf   "}.lst-kix_4864lfd9zdwy-6>li:before{content:"\0025cf   "}.lst-kix_3hobsm4w6nyk-2>li:before{content:"\0025a0   "}.lst-kix_lsygr38j1y0-7>li:before{content:"\0025cb   "}.lst-kix_awa32z9rxqor-6>li{counter-increment:lst-ctn-kix_awa32z9rxqor-6}.lst-kix_n23m8p7c7hw6-3>li:before{content:"\0025cf   "}.lst-kix_tibxkvatc3h0-7>li:before{content:"\0025cb   "}.lst-kix_n23m8p7c7hw6-6>li:before{content:"\0025cf   "}.lst-kix_lsygr38j1y0-4>li:before{content:"\0025cb   "}.lst-kix_84yjavmdod6-2>li:before{content:"\0025a0   "}.lst-kix_70tvr3ungtsg-7>li:before{content:"\0025cb   "}.lst-kix_kruy5m9jmo50-3>li:before{content:"\0025cf   "}.lst-kix_70tvr3ungtsg-4>li:before{content:"\0025cb   "}.lst-kix_q432akg4led-3>li:before{content:"\0025cf   "}ul.lst-kix_tut3ual308bf-8{list-style-type:none}.lst-kix_8joc3jsoa7de-8>li:before{content:"" counter(lst-ctn-kix_8joc3jsoa7de-8,lower-roman) ". "}ul.lst-kix_tut3ual308bf-7{list-style-type:none}ul.lst-kix_tut3ual308bf-6{list-style-type:none}ul.lst-kix_tut3ual308bf-5{list-style-type:none}ul.lst-kix_tut3ual308bf-4{list-style-type:none}.lst-kix_8joc3jsoa7de-5>li:before{content:"" counter(lst-ctn-kix_8joc3jsoa7de-5,lower-roman) ". "}ul.lst-kix_tut3ual308bf-3{list-style-type:none}ul.lst-kix_tut3ual308bf-2{list-style-type:none}.lst-kix_8wqgtnyy1nox-8>li:before{content:"\0025a0   "}.lst-kix_8joc3jsoa7de-6>li{counter-increment:lst-ctn-kix_8joc3jsoa7de-6}.lst-kix_473lniz6s057-4>li:before{content:"\0025cb   "}.lst-kix_6pozql3s9ehi-3>li:before{content:"\0025cf   "}ul.lst-kix_tut3ual308bf-1{list-style-type:none}ul.lst-kix_tut3ual308bf-0{list-style-type:none}.lst-kix_kfepckt4l6hg-7>li{counter-increment:lst-ctn-kix_kfepckt4l6hg-7}.lst-kix_arstwhb1c9gy-7>li:before{content:"\0025cb   "}.lst-kix_tj1ix0gi2ui9-0>li:before{content:"\0025cf   "}.lst-kix_pb595s4qvcov-0>li:before{content:"\0025cf   "}.lst-kix_8luyzq1gqi6-1>li:before{content:"\0025cb   "}.lst-kix_auto7t5cf4tl-2>li:before{content:"\0025a0   "}.lst-kix_6pozql3s9ehi-6>li:before{content:"\0025cf   "}.lst-kix_hlp788b4qti1-7>li:before{content:"\0025cb   "}.lst-kix_h8tfxvbo4u2a-0>li:before{content:"\0025cf   "}.lst-kix_k2lnp2ex00id-8>li:before{content:"\0025a0   "}.lst-kix_473lniz6s057-7>li:before{content:"\0025cb   "}.lst-kix_q432akg4led-6>li:before{content:"\0025cf   "}.lst-kix_p73ghpm47z3w-4>li:before{content:"\0025cb   "}.lst-kix_kruy5m9jmo50-0>li:before{content:"\0025cf   "}.lst-kix_p73ghpm47z3w-7>li:before{content:"\0025cb   "}.lst-kix_haoc7b4kfhm3-4>li:before{content:"\0025cb   "}.lst-kix_tx9us7pz8zze-1>li:before{content:"\0025cb   "}.lst-kix_upyk6it0f1tq-1>li:before{content:"\0025cb   "}ul.lst-kix_6ay9chlfxidr-8{list-style-type:none}ul.lst-kix_6ay9chlfxidr-4{list-style-type:none}.lst-kix_b4xlo74k6gd0-8>li:before{content:"\0025a0   "}ul.lst-kix_6ay9chlfxidr-5{list-style-type:none}ul.lst-kix_6ay9chlfxidr-6{list-style-type:none}.lst-kix_t27vnxuzkttq-0>li:before{content:"\0025cf   "}ul.lst-kix_6ay9chlfxidr-7{list-style-type:none}ul.lst-kix_6ay9chlfxidr-0{list-style-type:none}ul.lst-kix_6ay9chlfxidr-1{list-style-type:none}.lst-kix_haoc7b4kfhm3-7>li:before{content:"\0025cb   "}.lst-kix_8yxwyie669z2-1>li:before{content:"\0025cb   "}ul.lst-kix_6ay9chlfxidr-2{list-style-type:none}ul.lst-kix_6ay9chlfxidr-3{list-style-type:none}.lst-kix_8i9khg2xgulr-0>li:before{content:"\0025cf   "}.lst-kix_ht2xpfpspp5-8>li:before{content:"\0025a0   "}ul.lst-kix_8joc3jsoa7de-1{list-style-type:none}.lst-kix_mktumxqlukrw-4>li:before{content:"\0025cb   "}.lst-kix_gp2qqn5s6so6-6>li:before{content:"\0025cf   "}.lst-kix_4y6tqwnm2166-5>li:before{content:"\0025a0   "}ul.lst-kix_wk2xuqsimxh-7{list-style-type:none}.lst-kix_qxeaj9fcw57j-4>li:before{content:"\0025cb   "}ul.lst-kix_wk2xuqsimxh-6{list-style-type:none}.lst-kix_9fa76bs6q7z7-7>li:before{content:"\0025cb   "}ul.lst-kix_wk2xuqsimxh-5{list-style-type:none}ul.lst-kix_wk2xuqsimxh-4{list-style-type:none}.lst-kix_etn6l1m36dy-1>li:before{content:"\0025cb   "}.lst-kix_w12y2pivfx64-5>li:before{content:"\0025a0   "}ul.lst-kix_wk2xuqsimxh-3{list-style-type:none}ul.lst-kix_wk2xuqsimxh-2{list-style-type:none}ul.lst-kix_wk2xuqsimxh-1{list-style-type:none}ul.lst-kix_wk2xuqsimxh-0{list-style-type:none}ul.lst-kix_9ei06drf1sg0-0{list-style-type:none}.lst-kix_ihv36zxyu5h9-2>li:before{content:"\0025a0   "}.lst-kix_7g6nqlfa7rn7-1>li:before{content:"\0025cb   "}.lst-kix_upyk6it0f1tq-4>li:before{content:"\0025cb   "}.lst-kix_iprbjac0k1x3-6>li:before{content:"\0025cf   "}.lst-kix_ytu3x5e7mxt6-2>li{counter-increment:lst-ctn-kix_ytu3x5e7mxt6-2}.lst-kix_wje7sh1pcpc1-0>li:before{content:"\0025cf   "}.lst-kix_dgmlm9mxaa4-2>li{counter-increment:lst-ctn-kix_dgmlm9mxaa4-2}.lst-kix_x3csall36p7p-7>li:before{content:"\0025cb   "}.lst-kix_5mbqwckh3266-7>li:before{content:"\0025cb   "}.lst-kix_h8tfxvbo4u2a-3>li:before{content:"\0025cf   "}.lst-kix_s4u6wv582l9n-3>li:before{content:"\0025cf   "}.lst-kix_esoh3w8sgdnn-7>li:before{content:"" counter(lst-ctn-kix_esoh3w8sgdnn-7,lower-latin) ". "}.lst-kix_v0sksjnof0cj-0>li:before{content:"\0025cf   "}.lst-kix_6vg7rekv2ht5-4>li:before{content:"\0025cb   "}ul.lst-kix_129uykneptsf-1{list-style-type:none}ul.lst-kix_1ip3hsr9v1h5-0{list-style-type:none}ul.lst-kix_129uykneptsf-0{list-style-type:none}ul.lst-kix_1ip3hsr9v1h5-1{list-style-type:none}.lst-kix_b0v72ufkqzg3-4>li:before{content:"\0025cb   "}ul.lst-kix_129uykneptsf-3{list-style-type:none}ul.lst-kix_1ip3hsr9v1h5-2{list-style-type:none}ul.lst-kix_129uykneptsf-2{list-style-type:none}ul.lst-kix_1ip3hsr9v1h5-3{list-style-type:none}.lst-kix_2tnhjv96pndg-4>li:before{content:"\0025cb   "}ul.lst-kix_1ip3hsr9v1h5-8{list-style-type:none}ul.lst-kix_arstwhb1c9gy-8{list-style-type:none}ul.lst-kix_arstwhb1c9gy-7{list-style-type:none}ul.lst-kix_1ip3hsr9v1h5-4{list-style-type:none}ul.lst-kix_1ip3hsr9v1h5-5{list-style-type:none}ul.lst-kix_1ip3hsr9v1h5-6{list-style-type:none}ul.lst-kix_1ip3hsr9v1h5-7{list-style-type:none}.lst-kix_sr3soljrf7bf-0>li:before{content:"\0025cf   "}.lst-kix_5mkgtms2qzgk-1>li:before{content:"\0025cb   "}.lst-kix_ftoogxlvmqxr-3>li:before{content:"\0025cf   "}ul.lst-kix_ls0ws55dysw8-6{list-style-type:none}ul.lst-kix_ls0ws55dysw8-5{list-style-type:none}ul.lst-kix_ls0ws55dysw8-4{list-style-type:none}.lst-kix_fd9jvhtv6teb-2>li:before{content:"\0025a0   "}ul.lst-kix_ls0ws55dysw8-3{list-style-type:none}ul.lst-kix_ls0ws55dysw8-2{list-style-type:none}ul.lst-kix_ls0ws55dysw8-1{list-style-type:none}.lst-kix_umwvyipzf505-4>li:before{content:"\0025cb   "}ul.lst-kix_ls0ws55dysw8-0{list-style-type:none}.lst-kix_8r3w2rcyu87a-0>li:before{content:"\0025cf   "}.lst-kix_vams35lnoe4-2>li:before{content:"\0025a0   "}ul.lst-kix_ls0ws55dysw8-8{list-style-type:none}ul.lst-kix_ls0ws55dysw8-7{list-style-type:none}ul.lst-kix_9ei06drf1sg0-7{list-style-type:none}ul.lst-kix_9ei06drf1sg0-8{list-style-type:none}.lst-kix_esoh3w8sgdnn-0>li{counter-increment:lst-ctn-kix_esoh3w8sgdnn-0}.lst-kix_6ay9chlfxidr-3>li:before{content:"\0025cf   "}ul.lst-kix_9ei06drf1sg0-5{list-style-type:none}.lst-kix_tibxkvatc3h0-4>li:before{content:"\0025cb   "}ul.lst-kix_9ei06drf1sg0-6{list-style-type:none}ul.lst-kix_9ei06drf1sg0-3{list-style-type:none}ul.lst-kix_9ei06drf1sg0-4{list-style-type:none}ul.lst-kix_9ei06drf1sg0-1{list-style-type:none}.lst-kix_eag5a795mtbr-1>li:before{content:"" counter(lst-ctn-kix_eag5a795mtbr-1,lower-latin) ". "}ul.lst-kix_9ei06drf1sg0-2{list-style-type:none}.lst-kix_2fh1qudxwat7-8>li:before{content:"\0025a0   "}.lst-kix_eag5a795mtbr-7>li{counter-increment:lst-ctn-kix_eag5a795mtbr-7}.lst-kix_1lds0wnd5zeb-6>li:before{content:"\0025cf   "}ol.lst-kix_eag5a795mtbr-8{list-style-type:none}ol.lst-kix_eag5a795mtbr-5{list-style-type:none}ol.lst-kix_eag5a795mtbr-4{list-style-type:none}ol.lst-kix_eag5a795mtbr-7{list-style-type:none}ol.lst-kix_eag5a795mtbr-6{list-style-type:none}ol.lst-kix_eag5a795mtbr-1{list-style-type:none}ol.lst-kix_eag5a795mtbr-0{list-style-type:none}ol.lst-kix_eag5a795mtbr-3{list-style-type:none}ol.lst-kix_eag5a795mtbr-2{list-style-type:none}ul.lst-kix_p73ghpm47z3w-8{list-style-type:none}.lst-kix_lkbbmj5sol47-6>li:before{content:"\0025cf   "}ul.lst-kix_k2lnp2ex00id-1{list-style-type:none}ul.lst-kix_p73ghpm47z3w-4{list-style-type:none}.lst-kix_lkbbmj5sol47-8>li:before{content:"\0025a0   "}ul.lst-kix_k2lnp2ex00id-0{list-style-type:none}ul.lst-kix_p73ghpm47z3w-5{list-style-type:none}ul.lst-kix_k2lnp2ex00id-3{list-style-type:none}ul.lst-kix_p73ghpm47z3w-6{list-style-type:none}ul.lst-kix_k2lnp2ex00id-2{list-style-type:none}ul.lst-kix_p73ghpm47z3w-7{list-style-type:none}ul.lst-kix_k2lnp2ex00id-5{list-style-type:none}ul.lst-kix_p73ghpm47z3w-0{list-style-type:none}ul.lst-kix_k2lnp2ex00id-4{list-style-type:none}ul.lst-kix_p73ghpm47z3w-1{list-style-type:none}ol.lst-kix_eag5a795mtbr-5.start{counter-reset:lst-ctn-kix_eag5a795mtbr-5 0}ul.lst-kix_k2lnp2ex00id-7{list-style-type:none}ul.lst-kix_p73ghpm47z3w-2{list-style-type:none}.lst-kix_rggdlfkt6mfq-2>li{counter-increment:lst-ctn-kix_rggdlfkt6mfq-2}ul.lst-kix_k2lnp2ex00id-6{list-style-type:none}ul.lst-kix_p73ghpm47z3w-3{list-style-type:none}.lst-kix_rggdlfkt6mfq-6>li:before{content:"" counter(lst-ctn-kix_rggdlfkt6mfq-6,decimal) ". "}ul.lst-kix_k2lnp2ex00id-8{list-style-type:none}ul.lst-kix_73hytacj4fgx-5{list-style-type:none}ul.lst-kix_73hytacj4fgx-6{list-style-type:none}ul.lst-kix_73hytacj4fgx-7{list-style-type:none}.lst-kix_lcass77ih41v-1>li:before{content:"\0025cb   "}ul.lst-kix_73hytacj4fgx-8{list-style-type:none}.lst-kix_uilbxigghq1p-4>li:before{content:"\0025cb   "}.lst-kix_uilbxigghq1p-8>li:before{content:"\0025a0   "}ul.lst-kix_73hytacj4fgx-1{list-style-type:none}ul.lst-kix_73hytacj4fgx-2{list-style-type:none}.lst-kix_rggdlfkt6mfq-2>li:before{content:"" counter(lst-ctn-kix_rggdlfkt6mfq-2,lower-roman) ". "}ul.lst-kix_73hytacj4fgx-3{list-style-type:none}ul.lst-kix_73hytacj4fgx-4{list-style-type:none}.lst-kix_rggdlfkt6mfq-4>li:before{content:"" counter(lst-ctn-kix_rggdlfkt6mfq-4,lower-latin) ". "}.lst-kix_lcass77ih41v-3>li:before{content:"\0025cf   "}ul.lst-kix_73hytacj4fgx-0{list-style-type:none}.lst-kix_uilbxigghq1p-6>li:before{content:"\0025cf   "}ul.lst-kix_ht2xpfpspp5-8{list-style-type:none}ul.lst-kix_ht2xpfpspp5-7{list-style-type:none}.lst-kix_lcass77ih41v-5>li:before{content:"\0025a0   "}.lst-kix_uilbxigghq1p-0>li:before{content:"\0025cf   "}ul.lst-kix_ht2xpfpspp5-0{list-style-type:none}ul.lst-kix_ht2xpfpspp5-2{list-style-type:none}ul.lst-kix_ht2xpfpspp5-1{list-style-type:none}ul.lst-kix_ht2xpfpspp5-4{list-style-type:none}.lst-kix_rggdlfkt6mfq-8>li:before{content:"" counter(lst-ctn-kix_rggdlfkt6mfq-8,lower-roman) ". "}ul.lst-kix_ht2xpfpspp5-3{list-style-type:none}ul.lst-kix_ht2xpfpspp5-6{list-style-type:none}.lst-kix_lcass77ih41v-7>li:before{content:"\0025cb   "}ul.lst-kix_ht2xpfpspp5-5{list-style-type:none}.lst-kix_uilbxigghq1p-2>li:before{content:"\0025a0   "}.lst-kix_rggdlfkt6mfq-0>li:before{content:"" counter(lst-ctn-kix_rggdlfkt6mfq-0,decimal) ". "}.lst-kix_3qoh0wbylc0j-7>li:before{content:"" counter(lst-ctn-kix_3qoh0wbylc0j-7,lower-latin) ". "}ul.lst-kix_n23m8p7c7hw6-7{list-style-type:none}ul.lst-kix_n23m8p7c7hw6-8{list-style-type:none}ul.lst-kix_n23m8p7c7hw6-5{list-style-type:none}.lst-kix_3qoh0wbylc0j-3>li:before{content:"" counter(lst-ctn-kix_3qoh0wbylc0j-3,decimal) ". "}ul.lst-kix_n23m8p7c7hw6-6{list-style-type:none}ul.lst-kix_n23m8p7c7hw6-3{list-style-type:none}ul.lst-kix_n23m8p7c7hw6-4{list-style-type:none}ul.lst-kix_n23m8p7c7hw6-1{list-style-type:none}.lst-kix_3qoh0wbylc0j-5>li:before{content:"" counter(lst-ctn-kix_3qoh0wbylc0j-5,lower-roman) ". "}ul.lst-kix_n23m8p7c7hw6-2{list-style-type:none}ul.lst-kix_n23m8p7c7hw6-0{list-style-type:none}.lst-kix_sbk9fqxb69uy-1>li{counter-increment:lst-ctn-kix_sbk9fqxb69uy-1}ul.lst-kix_4y6tqwnm2166-0{list-style-type:none}.lst-kix_lkbbmj5sol47-4>li:before{content:"\0025cb   "}ul.lst-kix_4y6tqwnm2166-1{list-style-type:none}.lst-kix_xmm4jxv5hwlq-3>li:before{content:"\0025cf   "}.lst-kix_lkbbmj5sol47-2>li:before{content:"\0025a0   "}ul.lst-kix_dgmlm9mxaa4-1{list-style-type:none}.lst-kix_awa32z9rxqor-7>li{counter-increment:lst-ctn-kix_awa32z9rxqor-7}ul.lst-kix_4y6tqwnm2166-8{list-style-type:none}.lst-kix_lkbbmj5sol47-0>li:before{content:"\0025cf   "}ul.lst-kix_4y6tqwnm2166-6{list-style-type:none}.lst-kix_xmm4jxv5hwlq-5>li:before{content:"\0025a0   "}.lst-kix_xmm4jxv5hwlq-7>li:before{content:"\0025cb   "}ul.lst-kix_4y6tqwnm2166-7{list-style-type:none}ul.lst-kix_4y6tqwnm2166-4{list-style-type:none}ul.lst-kix_4y6tqwnm2166-5{list-style-type:none}.lst-kix_3qoh0wbylc0j-1>li:before{content:"" counter(lst-ctn-kix_3qoh0wbylc0j-1,lower-latin) ". "}.lst-kix_gp2qqn5s6so6-8>li:before{content:"\0025a0   "}ul.lst-kix_4y6tqwnm2166-2{list-style-type:none}ul.lst-kix_4y6tqwnm2166-3{list-style-type:none}.lst-kix_ie2cjnul3ua-7>li:before{content:"\0025cb   "}.lst-kix_ie2cjnul3ua-5>li:before{content:"\0025a0   "}ol.lst-kix_3qoh0wbylc0j-0.start{counter-reset:lst-ctn-kix_3qoh0wbylc0j-0 0}.lst-kix_dgmlm9mxaa4-0>li{counter-increment:lst-ctn-kix_dgmlm9mxaa4-0}.lst-kix_6z3d30mc5xph-2>li:before{content:"\0025a0   "}ol.lst-kix_sbk9fqxb69uy-5.start{counter-reset:lst-ctn-kix_sbk9fqxb69uy-5 0}.lst-kix_ytu3x5e7mxt6-5>li{counter-increment:lst-ctn-kix_ytu3x5e7mxt6-5}.lst-kix_iprbjac0k1x3-3>li:before{content:"\0025cf   "}.lst-kix_iprbjac0k1x3-7>li:before{content:"\0025cb   "}.lst-kix_umwvyipzf505-1>li:before{content:"\0025cb   "}.lst-kix_6z3d30mc5xph-4>li:before{content:"\0025cb   "}.lst-kix_iprbjac0k1x3-1>li:before{content:"\0025cb   "}.lst-kix_wje7sh1pcpc1-1>li:before{content:"\0025cb   "}.lst-kix_tkg3l6cy3bnc-7>li{counter-increment:lst-ctn-kix_tkg3l6cy3bnc-7}.lst-kix_s4u6wv582l9n-0>li:before{content:"\0025cf   "}.lst-kix_wje7sh1pcpc1-3>li:before{content:"\0025cf   "}.lst-kix_wje7sh1pcpc1-7>li:before{content:"\0025cb   "}.lst-kix_6z3d30mc5xph-0>li:before{content:"\0025cf   "}.lst-kix_iprbjac0k1x3-5>li:before{content:"\0025a0   "}.lst-kix_s4u6wv582l9n-2>li:before{content:"\0025a0   "}.lst-kix_wje7sh1pcpc1-5>li:before{content:"\0025a0   "}.lst-kix_ftoogxlvmqxr-8>li:before{content:"\0025a0   "}.lst-kix_s4u6wv582l9n-4>li:before{content:"\0025cb   "}.lst-kix_s4u6wv582l9n-8>li:before{content:"\0025a0   "}ol.lst-kix_tkg3l6cy3bnc-3.start{counter-reset:lst-ctn-kix_tkg3l6cy3bnc-3 0}ol.lst-kix_awa32z9rxqor-4.start{counter-reset:lst-ctn-kix_awa32z9rxqor-4 0}.lst-kix_ftoogxlvmqxr-4>li:before{content:"\0025cb   "}.lst-kix_ftoogxlvmqxr-6>li:before{content:"\0025cf   "}.lst-kix_umwvyipzf505-7>li:before{content:"\0025cb   "}.lst-kix_s4u6wv582l9n-6>li:before{content:"\0025cf   "}.lst-kix_umwvyipzf505-5>li:before{content:"\0025a0   "}.lst-kix_ftoogxlvmqxr-0>li:before{content:"\0025cf   "}.lst-kix_ftoogxlvmqxr-2>li:before{content:"\0025a0   "}.lst-kix_6z3d30mc5xph-6>li:before{content:"\0025cf   "}.lst-kix_vams35lnoe4-7>li:before{content:"\0025cb   "}.lst-kix_umwvyipzf505-3>li:before{content:"\0025cf   "}.lst-kix_6z3d30mc5xph-8>li:before{content:"\0025a0   "}.lst-kix_vams35lnoe4-3>li:before{content:"\0025cf   "}.lst-kix_vams35lnoe4-1>li:before{content:"\0025cb   "}.lst-kix_vams35lnoe4-5>li:before{content:"\0025a0   "}.lst-kix_tkg3l6cy3bnc-0>li{counter-increment:lst-ctn-kix_tkg3l6cy3bnc-0}.lst-kix_lsygr38j1y0-1>li:before{content:"\0025cb   "}ol.lst-kix_8joc3jsoa7de-3.start{counter-reset:lst-ctn-kix_8joc3jsoa7de-3 0}ol.lst-kix_ytu3x5e7mxt6-5.start{counter-reset:lst-ctn-kix_ytu3x5e7mxt6-5 0}.lst-kix_n23m8p7c7hw6-0>li:before{content:"\0025cf   "}ul.lst-kix_d7lufd20rbm5-0{list-style-type:none}.lst-kix_n23m8p7c7hw6-4>li:before{content:"\0025cb   "}.lst-kix_4864lfd9zdwy-0>li:before{content:"\0025cf   "}.lst-kix_4864lfd9zdwy-8>li:before{content:"\0025a0   "}ul.lst-kix_d7lufd20rbm5-6{list-style-type:none}ul.lst-kix_d7lufd20rbm5-5{list-style-type:none}ul.lst-kix_d7lufd20rbm5-8{list-style-type:none}.lst-kix_n23m8p7c7hw6-8>li:before{content:"\0025a0   "}ul.lst-kix_d7lufd20rbm5-7{list-style-type:none}ul.lst-kix_d7lufd20rbm5-2{list-style-type:none}ul.lst-kix_d7lufd20rbm5-1{list-style-type:none}ul.lst-kix_d7lufd20rbm5-4{list-style-type:none}ul.lst-kix_d7lufd20rbm5-3{list-style-type:none}.lst-kix_lsygr38j1y0-5>li:before{content:"\0025a0   "}.lst-kix_4864lfd9zdwy-4>li:before{content:"\0025cb   "}.lst-kix_eag5a795mtbr-8>li{counter-increment:lst-ctn-kix_eag5a795mtbr-8}.lst-kix_kruy5m9jmo50-2>li:before{content:"\0025a0   "}.lst-kix_z4fu53gdnrww-6>li:before{content:"\0025cf   "}.lst-kix_q432akg4led-0>li:before{content:"\0025cf   "}.lst-kix_kruy5m9jmo50-6>li:before{content:"\0025cf   "}ul.lst-kix_v0sksjnof0cj-8{list-style-type:none}.lst-kix_z4fu53gdnrww-2>li:before{content:"\0025a0   "}ul.lst-kix_v0sksjnof0cj-6{list-style-type:none}ul.lst-kix_v0sksjnof0cj-7{list-style-type:none}ul.lst-kix_s4u6wv582l9n-5{list-style-type:none}ul.lst-kix_v0sksjnof0cj-0{list-style-type:none}ul.lst-kix_s4u6wv582l9n-4{list-style-type:none}ul.lst-kix_v0sksjnof0cj-1{list-style-type:none}ul.lst-kix_s4u6wv582l9n-3{list-style-type:none}ul.lst-kix_s4u6wv582l9n-2{list-style-type:none}ol.lst-kix_rggdlfkt6mfq-4.start{counter-reset:lst-ctn-kix_rggdlfkt6mfq-4 0}ul.lst-kix_s4u6wv582l9n-1{list-style-type:none}ul.lst-kix_v0sksjnof0cj-4{list-style-type:none}.lst-kix_8luyzq1gqi6-0>li:before{content:"\0025cf   "}ul.lst-kix_s4u6wv582l9n-0{list-style-type:none}ul.lst-kix_v0sksjnof0cj-5{list-style-type:none}ul.lst-kix_v0sksjnof0cj-2{list-style-type:none}ol.lst-kix_ytu3x5e7mxt6-0.start{counter-reset:lst-ctn-kix_ytu3x5e7mxt6-0 0}ul.lst-kix_v0sksjnof0cj-3{list-style-type:none}.lst-kix_auto7t5cf4tl-0>li:before{content:"\0025cf   "}.lst-kix_t27vnxuzkttq-6>li:before{content:"\0025cf   "}ul.lst-kix_s4u6wv582l9n-8{list-style-type:none}.lst-kix_tj1ix0gi2ui9-7>li:before{content:"\0025cb   "}ul.lst-kix_s4u6wv582l9n-7{list-style-type:none}ul.lst-kix_s4u6wv582l9n-6{list-style-type:none}ol.lst-kix_zqq05ofg66-0.start{counter-reset:lst-ctn-kix_zqq05ofg66-0 0}.lst-kix_h8tfxvbo4u2a-1>li:before{content:"\0025cb   "}.lst-kix_q432akg4led-4>li:before{content:"\0025cb   "}.lst-kix_8luyzq1gqi6-4>li:before{content:"\0025cb   "}.lst-kix_8luyzq1gqi6-8>li:before{content:"\0025a0   "}.lst-kix_v0sksjnof0cj-5>li:before{content:"\0025a0   "}.lst-kix_p73ghpm47z3w-1>li:before{content:"\0025cb   "}.lst-kix_x3csall36p7p-1>li:before{content:"\0025cb   "}.lst-kix_tj1ix0gi2ui9-3>li:before{content:"\0025cf   "}.lst-kix_haoc7b4kfhm3-2>li:before{content:"\0025a0   "}.lst-kix_p73ghpm47z3w-5>li:before{content:"\0025a0   "}.lst-kix_q432akg4led-8>li:before{content:"\0025a0   "}.lst-kix_an05xfuo7mar-5>li:before{content:"\0025a0   "}.lst-kix_sbk9fqxb69uy-2>li:before{content:"" counter(lst-ctn-kix_sbk9fqxb69uy-2,lower-roman) ". "}.lst-kix_auto7t5cf4tl-8>li:before{content:"\0025a0   "}ol.lst-kix_zqq05ofg66-5.start{counter-reset:lst-ctn-kix_zqq05ofg66-5 0}ul.lst-kix_blb26b155eyp-7{list-style-type:none}ul.lst-kix_blb26b155eyp-6{list-style-type:none}.lst-kix_auto7t5cf4tl-4>li:before{content:"\0025cb   "}.lst-kix_t27vnxuzkttq-2>li:before{content:"\0025a0   "}ul.lst-kix_blb26b155eyp-5{list-style-type:none}ul.lst-kix_blb26b155eyp-4{list-style-type:none}ul.lst-kix_blb26b155eyp-3{list-style-type:none}.lst-kix_an05xfuo7mar-1>li:before{content:"\0025cb   "}ul.lst-kix_blb26b155eyp-2{list-style-type:none}.lst-kix_sbk9fqxb69uy-6>li:before{content:"" counter(lst-ctn-kix_sbk9fqxb69uy-6,decimal) ". "}ul.lst-kix_blb26b155eyp-1{list-style-type:none}ul.lst-kix_blb26b155eyp-0{list-style-type:none}.lst-kix_haoc7b4kfhm3-6>li:before{content:"\0025cf   "}.lst-kix_ie2cjnul3ua-3>li:before{content:"\0025cf   "}ol.lst-kix_esoh3w8sgdnn-6.start{counter-reset:lst-ctn-kix_esoh3w8sgdnn-6 0}ul.lst-kix_blb26b155eyp-8{list-style-type:none}ul.lst-kix_esoh3w8sgdnn-1{list-style-type:none}ul.lst-kix_xq8kn6xuala3-0{list-style-type:none}ul.lst-kix_xq8kn6xuala3-1{list-style-type:none}ul.lst-kix_xq8kn6xuala3-2{list-style-type:none}ul.lst-kix_mktumxqlukrw-0{list-style-type:none}.lst-kix_mktumxqlukrw-6>li:before{content:"\0025cf   "}.lst-kix_gp2qqn5s6so6-4>li:before{content:"\0025cb   "}.lst-kix_tkg3l6cy3bnc-1>li:before{content:"\0025cb   "}.lst-kix_vqsmxaeqhsi-3>li:before{content:"\0025cf   "}ul.lst-kix_9fa76bs6q7z7-3{list-style-type:none}ul.lst-kix_9fa76bs6q7z7-2{list-style-type:none}ul.lst-kix_9fa76bs6q7z7-1{list-style-type:none}ul.lst-kix_9fa76bs6q7z7-0{list-style-type:none}ul.lst-kix_9fa76bs6q7z7-7{list-style-type:none}.lst-kix_vqsmxaeqhsi-7>li:before{content:"\0025cb   "}ul.lst-kix_xq8kn6xuala3-3{list-style-type:none}ul.lst-kix_9fa76bs6q7z7-6{list-style-type:none}ul.lst-kix_xq8kn6xuala3-4{list-style-type:none}ul.lst-kix_9fa76bs6q7z7-5{list-style-type:none}ul.lst-kix_xq8kn6xuala3-5{list-style-type:none}ul.lst-kix_9fa76bs6q7z7-4{list-style-type:none}ul.lst-kix_xq8kn6xuala3-6{list-style-type:none}ul.lst-kix_xq8kn6xuala3-7{list-style-type:none}.lst-kix_mktumxqlukrw-2>li:before{content:"\0025a0   "}.lst-kix_gp2qqn5s6so6-0>li:before{content:"\0025cf   "}ul.lst-kix_xq8kn6xuala3-8{list-style-type:none}ul.lst-kix_9fa76bs6q7z7-8{list-style-type:none}.lst-kix_x3csall36p7p-5>li:before{content:"\0025a0   "}ul.lst-kix_5s9vwy7hbq8r-0{list-style-type:none}ul.lst-kix_5s9vwy7hbq8r-1{list-style-type:none}ul.lst-kix_5s9vwy7hbq8r-4{list-style-type:none}ul.lst-kix_5s9vwy7hbq8r-5{list-style-type:none}ul.lst-kix_5s9vwy7hbq8r-2{list-style-type:none}ul.lst-kix_5s9vwy7hbq8r-3{list-style-type:none}.lst-kix_etn6l1m36dy-7>li:before{content:"\0025cb   "}ul.lst-kix_5s9vwy7hbq8r-8{list-style-type:none}ol.lst-kix_kfepckt4l6hg-2.start{counter-reset:lst-ctn-kix_kfepckt4l6hg-2 0}ul.lst-kix_5s9vwy7hbq8r-6{list-style-type:none}.lst-kix_sr3soljrf7bf-2>li:before{content:"\0025a0   "}ul.lst-kix_5s9vwy7hbq8r-7{list-style-type:none}ul.lst-kix_mktumxqlukrw-7{list-style-type:none}ul.lst-kix_mktumxqlukrw-8{list-style-type:none}ul.lst-kix_mktumxqlukrw-5{list-style-type:none}ul.lst-kix_mktumxqlukrw-6{list-style-type:none}.lst-kix_etn6l1m36dy-3>li:before{content:"\0025cf   "}.lst-kix_xq8kn6xuala3-5>li:before{content:"\0025a0   "}ul.lst-kix_mktumxqlukrw-3{list-style-type:none}ul.lst-kix_mktumxqlukrw-4{list-style-type:none}ul.lst-kix_mktumxqlukrw-1{list-style-type:none}.lst-kix_awa32z9rxqor-0>li{counter-increment:lst-ctn-kix_awa32z9rxqor-0}ul.lst-kix_mktumxqlukrw-2{list-style-type:none}.lst-kix_esoh3w8sgdnn-5>li:before{content:"" counter(lst-ctn-kix_esoh3w8sgdnn-5,lower-roman) ". "}.lst-kix_h8tfxvbo4u2a-5>li:before{content:"\0025a0   "}.lst-kix_sr3soljrf7bf-6>li:before{content:"\0025cf   "}.lst-kix_tkg3l6cy3bnc-5>li:before{content:"" counter(lst-ctn-kix_tkg3l6cy3bnc-5,lower-roman) ". "}.lst-kix_esoh3w8sgdnn-1>li:before{content:"\0025cb   "}.lst-kix_fd9jvhtv6teb-4>li:before{content:"\0025cb   "}.lst-kix_8joc3jsoa7de-5>li{counter-increment:lst-ctn-kix_8joc3jsoa7de-5}ol.lst-kix_8joc3jsoa7de-8.start{counter-reset:lst-ctn-kix_8joc3jsoa7de-8 0}.lst-kix_blb26b155eyp-0>li:before{content:"\0025cf   "}ul.lst-kix_81hzypied9ul-8{list-style-type:none}ul.lst-kix_81hzypied9ul-7{list-style-type:none}ul.lst-kix_dmy2ekb87fbb-5{list-style-type:none}ul.lst-kix_dmy2ekb87fbb-4{list-style-type:none}ul.lst-kix_dmy2ekb87fbb-7{list-style-type:none}.lst-kix_fd9jvhtv6teb-0>li:before{content:"\0025cf   "}.lst-kix_fd9jvhtv6teb-8>li:before{content:"\0025a0   "}ul.lst-kix_dmy2ekb87fbb-6{list-style-type:none}ul.lst-kix_dmy2ekb87fbb-8{list-style-type:none}ul.lst-kix_dmy2ekb87fbb-1{list-style-type:none}.lst-kix_blb26b155eyp-8>li:before{content:"\0025a0   "}ul.lst-kix_dmy2ekb87fbb-0{list-style-type:none}.lst-kix_blb26b155eyp-4>li:before{content:"\0025cb   "}ul.lst-kix_dmy2ekb87fbb-3{list-style-type:none}ol.lst-kix_3qoh0wbylc0j-5.start{counter-reset:lst-ctn-kix_3qoh0wbylc0j-5 0}ul.lst-kix_dmy2ekb87fbb-2{list-style-type:none}ul.lst-kix_81hzypied9ul-6{list-style-type:none}ol.lst-kix_kfepckt4l6hg-0.start{counter-reset:lst-ctn-kix_kfepckt4l6hg-0 0}ul.lst-kix_81hzypied9ul-5{list-style-type:none}ol.lst-kix_eag5a795mtbr-0.start{counter-reset:lst-ctn-kix_eag5a795mtbr-0 0}ul.lst-kix_81hzypied9ul-4{list-style-type:none}ul.lst-kix_81hzypied9ul-3{list-style-type:none}.lst-kix_xq8kn6xuala3-1>li:before{content:"\0025cb   "}ul.lst-kix_81hzypied9ul-2{list-style-type:none}ul.lst-kix_81hzypied9ul-1{list-style-type:none}ol.lst-kix_rggdlfkt6mfq-1.start{counter-reset:lst-ctn-kix_rggdlfkt6mfq-1 0}ul.lst-kix_81hzypied9ul-0{list-style-type:none}ul.lst-kix_z4fu53gdnrww-8{list-style-type:none}ul.lst-kix_z4fu53gdnrww-7{list-style-type:none}ul.lst-kix_z4fu53gdnrww-6{list-style-type:none}ul.lst-kix_z4fu53gdnrww-5{list-style-type:none}ul.lst-kix_z4fu53gdnrww-4{list-style-type:none}ul.lst-kix_z4fu53gdnrww-3{list-style-type:none}ul.lst-kix_z4fu53gdnrww-2{list-style-type:none}ul.lst-kix_z4fu53gdnrww-1{list-style-type:none}ul.lst-kix_z4fu53gdnrww-0{list-style-type:none}.lst-kix_sp4bx5if0jbk-5>li:before{content:"\0025a0   "}.lst-kix_eag5a795mtbr-1>li{counter-increment:lst-ctn-kix_eag5a795mtbr-1}.lst-kix_90vpss294crz-2>li:before{content:"\0025a0   "}.lst-kix_sp4bx5if0jbk-1>li:before{content:"\0025cb   "}.lst-kix_2fh1qudxwat7-2>li:before{content:"\0025a0   "}.lst-kix_2fh1qudxwat7-6>li:before{content:"\0025cf   "}ol.lst-kix_zqq05ofg66-2.start{counter-reset:lst-ctn-kix_zqq05ofg66-2 0}.lst-kix_90vpss294crz-6>li:before{content:"\0025cf   "}.lst-kix_i5y1isusbf26-1>li:before{content:"\0025cb   "}ul.lst-kix_h8tfxvbo4u2a-8{list-style-type:none}.lst-kix_i5y1isusbf26-5>li:before{content:"\0025a0   "}.lst-kix_xq7wzd2lr8u5-8>li:before{content:"\0025a0   "}ul.lst-kix_h8tfxvbo4u2a-5{list-style-type:none}ul.lst-kix_5mbqwckh3266-0{list-style-type:none}ul.lst-kix_h8tfxvbo4u2a-4{list-style-type:none}.lst-kix_i5y1isusbf26-7>li:before{content:"\0025cb   "}ul.lst-kix_5mbqwckh3266-1{list-style-type:none}ul.lst-kix_h8tfxvbo4u2a-7{list-style-type:none}ul.lst-kix_h8tfxvbo4u2a-6{list-style-type:none}.lst-kix_jyqknjqk01cc-6>li:before{content:"\0025cf   "}ul.lst-kix_e134675mgdj3-6{list-style-type:none}.lst-kix_2najoyx65arn-4>li:before{content:"\0025cb   "}ul.lst-kix_e134675mgdj3-5{list-style-type:none}.lst-kix_wk2xuqsimxh-5>li:before{content:"\0025a0   "}ul.lst-kix_e134675mgdj3-8{list-style-type:none}ul.lst-kix_e134675mgdj3-7{list-style-type:none}.lst-kix_wk2xuqsimxh-7>li:before{content:"\0025cb   "}.lst-kix_xq7wzd2lr8u5-0>li:before{content:"\0025cf   "}.lst-kix_xq7wzd2lr8u5-2>li:before{content:"\0025a0   "}ul.lst-kix_e134675mgdj3-0{list-style-type:none}.lst-kix_129uykneptsf-6>li:before{content:"\0025cf   "}.lst-kix_129uykneptsf-8>li:before{content:"\0025a0   "}.lst-kix_8joc3jsoa7de-7>li{counter-increment:lst-ctn-kix_8joc3jsoa7de-7}ul.lst-kix_e134675mgdj3-2{list-style-type:none}ul.lst-kix_e134675mgdj3-1{list-style-type:none}ul.lst-kix_e134675mgdj3-4{list-style-type:none}ol.lst-kix_ytu3x5e7mxt6-2.start{counter-reset:lst-ctn-kix_ytu3x5e7mxt6-2 0}ul.lst-kix_e134675mgdj3-3{list-style-type:none}ul.lst-kix_5mbqwckh3266-4{list-style-type:none}.lst-kix_2fszuplxt2n9-8>li:before{content:"\0025a0   "}ul.lst-kix_5mbqwckh3266-5{list-style-type:none}ul.lst-kix_5mbqwckh3266-2{list-style-type:none}ul.lst-kix_5mbqwckh3266-3{list-style-type:none}.lst-kix_jyqknjqk01cc-0>li:before{content:"\0025cf   "}ul.lst-kix_5mbqwckh3266-8{list-style-type:none}ul.lst-kix_5mbqwckh3266-6{list-style-type:none}.lst-kix_2ws5k2m4lsl8-8>li:before{content:"\0025a0   "}ul.lst-kix_5mbqwckh3266-7{list-style-type:none}.lst-kix_awa32z9rxqor-5>li{counter-increment:lst-ctn-kix_awa32z9rxqor-5}.lst-kix_73hytacj4fgx-3>li:before{content:"\0025cf   "}.lst-kix_73hytacj4fgx-5>li:before{content:"\0025a0   "}.lst-kix_cwtqd0lozkzu-4>li:before{content:"\0025cb   "}.lst-kix_cwtqd0lozkzu-6>li:before{content:"\0025cf   "}ol.lst-kix_dgmlm9mxaa4-8.start{counter-reset:lst-ctn-kix_dgmlm9mxaa4-8 0}.lst-kix_yxgmy3rkq7bq-3>li:before{content:"\0025cf   "}.lst-kix_nqcns3qrnch2-7>li:before{content:"\0025cb   "}.lst-kix_yxgmy3rkq7bq-5>li:before{content:"\0025a0   "}.lst-kix_85agg0xrrcik-1>li:before{content:"\0025cb   "}ul.lst-kix_h8tfxvbo4u2a-1{list-style-type:none}ul.lst-kix_h8tfxvbo4u2a-0{list-style-type:none}ul.lst-kix_h8tfxvbo4u2a-3{list-style-type:none}ul.lst-kix_h8tfxvbo4u2a-2{list-style-type:none}.lst-kix_jyqknjqk01cc-8>li:before{content:"\0025a0   "}ol.lst-kix_zqq05ofg66-7.start{counter-reset:lst-ctn-kix_zqq05ofg66-7 0}ul.lst-kix_b01r1xbffk2a-0{list-style-type:none}.lst-kix_85agg0xrrcik-7>li:before{content:"\0025cb   "}ul.lst-kix_b01r1xbffk2a-1{list-style-type:none}ul.lst-kix_b01r1xbffk2a-2{list-style-type:none}.lst-kix_eb0mkv6he1ge-0>li:before{content:"\0025cf   "}.lst-kix_eb0mkv6he1ge-2>li:before{content:"\0025a0   "}.lst-kix_6k838qjxgg72-5>li:before{content:"\0025a0   "}.lst-kix_9ei06drf1sg0-1>li:before{content:"\0025cb   "}ol.lst-kix_tkg3l6cy3bnc-0.start{counter-reset:lst-ctn-kix_tkg3l6cy3bnc-0 0}.lst-kix_6k838qjxgg72-3>li:before{content:"\0025cf   "}ul.lst-kix_b01r1xbffk2a-7{list-style-type:none}.lst-kix_d7lufd20rbm5-3>li:before{content:"\0025cf   "}.lst-kix_ytu3x5e7mxt6-0>li:before{content:"" counter(lst-ctn-kix_ytu3x5e7mxt6-0,decimal) ". "}ul.lst-kix_b01r1xbffk2a-8{list-style-type:none}ul.lst-kix_b01r1xbffk2a-3{list-style-type:none}ul.lst-kix_b01r1xbffk2a-4{list-style-type:none}ul.lst-kix_b01r1xbffk2a-5{list-style-type:none}ul.lst-kix_b01r1xbffk2a-6{list-style-type:none}ul.lst-kix_awa32z9rxqor-1{list-style-type:none}.lst-kix_ytu3x5e7mxt6-6>li:before{content:"" counter(lst-ctn-kix_ytu3x5e7mxt6-6,decimal) ". "}.lst-kix_3aierr3stws2-7>li:before{content:"\0025cb   "}.lst-kix_nqcns3qrnch2-1>li:before{content:"\0025cb   "}.lst-kix_74bj55g8eq2b-3>li:before{content:"\0025cf   "}.lst-kix_9ei06drf1sg0-3>li:before{content:"\0025cf   "}.lst-kix_vqsmxaeqhsi-1>li:before{content:"\0025cb   "}.lst-kix_2fszuplxt2n9-2>li:before{content:"\0025a0   "}.lst-kix_on63e6od9ck4-7>li:before{content:"\0025cb   "}.lst-kix_74bj55g8eq2b-5>li:before{content:"\0025a0   "}ol.lst-kix_8joc3jsoa7de-6.start{counter-reset:lst-ctn-kix_8joc3jsoa7de-6 0}.lst-kix_2ws5k2m4lsl8-2>li:before{content:"\0025a0   "}.lst-kix_ytu3x5e7mxt6-8>li:before{content:"" counter(lst-ctn-kix_ytu3x5e7mxt6-8,lower-roman) ". "}.lst-kix_2fszuplxt2n9-0>li:before{content:"\0025cf   "}.lst-kix_on63e6od9ck4-5>li:before{content:"\0025a0   "}.lst-kix_2ws5k2m4lsl8-0>li:before{content:"\0025cf   "}.lst-kix_z1b7uf8mlquv-2>li:before{content:"\0025a0   "}.lst-kix_zqq05ofg66-7>li:before{content:"" counter(lst-ctn-kix_zqq05ofg66-7,lower-latin) ". "}.lst-kix_9fa76bs6q7z7-8>li:before{content:"\0025a0   "}ol.lst-kix_tkg3l6cy3bnc-5.start{counter-reset:lst-ctn-kix_tkg3l6cy3bnc-5 0}.lst-kix_ht2xpfpspp5-1>li:before{content:"\0025cb   "}.lst-kix_zqq05ofg66-5>li:before{content:"" counter(lst-ctn-kix_zqq05ofg66-5,lower-roman) ". "}.lst-kix_4hyg4ehfafxi-5>li:before{content:"\0025a0   "}.lst-kix_9fa76bs6q7z7-6>li:before{content:"\0025cf   "}.lst-kix_awa32z9rxqor-8>li:before{content:"" counter(lst-ctn-kix_awa32z9rxqor-8,lower-roman) ". "}.lst-kix_qxeaj9fcw57j-3>li:before{content:"\0025cf   "}.lst-kix_qxeaj9fcw57j-5>li:before{content:"\0025a0   "}.lst-kix_k38n94udk0el-8>li:before{content:"\0025a0   "}.lst-kix_b01r1xbffk2a-3>li:before{content:"\0025cf   "}.lst-kix_4hyg4ehfafxi-3>li:before{content:"\0025cf   "}.lst-kix_tc68vt7q79wj-6>li:before{content:"\0025cf   "}.lst-kix_awa32z9rxqor-2>li:before{content:"" counter(lst-ctn-kix_awa32z9rxqor-2,lower-roman) ". "}.lst-kix_ytu3x5e7mxt6-3>li{counter-increment:lst-ctn-kix_ytu3x5e7mxt6-3}.lst-kix_7g6nqlfa7rn7-0>li:before{content:"\0025cf   "}.lst-kix_7g6nqlfa7rn7-2>li:before{content:"\0025a0   "}.lst-kix_z1b7uf8mlquv-4>li:before{content:"\0025cb   "}.lst-kix_tc68vt7q79wj-4>li:before{content:"\0025cb   "}.lst-kix_ij4fg4tarntm-3>li:before{content:"\0025cf   "}ul.lst-kix_umwvyipzf505-5{list-style-type:none}ul.lst-kix_umwvyipzf505-4{list-style-type:none}ul.lst-kix_umwvyipzf505-7{list-style-type:none}ul.lst-kix_umwvyipzf505-6{list-style-type:none}.lst-kix_6vg7rekv2ht5-5>li:before{content:"\0025a0   "}.lst-kix_9fa76bs6q7z7-0>li:before{content:"\0025cf   "}ol.lst-kix_esoh3w8sgdnn-4.start{counter-reset:lst-ctn-kix_esoh3w8sgdnn-4 0}ul.lst-kix_umwvyipzf505-8{list-style-type:none}.lst-kix_v0sksjnof0cj-1>li:before{content:"\0025cb   "}.lst-kix_eb0mkv6he1ge-8>li:before{content:"\0025a0   "}.lst-kix_6vg7rekv2ht5-3>li:before{content:"\0025cf   "}ul.lst-kix_umwvyipzf505-1{list-style-type:none}ul.lst-kix_umwvyipzf505-0{list-style-type:none}ul.lst-kix_umwvyipzf505-3{list-style-type:none}ul.lst-kix_umwvyipzf505-2{list-style-type:none}.lst-kix_b0v72ufkqzg3-3>li:before{content:"\0025cf   "}.lst-kix_b0v72ufkqzg3-5>li:before{content:"\0025a0   "}ol.lst-kix_zqq05ofg66-8{list-style-type:none}ol.lst-kix_zqq05ofg66-7{list-style-type:none}ol.lst-kix_zqq05ofg66-6{list-style-type:none}.lst-kix_5mkgtms2qzgk-0>li:before{content:"\0025cf   "}ol.lst-kix_dgmlm9mxaa4-3.start{counter-reset:lst-ctn-kix_dgmlm9mxaa4-3 0}ol.lst-kix_zqq05ofg66-0{list-style-type:none}ol.lst-kix_sbk9fqxb69uy-8.start{counter-reset:lst-ctn-kix_sbk9fqxb69uy-8 0}ol.lst-kix_tkg3l6cy3bnc-8{list-style-type:none}ol.lst-kix_zqq05ofg66-5{list-style-type:none}.lst-kix_5mkgtms2qzgk-2>li:before{content:"\0025a0   "}ol.lst-kix_zqq05ofg66-4{list-style-type:none}.lst-kix_kfepckt4l6hg-7>li:before{content:"" counter(lst-ctn-kix_kfepckt4l6hg-7,lower-latin) ". "}ol.lst-kix_zqq05ofg66-3{list-style-type:none}.lst-kix_ij4fg4tarntm-5>li:before{content:"\0025a0   "}ol.lst-kix_zqq05ofg66-2{list-style-type:none}ol.lst-kix_tkg3l6cy3bnc-3{list-style-type:none}ol.lst-kix_tkg3l6cy3bnc-2{list-style-type:none}ol.lst-kix_tkg3l6cy3bnc-0{list-style-type:none}ol.lst-kix_tkg3l6cy3bnc-7{list-style-type:none}ol.lst-kix_tkg3l6cy3bnc-6{list-style-type:none}.lst-kix_8r3w2rcyu87a-7>li:before{content:"\0025cb   "}ol.lst-kix_tkg3l6cy3bnc-5{list-style-type:none}.lst-kix_kfepckt4l6hg-5>li:before{content:"" counter(lst-ctn-kix_kfepckt4l6hg-5,lower-roman) ". "}ol.lst-kix_tkg3l6cy3bnc-4{list-style-type:none}.lst-kix_5mkgtms2qzgk-8>li:before{content:"\0025a0   "}.lst-kix_8r3w2rcyu87a-1>li:before{content:"\0025cb   "}.lst-kix_129uykneptsf-0>li:before{content:"\0025cf   "}.lst-kix_k38n94udk0el-0>li:before{content:"\0025cf   "}.lst-kix_6ay9chlfxidr-4>li:before{content:"\0025cb   "}.lst-kix_tibxkvatc3h0-3>li:before{content:"\0025cf   "}.lst-kix_6ay9chlfxidr-2>li:before{content:"\0025a0   "}.lst-kix_awa32z9rxqor-0>li:before{content:"" counter(lst-ctn-kix_awa32z9rxqor-0,decimal) ". "}.lst-kix_k38n94udk0el-2>li:before{content:"\0025a0   "}ul.lst-kix_haoc7b4kfhm3-1{list-style-type:none}ul.lst-kix_haoc7b4kfhm3-0{list-style-type:none}ul.lst-kix_sp4bx5if0jbk-0{list-style-type:none}ul.lst-kix_6k838qjxgg72-7{list-style-type:none}ul.lst-kix_6k838qjxgg72-8{list-style-type:none}.lst-kix_j5h2oeedi37x-7>li:before{content:"\0025cb   "}ul.lst-kix_6k838qjxgg72-5{list-style-type:none}ul.lst-kix_haoc7b4kfhm3-5{list-style-type:none}ul.lst-kix_sp4bx5if0jbk-3{list-style-type:none}ul.lst-kix_6k838qjxgg72-6{list-style-type:none}.lst-kix_b01r1xbffk2a-1>li:before{content:"\0025cb   "}ul.lst-kix_haoc7b4kfhm3-4{list-style-type:none}ul.lst-kix_sp4bx5if0jbk-4{list-style-type:none}ul.lst-kix_6k838qjxgg72-3{list-style-type:none}ul.lst-kix_haoc7b4kfhm3-3{list-style-type:none}ul.lst-kix_sp4bx5if0jbk-1{list-style-type:none}ul.lst-kix_6k838qjxgg72-4{list-style-type:none}ul.lst-kix_haoc7b4kfhm3-2{list-style-type:none}ul.lst-kix_sp4bx5if0jbk-2{list-style-type:none}ul.lst-kix_6k838qjxgg72-1{list-style-type:none}ul.lst-kix_sp4bx5if0jbk-7{list-style-type:none}ul.lst-kix_6k838qjxgg72-2{list-style-type:none}ul.lst-kix_haoc7b4kfhm3-8{list-style-type:none}ul.lst-kix_sp4bx5if0jbk-8{list-style-type:none}.lst-kix_3hobsm4w6nyk-0>li:before{content:"\0025cf   "}ul.lst-kix_haoc7b4kfhm3-7{list-style-type:none}ul.lst-kix_sp4bx5if0jbk-5{list-style-type:none}ul.lst-kix_6k838qjxgg72-0{list-style-type:none}ul.lst-kix_haoc7b4kfhm3-6{list-style-type:none}ul.lst-kix_sp4bx5if0jbk-6{list-style-type:none}.lst-kix_j5h2oeedi37x-5>li:before{content:"\0025a0   "}.lst-kix_lsygr38j1y0-3>li:before{content:"\0025cf   "}.lst-kix_n23m8p7c7hw6-2>li:before{content:"\0025a0   "}.lst-kix_4864lfd9zdwy-2>li:before{content:"\0025a0   "}ul.lst-kix_iprbjac0k1x3-6{list-style-type:none}ul.lst-kix_iprbjac0k1x3-5{list-style-type:none}ul.lst-kix_iprbjac0k1x3-8{list-style-type:none}ul.lst-kix_iprbjac0k1x3-7{list-style-type:none}.lst-kix_84yjavmdod6-6>li:before{content:"\0025cf   "}.lst-kix_dgmlm9mxaa4-7>li{counter-increment:lst-ctn-kix_dgmlm9mxaa4-7}.lst-kix_3hobsm4w6nyk-6>li:before{content:"\0025cf   "}.lst-kix_mhfweobdea1s-0>li:before{content:"\0025cf   "}ol.lst-kix_sbk9fqxb69uy-3.start{counter-reset:lst-ctn-kix_sbk9fqxb69uy-3 0}.lst-kix_vx5oc7tbpucq-0>li:before{content:"\0025cf   "}.lst-kix_mhfweobdea1s-8>li:before{content:"\0025a0   "}.lst-kix_tut3ual308bf-3>li:before{content:"\0025cf   "}.lst-kix_8joc3jsoa7de-4>li:before{content:"" counter(lst-ctn-kix_8joc3jsoa7de-4,lower-latin) ". "}.lst-kix_8wqgtnyy1nox-4>li:before{content:"\0025cb   "}.lst-kix_kruy5m9jmo50-4>li:before{content:"\0025cb   "}.lst-kix_q432akg4led-2>li:before{content:"\0025a0   "}.lst-kix_z4fu53gdnrww-4>li:before{content:"\0025cb   "}.lst-kix_70tvr3ungtsg-8>li:before{content:"\0025a0   "}.lst-kix_ss89m2334v3w-5>li:before{content:"\0025a0   "}.lst-kix_473lniz6s057-3>li:before{content:"\0025cf   "}.lst-kix_tj1ix0gi2ui9-1>li:before{content:"\0025cb   "}.lst-kix_arstwhb1c9gy-6>li:before{content:"\0025cf   "}.lst-kix_8luyzq1gqi6-2>li:before{content:"\0025a0   "}.lst-kix_pb595s4qvcov-1>li:before{content:"\0025cb   "}.lst-kix_ytu3x5e7mxt6-0>li{counter-increment:lst-ctn-kix_ytu3x5e7mxt6-0}ol.lst-kix_awa32z9rxqor-7.start{counter-reset:lst-ctn-kix_awa32z9rxqor-7 0}.lst-kix_dmy2ekb87fbb-5>li:before{content:"\0025a0   "}.lst-kix_8yxwyie669z2-5>li:before{content:"\0025a0   "}.lst-kix_6pozql3s9ehi-7>li:before{content:"\0025cb   "}.lst-kix_p73ghpm47z3w-3>li:before{content:"\0025cf   "}.lst-kix_hlp788b4qti1-6>li:before{content:"\0025cf   "}.lst-kix_esoh3w8sgdnn-5>li{counter-increment:lst-ctn-kix_esoh3w8sgdnn-5}.lst-kix_v0sksjnof0cj-7>li:before{content:"\0025cb   "}.lst-kix_70tvr3ungtsg-0>li:before{content:"\0025cf   "}.lst-kix_x3csall36p7p-3>li:before{content:"\0025cf   "}.lst-kix_tx9us7pz8zze-5>li:before{content:"\0025a0   "}ol.lst-kix_dgmlm9mxaa4-6.start{counter-reset:lst-ctn-kix_dgmlm9mxaa4-6 0}.lst-kix_haoc7b4kfhm3-0>li:before{content:"\0025cf   "}.lst-kix_upyk6it0f1tq-0>li:before{content:"\0025cf   "}.lst-kix_sbk9fqxb69uy-0>li:before{content:"" counter(lst-ctn-kix_sbk9fqxb69uy-0,decimal) ". "}.lst-kix_sbk9fqxb69uy-8>li:before{content:"" counter(lst-ctn-kix_sbk9fqxb69uy-8,lower-roman) ". "}.lst-kix_k2lnp2ex00id-4>li:before{content:"\0025cb   "}ol.lst-kix_awa32z9rxqor-6.start{counter-reset:lst-ctn-kix_awa32z9rxqor-6 0}.lst-kix_haoc7b4kfhm3-8>li:before{content:"\0025a0   "}.lst-kix_8i9khg2xgulr-1>li:before{content:"\0025cb   "}ol.lst-kix_kfepckt4l6hg-8{list-style-type:none}ol.lst-kix_kfepckt4l6hg-5{list-style-type:none}ol.lst-kix_kfepckt4l6hg-4{list-style-type:none}.lst-kix_an05xfuo7mar-3>li:before{content:"\0025cf   "}.lst-kix_mktumxqlukrw-8>li:before{content:"\0025a0   "}.lst-kix_7g6nqlfa7rn7-8>li:before{content:"\0025a0   "}ol.lst-kix_kfepckt4l6hg-7{list-style-type:none}.lst-kix_t27vnxuzkttq-4>li:before{content:"\0025cb   "}ol.lst-kix_kfepckt4l6hg-6{list-style-type:none}ol.lst-kix_kfepckt4l6hg-1{list-style-type:none}.lst-kix_vx5oc7tbpucq-8>li:before{content:"\0025a0   "}ol.lst-kix_kfepckt4l6hg-0{list-style-type:none}.lst-kix_auto7t5cf4tl-6>li:before{content:"\0025cf   "}ol.lst-kix_kfepckt4l6hg-3{list-style-type:none}.lst-kix_b4xlo74k6gd0-7>li:before{content:"\0025cb   "}ol.lst-kix_kfepckt4l6hg-2{list-style-type:none}.lst-kix_ht2xpfpspp5-7>li:before{content:"\0025cb   "}ul.lst-kix_vx5oc7tbpucq-8{list-style-type:none}.lst-kix_d8i9v1tvihft-8>li:before{content:"\0025a0   "}.lst-kix_w12y2pivfx64-1>li:before{content:"\0025cb   "}ul.lst-kix_vx5oc7tbpucq-2{list-style-type:none}ul.lst-kix_vx5oc7tbpucq-3{list-style-type:none}.lst-kix_gp2qqn5s6so6-2>li:before{content:"\0025a0   "}ul.lst-kix_vx5oc7tbpucq-0{list-style-type:none}ul.lst-kix_vx5oc7tbpucq-1{list-style-type:none}ul.lst-kix_vx5oc7tbpucq-6{list-style-type:none}ul.lst-kix_vx5oc7tbpucq-7{list-style-type:none}.lst-kix_upyk6it0f1tq-8>li:before{content:"\0025a0   "}ul.lst-kix_vx5oc7tbpucq-4{list-style-type:none}ul.lst-kix_vx5oc7tbpucq-5{list-style-type:none}.lst-kix_xmm4jxv5hwlq-1>li:before{content:"\0025cb   "}ol.lst-kix_tkg3l6cy3bnc-8.start{counter-reset:lst-ctn-kix_tkg3l6cy3bnc-8 0}.lst-kix_d8i9v1tvihft-0>li:before{content:"\0025cf   "}.lst-kix_kfepckt4l6hg-5>li{counter-increment:lst-ctn-kix_kfepckt4l6hg-5}.lst-kix_ihv36zxyu5h9-6>li:before{content:"\0025cf   "}.lst-kix_sbk9fqxb69uy-8>li{counter-increment:lst-ctn-kix_sbk9fqxb69uy-8}.lst-kix_mktumxqlukrw-0>li:before{content:"\0025cf   "}.lst-kix_m75akhxjyj9r-5>li:before{content:"\0025a0   "}ul.lst-kix_3hobsm4w6nyk-1{list-style-type:none}ul.lst-kix_3hobsm4w6nyk-0{list-style-type:none}.lst-kix_5s9vwy7hbq8r-2>li:before{content:"\0025a0   "}.lst-kix_sr3soljrf7bf-4>li:before{content:"\0025cb   "}.lst-kix_3aierr3stws2-1>li:before{content:"\0025cb   "}ol.lst-kix_sbk9fqxb69uy-2.start{counter-reset:lst-ctn-kix_sbk9fqxb69uy-2 0}.lst-kix_etn6l1m36dy-5>li:before{content:"\0025a0   "}.lst-kix_xq8kn6xuala3-7>li:before{content:"\0025cb   "}ul.lst-kix_cb1r7mwbu1gi-1{list-style-type:none}.lst-kix_tkg3l6cy3bnc-7>li:before{content:"" counter(lst-ctn-kix_tkg3l6cy3bnc-7,lower-latin) ". "}ul.lst-kix_cb1r7mwbu1gi-0{list-style-type:none}ul.lst-kix_3hobsm4w6nyk-7{list-style-type:none}.lst-kix_ehxsaivn684z-5>li:before{content:"\0025a0   "}ul.lst-kix_3hobsm4w6nyk-6{list-style-type:none}.lst-kix_19nen5usjyzs-3>li:before{content:"\0025cf   "}.lst-kix_h8tfxvbo4u2a-7>li:before{content:"\0025cb   "}ul.lst-kix_3hobsm4w6nyk-8{list-style-type:none}ul.lst-kix_3hobsm4w6nyk-3{list-style-type:none}ul.lst-kix_3hobsm4w6nyk-2{list-style-type:none}ul.lst-kix_3hobsm4w6nyk-5{list-style-type:none}ul.lst-kix_3hobsm4w6nyk-4{list-style-type:none}.lst-kix_fd9jvhtv6teb-6>li:before{content:"\0025cf   "}.lst-kix_1lds0wnd5zeb-2>li:before{content:"\0025a0   "}.lst-kix_2tnhjv96pndg-0>li:before{content:"\0025cf   "}ul.lst-kix_cb1r7mwbu1gi-3{list-style-type:none}.lst-kix_blb26b155eyp-2>li:before{content:"\0025a0   "}ul.lst-kix_cb1r7mwbu1gi-2{list-style-type:none}.lst-kix_e2h0cgtfdf3g-3>li:before{content:"\0025cf   "}ul.lst-kix_cb1r7mwbu1gi-5{list-style-type:none}ul.lst-kix_cb1r7mwbu1gi-4{list-style-type:none}ul.lst-kix_cb1r7mwbu1gi-7{list-style-type:none}.lst-kix_esoh3w8sgdnn-3>li:before{content:"" counter(lst-ctn-kix_esoh3w8sgdnn-3,decimal) ". "}ul.lst-kix_cb1r7mwbu1gi-6{list-style-type:none}ul.lst-kix_cb1r7mwbu1gi-8{list-style-type:none}.lst-kix_ytu3x5e7mxt6-7>li{counter-increment:lst-ctn-kix_ytu3x5e7mxt6-7}.lst-kix_90vpss294crz-0>li:before{content:"\0025cf   "}.lst-kix_zqq05ofg66-4>li{counter-increment:lst-ctn-kix_zqq05ofg66-4}ul.lst-kix_6pozql3s9ehi-8{list-style-type:none}ul.lst-kix_6pozql3s9ehi-6{list-style-type:none}ul.lst-kix_6pozql3s9ehi-7{list-style-type:none}ul.lst-kix_6pozql3s9ehi-4{list-style-type:none}ul.lst-kix_6pozql3s9ehi-5{list-style-type:none}ul.lst-kix_19nen5usjyzs-0{list-style-type:none}.lst-kix_sp4bx5if0jbk-3>li:before{content:"\0025cf   "}ul.lst-kix_19nen5usjyzs-4{list-style-type:none}ol.lst-kix_sbk9fqxb69uy-0.start{counter-reset:lst-ctn-kix_sbk9fqxb69uy-0 0}ul.lst-kix_19nen5usjyzs-3{list-style-type:none}.lst-kix_6kne4t6ii93d-4>li:before{content:"\0025cb   "}li.li-bullet-0:before{margin-left:-18pt;white-space:nowrap;display:inline-block;min-width:18pt}ul.lst-kix_19nen5usjyzs-2{list-style-type:none}ul.lst-kix_19nen5usjyzs-1{list-style-type:none}ul.lst-kix_19nen5usjyzs-8{list-style-type:none}ul.lst-kix_19nen5usjyzs-7{list-style-type:none}ul.lst-kix_19nen5usjyzs-6{list-style-type:none}ul.lst-kix_iprbjac0k1x3-0{list-style-type:none}.lst-kix_bo4w5l7q0tq9-7>li:before{content:"\0025cb   "}.lst-kix_ls0ws55dysw8-3>li:before{content:"\0025cf   "}ul.lst-kix_19nen5usjyzs-5{list-style-type:none}ul.lst-kix_iprbjac0k1x3-2{list-style-type:none}ul.lst-kix_iprbjac0k1x3-1{list-style-type:none}ul.lst-kix_iprbjac0k1x3-4{list-style-type:none}ul.lst-kix_iprbjac0k1x3-3{list-style-type:none}.lst-kix_2tnhjv96pndg-8>li:before{content:"\0025a0   "}ul.lst-kix_6pozql3s9ehi-2{list-style-type:none}.lst-kix_90vpss294crz-8>li:before{content:"\0025a0   "}ul.lst-kix_6pozql3s9ehi-3{list-style-type:none}.lst-kix_3qoh0wbylc0j-2>li{counter-increment:lst-ctn-kix_3qoh0wbylc0j-2}ul.lst-kix_6pozql3s9ehi-0{list-style-type:none}ul.lst-kix_6pozql3s9ehi-1{list-style-type:none}.lst-kix_eag5a795mtbr-5>li:before{content:"" counter(lst-ctn-kix_eag5a795mtbr-5,lower-roman) ". "}.lst-kix_2fh1qudxwat7-4>li:before{content:"\0025cb   "}ul.lst-kix_i5y1isusbf26-5{list-style-type:none}ul.lst-kix_i5y1isusbf26-6{list-style-type:none}ul.lst-kix_i5y1isusbf26-7{list-style-type:none}ul.lst-kix_i5y1isusbf26-8{list-style-type:none}.lst-kix_skp7x04r24av-8>li:before{content:"\0025a0   "}ul.lst-kix_i5y1isusbf26-0{list-style-type:none}ul.lst-kix_i5y1isusbf26-1{list-style-type:none}.lst-kix_cb1r7mwbu1gi-2>li:before{content:"\0025a0   "}ul.lst-kix_i5y1isusbf26-2{list-style-type:none}ul.lst-kix_i5y1isusbf26-3{list-style-type:none}ul.lst-kix_i5y1isusbf26-4{list-style-type:none}.lst-kix_1ip3hsr9v1h5-1>li:before{content:"\0025cb   "}.lst-kix_1ip3hsr9v1h5-3>li:before{content:"\0025cf   "}.lst-kix_6kne4t6ii93d-7>li:before{content:"\0025cb   "}.lst-kix_skp7x04r24av-1>li:before{content:"\0025cb   "}.lst-kix_1ip3hsr9v1h5-0>li:before{content:"\0025cf   "}.lst-kix_1ip3hsr9v1h5-4>li:before{content:"\0025cb   "}.lst-kix_cb1r7mwbu1gi-0>li:before{content:"\0025cf   "}.lst-kix_esoh3w8sgdnn-2>li{counter-increment:lst-ctn-kix_esoh3w8sgdnn-2}.lst-kix_ehpc0o1sivs1-6>li:before{content:"\0025cf   "}.lst-kix_ehpc0o1sivs1-3>li:before{content:"\0025cf   "}.lst-kix_eunro7k2bn6e-1>li:before{content:"\0025cb   "}.lst-kix_eunro7k2bn6e-8>li:before{content:"\0025a0   "}.lst-kix_ehpc0o1sivs1-5>li:before{content:"\0025a0   "}.lst-kix_e134675mgdj3-4>li:before{content:"\0025cb   "}.lst-kix_cb1r7mwbu1gi-3>li:before{content:"\0025cf   "}.lst-kix_cb1r7mwbu1gi-5>li:before{content:"\0025a0   "}.lst-kix_eunro7k2bn6e-2>li:before{content:"\0025a0   "}.lst-kix_e134675mgdj3-7>li:before{content:"\0025cb   "}.lst-kix_ehpc0o1sivs1-0>li:before{content:"\0025cf   "}.lst-kix_ehpc0o1sivs1-2>li:before{content:"\0025a0   "}.lst-kix_eunro7k2bn6e-5>li:before{content:"\0025a0   "}.lst-kix_cb1r7mwbu1gi-6>li:before{content:"\0025cf   "}ol.lst-kix_ytu3x5e7mxt6-8.start{counter-reset:lst-ctn-kix_ytu3x5e7mxt6-8 0}.lst-kix_eunro7k2bn6e-4>li:before{content:"\0025cb   "}ol.lst-kix_8joc3jsoa7de-0{list-style-type:none}.lst-kix_81hzypied9ul-5>li:before{content:"\0025a0   "}.lst-kix_w9ink2jq27yv-0>li:before{content:"\0025cf   "}.lst-kix_w9ink2jq27yv-1>li:before{content:"\0025cb   "}.lst-kix_ehxsaivn684z-0>li:before{content:"\0025cf   "}.lst-kix_ehxsaivn684z-2>li:before{content:"\0025a0   "}ol.lst-kix_8joc3jsoa7de-2{list-style-type:none}.lst-kix_eag5a795mtbr-5>li{counter-increment:lst-ctn-kix_eag5a795mtbr-5}.lst-kix_ehxsaivn684z-3>li:before{content:"\0025cf   "}ol.lst-kix_8joc3jsoa7de-4{list-style-type:none}.lst-kix_81hzypied9ul-4>li:before{content:"\0025cb   "}.lst-kix_81hzypied9ul-7>li:before{content:"\0025cb   "}.lst-kix_81hzypied9ul-8>li:before{content:"\0025a0   "}.lst-kix_w9ink2jq27yv-3>li:before{content:"\0025cf   "}ol.lst-kix_8joc3jsoa7de-3{list-style-type:none}.lst-kix_fsz4dbursko6-3>li:before{content:"\0025cf   "}ol.lst-kix_8joc3jsoa7de-6{list-style-type:none}ol.lst-kix_8joc3jsoa7de-5{list-style-type:none}.lst-kix_81hzypied9ul-1>li:before{content:"\0025cb   "}.lst-kix_81hzypied9ul-2>li:before{content:"\0025a0   "}.lst-kix_fsz4dbursko6-6>li:before{content:"\0025cf   "}ul.lst-kix_90vpss294crz-0{list-style-type:none}ol.lst-kix_8joc3jsoa7de-5.start{counter-reset:lst-ctn-kix_8joc3jsoa7de-5 0}ul.lst-kix_90vpss294crz-1{list-style-type:none}.lst-kix_dgmlm9mxaa4-4>li{counter-increment:lst-ctn-kix_dgmlm9mxaa4-4}ul.lst-kix_90vpss294crz-4{list-style-type:none}ul.lst-kix_90vpss294crz-5{list-style-type:none}ul.lst-kix_w9ink2jq27yv-1{list-style-type:none}ul.lst-kix_90vpss294crz-2{list-style-type:none}ul.lst-kix_w9ink2jq27yv-0{list-style-type:none}ul.lst-kix_90vpss294crz-3{list-style-type:none}ul.lst-kix_90vpss294crz-8{list-style-type:none}.lst-kix_w9ink2jq27yv-7>li:before{content:"\0025cb   "}.lst-kix_565pscofcl0-5>li:before{content:"\0025a0   "}ul.lst-kix_90vpss294crz-6{list-style-type:none}ul.lst-kix_90vpss294crz-7{list-style-type:none}.lst-kix_fsz4dbursko6-0>li:before{content:"\0025cf   "}ul.lst-kix_w9ink2jq27yv-7{list-style-type:none}.lst-kix_e134675mgdj3-0>li:before{content:"\0025cf   "}ol.lst-kix_8joc3jsoa7de-8{list-style-type:none}ol.lst-kix_tkg3l6cy3bnc-6.start{counter-reset:lst-ctn-kix_tkg3l6cy3bnc-6 0}.lst-kix_w9ink2jq27yv-4>li:before{content:"\0025cb   "}ul.lst-kix_w9ink2jq27yv-6{list-style-type:none}.lst-kix_565pscofcl0-8>li:before{content:"\0025a0   "}ol.lst-kix_8joc3jsoa7de-7{list-style-type:none}.lst-kix_e134675mgdj3-1>li:before{content:"\0025cb   "}ul.lst-kix_w9ink2jq27yv-8{list-style-type:none}ul.lst-kix_w9ink2jq27yv-3{list-style-type:none}ul.lst-kix_w9ink2jq27yv-2{list-style-type:none}ul.lst-kix_w9ink2jq27yv-5{list-style-type:none}ul.lst-kix_w9ink2jq27yv-4{list-style-type:none}.lst-kix_1qqoijhe7jmx-4>li:before{content:"\0025cb   "}.lst-kix_1qqoijhe7jmx-2>li:before{content:"\0025a0   "}.lst-kix_1qqoijhe7jmx-7>li:before{content:"\0025cb   "}.lst-kix_1qqoijhe7jmx-8>li:before{content:"\0025a0   "}.lst-kix_1ip3hsr9v1h5-7>li:before{content:"\0025cb   "}.lst-kix_zqq05ofg66-0>li{counter-increment:lst-ctn-kix_zqq05ofg66-0}.lst-kix_1qqoijhe7jmx-1>li:before{content:"\0025cb   "}.lst-kix_1ip3hsr9v1h5-6>li:before{content:"\0025cf   "}ol.lst-kix_8joc3jsoa7de-0.start{counter-reset:lst-ctn-kix_8joc3jsoa7de-0 0}.lst-kix_1qqoijhe7jmx-5>li:before{content:"\0025a0   "}.lst-kix_4y6tqwnm2166-6>li:before{content:"\0025cf   "}ol.lst-kix_sbk9fqxb69uy-6{list-style-type:none}.lst-kix_3qoh0wbylc0j-5>li{counter-increment:lst-ctn-kix_3qoh0wbylc0j-5}ol.lst-kix_sbk9fqxb69uy-7{list-style-type:none}ol.lst-kix_sbk9fqxb69uy-8{list-style-type:none}.lst-kix_w12y2pivfx64-0>li:before{content:"\0025cf   "}.lst-kix_w12y2pivfx64-4>li:before{content:"\0025cb   "}ol.lst-kix_sbk9fqxb69uy-2{list-style-type:none}ol.lst-kix_sbk9fqxb69uy-3{list-style-type:none}ol.lst-kix_sbk9fqxb69uy-4{list-style-type:none}ol.lst-kix_sbk9fqxb69uy-5{list-style-type:none}.lst-kix_ihv36zxyu5h9-1>li:before{content:"\0025cb   "}.lst-kix_4y6tqwnm2166-4>li:before{content:"\0025cb   "}ol.lst-kix_sbk9fqxb69uy-0{list-style-type:none}ol.lst-kix_sbk9fqxb69uy-1{list-style-type:none}.lst-kix_8joc3jsoa7de-2>li{counter-increment:lst-ctn-kix_8joc3jsoa7de-2}.lst-kix_w12y2pivfx64-6>li:before{content:"\0025cf   "}.lst-kix_ihv36zxyu5h9-3>li:before{content:"\0025cf   "}.lst-kix_4y6tqwnm2166-0>li:before{content:"\0025cf   "}.lst-kix_kfepckt4l6hg-3>li{counter-increment:lst-ctn-kix_kfepckt4l6hg-3}.lst-kix_5mbqwckh3266-2>li:before{content:"\0025a0   "}.lst-kix_ihv36zxyu5h9-7>li:before{content:"\0025cb   "}.lst-kix_565pscofcl0-2>li:before{content:"\0025a0   "}ul.lst-kix_e2h0cgtfdf3g-2{list-style-type:none}ul.lst-kix_e2h0cgtfdf3g-3{list-style-type:none}.lst-kix_3qoh0wbylc0j-6>li{counter-increment:lst-ctn-kix_3qoh0wbylc0j-6}ul.lst-kix_e2h0cgtfdf3g-0{list-style-type:none}.lst-kix_pb595s4qvcov-5>li:before{content:"\0025a0   "}ul.lst-kix_e2h0cgtfdf3g-1{list-style-type:none}ul.lst-kix_e2h0cgtfdf3g-6{list-style-type:none}ul.lst-kix_e2h0cgtfdf3g-7{list-style-type:none}ol.lst-kix_rggdlfkt6mfq-7.start{counter-reset:lst-ctn-kix_rggdlfkt6mfq-7 0}ul.lst-kix_e2h0cgtfdf3g-4{list-style-type:none}.lst-kix_5mbqwckh3266-6>li:before{content:"\0025cf   "}ul.lst-kix_e2h0cgtfdf3g-5{list-style-type:none}ul.lst-kix_e2h0cgtfdf3g-8{list-style-type:none}.lst-kix_ehxsaivn684z-6>li:before{content:"\0025cf   "}.lst-kix_5mbqwckh3266-8>li:before{content:"\0025a0   "}ul.lst-kix_tibxkvatc3h0-3{list-style-type:none}.lst-kix_2tnhjv96pndg-3>li:before{content:"\0025cf   "}ul.lst-kix_tibxkvatc3h0-2{list-style-type:none}ul.lst-kix_tibxkvatc3h0-1{list-style-type:none}ul.lst-kix_tibxkvatc3h0-0{list-style-type:none}ul.lst-kix_tibxkvatc3h0-7{list-style-type:none}ul.lst-kix_tibxkvatc3h0-6{list-style-type:none}ul.lst-kix_tibxkvatc3h0-5{list-style-type:none}ul.lst-kix_tibxkvatc3h0-4{list-style-type:none}.lst-kix_1lds0wnd5zeb-7>li:before{content:"\0025cb   "}ul.lst-kix_tibxkvatc3h0-8{list-style-type:none}.lst-kix_1lds0wnd5zeb-1>li:before{content:"\0025cb   "}.lst-kix_ls0ws55dysw8-4>li:before{content:"\0025cb   "}ul.lst-kix_gp2qqn5s6so6-8{list-style-type:none}ul.lst-kix_gp2qqn5s6so6-7{list-style-type:none}ul.lst-kix_gp2qqn5s6so6-6{list-style-type:none}ol.lst-kix_dgmlm9mxaa4-4.start{counter-reset:lst-ctn-kix_dgmlm9mxaa4-4 0}ul.lst-kix_gp2qqn5s6so6-5{list-style-type:none}ul.lst-kix_gp2qqn5s6so6-4{list-style-type:none}ul.lst-kix_gp2qqn5s6so6-3{list-style-type:none}ul.lst-kix_gp2qqn5s6so6-2{list-style-type:none}.lst-kix_sbk9fqxb69uy-5>li{counter-increment:lst-ctn-kix_sbk9fqxb69uy-5}ul.lst-kix_gp2qqn5s6so6-1{list-style-type:none}ul.lst-kix_gp2qqn5s6so6-0{list-style-type:none}.lst-kix_bo4w5l7q0tq9-0>li:before{content:"\0025cf   "}.lst-kix_6kne4t6ii93d-3>li:before{content:"\0025cf   "}.lst-kix_rggdlfkt6mfq-6>li{counter-increment:lst-ctn-kix_rggdlfkt6mfq-6}.lst-kix_dgmlm9mxaa4-8>li:before{content:"" counter(lst-ctn-kix_dgmlm9mxaa4-8,lower-roman) ". "}.lst-kix_skp7x04r24av-4>li:before{content:"\0025cb   "}.lst-kix_bo4w5l7q0tq9-6>li:before{content:"\0025cf   "}.lst-kix_dgmlm9mxaa4-4>li:before{content:"" counter(lst-ctn-kix_dgmlm9mxaa4-4,lower-latin) ". "}ol.lst-kix_sbk9fqxb69uy-7.start{counter-reset:lst-ctn-kix_sbk9fqxb69uy-7 0}.lst-kix_2tnhjv96pndg-5>li:before{content:"\0025a0   "}.lst-kix_1lds0wnd5zeb-5>li:before{content:"\0025a0   "}.lst-kix_ls0ws55dysw8-0>li:before{content:"\0025cf   "}.lst-kix_3hobsm4w6nyk-1>li:before{content:"\0025cb   "}.lst-kix_84yjavmdod6-4>li:before{content:"\0025cb   "}.lst-kix_3hobsm4w6nyk-4>li:before{content:"\0025cb   "}ul.lst-kix_6z3d30mc5xph-1{list-style-type:none}ul.lst-kix_6vg7rekv2ht5-8{list-style-type:none}ul.lst-kix_6z3d30mc5xph-0{list-style-type:none}.lst-kix_tibxkvatc3h0-6>li:before{content:"\0025cf   "}.lst-kix_mhfweobdea1s-1>li:before{content:"\0025cb   "}.lst-kix_84yjavmdod6-1>li:before{content:"\0025cb   "}ul.lst-kix_6z3d30mc5xph-5{list-style-type:none}.lst-kix_tibxkvatc3h0-5>li:before{content:"\0025a0   "}ul.lst-kix_6vg7rekv2ht5-4{list-style-type:none}ul.lst-kix_6z3d30mc5xph-4{list-style-type:none}ul.lst-kix_6vg7rekv2ht5-5{list-style-type:none}.lst-kix_84yjavmdod6-0>li:before{content:"\0025cf   "}ul.lst-kix_6z3d30mc5xph-3{list-style-type:none}ul.lst-kix_6vg7rekv2ht5-6{list-style-type:none}ul.lst-kix_6z3d30mc5xph-2{list-style-type:none}ul.lst-kix_6vg7rekv2ht5-7{list-style-type:none}ul.lst-kix_6vg7rekv2ht5-0{list-style-type:none}ul.lst-kix_6z3d30mc5xph-8{list-style-type:none}ul.lst-kix_6vg7rekv2ht5-1{list-style-type:none}ul.lst-kix_6z3d30mc5xph-7{list-style-type:none}ul.lst-kix_6vg7rekv2ht5-2{list-style-type:none}ul.lst-kix_6z3d30mc5xph-6{list-style-type:none}ul.lst-kix_6vg7rekv2ht5-3{list-style-type:none}.lst-kix_70tvr3ungtsg-6>li:before{content:"\0025cf   "}ul.lst-kix_6kne4t6ii93d-0{list-style-type:none}.lst-kix_70tvr3ungtsg-5>li:before{content:"\0025a0   "}ol.lst-kix_rggdlfkt6mfq-3.start{counter-reset:lst-ctn-kix_rggdlfkt6mfq-3 0}.lst-kix_tut3ual308bf-1>li:before{content:"\0025cb   "}.lst-kix_8wqgtnyy1nox-6>li:before{content:"\0025cf   "}.lst-kix_70tvr3ungtsg-2>li:before{content:"\0025a0   "}.lst-kix_8joc3jsoa7de-3>li:before{content:"" counter(lst-ctn-kix_8joc3jsoa7de-3,decimal) ". "}.lst-kix_8joc3jsoa7de-7>li:before{content:"" counter(lst-ctn-kix_8joc3jsoa7de-7,lower-latin) ". "}.lst-kix_8joc3jsoa7de-6>li:before{content:"" counter(lst-ctn-kix_8joc3jsoa7de-6,decimal) ". "}.lst-kix_eag5a795mtbr-6>li{counter-increment:lst-ctn-kix_eag5a795mtbr-6}.lst-kix_rggdlfkt6mfq-5>li{counter-increment:lst-ctn-kix_rggdlfkt6mfq-5}.lst-kix_473lniz6s057-2>li:before{content:"\0025a0   "}.lst-kix_473lniz6s057-6>li:before{content:"\0025cf   "}ol.lst-kix_dgmlm9mxaa4-0.start{counter-reset:lst-ctn-kix_dgmlm9mxaa4-0 0}ul.lst-kix_d8i9v1tvihft-1{list-style-type:none}.lst-kix_pb595s4qvcov-2>li:before{content:"\0025a0   "}ul.lst-kix_d8i9v1tvihft-2{list-style-type:none}ul.lst-kix_d8i9v1tvihft-0{list-style-type:none}.lst-kix_6pozql3s9ehi-4>li:before{content:"\0025cb   "}.lst-kix_k2lnp2ex00id-6>li:before{content:"\0025cf   "}.lst-kix_473lniz6s057-5>li:before{content:"\0025a0   "}.lst-kix_8yxwyie669z2-3>li:before{content:"\0025cf   "}.lst-kix_6pozql3s9ehi-5>li:before{content:"\0025a0   "}.lst-kix_arstwhb1c9gy-8>li:before{content:"\0025a0   "}ul.lst-kix_d8i9v1tvihft-7{list-style-type:none}ul.lst-kix_d8i9v1tvihft-8{list-style-type:none}.lst-kix_6pozql3s9ehi-8>li:before{content:"\0025a0   "}ul.lst-kix_d8i9v1tvihft-5{list-style-type:none}ul.lst-kix_d8i9v1tvihft-6{list-style-type:none}ul.lst-kix_d8i9v1tvihft-3{list-style-type:none}ul.lst-kix_d8i9v1tvihft-4{list-style-type:none}.lst-kix_hlp788b4qti1-5>li:before{content:"\0025a0   "}.lst-kix_arstwhb1c9gy-5>li:before{content:"\0025a0   "}.lst-kix_dmy2ekb87fbb-7>li:before{content:"\0025cb   "}ul.lst-kix_85agg0xrrcik-1{list-style-type:none}ul.lst-kix_85agg0xrrcik-0{list-style-type:none}ul.lst-kix_85agg0xrrcik-3{list-style-type:none}ul.lst-kix_85agg0xrrcik-2{list-style-type:none}ul.lst-kix_85agg0xrrcik-5{list-style-type:none}.lst-kix_tx9us7pz8zze-3>li:before{content:"\0025cf   "}ul.lst-kix_85agg0xrrcik-4{list-style-type:none}ul.lst-kix_85agg0xrrcik-7{list-style-type:none}ul.lst-kix_85agg0xrrcik-6{list-style-type:none}.lst-kix_upyk6it0f1tq-2>li:before{content:"\0025a0   "}.lst-kix_tx9us7pz8zze-0>li:before{content:"\0025cf   "}.lst-kix_hlp788b4qti1-8>li:before{content:"\0025a0   "}.lst-kix_sbk9fqxb69uy-4>li{counter-increment:lst-ctn-kix_sbk9fqxb69uy-4}.lst-kix_8i9khg2xgulr-2>li:before{content:"\0025a0   "}.lst-kix_8yxwyie669z2-0>li:before{content:"\0025cf   "}ol.lst-kix_rggdlfkt6mfq-8{list-style-type:none}ol.lst-kix_rggdlfkt6mfq-7{list-style-type:none}.lst-kix_vx5oc7tbpucq-7>li:before{content:"\0025cb   "}ol.lst-kix_rggdlfkt6mfq-6{list-style-type:none}ol.lst-kix_rggdlfkt6mfq-5{list-style-type:none}ol.lst-kix_rggdlfkt6mfq-4{list-style-type:none}ol.lst-kix_rggdlfkt6mfq-3{list-style-type:none}.lst-kix_b4xlo74k6gd0-6>li:before{content:"\0025cf   "}ol.lst-kix_rggdlfkt6mfq-2{list-style-type:none}ol.lst-kix_rggdlfkt6mfq-1{list-style-type:none}.lst-kix_6pozql3s9ehi-1>li:before{content:"\0025cb   "}ol.lst-kix_rggdlfkt6mfq-0{list-style-type:none}.lst-kix_4y6tqwnm2166-7>li:before{content:"\0025cb   "}.lst-kix_7g6nqlfa7rn7-3>li:before{content:"\0025cf   "}.lst-kix_qxeaj9fcw57j-2>li:before{content:"\0025a0   "}.lst-kix_awa32z9rxqor-7>li:before{content:"" counter(lst-ctn-kix_awa32z9rxqor-7,lower-latin) ". "}.lst-kix_4y6tqwnm2166-3>li:before{content:"\0025cf   "}.lst-kix_9fa76bs6q7z7-5>li:before{content:"\0025a0   "}.lst-kix_ihv36zxyu5h9-0>li:before{content:"\0025cf   "}.lst-kix_upyk6it0f1tq-6>li:before{content:"\0025cf   "}ul.lst-kix_85agg0xrrcik-8{list-style-type:none}ul.lst-kix_auto7t5cf4tl-0{list-style-type:none}.lst-kix_w12y2pivfx64-3>li:before{content:"\0025cf   "}.lst-kix_ihv36zxyu5h9-4>li:before{content:"\0025cb   "}.lst-kix_5s9vwy7hbq8r-0>li:before{content:"\0025cf   "}.lst-kix_w12y2pivfx64-7>li:before{content:"\0025cb   "}ul.lst-kix_vqsmxaeqhsi-0{list-style-type:none}ul.lst-kix_473lniz6s057-8{list-style-type:none}.lst-kix_5mbqwckh3266-5>li:before{content:"\0025a0   "}ul.lst-kix_vqsmxaeqhsi-1{list-style-type:none}ul.lst-kix_473lniz6s057-7{list-style-type:none}ul.lst-kix_vqsmxaeqhsi-2{list-style-type:none}ul.lst-kix_vqsmxaeqhsi-3{list-style-type:none}.lst-kix_6vg7rekv2ht5-2>li:before{content:"\0025a0   "}.lst-kix_6vg7rekv2ht5-6>li:before{content:"\0025cf   "}ul.lst-kix_vqsmxaeqhsi-4{list-style-type:none}ul.lst-kix_473lniz6s057-4{list-style-type:none}ul.lst-kix_473lniz6s057-3{list-style-type:none}ul.lst-kix_473lniz6s057-6{list-style-type:none}ul.lst-kix_473lniz6s057-5{list-style-type:none}ul.lst-kix_473lniz6s057-0{list-style-type:none}ul.lst-kix_473lniz6s057-2{list-style-type:none}ul.lst-kix_473lniz6s057-1{list-style-type:none}ul.lst-kix_wje7sh1pcpc1-5{list-style-type:none}ul.lst-kix_wje7sh1pcpc1-6{list-style-type:none}.lst-kix_19nen5usjyzs-1>li:before{content:"\0025cb   "}ul.lst-kix_wje7sh1pcpc1-7{list-style-type:none}.lst-kix_1lds0wnd5zeb-4>li:before{content:"\0025cb   "}ul.lst-kix_wje7sh1pcpc1-8{list-style-type:none}.lst-kix_e2h0cgtfdf3g-1>li:before{content:"\0025cb   "}.lst-kix_ytu3x5e7mxt6-1>li{counter-increment:lst-ctn-kix_ytu3x5e7mxt6-1}ul.lst-kix_vqsmxaeqhsi-5{list-style-type:none}ul.lst-kix_skp7x04r24av-0{list-style-type:none}ul.lst-kix_vqsmxaeqhsi-6{list-style-type:none}.lst-kix_1lds0wnd5zeb-8>li:before{content:"\0025a0   "}ul.lst-kix_vqsmxaeqhsi-7{list-style-type:none}ul.lst-kix_vqsmxaeqhsi-8{list-style-type:none}ul.lst-kix_wje7sh1pcpc1-0{list-style-type:none}ul.lst-kix_skp7x04r24av-3{list-style-type:none}ul.lst-kix_wje7sh1pcpc1-1{list-style-type:none}ul.lst-kix_skp7x04r24av-4{list-style-type:none}ul.lst-kix_wje7sh1pcpc1-2{list-style-type:none}ul.lst-kix_skp7x04r24av-1{list-style-type:none}.lst-kix_2tnhjv96pndg-2>li:before{content:"\0025a0   "}ul.lst-kix_wje7sh1pcpc1-3{list-style-type:none}ul.lst-kix_skp7x04r24av-2{list-style-type:none}ul.lst-kix_wje7sh1pcpc1-4{list-style-type:none}ul.lst-kix_8luyzq1gqi6-0{list-style-type:none}ul.lst-kix_skp7x04r24av-7{list-style-type:none}ul.lst-kix_8luyzq1gqi6-1{list-style-type:none}ul.lst-kix_skp7x04r24av-8{list-style-type:none}ul.lst-kix_8luyzq1gqi6-2{list-style-type:none}ul.lst-kix_skp7x04r24av-5{list-style-type:none}ul.lst-kix_8luyzq1gqi6-3{list-style-type:none}ul.lst-kix_skp7x04r24av-6{list-style-type:none}ul.lst-kix_8luyzq1gqi6-4{list-style-type:none}ul.lst-kix_8luyzq1gqi6-5{list-style-type:none}ul.lst-kix_8luyzq1gqi6-6{list-style-type:none}ul.lst-kix_8luyzq1gqi6-7{list-style-type:none}.lst-kix_dgmlm9mxaa4-3>li{counter-increment:lst-ctn-kix_dgmlm9mxaa4-3}ul.lst-kix_8luyzq1gqi6-8{list-style-type:none}ol.lst-kix_ytu3x5e7mxt6-4.start{counter-reset:lst-ctn-kix_ytu3x5e7mxt6-4 0}ul.lst-kix_auto7t5cf4tl-7{list-style-type:none}ul.lst-kix_auto7t5cf4tl-8{list-style-type:none}.lst-kix_m75akhxjyj9r-7>li:before{content:"\0025cb   "}ul.lst-kix_auto7t5cf4tl-5{list-style-type:none}ul.lst-kix_auto7t5cf4tl-6{list-style-type:none}ul.lst-kix_auto7t5cf4tl-3{list-style-type:none}ul.lst-kix_auto7t5cf4tl-4{list-style-type:none}ul.lst-kix_auto7t5cf4tl-1{list-style-type:none}ul.lst-kix_auto7t5cf4tl-2{list-style-type:none}.lst-kix_6kne4t6ii93d-6>li:before{content:"\0025cf   "}.lst-kix_tibxkvatc3h0-2>li:before{content:"\0025a0   "}.lst-kix_ls0ws55dysw8-1>li:before{content:"\0025cb   "}.lst-kix_qxeaj9fcw57j-6>li:before{content:"\0025cf   "}.lst-kix_dgmlm9mxaa4-7>li:before{content:"" counter(lst-ctn-kix_dgmlm9mxaa4-7,lower-latin) ". "}.lst-kix_k38n94udk0el-7>li:before{content:"\0025cb   "}.lst-kix_eag5a795mtbr-3>li:before{content:"" counter(lst-ctn-kix_eag5a795mtbr-3,decimal) ". "}ol.lst-kix_rggdlfkt6mfq-2.start{counter-reset:lst-ctn-kix_rggdlfkt6mfq-2 0}ol.lst-kix_ytu3x5e7mxt6-3.start{counter-reset:lst-ctn-kix_ytu3x5e7mxt6-3 0}.lst-kix_skp7x04r24av-7>li:before{content:"\0025cb   "}.lst-kix_129uykneptsf-1>li:before{content:"\0025cb   "}.lst-kix_2tnhjv96pndg-6>li:before{content:"\0025cf   "}.lst-kix_i5y1isusbf26-6>li:before{content:"\0025cf   "}.lst-kix_wk2xuqsimxh-0>li:before{content:"\0025cf   "}ul.lst-kix_ehxsaivn684z-8{list-style-type:none}ul.lst-kix_ehxsaivn684z-7{list-style-type:none}ul.lst-kix_ehxsaivn684z-6{list-style-type:none}.lst-kix_i5y1isusbf26-3>li:before{content:"\0025cf   "}ul.lst-kix_ehxsaivn684z-5{list-style-type:none}.lst-kix_dgmlm9mxaa4-6>li{counter-increment:lst-ctn-kix_dgmlm9mxaa4-6}ul.lst-kix_ehxsaivn684z-4{list-style-type:none}ul.lst-kix_ehxsaivn684z-3{list-style-type:none}ul.lst-kix_2fszuplxt2n9-0{list-style-type:none}ul.lst-kix_ehxsaivn684z-2{list-style-type:none}ul.lst-kix_2fszuplxt2n9-1{list-style-type:none}ul.lst-kix_ehxsaivn684z-1{list-style-type:none}.lst-kix_xq7wzd2lr8u5-6>li:before{content:"\0025cf   "}ul.lst-kix_ehxsaivn684z-0{list-style-type:none}ul.lst-kix_2fszuplxt2n9-4{list-style-type:none}.lst-kix_jyqknjqk01cc-4>li:before{content:"\0025cb   "}ul.lst-kix_2fszuplxt2n9-5{list-style-type:none}.lst-kix_2najoyx65arn-6>li:before{content:"\0025cf   "}.lst-kix_wk2xuqsimxh-6>li:before{content:"\0025cf   "}ul.lst-kix_2fszuplxt2n9-2{list-style-type:none}ul.lst-kix_2fszuplxt2n9-3{list-style-type:none}.lst-kix_2fh1qudxwat7-0>li:before{content:"\0025cf   "}ul.lst-kix_2fszuplxt2n9-8{list-style-type:none}ul.lst-kix_fd9jvhtv6teb-6{list-style-type:none}.lst-kix_3qoh0wbylc0j-0>li{counter-increment:lst-ctn-kix_3qoh0wbylc0j-0}ul.lst-kix_fd9jvhtv6teb-5{list-style-type:none}ul.lst-kix_2fszuplxt2n9-6{list-style-type:none}ul.lst-kix_fd9jvhtv6teb-4{list-style-type:none}ol.lst-kix_rggdlfkt6mfq-6.start{counter-reset:lst-ctn-kix_rggdlfkt6mfq-6 0}ul.lst-kix_2fszuplxt2n9-7{list-style-type:none}ul.lst-kix_fd9jvhtv6teb-3{list-style-type:none}.lst-kix_2fszuplxt2n9-6>li:before{content:"\0025cf   "}ul.lst-kix_fd9jvhtv6teb-8{list-style-type:none}.lst-kix_129uykneptsf-7>li:before{content:"\0025cb   "}ul.lst-kix_fd9jvhtv6teb-7{list-style-type:none}.lst-kix_nqcns3qrnch2-5>li:before{content:"\0025a0   "}.lst-kix_6ay9chlfxidr-8>li:before{content:"\0025a0   "}.lst-kix_73hytacj4fgx-6>li:before{content:"\0025cf   "}.lst-kix_nqcns3qrnch2-8>li:before{content:"\0025a0   "}.lst-kix_yxgmy3rkq7bq-4>li:before{content:"\0025cb   "}.lst-kix_cwtqd0lozkzu-7>li:before{content:"\0025cb   "}.lst-kix_jyqknjqk01cc-7>li:before{content:"\0025cb   "}.lst-kix_yxgmy3rkq7bq-7>li:before{content:"\0025cb   "}ul.lst-kix_74bj55g8eq2b-5{list-style-type:none}ul.lst-kix_5mkgtms2qzgk-5{list-style-type:none}.lst-kix_6k838qjxgg72-1>li:before{content:"\0025cb   "}.lst-kix_esoh3w8sgdnn-4>li{counter-increment:lst-ctn-kix_esoh3w8sgdnn-4}ul.lst-kix_74bj55g8eq2b-6{list-style-type:none}ul.lst-kix_5mkgtms2qzgk-4{list-style-type:none}ul.lst-kix_74bj55g8eq2b-3{list-style-type:none}ul.lst-kix_5mkgtms2qzgk-7{list-style-type:none}ul.lst-kix_74bj55g8eq2b-4{list-style-type:none}ul.lst-kix_5mkgtms2qzgk-6{list-style-type:none}ul.lst-kix_74bj55g8eq2b-1{list-style-type:none}ul.lst-kix_5mkgtms2qzgk-1{list-style-type:none}.lst-kix_85agg0xrrcik-5>li:before{content:"\0025a0   "}.lst-kix_85agg0xrrcik-8>li:before{content:"\0025a0   "}ul.lst-kix_74bj55g8eq2b-2{list-style-type:none}ul.lst-kix_5mkgtms2qzgk-0{list-style-type:none}ul.lst-kix_5mkgtms2qzgk-3{list-style-type:none}ul.lst-kix_74bj55g8eq2b-0{list-style-type:none}ul.lst-kix_5mkgtms2qzgk-2{list-style-type:none}.lst-kix_9ei06drf1sg0-0>li:before{content:"\0025cf   "}.lst-kix_19nen5usjyzs-4>li:before{content:"\0025cb   "}.lst-kix_eb0mkv6he1ge-3>li:before{content:"\0025cf   "}.lst-kix_6k838qjxgg72-4>li:before{content:"\0025cb   "}.lst-kix_ytu3x5e7mxt6-6>li{counter-increment:lst-ctn-kix_ytu3x5e7mxt6-6}ul.lst-kix_74bj55g8eq2b-7{list-style-type:none}ul.lst-kix_74bj55g8eq2b-8{list-style-type:none}.lst-kix_19nen5usjyzs-7>li:before{content:"\0025cb   "}.lst-kix_d7lufd20rbm5-1>li:before{content:"\0025cb   "}.lst-kix_tkg3l6cy3bnc-5>li{counter-increment:lst-ctn-kix_tkg3l6cy3bnc-5}.lst-kix_yxgmy3rkq7bq-1>li:before{content:"\0025cb   "}ol.lst-kix_ytu3x5e7mxt6-7.start{counter-reset:lst-ctn-kix_ytu3x5e7mxt6-7 0}.lst-kix_nqcns3qrnch2-2>li:before{content:"\0025a0   "}.lst-kix_d7lufd20rbm5-4>li:before{content:"\0025cb   "}.lst-kix_6k838qjxgg72-7>li:before{content:"\0025cb   "}.lst-kix_3aierr3stws2-8>li:before{content:"\0025a0   "}.lst-kix_zqq05ofg66-6>li{counter-increment:lst-ctn-kix_zqq05ofg66-6}.lst-kix_cwtqd0lozkzu-0>li:before{content:"\0025cf   "}.lst-kix_ytu3x5e7mxt6-2>li:before{content:"" counter(lst-ctn-kix_ytu3x5e7mxt6-2,lower-roman) ". "}.lst-kix_74bj55g8eq2b-1>li:before{content:"\0025cb   "}.lst-kix_ytu3x5e7mxt6-5>li:before{content:"" counter(lst-ctn-kix_ytu3x5e7mxt6-5,lower-roman) ". "}.lst-kix_2ws5k2m4lsl8-6>li:before{content:"\0025cf   "}.lst-kix_74bj55g8eq2b-4>li:before{content:"\0025cb   "}.lst-kix_2ws5k2m4lsl8-3>li:before{content:"\0025cf   "}ul.lst-kix_fd9jvhtv6teb-2{list-style-type:none}ul.lst-kix_fd9jvhtv6teb-1{list-style-type:none}.lst-kix_on63e6od9ck4-1>li:before{content:"\0025cb   "}ul.lst-kix_fd9jvhtv6teb-0{list-style-type:none}.lst-kix_sbk9fqxb69uy-7>li{counter-increment:lst-ctn-kix_sbk9fqxb69uy-7}.lst-kix_on63e6od9ck4-4>li:before{content:"\0025cb   "}.lst-kix_9ei06drf1sg0-7>li:before{content:"\0025cb   "}.lst-kix_wk2xuqsimxh-3>li:before{content:"\0025cf   "}.lst-kix_74bj55g8eq2b-7>li:before{content:"\0025cb   "}ul.lst-kix_sr3soljrf7bf-0{list-style-type:none}.lst-kix_ht2xpfpspp5-3>li:before{content:"\0025cf   "}ul.lst-kix_b4xlo74k6gd0-7{list-style-type:none}ul.lst-kix_sr3soljrf7bf-1{list-style-type:none}ul.lst-kix_b4xlo74k6gd0-8{list-style-type:none}.lst-kix_7g6nqlfa7rn7-6>li:before{content:"\0025cf   "}ul.lst-kix_sr3soljrf7bf-2{list-style-type:none}ul.lst-kix_sr3soljrf7bf-3{list-style-type:none}ul.lst-kix_sr3soljrf7bf-4{list-style-type:none}ul.lst-kix_sr3soljrf7bf-5{list-style-type:none}ul.lst-kix_sr3soljrf7bf-6{list-style-type:none}ul.lst-kix_84yjavmdod6-0{list-style-type:none}ul.lst-kix_sr3soljrf7bf-7{list-style-type:none}.lst-kix_d8i9v1tvihft-7>li:before{content:"\0025cb   "}ul.lst-kix_b4xlo74k6gd0-0{list-style-type:none}ul.lst-kix_b4xlo74k6gd0-1{list-style-type:none}ul.lst-kix_b4xlo74k6gd0-2{list-style-type:none}.lst-kix_z1b7uf8mlquv-0>li:before{content:"\0025cf   "}ul.lst-kix_b4xlo74k6gd0-3{list-style-type:none}ul.lst-kix_b4xlo74k6gd0-4{list-style-type:none}.lst-kix_vqsmxaeqhsi-2>li:before{content:"\0025a0   "}ul.lst-kix_b4xlo74k6gd0-5{list-style-type:none}.lst-kix_d8i9v1tvihft-1>li:before{content:"\0025cb   "}ul.lst-kix_b4xlo74k6gd0-6{list-style-type:none}.lst-kix_b01r1xbffk2a-5>li:before{content:"\0025a0   "}.lst-kix_awa32z9rxqor-4>li:before{content:"" counter(lst-ctn-kix_awa32z9rxqor-4,lower-latin) ". "}ul.lst-kix_on63e6od9ck4-3{list-style-type:none}ul.lst-kix_on63e6od9ck4-2{list-style-type:none}ul.lst-kix_on63e6od9ck4-1{list-style-type:none}.lst-kix_4hyg4ehfafxi-1>li:before{content:"\0025cb   "}ul.lst-kix_on63e6od9ck4-0{list-style-type:none}ul.lst-kix_on63e6od9ck4-7{list-style-type:none}ul.lst-kix_84yjavmdod6-2{list-style-type:none}.lst-kix_m75akhxjyj9r-4>li:before{content:"\0025cb   "}ul.lst-kix_on63e6od9ck4-6{list-style-type:none}.lst-kix_z1b7uf8mlquv-6>li:before{content:"\0025cf   "}ul.lst-kix_84yjavmdod6-1{list-style-type:none}ul.lst-kix_on63e6od9ck4-5{list-style-type:none}ul.lst-kix_84yjavmdod6-4{list-style-type:none}ul.lst-kix_on63e6od9ck4-4{list-style-type:none}ul.lst-kix_84yjavmdod6-3{list-style-type:none}ul.lst-kix_84yjavmdod6-6{list-style-type:none}.lst-kix_tc68vt7q79wj-8>li:before{content:"\0025a0   "}ul.lst-kix_84yjavmdod6-5{list-style-type:none}ul.lst-kix_84yjavmdod6-8{list-style-type:none}ul.lst-kix_on63e6od9ck4-8{list-style-type:none}.lst-kix_awa32z9rxqor-2>li{counter-increment:lst-ctn-kix_awa32z9rxqor-2}ul.lst-kix_84yjavmdod6-7{list-style-type:none}.lst-kix_sr3soljrf7bf-5>li:before{content:"\0025a0   "}.lst-kix_vqsmxaeqhsi-8>li:before{content:"\0025a0   "}.lst-kix_ij4fg4tarntm-1>li:before{content:"\0025cb   "}.lst-kix_tc68vt7q79wj-2>li:before{content:"\0025a0   "}.lst-kix_3aierr3stws2-2>li:before{content:"\0025a0   "}.lst-kix_9fa76bs6q7z7-2>li:before{content:"\0025a0   "}ul.lst-kix_j5h2oeedi37x-0{list-style-type:none}ul.lst-kix_j5h2oeedi37x-1{list-style-type:none}ul.lst-kix_j5h2oeedi37x-2{list-style-type:none}ul.lst-kix_j5h2oeedi37x-3{list-style-type:none}.lst-kix_etn6l1m36dy-6>li:before{content:"\0025cf   "}.lst-kix_xq8kn6xuala3-6>li:before{content:"\0025cf   "}ul.lst-kix_sr3soljrf7bf-8{list-style-type:none}ul.lst-kix_j5h2oeedi37x-4{list-style-type:none}ul.lst-kix_j5h2oeedi37x-5{list-style-type:none}.lst-kix_zqq05ofg66-3>li:before{content:"" counter(lst-ctn-kix_zqq05ofg66-3,decimal) ". "}.lst-kix_5s9vwy7hbq8r-3>li:before{content:"\0025cf   "}ul.lst-kix_j5h2oeedi37x-6{list-style-type:none}.lst-kix_eag5a795mtbr-3>li{counter-increment:lst-ctn-kix_eag5a795mtbr-3}ul.lst-kix_j5h2oeedi37x-7{list-style-type:none}ul.lst-kix_j5h2oeedi37x-8{list-style-type:none}.lst-kix_eb0mkv6he1ge-6>li:before{content:"\0025cf   "}.lst-kix_ss89m2334v3w-7>li:before{content:"\0025cb   "}.lst-kix_kfepckt4l6hg-3>li:before{content:"" counter(lst-ctn-kix_kfepckt4l6hg-3,decimal) ". "}ul.lst-kix_tx9us7pz8zze-8{list-style-type:none}ul.lst-kix_tx9us7pz8zze-7{list-style-type:none}ul.lst-kix_tx9us7pz8zze-6{list-style-type:none}ul.lst-kix_8yxwyie669z2-8{list-style-type:none}ul.lst-kix_8yxwyie669z2-7{list-style-type:none}ul.lst-kix_8yxwyie669z2-6{list-style-type:none}ul.lst-kix_8yxwyie669z2-5{list-style-type:none}.lst-kix_ij4fg4tarntm-7>li:before{content:"\0025cb   "}.lst-kix_xq8kn6xuala3-0>li:before{content:"\0025cf   "}.lst-kix_5mkgtms2qzgk-6>li:before{content:"\0025cf   "}.lst-kix_j5h2oeedi37x-3>li:before{content:"\0025cf   "}ul.lst-kix_8yxwyie669z2-4{list-style-type:none}.lst-kix_sp4bx5if0jbk-4>li:before{content:"\0025cb   "}ol.lst-kix_rggdlfkt6mfq-8.start{counter-reset:lst-ctn-kix_rggdlfkt6mfq-8 0}ul.lst-kix_8yxwyie669z2-3{list-style-type:none}.lst-kix_8r3w2rcyu87a-5>li:before{content:"\0025a0   "}ul.lst-kix_8yxwyie669z2-2{list-style-type:none}.lst-kix_129uykneptsf-4>li:before{content:"\0025cb   "}ul.lst-kix_8yxwyie669z2-1{list-style-type:none}ul.lst-kix_8yxwyie669z2-0{list-style-type:none}.lst-kix_eag5a795mtbr-6>li:before{content:"" counter(lst-ctn-kix_eag5a795mtbr-6,decimal) ". "}ul.lst-kix_tx9us7pz8zze-1{list-style-type:none}ul.lst-kix_tx9us7pz8zze-0{list-style-type:none}ul.lst-kix_uilbxigghq1p-8{list-style-type:none}ul.lst-kix_uilbxigghq1p-7{list-style-type:none}.lst-kix_2fh1qudxwat7-3>li:before{content:"\0025cf   "}ul.lst-kix_uilbxigghq1p-6{list-style-type:none}.lst-kix_e2h0cgtfdf3g-4>li:before{content:"\0025cb   "}ul.lst-kix_tx9us7pz8zze-5{list-style-type:none}.lst-kix_2najoyx65arn-3>li:before{content:"\0025cf   "}ul.lst-kix_tx9us7pz8zze-4{list-style-type:none}ul.lst-kix_tx9us7pz8zze-3{list-style-type:none}.lst-kix_rggdlfkt6mfq-0>li{counter-increment:lst-ctn-kix_rggdlfkt6mfq-0}ul.lst-kix_tx9us7pz8zze-2{list-style-type:none}ul.lst-kix_uilbxigghq1p-1{list-style-type:none}ul.lst-kix_uilbxigghq1p-0{list-style-type:none}.lst-kix_k38n94udk0el-4>li:before{content:"\0025cb   "}.lst-kix_i5y1isusbf26-0>li:before{content:"\0025cf   "}ul.lst-kix_uilbxigghq1p-5{list-style-type:none}ul.lst-kix_uilbxigghq1p-4{list-style-type:none}.lst-kix_4hyg4ehfafxi-7>li:before{content:"\0025cb   "}ul.lst-kix_uilbxigghq1p-3{list-style-type:none}ul.lst-kix_uilbxigghq1p-2{list-style-type:none}.lst-kix_84yjavmdod6-7>li:before{content:"\0025cb   "}.lst-kix_4864lfd9zdwy-1>li:before{content:"\0025cb   "}ul.lst-kix_2najoyx65arn-7{list-style-type:none}.lst-kix_3hobsm4w6nyk-7>li:before{content:"\0025cb   "}ul.lst-kix_2najoyx65arn-6{list-style-type:none}ul.lst-kix_2najoyx65arn-5{list-style-type:none}ul.lst-kix_2najoyx65arn-4{list-style-type:none}ul.lst-kix_vams35lnoe4-8{list-style-type:none}.lst-kix_z4fu53gdnrww-8>li:before{content:"\0025a0   "}.lst-kix_mhfweobdea1s-4>li:before{content:"\0025cb   "}ul.lst-kix_2najoyx65arn-8{list-style-type:none}ul.lst-kix_vams35lnoe4-5{list-style-type:none}ul.lst-kix_vams35lnoe4-4{list-style-type:none}ul.lst-kix_vams35lnoe4-7{list-style-type:none}ul.lst-kix_vams35lnoe4-6{list-style-type:none}ul.lst-kix_vams35lnoe4-1{list-style-type:none}ul.lst-kix_2najoyx65arn-3{list-style-type:none}.lst-kix_vx5oc7tbpucq-1>li:before{content:"\0025cb   "}ul.lst-kix_vams35lnoe4-0{list-style-type:none}ul.lst-kix_2najoyx65arn-2{list-style-type:none}ul.lst-kix_vams35lnoe4-3{list-style-type:none}ul.lst-kix_2najoyx65arn-1{list-style-type:none}ul.lst-kix_vams35lnoe4-2{list-style-type:none}ul.lst-kix_2najoyx65arn-0{list-style-type:none}.lst-kix_hlp788b4qti1-2>li:before{content:"\0025a0   "}.lst-kix_8joc3jsoa7de-0>li:before{content:"" counter(lst-ctn-kix_8joc3jsoa7de-0,decimal) ". "}.lst-kix_kfepckt4l6hg-0>li{counter-increment:lst-ctn-kix_kfepckt4l6hg-0}.lst-kix_tkg3l6cy3bnc-2>li{counter-increment:lst-ctn-kix_tkg3l6cy3bnc-2}.lst-kix_mhfweobdea1s-7>li:before{content:"\0025cb   "}.lst-kix_8wqgtnyy1nox-3>li:before{content:"\0025cf   "}.lst-kix_tut3ual308bf-4>li:before{content:"\0025cb   "}.lst-kix_z4fu53gdnrww-5>li:before{content:"\0025a0   "}.lst-kix_ss89m2334v3w-4>li:before{content:"\0025cb   "}.lst-kix_kruy5m9jmo50-8>li:before{content:"\0025a0   "}.lst-kix_b4xlo74k6gd0-0>li:before{content:"\0025cf   "}.lst-kix_t27vnxuzkttq-8>li:before{content:"\0025a0   "}.lst-kix_tj1ix0gi2ui9-5>li:before{content:"\0025a0   "}.lst-kix_arstwhb1c9gy-2>li:before{content:"\0025a0   "}.lst-kix_tj1ix0gi2ui9-8>li:before{content:"\0025a0   "}.lst-kix_ss89m2334v3w-1>li:before{content:"\0025cb   "}.lst-kix_b4xlo74k6gd0-3>li:before{content:"\0025cf   "}.lst-kix_dmy2ekb87fbb-4>li:before{content:"\0025cb   "}.lst-kix_8luyzq1gqi6-6>li:before{content:"\0025cf   "}.lst-kix_8i9khg2xgulr-5>li:before{content:"\0025a0   "}.lst-kix_8yxwyie669z2-6>li:before{content:"\0025cf   "}.lst-kix_8i9khg2xgulr-8>li:before{content:"\0025a0   "}.lst-kix_v0sksjnof0cj-8>li:before{content:"\0025a0   "}.lst-kix_8wqgtnyy1nox-0>li:before{content:"\0025cf   "}.lst-kix_sbk9fqxb69uy-1>li:before{content:"" counter(lst-ctn-kix_sbk9fqxb69uy-1,lower-latin) ". "}.lst-kix_an05xfuo7mar-7>li:before{content:"\0025cb   "}.lst-kix_tx9us7pz8zze-6>li:before{content:"\0025cf   "}.lst-kix_an05xfuo7mar-4>li:before{content:"\0025cb   "}.lst-kix_vx5oc7tbpucq-4>li:before{content:"\0025cb   "}.lst-kix_ie2cjnul3ua-1>li:before{content:"\0025cb   "}.lst-kix_k2lnp2ex00id-3>li:before{content:"\0025cf   "}.lst-kix_dmy2ekb87fbb-1>li:before{content:"\0025cb   "}.lst-kix_k2lnp2ex00id-0>li:before{content:"\0025cf   "}.lst-kix_t27vnxuzkttq-5>li:before{content:"\0025a0   "}.lst-kix_auto7t5cf4tl-7>li:before{content:"\0025cb   "}.lst-kix_sbk9fqxb69uy-4>li:before{content:"" counter(lst-ctn-kix_sbk9fqxb69uy-4,lower-latin) ". "}.lst-kix_zqq05ofg66-6>li:before{content:"" counter(lst-ctn-kix_zqq05ofg66-6,decimal) ". "}.lst-kix_z1b7uf8mlquv-3>li:before{content:"\0025cf   "}.lst-kix_ie2cjnul3ua-4>li:before{content:"\0025cb   "}.lst-kix_tkg3l6cy3bnc-3>li:before{content:"" counter(lst-ctn-kix_tkg3l6cy3bnc-3,decimal) ". "}.lst-kix_b01r1xbffk2a-2>li:before{content:"\0025a0   "}.lst-kix_4hyg4ehfafxi-4>li:before{content:"\0025cb   "}.lst-kix_tc68vt7q79wj-5>li:before{content:"\0025a0   "}.lst-kix_m75akhxjyj9r-1>li:before{content:"\0025cb   "}.lst-kix_awa32z9rxqor-1>li:before{content:"\0025cb   "}ul.lst-kix_zqq05ofg66-1{list-style-type:none}.lst-kix_vqsmxaeqhsi-5>li:before{content:"\0025a0   "}.lst-kix_ht2xpfpspp5-0>li:before{content:"\0025cf   "}.lst-kix_3aierr3stws2-5>li:before{content:"\0025a0   "}.lst-kix_d7lufd20rbm5-7>li:before{content:"\0025cb   "}.lst-kix_sr3soljrf7bf-8>li:before{content:"\0025a0   "}ul.lst-kix_nqcns3qrnch2-8{list-style-type:none}ul.lst-kix_nqcns3qrnch2-7{list-style-type:none}.lst-kix_ij4fg4tarntm-4>li:before{content:"\0025cb   "}.lst-kix_xq8kn6xuala3-3>li:before{content:"\0025cf   "}ul.lst-kix_nqcns3qrnch2-4{list-style-type:none}.lst-kix_pb595s4qvcov-8>li:before{content:"\0025a0   "}ul.lst-kix_nqcns3qrnch2-3{list-style-type:none}.lst-kix_6z3d30mc5xph-3>li:before{content:"\0025cf   "}ul.lst-kix_nqcns3qrnch2-6{list-style-type:none}ul.lst-kix_nqcns3qrnch2-5{list-style-type:none}ul.lst-kix_nqcns3qrnch2-0{list-style-type:none}ul.lst-kix_nqcns3qrnch2-2{list-style-type:none}ul.lst-kix_nqcns3qrnch2-1{list-style-type:none}.lst-kix_wje7sh1pcpc1-8>li:before{content:"\0025a0   "}.lst-kix_5s9vwy7hbq8r-6>li:before{content:"\0025cf   "}.lst-kix_d8i9v1tvihft-4>li:before{content:"\0025cb   "}.lst-kix_eag5a795mtbr-0>li{counter-increment:lst-ctn-kix_eag5a795mtbr-0}.lst-kix_esoh3w8sgdnn-7>li{counter-increment:lst-ctn-kix_esoh3w8sgdnn-7}.lst-kix_blb26b155eyp-6>li:before{content:"\0025cf   "}.lst-kix_tut3ual308bf-7>li:before{content:"\0025cb   "}.lst-kix_8r3w2rcyu87a-8>li:before{content:"\0025a0   "}.lst-kix_kfepckt4l6hg-6>li:before{content:"" counter(lst-ctn-kix_kfepckt4l6hg-6,decimal) ". "}.lst-kix_ls0ws55dysw8-7>li:before{content:"\0025cb   "}.lst-kix_dgmlm9mxaa4-1>li:before{content:"\0025cb   "}.lst-kix_sp4bx5if0jbk-7>li:before{content:"\0025cb   "}.lst-kix_bo4w5l7q0tq9-3>li:before{content:"\0025cf   "}.lst-kix_k38n94udk0el-1>li:before{content:"\0025cb   "}.lst-kix_90vpss294crz-4>li:before{content:"\0025cb   "}.lst-kix_e2h0cgtfdf3g-7>li:before{content:"\0025cb   "}.lst-kix_j5h2oeedi37x-6>li:before{content:"\0025cf   "}.lst-kix_2najoyx65arn-0>li:before{content:"\0025cf   "}.lst-kix_6kne4t6ii93d-0>li:before{content:"\0025cf   "}ol{margin:0;padding:0}table td,table th{padding:0}.c7{border-right-style:solid;padding:5pt 5pt 5pt 5pt;border-bottom-color:#000000;border-top-width:0pt;border-right-width:0pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:0pt;border-top-style:solid;border-left-style:solid;border-bottom-width:0pt;width:156pt;border-top-color:#000000;border-bottom-style:solid}.c21{border-right-style:solid;padding:5pt 5pt 5pt 5pt;border-bottom-color:#000000;border-top-width:0pt;border-right-width:0pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:0pt;border-top-style:solid;border-left-style:solid;border-bottom-width:0pt;width:234pt;border-top-color:#000000;border-bottom-style:solid}.c1{border-right-style:solid;padding:5pt 5pt 5pt 5pt;border-bottom-color:#000000;border-top-width:0pt;border-right-width:0pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:0pt;border-top-style:solid;border-left-style:solid;border-bottom-width:0pt;width:78pt;border-top-color:#000000;border-bottom-style:solid}.c16{border-right-style:solid;padding:5pt 5pt 5pt 5pt;border-bottom-color:#000000;border-top-width:0pt;border-right-width:0pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:0pt;border-top-style:solid;border-left-style:solid;border-bottom-width:0pt;width:66.9pt;border-top-color:#000000;border-bottom-style:solid}.c5{border-right-style:solid;padding:5pt 5pt 5pt 5pt;border-bottom-color:#000000;border-top-width:0pt;border-right-width:0pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:0pt;border-top-style:solid;border-left-style:solid;border-bottom-width:0pt;width:117pt;border-top-color:#000000;border-bottom-style:solid}.c3{margin-left:72pt;padding-top:12pt;padding-left:0pt;padding-bottom:12pt;line-height:1.15;orphans:2;widows:2;text-align:left}.c0{margin-left:36pt;padding-top:12pt;padding-left:0pt;padding-bottom:12pt;line-height:1.15;orphans:2;widows:2;text-align:left}.c6{margin-left:108pt;padding-top:12pt;padding-left:0pt;padding-bottom:12pt;line-height:1.15;orphans:2;widows:2;text-align:left}.c2{color:#000000;font-weight:400;text-decoration:none;vertical-align:baseline;font-size:11pt;font-family:"Arial";font-style:normal}.c14{color:#ffffff;font-weight:400;text-decoration:none;vertical-align:baseline;font-size:13pt;font-family:"Arial";font-style:normal}.c8{color:#000000;font-weight:700;text-decoration:none;vertical-align:baseline;font-size:13pt;font-family:"Arial";font-style:normal}.c13{color:#000000;font-weight:700;text-decoration:none;vertical-align:baseline;font-size:11pt;font-family:"Arial";font-style:normal}.c20{color:#000000;font-weight:700;text-decoration:none;vertical-align:baseline;font-size:17pt;font-family:"Arial";font-style:normal}.c17{color:#000000;font-weight:700;text-decoration:none;vertical-align:baseline;font-size:23pt;font-family:"Arial";font-style:normal}.c15{padding-top:0pt;padding-bottom:0pt;line-height:1.15;orphans:2;widows:2;text-align:left}.c18{padding-top:12pt;padding-bottom:12pt;line-height:1.15;orphans:2;widows:2;text-align:center}.c19{padding-top:18pt;padding-bottom:4pt;line-height:1.15;orphans:2;widows:2;text-align:left}.c22{padding-top:12pt;padding-bottom:2pt;line-height:1.15;orphans:2;widows:2;text-align:left}.c9{padding-top:12pt;padding-bottom:12pt;line-height:1.15;orphans:2;widows:2;text-align:left}.c12{padding-top:14pt;padding-bottom:4pt;line-height:1.15;orphans:2;widows:2;text-align:left}.c10{padding-top:24pt;padding-bottom:6pt;line-height:1.15;orphans:2;widows:2;text-align:left}.c25{border-spacing:0;border-collapse:collapse;margin-right:auto}.c23{background-color:#ffffff;max-width:468pt;padding:72pt 72pt 72pt 72pt}.c4{padding:0;margin:0}.c11{height:11pt}.c24{height:0pt}.c26{margin-left:22pt}.title{padding-top:0pt;color:#000000;font-size:26pt;padding-bottom:3pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}.subtitle{padding-top:0pt;color:#666666;font-size:15pt;padding-bottom:16pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}li{color:#000000;font-size:11pt;font-family:"Arial"}p{margin:0;color:#000000;font-size:11pt;font-family:"Arial"}h1{padding-top:20pt;color:#000000;font-size:20pt;padding-bottom:6pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h2{padding-top:18pt;color:#000000;font-size:16pt;padding-bottom:6pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h3{padding-top:16pt;color:#434343;font-size:14pt;padding-bottom:4pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h4{padding-top:14pt;color:#666666;font-size:12pt;padding-bottom:4pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h5{padding-top:12pt;color:#666666;font-size:11pt;padding-bottom:4pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h6{padding-top:12pt;color:#666666;font-size:11pt;padding-bottom:4pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;font-style:italic;orphans:2;widows:2;text-align:left}</style></head><body class="c23 doc-content"><p class="c15"><span class="c2">EXAMPLE 1:</span></p><p class="c15 c11"><span class="c2"></span></p><p class="c15"><span class="c2">Prompt- </span></p><p class="c9"><span class="c2">Write detailed study designs and GLP protocols for all required IND enabling non clinical studies for an mRNA vaccine for Influenza.</span></p><p class="c15"><span class="c2">ChatGPT- </span></p><p class="c9"><span class="c2">To provide a comprehensive study design and GLP protocols for IND-enabling non-clinical studies for an mRNA vaccine for Influenza, could you clarify:</span></p><ul class="c4 lst-kix_1qqoijhe7jmx-0 start"><li class="c0 li-bullet-0"><span class="c2">Do you need specific species selection for toxicology and pharmacology studies, or should I provide general recommendations based on regulatory expectations?<br></span></li><li class="c0 li-bullet-0"><span class="c2">Should the designs include both in vitro and in vivo studies?<br></span></li><li class="c0 li-bullet-0"><span class="c2">Would you like a section on biodistribution and pharmacokinetics?<br></span></li><li class="c0 li-bullet-0"><span class="c2">Should I include immunogenicity and efficacy models?<br></span></li><li class="c0 li-bullet-0"><span class="c2">Are you looking for a full regulatory framework, including adherence to FDA, EMA, or ICH guidelines?<br></span></li><li class="c0 li-bullet-0"><span class="c2">Any particular format or level of detail required (e.g., tabulated study plans, full narrative protocols)?<br></span></li></ul><p class="c9"><span class="c2">Let me know so I can tailor the document accordingly.</span></p><p class="c9"><span class="c2">Author: </span></p><p class="c9"><span class="c2">Yes to all. Need you to provide recommendations on species, dosage, include both in vitro and in vivo, yes to biodistribution and PK, include efficacy and immunogenicity models, looking for a full reg framework, provide tabulated study plans and fill narrative protocols.</span></p><p class="c9"><span class="c2">ChatGPT:</span></p><p class="c9"><span class="c2">Got it! I will prepare detailed study designs and GLP protocols for all required IND-enabling non-clinical studies for an mRNA vaccine for Influenza. This will include:</span></p><ul class="c4 lst-kix_ftoogxlvmqxr-0 start"><li class="c0 li-bullet-0"><span class="c2">Recommendations on species selection and dosing strategies<br></span></li><li class="c0 li-bullet-0"><span class="c2">Both in vitro and in vivo study designs<br></span></li><li class="c0 li-bullet-0"><span class="c2">Biodistribution and pharmacokinetics studies<br></span></li><li class="c0 li-bullet-0"><span class="c2">Immunogenicity and efficacy models<br></span></li><li class="c0 li-bullet-0"><span class="c2">A full regulatory framework aligned with FDA, EMA, and ICH guidelines<br></span></li><li class="c0 li-bullet-0"><span class="c2">Tabulated study plans for easy reference<br></span></li><li class="c0 li-bullet-0"><span class="c2">Fully developed narrative protocols<br></span></li></ul><p class="c9"><span class="c2">I will let you know when the comprehensive document is ready for your review.</span></p><p class="c9"><span class="c2">Final Response: </span></p><h1 class="c10" id="h.8p1ds0nmo555"><span class="c17">IND-Enabling Non-Clinical Study Plan for an mRNA Influenza Vaccine</span></h1><h2 class="c19" id="h.oxe2j0a8cufs"><span class="c20">Species Selection and Dosage Recommendations</span></h2><p class="c9"><span class="c2">Toxicology Species: For repeat-dose toxicology studies, a single relevant animal species is generally sufficient, provided it mounts an immune response to the influenza antigen . Rodents (e.g. Sprague-Dawley or Wistar rats) are commonly used due to extensive historical data and proven sensitivity to vaccine-induced immune effects . Rats have been utilized in prior mRNA vaccine programs (e.g. COVID-19 vaccines) to successfully assess safety . A second species is not usually required for initial toxicity testing , unless warranted by unique aspects of the vaccine (e.g. a novel adjuvant or if rodent data are insufficient) . Non-human primates (NHPs, such as cynomolgus macaques) may be considered as a supplemental species for immunotoxicity or if the immune mechanism needs further characterization, but are not mandated if the rodent model is immunologically relevant . Both sexes should be included in toxicology studies (typically 5-10/sex/group in rodents) to detect any sex-specific effects.</span></p><p class="c9"><span class="c2">Biodistribution Species: Small animals are recommended for biodistribution studies, with mice or rats being appropriate. Mice allow use of larger group sizes and potential transgene reporter systems (e.g. luciferase) for imaging , whereas rats provide greater blood and tissue volumes for analytical assays. Prior mRNA vaccine biodistribution studies have used rodents to map tissue distribution . Rodents are immunologically responsive to mRNA-LNP vaccines, ensuring that distribution data reflect relevant uptake (e.g. via lymphatics to lymph nodes) . A non-rodent species for biodistribution is generally not required; biodistribution patterns of LNP-formulated mRNA are largely determined by formulation and are similar across mammals .</span></p><p class="c9"><span class="c2">Efficacy Model Species: Efficacy and immunogenicity evaluations should use influenza-susceptible species. The ferret (Mustela putorius furo) is the gold-standard small animal model for human influenza, as ferrets are naturally susceptible to human influenza viruses and exhibit human-like symptoms (fever, respiratory signs) upon challenge. Ferret models are routinely used to assess vaccine-mediated protection, including reduction of viral load and clinical signs after intranasal challenge . We recommend ferrets for viral challenge studies to demonstrate protection; for example, ferrets immunized with an mRNA flu vaccine showed protection from disease and death upon H5N1 or H7N9 challenge . Mice (e.g. Balb/c or C57BL/6) are also useful for immunogenicity and preliminary efficacy screening. Mice can be infected with adapted or high-dose influenza strains, providing a convenient model to evaluate dose-response of immune outcomes and even survival in lethal challenge models. Indeed, robust antibody responses and neutralization have been observed in mice given mRNA influenza vaccines . NHPs (such as rhesus macaques) may be considered to characterize immune responses in a phylogenetically closer model and to assess tolerability, but they often exhibit mild disease. Thus, NHP use can be reserved for cases where translation of immunogenicity needs confirmation or to support dose selection, rather than as a primary efficacy model.</span></p><p class="c9"><span class="c2">Dosage Level Selection: Dose levels for animal studies should bracket the anticipated clinical dose and include a safety margin. Regulatory guidance recommends using a high dose that elicits maximal immune response in the test species . This ensures the vaccine&rsquo;s pharmacologic effect is stressed, maximizing the chance to observe any immune-mediated toxicities. Typically, three dose levels are included in toxicology studies (in addition to a placebo control):</span></p><ul class="c4 lst-kix_v0sksjnof0cj-0 start"><li class="c0 li-bullet-0"><span class="c2">Low dose: a dose at or slightly above the projected human dose (on a &micro;g/mouse or &micro;g/kg basis). This provides relevance to clinical exposure.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Mid dose: an intermediate level (e.g. ~3&times; the human dose equivalent) to explore dose-response.<br></span></li><li class="c0 li-bullet-0"><span class="c2">High dose: a high-exposure dose (e.g. ~10&times; human dose, or the dose that saturates antibody response in animals ). For mRNA vaccines, this could be limited by formulation tolerability; however, use of the maximal immunogenic dose is advised even if it exceeds the planned clinical dose, to ensure a safety margin .<br></span></li></ul><p class="c9"><span class="c2">Dose unit selection (&micro;g of mRNA per animal vs. &micro;g/kg) should be justified. For vaccines, per-animal dosing (fixed dose per injection) is often used, mimicking the human regimen, since immune activation tends to depend on absolute dose per site rather than body weight. For example, if the human dose is 50 &micro;g, rat studies might use doses of ~50 &micro;g, 250 &micro;g, and 500&ndash;1000 &micro;g per injection to achieve 1&times;, 5&times;, and 10&ndash;20&times; the human amount, respectively. These levels have precedent: mRNA COVID-19 vaccine studies in rats used doses on the order of 30&ndash;100 &micro;g per dose (for a human dose of ~30 &micro;g) without serious toxicities . All animal dose levels will be delivered in the same formulation (LNP) and volume per injection as intended clinically (e.g. ~0.1&ndash;0.5 mL intramuscularly) to maintain comparable local exposure.</span></p><p class="c9"><span class="c2">Dosing Regimen: The dosing schedule in animals should reflect the intended clinical schedule . If the vaccine will be given as a prime-boost (two-dose) regimen in humans (~3&ndash;4 weeks apart), the repeat-dose tox study should mirror that (e.g. two IM injections 21 or 28 days apart) . This allows evaluation of both initial and booster dose tolerability and potential immune exacerbation on second dose. If the clinical plan is a single dose (as might be the case for a seasonal flu shot), a single-dose toxicology study with an appropriate observation period can suffice, though including a second dose is often still useful to simulate a worst-case scenario (multiple exposures). In efficacy models, at least a prime-boost regimen is recommended for novel vaccines in naive animals to ensure robust immunity prior to challenge , whereas in previously influenza-exposed animals a single booster may be evaluated . All animal studies will use the intramuscular (IM) route of administration, typically in the thigh or shoulder muscle, consistent with the intended human route . Using the clinical route in animals is required to ensure the local tissue exposure and immune trafficking are representative of human vaccination .</span></p><h2 class="c19" id="h.zb93emx5uyfs"><span class="c20">In Vitro and In Vivo Study Designs</span></h2><h3 class="c12" id="h.mbt90h213qun"><span class="c8">In Vitro Assays</span></h3><p class="c9"><span class="c2">Non-clinical development will include several in vitro assays to characterize the mRNA vaccine&rsquo;s activity and safety profile before in vivo testing:</span></p><ul class="c4 lst-kix_n23m8p7c7hw6-0 start"><li class="c0 li-bullet-0"><span class="c2">mRNA Expression and Protein Characterization: A cell-based expression assay will be used to confirm that the delivered mRNA produces the intended influenza antigen (e.g. hemagglutinin, HA) in vitro. For example, transfecting the LNP-formulated mRNA into cultured mammalian cells (such as HEK293 or CHO cells) and then measuring HA protein levels (by Western blot or ELISA) will verify proper translation and folding. The expressed HA protein&rsquo;s functionality can be assessed via a hemagglutination assay with red blood cells, confirming that the antigen can bind sialic acid receptors similar to wild-type viral HA.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Potency Assay: An in vitro potency or bioassay will be developed to quantify the vaccine&rsquo;s biological activity lot-to-lot. This may overlap with the expression assay above. For instance, a luciferase reporter mRNA of analogous design can serve as a surrogate: when LNP-encapsulated luciferase mRNA is added to cell culture, luminescence output correlates with mRNA delivery efficiency and translational potency . This helps set release specifications and ensures consistency in the formulated vaccine&rsquo;s capability to transduce cells.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Innate Immune Activation (Cytokine Release) Assay: Because mRNA and lipid nanoparticles can stimulate innate immunity, in vitro assays will evaluate cytokine responses. Human peripheral blood mononuclear cells (PBMCs) or whole blood will be incubated with the mRNA-LNP to measure rapid cytokine release (e.g. IL-6, TNF-&alpha;, type I interferons). This cytokine release assay helps predict acute reactogenicity or potential for cytokine storm. Acceptable levels are those comparable to or lower than known safe vaccines. Any ex vivo cytokine testing using human blood will follow recommended safety margins to ensure no excessive inflammatory signals prior to first-in-human trials .<br></span></li><li class="c0 li-bullet-0"><span class="c2">Inflammation/Complement Activation Assays: If the LNP contains novel excipients (e.g. PEGylated lipids), an in vitro complement activation test (e.g. measuring C5a in human serum after exposure) might be performed to assess risk of infusion reactions (such as CARPA). Additionally, in vitro pyroginicity (LAL endotoxin test) will be done on the vaccine preparation, and any specific assay for innate sensor activation (e.g. TLR assays using reporter cell lines) can characterize the immunostimulatory profile of the mRNA (which often includes nucleoside modifications to dampen innate sensing).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Stability and Protein Binding Studies: In vitro studies will also examine the stability of the mRNA-LNP in biological fluids. Incubating the vaccine in simulated physiological conditions (37&deg;C, in serum) and sampling over time for mRNA integrity (via RNA gel or RT-PCR) will estimate the half-life of the mRNA outside cells. This informs on how quickly the mRNA might be degraded by extracellular RNases after injection. Also, plasma protein binding and lipoprotein interactions of the LNP will be studied, as apolipoprotein association can influence biodistribution (e.g. ApoE binding leading to hepatic uptake ).<br></span></li></ul><h3 class="c12" id="h.28mkuq6e4i1o"><span class="c8">In Vivo Models</span></h3><p class="c9"><span class="c2">A variety of in vivo models will be used to evaluate the pharmacology and safety of the mRNA influenza vaccine:</span></p><ul class="c4 lst-kix_vqsmxaeqhsi-0 start"><li class="c0 li-bullet-0"><span class="c2">Rodent Toxicology Model (Rat): As described, rats will serve as the primary model for GLP toxicology. Groups of rats will receive the vaccine or control as per the dose regimen, and will be observed for toxicity endpoints (clinical signs, body weight, clinical pathology, and histopathology of all major organs). Rats are large enough for serial blood sampling and full necropsies, enabling thorough safety evaluation including local tolerability at the injection site. The rat model will also be used for toxicokinetics of the vaccine if needed &ndash; although classical pharmacokinetic studies are not required for vaccines, we may measure systemic exposure of the mRNA (by plasma qPCR) and any induced cytokines post-dose to characterize the acute phase response .<br></span></li><li class="c0 li-bullet-0"><span class="c2">Rodent Immunogenicity Model (Mouse): Mice will be used to rapidly assess immunogenicity and guide vaccine dose/formulation optimization. Inbred strains (such as Balb/c) allow detailed immune phenotyping. Mice (5-10 per group) will be immunized with various dose levels of the mRNA vaccine. Key readouts include antigen-specific antibody titers (total IgG and neutralizing antibodies) and T-cell responses (e.g. IFN-&gamma; ELISpot for CD8&#8314; T cells against influenza epitopes). This small animal model enables screening of immune responses to different vaccine variants (e.g. modified mRNA vs unmodified, or different lipid formulations) prior to advancing the final candidate. Mouse data will also confirm the Th1/Th2 bias of the immune response by isotyping antibodies (IgG2a vs IgG1) and measuring cytokines (IFN-&gamma;, IL-4). A Th1-biased response (indicative of cell-mediated immunity) is generally preferred for viral vaccines to avoid potential vaccine-associated enhanced respiratory disease (VAERD) that can occur with Th2-skewed profiles .<br></span></li><li class="c0 li-bullet-0"><span class="c2">Ferret Efficacy Model: Ferrets (outbred, young adults) will be employed for pathogen challenge studies to demonstrate vaccine efficacy. Ferrets are uniquely susceptible to human influenza viruses and exhibit respiratory illness similar to humans. In the efficacy study, ferrets will be vaccinated (two doses IM) and then challenged with live influenza virus (e.g. H1N1 or H3N2 strain matching the vaccine antigen, or a heterologous strain for breadth testing) via the intranasal route. Outcomes will include reduction in viral load in nasal washes and lung tissue, mitigation of clinical signs (fever, weight loss, nasal discharge, sneezing, lethargy scored via an index), and survival (if a high-dose, potentially lethal challenge is used). The ferret model also allows assessment of transmission, if needed, by co-housing vaccinated ferrets with naive sentinels after challenge to see if vaccine prevents virus spread. Ferret data will be pivotal in showing real-world protective effect, as evidenced in prior studies where mRNA influenza vaccines protected ferrets from disease after virulent virus challenge .<br></span></li><li class="c0 li-bullet-0"><span class="c2">Non-Human Primate Immunogenicity Model (Optional): While not strictly required, a NHP model (e.g. rhesus macaques) may be included to gather additional immunogenicity and safety data in a closer-to-human species. NHPs can receive the vaccine IM and be monitored for clinical observations and detailed immune responses, such as neutralizing antibody titers and T-cell memory responses in blood and bronchoalveolar lavage. Monkeys might not exhibit strong illness upon human influenza virus challenge (many strains cause only subclinical or mild infection), so NHPs would mainly serve to confirm the immunogenic profile (e.g. breadth of antibody response, durability of immunity) and to detect any primate-specific toxicities or exaggerated inflammatory responses that rodents/ferrets might not show. Given the expense and ethical considerations, NHP studies will be used judiciously, in line with the 3Rs (Replace, Reduce, Refine) principle, and possibly leveraged as part of a platform data package if similar mRNA vaccines have been tested in NHPs .<br></span></li><li class="c0 li-bullet-0"><span class="c2">Rabbit Model for Reproductive Toxicology: Although not part of initial IND-enabling work (unless first-in-human trials will involve pregnant individuals), ultimately a developmental and reproductive toxicity study in rabbits or rats will be planned in accordance with ICH S5 (R3). Rabbits are often chosen for developmental toxicity assessment of vaccines and were used for COVID-19 mRNA vaccine DART studies . This model would expose pregnant rabbits to the vaccine (e.g. before mating and during gestation) to evaluate embryo-fetal development and postnatal outcomes . However, this study is typically timed later (before Phase III or inclusion of women of childbearing potential in large trials) , not strictly for initial IND, and thus is mentioned for completeness.<br></span></li></ul><p class="c9"><span class="c2">Each in vivo model is designed to address specific questions (safety, pharmacokinetics, immunogenicity, efficacy), providing a comprehensive non-clinical dataset. All in vivo studies will be conducted with appropriate controls (e.g. placebo injection or comparator vaccine) and adhere to humane endpoints.</span></p><h2 class="c19" id="h.es7w52og0ena"><span class="c20">Biodistribution and Pharmacokinetics</span></h2><p class="c9"><span class="c2">A dedicated biodistribution study will characterize how the mRNA vaccine distributes and clears in vivo, since traditional pharmacokinetic (PK) studies (blood drug levels over time) are generally not required for vaccines . Instead, regulatory guidance for mRNA vaccines emphasizes tissue distribution and persistence studies to understand where the mRNA/LNP and encoded protein go after injection . Key parameters and design features for the biodistribution/PK assessment are as follows:</span></p><p class="c9"><span class="c2">Test Article and Species: The biodistribution study will use the fully formulated mRNA-LNP vaccine (with the same antigen sequence as the clinical candidate). Rodents (likely rats, e.g. 4&ndash;8 weeks old, female rats often used for distribution studies to also assess ovarian exposure) will be the test species. Rats are chosen for their size (enabling serial sacrifices and sufficient tissue per timepoint) and because prior biodistribution data on LNP-mRNA (e.g. COVID-19 vaccines) were generated in rats , allowing comparison. A small pilot in mice may be used if employing a reporter gene (like luciferase mRNA) for real-time imaging, but the definitive tissue quantitative study will be in rats with the actual vaccine construct.</span></p><p class="c9"><span class="c2">Dosing and Time Points: Rats will receive a single IM dose of the mRNA vaccine (typically the high dose used in tox studies, to maximize detectability). A single dose is sufficient for distribution profiling; if multiple clinical doses are planned, distribution after the first dose is usually representative, though we will also assess whether repeated dosing leads to any tissue accumulation by possibly evaluating tissues after the second dose in a subset. Multiple sacrifice time points will be included to cover the kinetics of clearance: e.g. 6 hours, 24 hours, 72 hours, 1 week, 2 weeks, and 4 weeks post-injection. At least 3 rats/sex per timepoint will be used (with possibly more at early times to capture variability). This schedule will capture the expected peak distribution (within hours) and the decline over days to weeks.</span></p><p class="c9"><span class="c2">Tissues Analyzed: A broad panel of tissues will be collected at necropsy for analysis of mRNA and protein: injection site muscle (and skin around it), draining lymph nodes (e.g. inguinal or popliteal LN near injection site), spleen, liver, kidneys, lungs, heart, brain, gonads (ovaries and testes), bone marrow, and blood plasma. These cover major organ systems and areas of interest (e.g. gonads for reproductive safety, brain to check if any crosses blood-brain barrier, etc.). Based on prior studies, we anticipate the highest mRNA concentrations at the injection site and in draining lymphatics, with some distribution to the liver and spleen via systemic circulation . Indeed, mRNA-LNP vaccines have shown that after IM injection, most of the expression (e.g. luciferase reporter) remains at the muscle and to a lesser extent in liver and spleen . We will confirm if the influenza mRNA behaves similarly.</span></p><p class="c9"><span class="c2">Analytics: Quantitative RT-PCR will be the primary method to detect and quantify mRNA in tissues. Tissue samples will be homogenized and total RNA extracted, then a sensitive PCR assay specific to the mRNA sequence (e.g. amplifying the untranslated region unique to the vaccine) will measure mRNA copies per &micro;g of RNA or per gram of tissue. This yields the concentration of mRNA in each tissue at each time. Additionally, if feasible, the LNP may be labeled (for example, with a trace of a radioisotope or a fluorescent lipid) to directly track nanoparticle distribution; the concentration of label in each tissue can be measured by scintillation counting or fluorescence, providing an independent measure of LNP biodistribution.</span></p><p class="c9"><span class="c2">Encoded Protein Detection: While the main focus is the mRNA, we will also assess the distribution of the expressed antigen (influenza HA protein). This can be done by ELISA on tissue lysates or immunohistochemistry on tissue sections to see which cells are producing the antigen. Prior experiments with luciferase reporters show that protein expression is largely local (muscle) with minor expression in liver/spleen . For HA, we will confirm that significant protein is present at the injection site muscle fibers and in antigen-presenting cells in lymph nodes, and minimal elsewhere.</span></p><p class="c9"><span class="c2">Pharmacokinetic Parameters: Key PK-like parameters to derive include:</span></p><ul class="c4 lst-kix_b4xlo74k6gd0-0 start"><li class="c0 li-bullet-0"><span class="c2">Peak tissue concentration (Cmax) of mRNA in key organs (and the time to peak, Tmax). We expect muscle at the injection site to have the highest Cmax, likely at the earliest time (within hours).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Half-life (t&frac12;) of mRNA in tissues: by plotting mRNA copy number decline over time, an apparent half-life can be estimated. We anticipate a rapid drop in most tissues, with clearance of the bulk of mRNA within a few days as mRNA is translated and degraded. For instance, luciferase mRNA signal in muscle disappears within ~4 days in mice . We will quantify if any low-level mRNA remains at later time points (e.g. day 14 or 28) in any tissue.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Area-under-curve (AUC): although classical AUC (over time in plasma) is less relevant for locally acting vaccines, we can calculate AUC for mRNA presence in the bloodstream to gauge total systemic exposure. Plasma will be tested for mRNA at multiple early times (e.g. 1h, 6h, 24h) to see if mRNA enters circulation transiently.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Persistence and Clearance: define when the mRNA is no longer detectable in each tissue. Complete clearance by a certain time is important for safety. If any tissue (e.g. lymph node or injection site) shows residual mRNA at later time points, that will be noted and investigated, though we expect clearance in the order of days to a couple of weeks at most.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Metabolism: The study will also consider how the mRNA and LNP are metabolized. mRNA is expected to break down into nucleotides/nucleosides that are naturally handled by cellular metabolism (and thus not persist or accumulate). The LNP lipids may be metabolized via normal lipid pathways in the liver . We will document evidence of mRNA degradation (e.g. shorter RNA fragments via PCR, or loss of signal over time indicating metabolism to inert components). Any novel metabolites are not expected given the nature of mRNA and lipid components, but the study will be mindful of any unexpected persistence of chemically modified nucleosides in tissues.<br></span></li></ul><p class="c9"><span class="c2">Special Considerations: We will evaluate if the mRNA or protein crosses the blood-brain barrier or placenta. While not expected (and prior studies indicate almost no distribution to brain for IM route ), a sensitive PCR on brain tissue and, if applicable, on fetal tissues (in case of a pregnant animal, though we are not including pregnant animals in this distribution study) can address these theoretical concerns. The same applies to breast milk secretion (not applicable in our study design, but noted as a consideration in guidelines ).</span></p><p class="c9"><span class="c2">All biodistribution work will adhere to good laboratory practice where possible. Absolute GLP compliance for biodistribution is not strictly required by regulators if the data are supportive (and not the primary safety determinant), but the study will be well-controlled and documented . If any part of the study is non-GLP (for example, use of an exploratory reporter construct), it will be justified and key portions (like qPCR quantification) may still be done in a GLP-compliant manner or with a QA oversight . The aim is to meet FDA/EMA expectations by providing a clear understanding of tissue distribution and clearance of the mRNA vaccine prior to human trials, confirming that no tissue retains significant levels of the product long-term and that the vaccine behaves like a typical non-replicating, non-persistent immunogen .</span></p><h2 class="c19" id="h.weycrr654fyn"><span class="c20">Immunogenicity and Efficacy Models</span></h2><p class="c9"><span class="c2">Robust evaluation of immunogenicity (the immune response generated by the vaccine) and efficacy (protection against influenza) in non-clinical models is a critical component of IND-enabling studies. These data provide proof-of-concept that the mRNA vaccine induces appropriate immunity and can protect against influenza infection, guiding dose selection and supporting the rationale for human trials .</span></p><p class="c9"><span class="c2">Immunogenicity Endpoints (Humoral and Cellular): The vaccine&rsquo;s ability to elicit both antibody- and cell-mediated immune responses will be assessed:</span></p><ul class="c4 lst-kix_6pozql3s9ehi-0 start"><li class="c0 li-bullet-0"><span class="c2">Humoral Immune Response: Serum samples from vaccinated animals (mice, ferrets, and optionally NHPs) will be tested for influenza-specific antibodies. The gold standard assay for flu vaccines is the hemagglutination inhibition (HAI) assay, which measures antibodies that prevent the virus from agglutinating red blood cells. An HAI titer of &ge;1:40 is often correlated with protection in humans; we will use this as a benchmark . Additionally, neutralizing antibody titers will be determined using live virus or pseudovirus neutralization assays. This involves incubating serial dilutions of animal sera with influenza virus and then infecting cells, observing the reduction in viral cytopathic effect or viral gene expression. High neutralization titers indicate the vaccine induced antibodies capable of blocking virus infection. We will evaluate responses against the specific strain encoded by the vaccine and potentially drifted strains to gauge breadth.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Isotype and Subclass: To understand the Th1/Th2 bias, IgG subclasses (in animals that have them, e.g. IgG2a/IgG1 in mice) will be measured. A Th1-biased response (IgG2a in mice, indicating strong IFN-&gamma; environment) is generally desirable for viral clearance, whereas a Th2-skewed profile could indicate propensity for allergic-type responses. The presence of mucosal IgA (in nasal washes or bronchoalveolar lavage) will also be checked in ferrets or NHPs, as mucosal antibodies are important for blocking infection at the respiratory entry point.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Cellular Immune Response: T-cell responses will be measured via assays such as ELISpot and intracellular cytokine staining (ICS). Spleen cells or blood lymphocytes from immunized animals will be stimulated with influenza antigen peptides (e.g. conserved epitopes from HA, or NP if included) to detect antigen-specific IFN-&gamma; producing T cells (CD4&#8314; and CD8&#8314;). We aim to show the vaccine induces a Th1-biased T cell response (characterized by IFN-&gamma;, IL-2, TNF-&alpha; production) rather than a Th2 response (IL-4, IL-5, IL-13). For example, a universal flu mRNA vaccine encoding internal proteins was shown to boost broadly reactive T cells in ferrets ; our HA-focused vaccine will mainly induce CD4&#8314; T helper and follicular helper responses to aid antibody production, but CD8&#8314; T cell responses to HA (and any other encoded proteins) will also be assessed. In NHPs, flow cytometry will be used to profile memory T cell subsets and their cytokine profiles (central vs effector memory, etc.). We will also monitor cytokines in serum after vaccination (transient IL-6, IFN-&alpha;, etc.) as an indication of innate immune activation which can shape the adaptive response.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Duration of Immunity: Although full duration studies may go beyond IND stage, we will collect immune readouts at multiple time points (e.g. 2 weeks after prime, 2 weeks after boost, and periodically up to a few months) to get a preliminary sense of how long antibodies and T cells persist. This can inform booster timing in clinical trials.<br></span></li></ul><p class="c9"><span class="c2">Efficacy (Challenge) Studies: To demonstrate that immune responses correspond to actual protection, we will conduct challenge studies in animals:</span></p><ul class="c4 lst-kix_74bj55g8eq2b-0 start"><li class="c0 li-bullet-0"><span class="c2">Ferret Influenza Challenge: As described, ferrets will be challenged with live virulent influenza after vaccination. Key efficacy endpoints include:<br></span></li></ul><ul class="c4 lst-kix_74bj55g8eq2b-1 start"><li class="c3 li-bullet-0"><span class="c2">Clinical Disease Scores: Ferrets will be observed daily post-challenge for signs of illness (sneeze frequency, nasal discharge, activity level, breathing rate). A scoring system (0 = normal, increasing to 3 or 4 for severe signs) will be used . Body weight will be measured daily; weight loss is a quantifiable indicator of disease severity in ferrets (e.g. &gt;10% loss is significant). Body temperature will be monitored (via implant or rectal thermometer) to detect fever responses; an elevated temperature from baseline indicates systemic infection. We will compare mean weight change and fever curves between vaccinated and control groups.<br></span></li><li class="c3 li-bullet-0"><span class="c2">Viral Load Reduction: Nasal wash samples will be collected from each ferret at days 1-7 post-challenge to measure virus titers (using TCID&#8325;&#8320; cell culture assay or RT-qPCR for viral RNA). We expect vaccinated ferrets to have significantly lower viral titers and a shorter duration of shedding than unvaccinated controls. At necropsy (around day 4-5 or earlier if humane endpoint criteria are met ), lung tissues will be collected to measure viral load in the lower respiratory tract. A successful vaccine should greatly reduce or eliminate virus in the lungs.<br></span></li><li class="c3 li-bullet-0"><span class="c2">Survival: If the challenge virus dose is high enough to cause mortality in unvaccinated animals (e.g. a lethal H5N1 dose), then survival rate is a critical endpoint. Vaccinated animals should have improved survival. For example, in one study a high-dose H7N9 challenge was lethal to mock-vaccinated ferrets, whereas mRNA-vaccinated ferrets survived, demonstrating efficacy .<br></span></li><li class="c3 li-bullet-0"><span class="c2">Pathology: Detailed pathology will be done on ferrets post-challenge. Lung tissues will be examined histologically for pneumonia, inflammation, and scored for severity of lesions. We anticipate that vaccinated ferrets will have minimal lung pathology compared to controls (which might show bronchopneumonia typical of influenza). The presence of any vaccine-associated enhanced disease (immunopathology) will be carefully looked for &ndash; e.g. an excessive eosinophilic or allergic inflammation would signal a problem (none is expected given an mRNA vaccine typically induces Th1 responses, but this was a concern historically for inactivated vaccines in naive models).<br></span></li><li class="c3 li-bullet-0"><span class="c2">Protection in Other Models: If a mouse challenge model is also used (e.g. using a mouse-adapted influenza strain), endpoints would be similar: body weight loss, survival, lung virus titers, and lung pathology in vaccinated vs. control mice.<br></span></li></ul><p class="c9"><span class="c2">Correlates of Protection: The data from immunogenicity and challenge studies will be analyzed to identify correlates of protection (e.g. a threshold HAI or neutralizing titer that predicts protection from infection). These correlates can guide human trial immunogenicity targets. For instance, if ferrets with HAI titers &gt;40 all avoid severe disease, that titer might be a goal for the vaccine in humans.</span></p><p class="c9"><span class="c2">Sample Size and Statistics: All immunogenicity and efficacy experiments will be designed with sufficient group sizes to detect meaningful differences. For ferret studies, 6&ndash;8 animals per group is typical to show statistically significant differences in viral load or weight loss. For mouse studies, groups of 10-15 per group (especially in survival experiments) may be used. Statistical analyses will include t-tests or ANOVA for parametric data (antibody titers, log-transformed viral loads), rank-sum tests for nonparametric data (scores), and Kaplan-Meier survival analysis with log-rank test (for survival curves). A p-value &lt; 0.05 will be considered significant. The study protocol will pre-define primary efficacy endpoints (e.g. day 4 lung viral titer difference) to power the study appropriately.</span></p><p class="c9"><span class="c2">Together, the immunogenicity and challenge studies will provide comprehensive evidence that the mRNA influenza vaccine can induce the desired immune profile and protect against influenza infection in relevant models. This satisfies regulatory expectations that a vaccine&rsquo;s pharmacology (mechanism of action and efficacy) is well-characterized in animals before proceeding to human trials .</span></p><h2 class="c19" id="h.pkwq0oiq9uwp"><span class="c20">Regulatory Framework (Compliance with FDA, EMA, ICH Guidelines)</span></h2><p class="c9"><span class="c2">All proposed studies are designed to comply with the applicable regulatory guidelines for IND-enabling non-clinical development of vaccines, including those from the U.S. Food and Drug Administration (FDA), European Medicines Agency (EMA), International Council for Harmonisation (ICH), and World Health Organization (WHO). Below is a summary of how the program aligns with these frameworks:</span></p><ul class="c4 lst-kix_uilbxigghq1p-0 start"><li class="c0 li-bullet-0"><span class="c2">Good Laboratory Practice (GLP): Pivotal safety studies &ndash; notably the repeat-dose toxicology study (and any safety pharmacology or local tolerance assessments) &ndash; will be conducted in compliance with GLP regulations (21 CFR Part 58) . FDA and EMA expect that toxicity studies supporting first-in-human trials are GLP-compliant . Our toxicology protocol and facility adhere to GLP requirements: including a formal protocol approval, QA monitoring, archiving of raw data, and a GLP statement in the final report. If any part of a study is not done under full GLP (for example, a preliminary range-finding study or certain exploratory endpoints), this will be justified and clearly stated with description of deviations . The overall non-clinical data package will highlight which studies were GLP. Generally, toxicology and DART studies will be GLP, whereas efficacy/immunogenicity studies may be non-GLP (conducted with scientific rigor but not under GLP, as is standard for pharmacology proof-of-concept data).<br></span></li><li class="c0 li-bullet-0"><span class="c2">ICH Guidelines: The program follows ICH M3(R2) guidance on timing of non-clinical studies relative to clinical trials, ensuring that required safety studies are completed before human exposure. ICH M3 recommends that repeat-dose toxicity (of appropriate duration) in one species is sufficient to support a short-duration clinical trial for vaccines, and that any reproduction toxicity studies (e.g. DART) can be conducted prior to Phase III or licensure for vaccines not initially intended for pregnant women . We will comply with ICH S6(R1) (Preclinical Safety Evaluation of Biotechnology-Derived Pharmaceuticals) principles for selecting relevant species &ndash; which in the case of vaccines, means an immunologically relevant species rather than strictly requiring two species. This is consistent with WHO guidance that one responsive species is acceptable . For immunotoxicity, ICH S8 guidance (Immunotoxicity Studies) will be considered: though vaccines by design stimulate the immune system, we will assess if there&rsquo;s any undue immunosuppression or autoimmunity potential (for example, by examining lymphoid organ histopathology and autoantibody screening in animals) to address S8 concerns. ICH S5(R3) (reproductive toxicology) is noted: as per S5 and WHO vaccine guidance, our developmental toxicity study (when conducted) will integrate fertility, embryofetal, and prenatal development endpoints into one study in one appropriate species (likely rat) .<br></span></li><li class="c0 li-bullet-0"><span class="c2">FDA/CBER Guidance: FDA&rsquo;s guidance &ldquo;Nonclinical Development of Preventive Vaccines&rdquo; (2002) and related documents (e.g. CBER&rsquo;s guidelines on adjuvants and COVID-19 vaccine guidances) emphasize that the scope of required studies depends on novelty of the product . For a novel mRNA vaccine platform, regulators expect a thorough demonstration of safety in animals prior to FIH . We will engage with FDA early (Pre-IND meeting) to confirm our plan, as FDA encourages discussing the specific nonclinical package needed for a given vaccine . Notably, FDA guidance suggests that if there is prior experience with a platform, some data can be leveraged, but for a new mRNA influenza vaccine (with perhaps prior mRNA COVID experience), we will still conduct product-specific tox and distribution studies. Our plan includes all elements FDA expects: immunogenicity characterization, dose-ranging in animals, GLP toxicity with full pathology, and characterization of distribution . We also abide by the FDA guidance that any toxicity and local tolerance studies used to support FIH must be completed and analyzed before dosing humans .<br></span></li><li class="c0 li-bullet-0"><span class="c2">EMA Requirements: EMA&rsquo;s guidance for vaccines (e.g. EMA &ldquo;Note for Guidance on Preclinical Pharmacological and Toxicological Testing of Vaccines&rdquo;) aligns with FDA and WHO in requiring one relevant species for toxicity, demonstration of immune response in animal models, and justification of any novel adjuvant components. EMA has also indicated openness to platform technology data &ndash; for instance, data from other LNP-mRNA vaccines could support aspects of our IND . We will cite any available toxicology data from similar mRNA vaccines (such as published COVID-19 vaccine tox data) as supportive information, while still providing our own product-specific studies as required. EMA and other regulators (via ICMRA) have encouraged using prior platform data to streamline development , a strategy we embrace to avoid unnecessary duplication and to uphold 3R principles . However, since this is a new indication (influenza), we will conduct all critical studies (platform data will be supplemental). EMA will also expect a environmental risk assessment for any GMO aspects; however, mRNA vaccines are not self-replicating and degrade quickly, so they pose minimal environmental risk (this will be documented as well).<br></span></li><li class="c0 li-bullet-0"><span class="c2">WHO Guidelines: We have closely followed WHO&rsquo;s &ldquo;Guidelines on Nonclinical Evaluation of Vaccines&rdquo; (WHO TRS 927, 2005) in crafting this program. For example, WHO notes that the animal species should develop an immune response to the vaccine , and that the toxicity study should incorporate endpoints like clinical pathology, antibody measurements, and histology with special attention to injection site and immune organs . Our GLP tox study is designed accordingly, including local tolerance and antibody measurements. WHO also indicates that safety pharmacology (e.g. neurotoxicity, cardiotoxicity) is generally not separately needed for vaccines unless specific concerns arise (we have not seen signals that mRNA vaccines have off-target pharmacologic effects, but we will monitor cardiovascular and neurological parameters within the tox study observations). Additionally, the WHO guideline (2013 update referenced in literature) states it&rsquo;s not necessary that the species used for efficacy be the same as for safety, as long as the safety species is immunocompetent . In our plan, rats (safety) and ferrets/mice (efficacy) are different, justified by their purposes and biological relevance.<br></span></li><li class="c0 li-bullet-0"><span class="c2">GLP vs. Non-GLP Studies: Regulators recognize that efficacy models and mechanistic studies may not be GLP. Our non-GLP studies (such as the ferret challenge, mouse immunogenicity, and certain in vitro tests) will still be conducted with robust scientific method and documented appropriately, but we will clarify in the IND that these were not under GLP (permissible as they are pharmacology/proof-of-concept studies). The IND submission&rsquo;s Pharmacology section will include these results to support the rationale and dosing for human trials, whereas the Toxicology section will contain the GLP tox report, clearly identifying its GLP status . Any deviations or exceptions (for instance, if a portion of the distribution study is non-GLP) will be justified in the IND in accordance with 21 CFR 312.23.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Ethical Compliance: All animal studies will be reviewed by Institutional Animal Care and Use Committees (IACUC) and conducted in AAALAC-accredited facilities, in line with FDA and EMA emphasis on animal welfare and the 3Rs. If any study can be refined or replaced (for example, using serology in lieu of some challenge studies), we will consider those to reduce animal use without compromising data integrity .<br></span></li></ul><p class="c9"><span class="c2">In summary, the non-clinical program is structured to meet or exceed regulatory expectations for an IND. By integrating guidance from FDA, EMA, ICH, and WHO, we ensure a thorough evaluation of safety (per GLP standards) and efficacy (per good scientific practices), providing regulators with confidence in the vaccine&rsquo;s profile. Key regulatory documents (e.g. WHO TRS 927 , FDA&rsquo;s guidance on vaccines ) will be cross-referenced in our IND to show alignment. The following sections present the study plans and protocols with this compliance framework in mind.</span></p><h2 class="c19" id="h.25qubqb65xlt"><span class="c20">Tabulated Study Plans</span></h2><p class="c9"><span class="c2">The table(s) below summarize the planned IND-enabling non-clinical studies for the mRNA influenza vaccine. Each study is listed with its purpose, design highlights, species, group sizes, dose levels, endpoints, and GLP status for clarity. These structured plans ensure all regulatory requirements are covered and make the development program easy to review.</span></p><p class="c9"><span class="c2">Table 1. Toxicology and Biodistribution Studies (Safety and PK)</span></p><p class="c9 c11"><span class="c2"></span></p><table class="c25"><tr class="c24"><td class="c16" colspan="1" rowspan="1"><p class="c18"><span class="c13">Study</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c18"><span class="c13">Objective</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c18"><span class="c13">Species (strain)</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c18"><span class="c13">Group Design</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c18"><span class="c13">Dose Levels &amp; Regimen</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c18"><span class="c13">Key Endpoints</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c18"><span class="c13">GLP</span></p></td></tr><tr class="c24"><td class="c16" colspan="1" rowspan="1"><p class="c9"><span class="c2">Repeat-Dose Toxicology (Repeat-Dose Toxicity &amp; Local Tolerance)</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c9"><span class="c2">Assess systemic toxicity, local tolerance, and immune effects from multiple doses. Supports FIH safety .</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c9"><span class="c2">Rat (Sprague-Dawley), 8-10/sex/group</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c9"><span class="c2">4 groups: control (PBS), Low, Mid, High dose; both sexes. Dosing on Day 1 and Day 22 (IM injection in thigh). 5 rats/sex will be necropsied at Day 29, remaining 5/sex after a 2-week recovery (Day 43) to assess reversibility.</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c9"><span class="c2">Low: ~1&times; human dose; Mid: ~3&times;; High: ~10&times; human dose (in &micro;g mRNA per injection). Two doses 3 weeks apart. Volume ~0.5 mL/immunization.</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c9"><span class="c2">- Clinical observations, cage-side exams daily- Injection site observations (redness, swelling scoring)- Body weights (weekly) and food consumption- Body temperature (measured post-dose in subset for fever)- Clinical pathology: hematology, chemistry on Day 29 and 43 - Organ weights (e.g. liver, spleen, kidneys, gonads)- Gross necropsy findings- Histopathology of all major organs and injection site &amp; draining LNs (microscopic exam for inflammation or tissue changes)- Serum anti-influenza antibody titers (to confirm immunogenicity in tox animals, without interfering with safety assessment)- Anti-drug antibodies to RNA/LNP (if applicable, to check for immune response against platform)</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c9"><span class="c2">Yes (GLP)</span></p></td></tr><tr class="c24"><td class="c16" colspan="1" rowspan="1"><p class="c9"><span class="c2">Biodistribution &amp; Persistence</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c9"><span class="c2">Determine tissue distribution, retention and clearance kinetics of mRNA and LNP components . Supports PK understanding and confirms no unexpected accumulation.</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c9"><span class="c2">Rat (Wistar) or Mouse (BALB/c) for specialized methods; ~4-6/timepoint</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c9"><span class="c2">Timepoints: ~6 per study (e.g. 6h, 24h, 72h, 7d, 14d, 28d post-dose). n=3 rats/sex per timepoint (total ~36 rats). Each animal receives single IM dose (high dose). Separate cohort for 2-dose scenario terminal collection if needed.</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c9"><span class="c2">Single IM administration (high-dose, worst-case, e.g. 100 &micro;g in rat). Optionally, an equivalent group given two doses (Day 1 and 22) with sacrifice at Day 23 to see distribution after boost.</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c9"><span class="c2">- mRNA levels in tissues over time (by qRT-PCR) in injection site muscle, draining LNs, spleen, liver, blood, ovaries/testes, heart, kidneys, lungs, brain - Detectable protein expression (by luciferase IVIS imaging in a subset using reporter, and by ELISA/IHC for HA protein in tissues)- Calculate half-life in key tissues (muscle, liver) from time-course - LNP component distribution (radio-label or fluorescent tag analysis in tissues)- Plasma mRNA concentration-time (early timepoints for systemic exposure)- Determine time to clearance (when mRNA is undetectable) in each tissue- Document if any persistence in lymphatic or reproductive organs</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c9"><span class="c2">Partially (Key analyses under GLP-like conditions; overall study non-GLP, per common practice, with rationale provided)</span></p></td></tr></table><p class="c9"><span class="c2">Note: Additional specialized studies (e.g. genotoxicity and integration assessments) are not routinely required for mRNA vaccines. A bacterial reverse mutation test and an in vitro micronucleus test will be conducted to address any theoretical genotoxic risk of novel LNP components, although the mRNA itself is not expected to be genotoxic (an mRNA in vitro micronucleus assay for a similar platform was negative ). Carcinogenicity studies are not applicable to prophylactic vaccines. A separate Developmental and Reproductive Toxicity study (per ICH S5) in rats or rabbits will be planned before later-stage trials, as indicated earlier, but is not included in the initial IND-enabling package.</span></p><p class="c9"><span class="c2">The above tables serve as a roadmap, ensuring each required study is well-defined and aligned with regulatory standards. All study protocols will be filed in the IND, and any changes will be documented via protocol amendments.</span></p><h2 class="c19" id="h.3hjxyllvip7w"><span class="c20">Narrative Study Protocols</span></h2><p class="c9"><span class="c2">Below are detailed narrative protocols for each of the key non-clinical studies. These protocols include the study objectives, design, methodology, endpoints to be measured, and statistical analysis plan. Each protocol is written in a manner consistent with GLP documentation standards (where applicable) and can be used to execute the studies and for inclusion in regulatory submissions.</span></p><h3 class="c12" id="h.w9gdhzef7pht"><span class="c8">Protocol 1: GLP Repeat-Dose Toxicology Study in Rats</span></h3><p class="c9"><span class="c2">Study Title: 28-Day Repeat-Dose Toxicity Study of an mRNA-LNP Influenza Vaccine in Rats (Intramuscular Administration) with a 2-Week Recovery Period.</span></p><p class="c9"><span class="c2">Study Objective: To evaluate the safety and tolerability of the mRNA influenza vaccine when administered intramuscularly in rats at doses up to a projected high clinical multiple. The study aims to identify any target organ toxicities or local reactogenicity, characterize the immune response, and determine if findings are reversible after a recovery period. This study will support the safety of the vaccine for first-in-human trials by fulfilling regulatory requirements for nonclinical toxicology .</span></p><p class="c9"><span class="c2">Test Article: mRNA influenza vaccine encoding hemagglutinin (HA) antigen of influenza A (H1N1 subtype, strain A/California/07/2009 or similar) encapsulated in lipid nanoparticles (ionizable lipid, phospholipid, cholesterol, PEG-lipid). Nucleoside-modified mRNA, formulated in citrate buffer with sucrose. Lot # and concentration will be specified. Stored at &ndash;70&deg;C and formulated to dosing concentrations as needed. The same lot is used for all doses to ensure consistency.</span></p><p class="c9"><span class="c2">Control Article: Phosphate-buffered saline (PBS) with empty lipid nanoparticles (without mRNA) &ndash; this matches the vaccine excipient composition, to isolate the effect of the mRNA. (If empty LNP itself may cause responses, a separate PBS-only group might be considered, but here we use &ldquo;placebo&rdquo; containing LNP to mimic the full formulation minus mRNA.)</span></p><p class="c9"><span class="c2">Species and Strain: Rat (Rattus norvegicus), Sprague-Dawley strain (Crl:CD(SD)). Rationale: Rats are a standard rodent model for toxicology with large historical control databases . They are outbred (genetic diversity) which aligns with WHO recommendations for toxicity testing . This strain is known to be robust for immunological studies as well, producing antibodies to vaccine antigens (ensuring relevance of immunotoxicological findings) .</span></p><p class="c9"><span class="c2">Number and Sex of Animals: 40 rats (20 male, 20 female) divided into 4 groups (10/sex/group). An additional 5/sex in control and high-dose groups will be included for recovery observations, for a total of 50/sex. Animals will be 8-10 weeks old at the start (young adult, sexually mature). Body weight ~200-250 g (females) and ~300-350 g (males) at dosing start. Animals will be individually identified by ear tag and housed 2-3 per cage (same sex) in an accredited vivarium. They will undergo an acclimation period of at least 5 days prior to dosing and be assessed by a veterinarian to ensure health status (no pre-existing conditions).</span></p><p class="c9"><span class="c2">Grouping and Dose Levels: 4 groups as below:</span></p><ul class="c4 lst-kix_6z3d30mc5xph-0 start"><li class="c0 li-bullet-0"><span class="c2">Group 1: Control (Placebo) &ndash; 0 &micro;g mRNA, volume equivalent of PBS+empty LNP.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Group 2: Low Dose &ndash; 0.1 &micro;g mRNA (per injection). This is ~1&times; the anticipated human dose on a per-weight basis (approximately equivalent to 0.0005 mg/kg in a 200 g rat, which correlates to human 50 &micro;g/70 kg ~0.0007 mg/kg, considering species differences).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Group 3: Mid Dose &ndash; 1 &micro;g mRNA (per injection). (~10&times; human dose per weight).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Group 4: High Dose &ndash; 5 &micro;g mRNA (per injection). (~50&times; human dose per weight; expected to be maximal immunostimulatory dose in rats). Rationale: Doses are scaled roughly to achieve a broad margin; 5 &micro;g in rat (~0.025 mg/kg) is anticipated to elicit a strong immune response . If formulation permits, we aim for at least a 10-fold multiple of the clinical dose in absolute amount .<br></span></li></ul><p class="c9"><span class="c2">Each dose will be given in a fixed volume of 0.25 mL per rat (to mimic human injection volume proportionally). Concentrations will be adjusted such that 0.25 mL contains the desired &micro;g of mRNA for each group. Both sexes will receive the same dose (not scaled by body weight, consistent with vaccine dosing strategy of fixed doses).</span></p><p class="c9"><span class="c2">Route and Regimen: Intramuscular (IM) injection into the quadriceps muscle of the hind leg. Dosing schedule: Day 1 (Prime) and Day 22 (Boost). These two administrations mimic a prime-boost regimen, which is relevant since many influenza vaccine schedules may involve two doses for naive individuals. The 21-day interval is chosen to allow immune response development and to test for any increased reactogenicity upon second exposure (potential immune-mediated effects). The injection site will alternate limbs for the two doses (e.g. left leg on Day 1, right leg on Day 22) to allow local site evaluation independently.</span></p><p class="c9"><span class="c2">All injections will be performed under light anesthesia (e.g. isoflurane) to minimize animal discomfort and ensure accurate IM delivery. Needles (e.g. 25G) will be inserted into the muscle belly. The actual injected dose (in &micro;g and mL) for each animal will be recorded.</span></p><p class="c9"><span class="c2">Observational Period: Animals will be observed up to Day 29 for main necropsy (1 week after second dose) and a subset (recovery groups) up to Day 43 (additional 2 weeks recovery without dosing).</span></p><p class="c9"><span class="c2">Clinical Observations:</span></p><ul class="c4 lst-kix_xmm4jxv5hwlq-0 start"><li class="c0 li-bullet-0"><span class="c2">Mortality/Moribundity: Checked at least twice daily. Any moribund animal will be humanely euthanized and considered as an unscheduled sacrifice; full necropsy will be performed.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Detailed Clinical Exams: Once daily, rats will be examined for general appearance, behavior, and any signs of illness or injection site reactions. Particular attention to: piloerection, posture, gait, respiratory distress, oral/nasal discharge, and any neurological signs.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Injection Site Reactions: The injected hindlimb will be inspected daily for redness, swelling, or warmth. A scoring system will be used (e.g. 0 = no reaction, 1 = slight redness/swelling, 2 = moderate, 3 = severe). Palpation will be done gently to detect any mass or induration.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Body Weight: Measured on Day 1 (pre-dose), Day 2, then at least weekly (Days 8, 15, 22 pre-second dose, 29, and 43 for recovery animals). More frequent weights (e.g. twice weekly) will be taken during the first week after each dose to monitor any acute weight changes.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Food Consumption: Measured weekly per cage. Not expected to change drastically, but any reduction might indicate systemic toxicity.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Body Temperature: In a subset of 3 rats/sex/group, rectal temperature will be measured at baseline and at 4-6 hours post-dose on Days 1 and 22 to detect acute pyrogenic response (fever). An increase of &gt;1.5&deg;C from baseline will be noted.<br></span></li></ul><p class="c9"><span class="c2">Clinical Pathology: Blood samples will be collected via retro-orbital sinus or tail vein (under brief anesthesia) from all main-study animals:</span></p><ul class="c4 lst-kix_d8i9v1tvihft-0 start"><li class="c0 li-bullet-0"><span class="c2">Interim (if applicable): If any concerning clinical signs arise, interim blood sampling may be done (e.g. 24-48h after first dose to check for cytokine storm or acute inflammatory response).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Terminal (Main necropsy Day 29): all animals (10/sex/group) euthanized on Day 29 will have blood drawn prior to euthanasia.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Recovery Termination (Day 43): remaining rats (5/sex in Control and High dose) will be sampled.<br></span></li></ul><p class="c9"><span class="c2">The following analyses will be performed:</span></p><ul class="c4 lst-kix_ie2cjnul3ua-0 start"><li class="c0 li-bullet-0"><span class="c2">Hematology: Complete blood count (CBC) with differential (RBC, hemoglobin, hematocrit, MCV, MCHC, platelet count, WBC count with neutrophils, lymphocytes, monocytes, eosinophils, basophils). We will pay special attention to neutrophil counts and acute phase reactants as indicators of inflammation.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Clinical Chemistry: Panel including ALT, AST, ALP (liver function), BUN, creatinine (renal), glucose, electrolytes, C-reactive protein or fibrinogen (if available, as an inflammation marker), and CK (muscle enzyme, for muscle injection-site damage).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Coagulation: PT and aPTT to detect any coagulopathy (e.g. if systemic inflammation triggers clotting changes).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Cytokines: Serum from a subset of animals (3/sex/group) at 6h post-dose (Day 1 and Day 22) will be frozen for cytokine analysis (IL-6, TNF-&alpha;, IFN-&gamma;, etc.), to characterize the innate immune activation profile. This is an exploratory endpoint to contextualize any acute reactions.<br></span></li></ul><p class="c9"><span class="c2">Immunogenicity Assessments: Although not a primary objective of a tox study, we will evaluate the immune response in these animals:</span></p><ul class="c4 lst-kix_m75akhxjyj9r-0 start"><li class="c0 li-bullet-0"><span class="c2">Antibody Titers: Serum collected at necropsy (Day 29) will be tested for anti-HA IgG by ELISA and HAI titer. This confirms the rats did mount an immune response (expected in high dose especially) and can help interpret any immune-mediated findings (e.g. immune complex deposition, if any, which is unlikely).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Neutralizing Antibodies: If sufficient serum is available, a microneutralization assay will be done.<br></span></li><li class="c0 li-bullet-0"><span class="c2">These immunological endpoints also address regulatory interest in understanding the product&rsquo;s activity in the tox species (ensuring the rat is a relevant model that responds to the vaccine).<br></span></li></ul><p class="c9"><span class="c2">Necropsy and Gross Pathology: On the scheduled necropsy days (Day 29 for main, Day 43 for recovery), animals will be euthanized by CO&#8322; asphyxiation (or overdose of anesthetic) followed by exsanguination. A complete gross examination of external surfaces, all orifices, and the cranial, thoracic, abdominal, and pelvic cavities will be performed. All organs will be examined in situ and then removed and weighed as appropriate.</span></p><ul class="c4 lst-kix_73hytacj4fgx-0 start"><li class="c0 li-bullet-0"><span class="c2">Organ Weights: Liver, kidneys, spleen, brain, heart, lungs, adrenal glands, gonads (testes or ovaries), and injection site muscle (in the form of dissected muscle plus draining lymph node) will be weighed for each animal. Thyroid (with parathyroid) and thymus will also be weighed if required (thymus weight can indicate stress/involution).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Gross Lesions: Any abnormal finding (e.g. nodules, discoloration) will be recorded and the tissue preserved for histology.<br></span></li></ul><p class="c9"><span class="c2">Histopathology: Tissues from all animals in control and high-dose groups will be processed for microscopic examination, and any tissues with macroscopic abnormalities from any group will also be examined. Additionally, because this is a vaccine, we will examine immune organs in all dose groups to catch dose-dependent changes. The tissue list includes:</span></p><ul class="c4 lst-kix_ehxsaivn684z-0 start"><li class="c0 li-bullet-0"><span class="c2">Injection site muscle (with surrounding tissue) &ndash; expect local inflammatory changes.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Draining lymph node (e.g. inguinal LN) &ndash; likely reactive hyperplasia expected due to antigen stimulation .<br></span></li><li class="c0 li-bullet-0"><span class="c2">Distant lymphoid organs: Spleen, distant lymph nodes (e.g. axillary if not directly draining).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Heart, lungs, liver, kidneys.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Stomach, intestines (to see if any incidental findings, though IM vaccine not expected to affect GI directly).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Brain (including sections of cerebrum, cerebellum) &ndash; to rule out any encephalitis or unexpected distribution; although unlikely, included for completeness.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Bone marrow (sternum or femur) &ndash; to assess hematopoiesis or any marrow activation/suppression.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Thymus (especially in younger rats) &ndash; check for lymphoid changes.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Endocrine: thyroid, adrenals.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Reproductive organs: testis, epididymis, ovaries, uterus &ndash; ensure no effect on reproductive tissues (important since LNP-mRNA in prior studies had slight distribution to ovaries in some cases).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Any site of note from clinical obs (e.g. if a rat had limping, examine that limb etc.).<br></span></li></ul><p class="c9"><span class="c2">All tissues will be fixed in 10% neutral buffered formalin (except eyes in Davidson&rsquo;s fixative if needed, but eyes are not target for IM vaccine typically). Bone tissues decalcified appropriately.</span></p><p class="c9"><span class="c2">Histological sections will be stained with H&amp;E and evaluated by a board-certified veterinary pathologist. Special stains may be employed if needed (e.g. Gram stain if an abscess is suspected, etc.). We will specifically look for:</span></p><ul class="c4 lst-kix_j5h2oeedi37x-0 start"><li class="c0 li-bullet-0"><span class="c2">Inflammation: at injection site (characterize cellular infiltrate: neutrophils vs lymphocytes, etc., and grade severity minimal/mild/moderate/severe ), and any evidence of systemic inflammation in organs.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Immune organ changes: spleen and LN hyperplasia (expected due to immune response; will be noted as an expected pharmacologic effect if present).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Liver: any hepatocellular degeneration or sinusoidal inflammation (LNPs often go to liver , we check for Kupffer cell aggregates, etc.).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Kidney: any glomerular changes or inflammation.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Heart: inflammatory cell infiltrates (myocarditis is unlikely but monitored given mRNA vaccines have rare association with myocarditis in humans; though typically not seen in rats).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Lungs: any inflammatory or eosinophilic infiltrates (to catch any potential hypersensitivity).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Injection site muscle: necrosis/regeneration patterns (some necrosis at injection site can occur from both needle and immune reaction, which typically resolves; regeneration will be noted).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Any other lesions: such as granulomas (possible at injection site if LNP causes a localized reaction), or any unusual findings (e.g. evidence of immune complex deposition in vessels or kidneys would be significant to note, though not anticipated).<br></span></li></ul><p class="c9"><span class="c2">For low- and mid-dose groups, histopathology will initially be done on any tissues that had treatment-related changes in the high dose, to evaluate dose-response. If high-dose has essentially no lesions beyond expected minor inflammation, lower doses may be examined only for injection site and lymphoid organs for completeness.</span></p><p class="c9"><span class="c2">Local Tolerance: Although integrated in observations and histopathology, we will specifically evaluate local tolerability. This includes the acute injection pain/response (none beyond transient prick expected), and chronic changes at injection site (like muscle fiber regeneration, fibrosis or granuloma formation). According to WHO, local tolerance should be part of tox study ; our design addresses this with detailed injection site exams and histology.</span></p><p class="c9"><span class="c2">Recovery Phase Evaluations: For the recovery subset (control and high-dose, 5/sex each):</span></p><ul class="c4 lst-kix_on63e6od9ck4-0 start"><li class="c0 li-bullet-0"><span class="c2">These animals will be observed for 14 additional days post main necropsy to see if any toxic effects reverse or if any delayed effects appear.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Body weights and clinical observations continue weekly.<br></span></li><li class="c0 li-bullet-0"><span class="c2">On Day 43, they undergo blood sampling for CBC/chemistry and then necropsy with the same protocol. Key organs previously affected in high-dose will be examined to see if they returned to normal (e.g. any injection site inflammation resolved).<br></span></li><li class="c0 li-bullet-0"><span class="c2">This will help distinguish persistent vs. transient findings (e.g. if liver enzyme was mildly high on Day 29 but normal by Day 43, it indicates reversibility).<br></span></li></ul><p class="c9"><span class="c2">Statistical Analysis: Data will be captured by electronic data capture where possible and compiled for statistical analysis:</span></p><ul class="c4 lst-kix_w9ink2jq27yv-0 start"><li class="c0 li-bullet-0"><span class="c2">For continuous endpoints (e.g. body weight, organ weight, clinical pathology values), group means and standard deviations will be calculated. We will perform a one-way ANOVA comparing treatment groups to control for each sex. If ANOVA is significant (p&lt;0.05), Dunnett&rsquo;s post hoc test will identify which dose groups differ from control. If data are not normally distributed or variance is heterogeneous, a non-parametric test (Kruskal-Wallis with Dunn&rsquo;s post test) will be used.<br></span></li><li class="c0 li-bullet-0"><span class="c2">For incidence data (like presence of a lesion, or proportion of animals with fever), a Fisher&rsquo;s exact test may be used.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Injection site score data, which are ordinal, might be compared via non-parametric tests (e.g. Kruskal-Wallis).<br></span></li><li class="c0 li-bullet-0"><span class="c2">The significance threshold is set at &alpha; = 0.05. All significant findings will be reported, but emphasis will be on consistent, dose-related trends considered likely treatment-related.<br></span></li><li class="c0 li-bullet-0"><span class="c2">We will also evaluate toxicologically significant changes (e.g. a small but statistically significant change in one enzyme might not be biologically meaningful if within historical control range ). The study report will integrate statistical results with expert judgment.<br></span></li></ul><p class="c9"><span class="c2">Quality Assurance: The study will be conducted in compliance with GLP. The protocol and any amendments will be reviewed by Quality Assurance (QA). QA will inspect critical phases (dosing, clinical obs, necropsy, data analysis) and audit the draft report. A compliance statement will be included. All raw data, specimens, and the final report will be archived per GLP.</span></p><p class="c9"><span class="c2">Expected Outcomes: Based on experience with other mRNA LNP vaccines, we anticipate no serious toxicity. Expected findings include:</span></p><ul class="c4 lst-kix_81hzypied9ul-0 start"><li class="c0 li-bullet-0"><span class="c2">Mild to moderate injection site inflammation (histologically, with possibly slight edema/redness clinically) .<br></span></li><li class="c0 li-bullet-0"><span class="c2">Reversible enlargement of draining lymph nodes/spleen due to immune stimulation .<br></span></li><li class="c0 li-bullet-0"><span class="c2">Transient increases in neutrophils or cytokines after injection (reflecting innate immune activation) .<br></span></li><li class="c0 li-bullet-0"><span class="c2">Possibly mild liver enzyme elevations in high dose, corresponding to LNP uptake in liver, but without histopathological liver damage .<br><br> No dose-limiting toxicities were seen in analogous rat studies for other mRNA vaccines (e.g. Moderna&rsquo;s rat tox showed only expected inflammatory changes and no organ toxicity ). This study will confirm that the influenza mRNA vaccine has a similarly benign non-clinical safety profile, supporting progression to human trials.<br></span></li></ul><h3 class="c12" id="h.v0454aa6st6i"><span class="c8">Protocol 2: Non-GLP Biodistribution and Pharmacokinetic Study in Rats</span></h3><p class="c9"><span class="c2">Study Title: Biodistribution and Persistence of mRNA-LNP Influenza Vaccine in Rats following Intramuscular Injection.</span></p><p class="c9"><span class="c2">Objectives: To quantitatively determine the tissue distribution, persistence, and clearance of the mRNA vaccine and its expressed antigen following IM administration in rats. This study will identify which tissues the mRNA and lipid nanoparticle travel to beyond the injection site, how long they last, and whether any accumulation occurs. It will also measure the kinetics of mRNA decay and protein expression, providing data on pharmacokinetic behavior (distribution and elimination) which is not obtainable via standard plasma PK for this modality .</span></p><p class="c9"><span class="c2">Rationale: As per regulatory expectations for novel nucleic acid vaccines, biodistribution data are required in lieu of traditional PK . This informs on safety (ensuring no unexpected organ exposure or long persistence) and helps interpret any toxicology findings by correlating them with distribution profiles (e.g. if liver changes were noted, did mRNA go to liver).</span></p><p class="c9"><span class="c2">Test Article: Same mRNA-LNP influenza vaccine as in tox study (Lot #X). For detection purposes, an identical formulation encoding a reporter (firefly luciferase mRNA) may be used in a subset of animals to visually confirm distribution via IVIS imaging . The primary data, however, will come from the actual HA mRNA formulation measured by PCR.</span></p><p class="c9"><span class="c2">Species: Rat (Wistar Han strain, 8-10 weeks old, female). Female rats are chosen to evaluate distribution to ovaries (a particular interest area); distribution in males is expected to be similar aside from reproductive organs, so primarily females suffice. If possible, a few males will be included to check testes distribution as well.</span></p><p class="c9"><span class="c2">Group Design: This is a serial sacrifice design. Approximately 42 rats (36 females + 6 males) will be used. Six timepoints are planned post-dose with n=6 females per time (except final time which may use remaining animals). Additionally, at the first timepoint, 2 males will be included, and at the last timepoint, 4 males included to examine male gonads at early and late times.</span></p><p class="c9"><span class="c2">Proposed timepoints (post injection):</span></p><ul class="c4 lst-kix_jyqknjqk01cc-0 start"><li class="c0 li-bullet-0"><span class="c2">0.25 days (6 hours)<br></span></li><li class="c0 li-bullet-0"><span class="c2">1 day (24 hours)<br></span></li><li class="c0 li-bullet-0"><span class="c2">3 days (72 hours)<br></span></li><li class="c0 li-bullet-0"><span class="c2">7 days<br></span></li><li class="c0 li-bullet-0"><span class="c2">14 days<br></span></li><li class="c0 li-bullet-0"><span class="c2">28 days<br></span></li></ul><p class="c9"><span class="c2">At each timepoint, 6 rats will be euthanized for tissue harvest. We might adjust timepoints based on preliminary data (e.g. add a 2-day or 21-day if needed for curve definition). The final timepoint (28d) is to ensure near complete clearance or to identify any very long persistence.</span></p><p class="c9"><span class="c2">All rats receive a single IM dose of the vaccine in the quadriceps (same site, no repeat dosing to keep data interpretation straightforward).</span></p><p class="c9"><span class="c2">Dose: A high dose of 5 &micro;g mRNA (the high dose used in tox) in 0.25 mL will be administered. Using the high dose maximizes detection sensitivity in tissues. Lower doses would simply scale down all signals and might risk some being below detection threshold. The dose is non-toxic as per tox study expectation, so it&rsquo;s safe to use for this purpose.</span></p><p class="c9"><span class="c2">Sample Collection and Methods: At each timepoint, rats are euthanized (overdose anesthesia followed by exsanguination). Blood is collected via cardiac puncture (for plasma) before perfusion. To reduce blood contamination in tissue, a PBS perfusion through the heart may be done for certain tissues like liver and spleen (optional if needed to distinguish blood vs tissue content of mRNA).</span></p><p class="c9"><span class="c2">Tissues collected:</span></p><ul class="c4 lst-kix_6kne4t6ii93d-0 start"><li class="c0 li-bullet-0"><span class="c2">Injection site muscle (with overlying skin): The entire area receiving the injection will be excised.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Draining lymph nodes (inguinal or iliac LN near injection).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Distant lymphoid organ: Spleen.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Liver.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Kidneys.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Heart.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Lungs.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Brain.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Ovaries (in females, both ovaries).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Uterus (females).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Testes (in included males).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Blood (plasma).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Additionally, urine and feces can be collected from cages (to see if any excretion of mRNA or its fragments, though unlikely significant since mRNA breaks down in tissues, not excreted intact).<br></span></li></ul><p class="c9"><span class="c2">Each tissue is divided: a portion snap-frozen for RNA analysis, a portion in formalin for possible histology or IHC (for protein).</span></p><p class="c9"><span class="c2">Analytical Procedures:</span></p><ul class="c4 lst-kix_t27vnxuzkttq-0 start"><li class="c0 li-bullet-0"><span class="c2">mRNA Quantification (qRT-PCR): Using a validated quantitative reverse-transcription PCR assay specific to the vaccine mRNA:<br></span></li></ul><ul class="c4 lst-kix_t27vnxuzkttq-1 start"><li class="c3 li-bullet-0"><span class="c2">We will design primers/probe for a unique sequence (e.g. the junction of 5&rsquo;UTR and coding region, or the poly(A) tail region) to avoid amplifying any endogenous transcripts.<br></span></li><li class="c3 li-bullet-0"><span class="c2">Standard curve will be made with known copies of the mRNA spiked into control tissue homogenates to quantify absolute copy numbers.<br></span></li><li class="c3 li-bullet-0"><span class="c2">Tissues will be homogenized (e.g. with a bead-beater or rotor-stator) in Trizol or appropriate lysis buffer to extract RNA. A fixed amount of an external RNA standard may be spiked in as QC.<br></span></li><li class="c3 li-bullet-0"><span class="c2">After extraction, reverse transcription to cDNA and real-time PCR will be performed. Results expressed as copies of mRNA per mg of tissue (or per mL of plasma).<br></span></li><li class="c3 li-bullet-0"><span class="c2">Sensitivity is expected to be &lt;100 copies per sample reliably. We will include negative controls (tissues from an uninjected rat) to ensure no false positives.<br></span></li><li class="c3 li-bullet-0"><span class="c2">Data will be reported as mean &plusmn; SD for n=6 at each time, for each tissue.<br></span></li></ul><ul class="c4 lst-kix_t27vnxuzkttq-0"><li class="c0 li-bullet-0"><span class="c2">LNP Lipid Quantification: If a radiolabeled lipid is used (for instance, trace ^3H or ^14C on the PEG-lipid or ionizable lipid), tissue samples will be processed by solvent extraction and scintillation counting to measure disintegrations per minute, converting to microgram-equivalents of lipid per gram tissue. Alternatively, if a fluorescently labeled lipid is available, we can measure fluorescence in tissue extracts (though autofluorescence may complicate). This will help confirm distribution patterns (e.g. mRNA vs lipid might show similar distribution, as expected since the complex likely stays together, except perhaps some lipid goes to liver via ApoE/LDL mechanisms ).<br></span></li></ul><ul class="c4 lst-kix_t27vnxuzkttq-1 start"><li class="c3 li-bullet-0"><span class="c2">Key interest is in liver and spleen: LNP lipids often accumulate in hepatocytes via ApoE coating , so a higher liver signal for lipid than mRNA might be seen (since mRNA may degrade, but lipid stays longer in liver).<br></span></li></ul><ul class="c4 lst-kix_t27vnxuzkttq-0"><li class="c0 li-bullet-0"><span class="c2">In Vivo Imaging (optional): A subset of 3 rats at each early timepoint could receive a co-formulated luciferase mRNA-LNP (in addition to the HA mRNA, or in separate matched groups) to visualize protein expression. These rats would be anesthetized and imaged with an IVIS at 6h, 24h, 3d for bioluminescence. As known, luminescence is expected at injection site and possibly in liver region . This non-GLP imaging data corroborates the PCR data in a qualitative way. After imaging, these rats would be euthanized and tissues processed same as others (hence they contribute to the qPCR data too).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Protein (HA) Detection:<br></span></li></ul><ul class="c4 lst-kix_t27vnxuzkttq-1 start"><li class="c3 li-bullet-0"><span class="c2">ELISA: A sandwich ELISA for HA antigen can be performed on tissue lysates (especially muscle at site and lymph node). We&rsquo;ll use a known anti-HA antibody pair to detect the HA. This indicates actual translation occurred in those tissues.<br></span></li><li class="c3 li-bullet-0"><span class="c2">Immunohistochemistry: Fixed tissues (injection site muscle, LN, spleen, liver) will be sectioned and stained with an anti-HA antibody to localize which cells express the antigen. We expect muscle fibers near injection site and antigen-presenting cells in LN and spleen might be positive. If found, that matches expectations of immune processing.<br></span></li></ul><ul class="c4 lst-kix_t27vnxuzkttq-0"><li class="c0 li-bullet-0"><span class="c2">Data Collected: For each tissue/timepoint:<br></span></li></ul><ul class="c4 lst-kix_t27vnxuzkttq-1 start"><li class="c3 li-bullet-0"><span class="c2">mRNA concentration (copies/mg tissue).<br></span></li><li class="c3 li-bullet-0"><span class="c2">Possibly % of injected dose (%ID) in whole organ if organ weights are known and distribution uniform (e.g. calculate how much of total dose resides in liver at peak).<br></span></li><li class="c3 li-bullet-0"><span class="c2">For plasma: mRNA ng/mL at various times to see any systemic exposure.<br></span></li><li class="c3 li-bullet-0"><span class="c2">For each, we will tabulate the decline over time.<br></span></li></ul><p class="c9"><span class="c2">Endpoints and Parameters:</span></p><ul class="c4 lst-kix_70tvr3ungtsg-0 start"><li class="c0 li-bullet-0"><span class="c2">Peak levels (Cmax) in each tissue and the corresponding time (Tmax). Likely:<br></span></li></ul><ul class="c4 lst-kix_70tvr3ungtsg-1 start"><li class="c3 li-bullet-0"><span class="c2">Muscle: highest at 6h or 24h (based on prior data showing peak ~6h ).<br></span></li><li class="c3 li-bullet-0"><span class="c2">Liver: possibly peak at 6h or 24h as well due to initial drainage of nanoparticles to liver via blood.<br></span></li><li class="c3 li-bullet-0"><span class="c2">LN: might peak slightly later if mRNA drains gradually via lymph (maybe 24-72h).<br></span></li></ul><ul class="c4 lst-kix_70tvr3ungtsg-0"><li class="c0 li-bullet-0"><span class="c2">Half-life in major tissues:<br></span></li></ul><ul class="c4 lst-kix_70tvr3ungtsg-1 start"><li class="c3 li-bullet-0"><span class="c2">We will calculate an apparent half-life by log-linear regression of the decline phase. For muscle, if it drops 90% by day 4 , half-life might be ~1 day. For liver, some slower decline might occur due to uptake by hepatocytes; we&rsquo;ll quantify it.<br></span></li></ul><ul class="c4 lst-kix_70tvr3ungtsg-0"><li class="c0 li-bullet-0"><span class="c2">Extent of Persistence: The last timepoint (28d) will show if any tissue still has detectable mRNA. Ideally, all are near or below the detection limit by 28d. If any tissue (like a lymph node or spleen) still has small amounts, we&rsquo;ll note it but presumably it will be extremely low (likely below functional relevance). Similarly, we&rsquo;ll see if any HA protein persists (unlikely beyond a few days since antigen will be cleared or metabolized by immune cells).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Comparative distribution: The relative exposure of tissues: we expect Injection site &gt; Liver &ge; Spleen &gt; LN &gt; others. Prior studies showed strong muscle signal and weaker liver . We will verify this pattern. We&rsquo;ll also confirm minimal to no mRNA in brain (should be none above background; any detected might be due to blood or very minor if any crosses).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Sex differences: If male data included, compare ovary vs testis uptake. Any high level in ovaries will be noted (previous LNP mRNA studies showed low but detectable in ovaries , but no effect on fertility in DART studies ).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Excretion: If we measure, likely no intact mRNA in urine (perhaps some nucleosides could be, but we won&rsquo;t specifically measure metabolite excretion unless needed; presumably no intact excretion as the kidneys filter small &lt;5nm molecules only).<br></span></li></ul><p class="c9"><span class="c2">Statistical Analysis: Mainly descriptive. We will compute means and SD for each tissue at each time. If modeling is needed, we might fit a two-phase exponential decay model to estimate half-lives. No hypothesis testing per se, but we will look at any differences (like male vs female gonad distribution) qualitatively.</span></p><p class="c9"><span class="c2">Quality and Compliance: This study is non-GLP, but critical phases (sample analysis) will follow SOPs similar to GLP. An internal QA review of the analytical method validation and data integrity will be done. The protocol deviations, if any, will be documented. Since it&rsquo;s supportive, regulators allow non-GLP with justification .</span></p><p class="c9"><span class="c2">Interpretation: We expect to demonstrate:</span></p><ul class="c4 lst-kix_6ay9chlfxidr-0 start"><li class="c0 li-bullet-0"><span class="c2">The mRNA largely stays local with partial systemic distribution to liver and spleen .<br></span></li><li class="c0 li-bullet-0"><span class="c2">Rapid clearance of mRNA (majority gone in days) .<br></span></li><li class="c0 li-bullet-0"><span class="c2">No long-term retention; by day 28 most tissues have no quantifiable mRNA.<br></span></li><li class="c0 li-bullet-0"><span class="c2">LNP lipids may persist slightly longer in liver (a class effect) but at levels not causing pathology.<br></span></li><li class="c0 li-bullet-0"><span class="c2">These results will be integrated with tox findings (e.g. if liver had slight immune cell infiltration in tox, we might attribute to the known liver uptake of LNP).<br><br> This protocol thus yields essential data to satisfy regulatory concerns about where the vaccine goes after injection and how long it remains .<br></span></li></ul><h3 class="c12" id="h.o6no86lf6ynb"><span class="c8">Protocol 3: Mouse Immunogenicity Study</span></h3><p class="c9"><span class="c2">Study Title: Immunogenicity Dose-Ranging Study of mRNA Influenza Vaccine in Mice.</span></p><p class="c9"><span class="c2">Objectives: To assess the immunogenic profile of the mRNA influenza vaccine at different dose levels in a mouse model. The study will determine the magnitude and type of immune response (humoral and cellular) elicited by low, medium, and high doses. The results will guide selection of an optimal dose for further development and provide early proof-of-concept of vaccine efficacy (neutralizing antibodies) in a small animal. Additionally, this study will evaluate the Th1/Th2 polarization of the response, addressing safety concerns such as potential Th2-biased immunopathology.</span></p><p class="c9"><span class="c2">Rationale: Mice are well-established for immunogenicity testing and allow multiple dose groups. The use of dose-ranging helps identify if a lower dose could still be effective, or if a higher dose provides significantly greater immunity (or plateaus) . It also offers insight into dose-response which can support the justification of the clinical starting dose. While mice are not the primary efficacy model, robust neutralizing antibody generation in mice has been correlated with protection in ferrets and humans .</span></p><p class="c9"><span class="c2">Test Article: mRNA-LNP influenza vaccine (same construct as above). Different dose levels will be achieved by diluting the formulation in buffer.</span></p><p class="c9"><span class="c2">Animals: Mus musculus (house mouse), female (to avoid fighting seen in group-housed males and because female immune responses in mice can be slightly higher antibody-wise, though either sex is acceptable). Strain: BALB/c, 6-8 weeks old. Chosen because BALB/c tends to have a Th2-biased baseline response; if our vaccine can drive a Th1 response even in BALB/c, it&rsquo;s a strong indicator of Th1 bias (on the other hand, C57BL/6 are Th1-prone; including BALB/c is a conservative test for potential Th2 skewing).</span></p><p class="c9"><span class="c2">Group Design: 4 groups of 10 mice each:</span></p><ul class="c4 lst-kix_5mbqwckh3266-0 start"><li class="c0 li-bullet-0"><span class="c2">Group 1: Placebo (PBS + empty LNP, 50 &micro;L IM).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Group 2: Low dose (0.1 &micro;g mRNA).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Group 3: Mid dose (1 &micro;g mRNA).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Group 4: High dose (5 &micro;g mRNA).<br></span></li></ul><p class="c9"><span class="c2">Doses selected similar to rat tox relative scaling (though absolute &micro;g in a mouse is smaller due to its size). A 5 &micro;g dose in a 20 g mouse is 0.25 mg/kg, which is a high but tolerable dose for a vaccine (some mouse studies of mRNA vaccines have used 10 &micro;g or more per mouse successfully ).</span></p><p class="c9"><span class="c2">Route/Regimen: IM injection into the quadriceps muscle. Two-dose regimen: Day 0 (prime) and Day 21 (boost). This regimen is chosen to mimic what might be done in humans (especially if aiming to test prime-boost). If the vaccine were intended as a single dose, we might still do a boost in mice to maximize the immune response for measurement (and to test for any enhancement or tolerance on second exposure).</span></p><p class="c9"><span class="c2">Injection volume: 50 &micro;L per leg. On Day 0 inject right leg, Day 21 left leg (to avoid local over-stressing one muscle).</span></p><p class="c9"><span class="c2">Procedures/Observations:</span></p><ul class="c4 lst-kix_2fh1qudxwat7-0 start"><li class="c0 li-bullet-0"><span class="c2">Monitor mice daily post-injection for any signs of adverse effects (lethargy, ruffled fur, injection site issues which in mice might appear as limping or redness).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Body weights weekly; significant weight drop would indicate an adverse effect, but none expected at these small doses.<br></span></li></ul><p class="c9"><span class="c2">Sample Collection:</span></p><ul class="c4 lst-kix_mhfweobdea1s-0 start"><li class="c0 li-bullet-0"><span class="c2">Blood: Collect ~100 &micro;L via submandibular bleed on Day 21 (just before boost) to measure primary response antibodies. Collect terminal blood via cardiac puncture at Day 35 (two weeks after boost).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Spleen: At termination (Day 35), spleens will be harvested aseptically for cellular immune assays.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Optionally, lymph nodes (draining) could be harvested for T-cell analysis if needed.<br></span></li></ul><p class="c9"><span class="c2">Immunological Assays:</span></p><p class="c9"><span class="c2">Humoral Immunity:</span></p><ul class="c4 lst-kix_z1b7uf8mlquv-0 start"><li class="c0 li-bullet-0"><span class="c2">ELISA for HA-specific IgG: Standard ELISA plates coated with recombinant HA protein from the vaccine strain. Diluted serum from each mouse (Day 21 and Day 35) will be tested. End-point titers or EC50 titers will be calculated. We expect dose-responsive titers, e.g. high dose yielding highest titers.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Hemagglutination Inhibition (HAI) Assay: Using turkey or guinea pig RBCs and the homologous influenza virus (if available). Sera (Day 35) will be treated (receptor-destroying enzyme and heat) and then tested for HAI titer. An HAI titer &ge;40 is often considered protective in humans; we will see what titers mice achieve. Often mice need adjuvants to get high HAI; but mRNA has been shown to induce functional antibodies.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Virus Neutralization Assay: Using live influenza virus (same strain as HA or a pseudovirus with HA and NA of that strain). Sera (Day 35) will be incubated with virus, then added to MDCK cells. After 2-3 days, we will read neutralization via plaque reduction or cytopathic effect inhibition. We will report 50% neutralization titers (NT50). This is a key measure of vaccine-induced protective antibodies.<br></span></li><li class="c0 li-bullet-0"><span class="c2">IgG Subclass Analysis: In mice, IgG1 vs IgG2a (or IgG2c in some strains) indicates Th2 vs Th1 skewing. We will perform ELISA that differentiates IgG1 and IgG2a specific to HA. A higher IgG2a:IgG1 ratio suggests Th1 dominance (driven by IFN-&gamma;), whereas high IgG1 would mean Th2 (driven by IL-4). We aim for balanced or Th1-biased response to avoid concern of Th2-dominated immunity which in some vaccines correlates with ERD (notably in RSV vaccine history).<br></span></li></ul><p class="c9"><span class="c2">Cellular Immunity:</span></p><ul class="c4 lst-kix_tx9us7pz8zze-0 start"><li class="c0 li-bullet-0"><span class="c2">Splenocyte IFN-&gamma; ELISpot: This will quantify the number of T cells producing IFN-&gamma; in response to influenza antigens. We will use a peptide library spanning HA (and NP if our construct included NP, but here likely just HA so focusing on HA T-cell epitopes). Plates coated with anti-IFN-&gamma; capture Ab, splenocytes added at 2-5&times;10^5/well, plus peptide pools (or live virus-infected APCs as stimulus). Spots counted after adding detection Ab and developing. A strong IFN-&gamma; ELISpot response indicates robust T helper 1 (or CD8) response.<br></span></li><li class="c0 li-bullet-0"><span class="c2">IL-4 ELISpot or IL-5: If resources allow, an IL-4 ELISpot can measure Th2 cells. Or an IL-5 cytokine secretion assay. This helps ensure we are not getting a predominant Th2 response.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Intracellular Cytokine Staining (ICS): Optionally, a subset of splenocytes can be stimulated with a dominant CD8 epitope (if known for HA in this mouse strain) and stained for IFN-&gamma;, TNF, etc., to examine CD8 T cell activation. Also, CD4 T cells for IL-2, IL-4, IFN-&gamma;. We expect an mRNA vaccine (with potent innate signals) will induce a mixed response with a lean towards Th1 cytokines .<br></span></li><li class="c0 li-bullet-0"><span class="c2">T cell memory markers: not in this short study, but if analyzing ICS, we might phenotype for central vs effector memory (CD62L, CD44 markers).<br></span></li></ul><p class="c9"><span class="c2">Other Immune Markers:</span></p><ul class="c4 lst-kix_dmy2ekb87fbb-0 start"><li class="c0 li-bullet-0"><span class="c2">Cytokines in serum post-vaccination: in a subset, bleed at 6h or 24h after prime to measure acute IL-6, MCP-1, etc. This is exploratory to see innate response intensity by dose. High dose may cause a transient higher cytokine level (which might correlate with reactogenicity but also adjuvant effect).<br></span></li><li class="c0 li-bullet-0"><span class="c2">B cell germinal center response: not doing, but could have done LN immunohistology. Possibly not in this small study.<br></span></li></ul><p class="c9"><span class="c2">Data Analysis:</span></p><ul class="c4 lst-kix_3hobsm4w6nyk-0 start"><li class="c0 li-bullet-0"><span class="c2">Compare dose groups for each immune measure. For example, use one-way ANOVA (or Kruskal-Wallis if non-normal) to see differences in ELISA titers or ELISpot counts across groups. If significant, compare each vaccine dose to control and to adjacent dose to see dose-responsiveness. We expect a plateau: mid vs high might not differ hugely if mid already saturated response.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Specifically check if low dose is significantly less immunogenic than mid/high (to find minimally effective dose).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Check subclass ratios: ideally, high dose yields IgG2a/IgG1 ratio &gt;1 (Th1-biased) whereas if we saw &lt;1 (Th2 bias) at any dose, that&rsquo;s noted.<br></span></li><li class="c0 li-bullet-0"><span class="c2">No GLP needed; this is discovery. Data will be presented as group means &plusmn; SEM.<br></span></li></ul><p class="c9"><span class="c2">Endpoints:</span></p><ul class="c4 lst-kix_ht2xpfpspp5-0 start"><li class="c0 li-bullet-0"><span class="c2">Primary: Neutralizing antibody titer on Day 35 (this is a surrogate for efficacy).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Secondary: HAI titer, ELISpot IFN-&gamma; count, IgG levels, etc.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Safety observations: ensuring no weight loss &gt;10%, no untoward clinical signs.<br></span></li></ul><p class="c9"><span class="c2">Ethics: All mouse procedures will be IACUC approved. Blood volumes follow guidelines (max 1% of body weight per bleed etc.). Terminal spleen harvest requires euthanasia (CO&#8322; or overdose pentobarbital).</span></p><p class="c9"><span class="c2">Expected Results: We anticipate the mid and high dose to induce high IgG and neutralizing titers. The low dose might be borderline in immunogenicity (which could indicate a need for a minimum amount of mRNA per dose). We expect a Th1 skew (given mRNA&rsquo;s innate immune activation tends to favor Th1). If we observe an adequate response even at 0.1 &micro;g, that suggests we might explore even lower dosing for safety margin, but likely the human dose ~50 &micro;g is already in the upper range ensuring strong response in humans. The findings from this study (e.g. that 1 &micro;g vs 5 &micro;g yields similar antibody levels) could help justify not exceeding a certain dose in humans to avoid diminishing returns.</span></p><h3 class="c12" id="h.5jz9is8sgjd1"><span class="c8">Protocol 4: Ferret Influenza Challenge Efficacy Study</span></h3><p class="c9"><span class="c2">Study Title: Efficacy of mRNA Influenza Vaccine in a Ferret Model: Immunogenicity and Protection Against Viral Challenge.</span></p><p class="c9"><span class="c2">Objectives: To evaluate the protective efficacy of the candidate mRNA influenza vaccine in ferrets by measuring its ability to prevent or mitigate influenza disease following a live virus challenge. The study will also characterize the immune responses in ferrets (both antibody and clinical outcomes) to confirm the vaccine&rsquo;s mechanism of action. Endpoints include reduction in viral load, disease severity, and induction of influenza-specific antibodies. This study serves as a critical proof-of-concept for vaccine effectiveness in a model that closely resembles human influenza infection.</span></p><p class="c9"><span class="c2">Rationale: Ferrets are highly susceptible to human influenza viruses and display similar symptoms (fever, sneezing, nasal discharge). Regulatory agencies expect demonstration of efficacy in a relevant animal model where feasible . Ferret data will strengthen the evidence of vaccine functionality beyond immunogenicity, showing actual disease prevention. It addresses any concerns of vaccine-induced disease enhancement (as ferrets have been used to identify enhanced disease in past vaccine studies by looking at pathology).</span></p><p class="c9"><span class="c2">Study Design: Randomized, controlled challenge study with two groups of ferrets:</span></p><ul class="c4 lst-kix_3aierr3stws2-0 start"><li class="c0 li-bullet-0"><span class="c2">Group 1: Vaccine Group &ndash; Ferrets immunized with mRNA vaccine.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Group 2: Control Group &ndash; Ferrets given a saline placebo (or placebo LNP).<br></span></li></ul><p class="c9"><span class="c2">Group size: 6 ferrets per group (n=6). This provides the power to see differences in viral titers and clinical scores, given ferret variability is moderate and effect sizes are expected large (e.g. ~2-3 log difference in viral titer). All ferrets will be females (~16-20 weeks old) or castrated males to avoid mating issues; weight ~0.8-1.2 kg at study start.</span></p><p class="c9"><span class="c2">Blinding: The personnel assessing clinical signs and analyzing samples will be blinded to group assignments to reduce bias.</span></p><p class="c9"><span class="c2">Vaccination Procedure: On Day 0, Vaccine group receives 50 &micro;g of mRNA vaccine IM (25 &micro;g in each hind leg muscle, 0.5 mL total volume). Control group receives 0.5 mL of saline IM split similarly. On Day 21, a second dose (booster) is given in the same manner. Injection sites will be monitored (though ferrets are quite tolerant to IM injections; only mild local reactions expected).</span></p><p class="c9"><span class="c2">Ferrets will be observed after vaccination for any immediate reactions for 1 hour (none anticipated aside from maybe transient discomfort). Over the 3-week interval, periodic blood samples (Day 21) will be taken to assess antibody development pre-boost.</span></p><p class="c9"><span class="c2">Immunogenicity Assessments Pre-Challenge: On Day 35 (two weeks after boost, which is ~ one week before challenge), ferrets are bled for:</span></p><ul class="c4 lst-kix_473lniz6s057-0 start"><li class="c0 li-bullet-0"><span class="c2">Serum HAI titer against the challenge strain.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Serum neutralization titer (if lab facilities for ferret serum testing; typically yes using microneutralization).<br></span></li><li class="c0 li-bullet-0"><span class="c2">These will confirm adequate immune response. We expect robust titers, similar to what effective human vaccines induce, in Vaccine group, and baseline or low titers in Controls.<br></span></li></ul><p class="c9"><span class="c2">Challenge Virus: A live influenza virus that the vaccine is designed to protect against, e.g. Influenza A H1N1 (pdm09) if the vaccine encodes H1 HA. We will use a wild-type human isolate (or minimally adapted) that is known to cause disease in ferrets. Titer ~10^6 TCID&#8325;&#8320; (tissue culture infectious dose 50) per ferret, administered in 1 mL total (0.5 mL per nostril) intranasally. This dose should reliably infect unvaccinated ferrets and cause moderate disease (weight loss, fever) but limited mortality (ferrets usually don&rsquo;t die from human flu strains, except highly pathogenic ones). If using a high-mortality strain (like H5N1 highly pathogenic), we&rsquo;d do intratracheal inoculation with a defined lethal dose to evaluate survival , but for a seasonal H1N1 we focus on morbidity reduction.</span></p><p class="c9"><span class="c2">Challenge is performed on Day 42 (3 weeks post-boost). Ferrets will be lightly anesthetized (ketamine/xylazine) during virus instillation to ensure it&rsquo;s inhaled properly and to minimize distress.</span></p><p class="c9"><span class="c2">Post-Challenge Monitoring:</span></p><ul class="c4 lst-kix_gp2qqn5s6so6-0 start"><li class="c0 li-bullet-0"><span class="c2">Clinical Signs and Scoring: A composite clinical score will be assigned daily for each ferret, assessing:<br></span></li></ul><ul class="c4 lst-kix_gp2qqn5s6so6-1 start"><li class="c3 li-bullet-0"><span class="c2">Activity level (0 = normal, 1 = slightly less playful, 2 = lethargic but responds, 3 = very lethargic or unresponsive).<br></span></li><li class="c3 li-bullet-0"><span class="c2">Respiratory signs (0 = normal, 1 = occasional sneeze, 2 = frequent sneezing and nasal discharge, 3 = labored breathing).<br></span></li><li class="c3 li-bullet-0"><span class="c2">Appetite (0 = normal, 1 = ate ~50%, 2 = ate &lt;25% or not eating).<br></span></li><li class="c3 li-bullet-0"><span class="c2">Each category can be scored and summed. If any ferret reaches a humane endpoint score (like total score &ge;4 out of maximum 6 or so , or loses &gt;15% body weight), it will be euthanized and counted as a severe disease outcome.<br></span></li></ul><ul class="c4 lst-kix_gp2qqn5s6so6-0"><li class="c0 li-bullet-0"><span class="c2">Body Weight: Measured daily. We expect control ferrets to lose weight (possibly 5-10% by day 5), and vaccinated ferrets to have minimal weight loss (or quicker recovery).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Body Temperature: Monitored by implantable temperature loggers or rectal thermometer at same time each day. Ferrets typically have baseline ~38&deg;C; fever defined as &gt;39&deg;C. Duration and peak of fever will be compared. Controls likely show fever on days 2-3; vaccinated maybe blunted or none.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Nasal discharge: qualitatively noted (clear, mucoid, bloody?).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Behavior: recorded (activity, grooming, sneezing frequency).<br><br> All clinical data recorded by blinded observers.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Viral Shedding: We will collect nasal washes or throat swabs on days 2, 4, 6, 8 post-challenge (or until virus is not detected). To do a nasal wash: a small volume (~1 mL) of sterile saline is instilled into one nostril and recovered (often ferrets sneeze it out into a collection bag or we aspirate gently). This sample is used for viral titer:<br></span></li></ul><ul class="c4 lst-kix_gp2qqn5s6so6-1 start"><li class="c3 li-bullet-0"><span class="c2">Quantitative RT-PCR for viral M gene to get genome copies.<br></span></li><li class="c3 li-bullet-0"><span class="c2">Virus titration on MDCK cells to get TCID&#8325;&#8320; or plaque-forming units. The latter is key to show live virus reduction. We expect vaccine group to have significantly lower titers at peak (day 2 or 4) and possibly clear virus faster (maybe cleared by day 6-8 vs controls still shedding).<br></span></li></ul><ul class="c4 lst-kix_gp2qqn5s6so6-0"><li class="c0 li-bullet-0"><span class="c2">Terminal Viral Load: All ferrets will be euthanized on day 5 post-challenge (unless survival is an endpoint; see below). Day 5 is chosen when unvaccinated ferrets typically still have virus in lungs for an H1N1 and peak illness is around day 2-4. At euthanasia:<br></span></li></ul><ul class="c4 lst-kix_gp2qqn5s6so6-1 start"><li class="c3 li-bullet-0"><span class="c2">Collect lung tissues (each lobe separately if needed) for virus titration (homogenize and titer).<br></span></li><li class="c3 li-bullet-0"><span class="c2">Collect nasal turbinates for virus titration to see local replication.<br></span></li><li class="c3 li-bullet-0"><span class="c2">The viral load in lungs is a critical measure of vaccine efficacy; vaccinated animals should have much lower lung viral titers .<br></span></li><li class="c3 li-bullet-0"><span class="c2">If the study is using a strain that can cause severe disease, we might extend observation to day 14 for survival analysis. In that case, viral load would be measured in any animals that succumb or at study end.<br></span></li></ul><ul class="c4 lst-kix_gp2qqn5s6so6-0"><li class="c0 li-bullet-0"><span class="c2">Pathology: On day 5, a necropsy is done:<br></span></li></ul><ul class="c4 lst-kix_gp2qqn5s6so6-1 start"><li class="c3 li-bullet-0"><span class="c2">Lungs are examined. We may score gross pathology (percentage of lung affected by consolidation). Then fix lungs for histology.<br></span></li><li class="c3 li-bullet-0"><span class="c2">Histopathology: lung sections (from all lobes or representative lobes) stained H&amp;E. Look for bronchopneumonia, inflammatory cell types, edema, etc. Possibly immunohistochemistry for influenza antigen to localize residual virus.<br></span></li><li class="c3 li-bullet-0"><span class="c2">Other organs: trachea, nasal turbinates histology, and any other organ if needed (brain for H5N1 if concerned about neuro invasion, etc.).<br></span></li><li class="c3 li-bullet-0"><span class="c2">Compare pathology severity between groups: e.g. average lung inflammation score (on 0-4 scale). Expect vaccine group to have minimal changes, control moderate bronchitis, bronchiolitis, alveolitis.<br></span></li></ul><ul class="c4 lst-kix_gp2qqn5s6so6-0"><li class="c0 li-bullet-0"><span class="c2">Immunologic analysis post-challenge:<br></span></li></ul><ul class="c4 lst-kix_gp2qqn5s6so6-1 start"><li class="c3 li-bullet-0"><span class="c2">On day 5, we can also collect blood for serology to see if anamnestic response started (likely too early though).<br></span></li><li class="c3 li-bullet-0"><span class="c2">Spleen or lymph nodes could be harvested for memory B/T analysis, though not necessary for primary endpoints.<br></span></li></ul><p class="c9"><span class="c2">If a survival aspect is included (like with a more pathogenic challenge requiring to see if vaccine prevents death), the study design would slightly adjust:</span></p><ul class="c4 lst-kix_w12y2pivfx64-0 start"><li class="c0 li-bullet-0"><span class="c2">Monitor up to 14 days post-challenge.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Humane endpoints for moribund animals.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Survival would be compared by Kaplan-Meier (log-rank test).<br><br> For standard seasonal strains, death is not expected, so survival may not be an endpoint, instead focusing on morbidity and viral clearance.<br></span></li></ul><p class="c9"><span class="c2">Outcome Measures and Analysis:</span></p><ul class="c4 lst-kix_6k838qjxgg72-0 start"><li class="c0 li-bullet-0"><span class="c2">Primary efficacy endpoint: Reduction in lung viral titer at day 5 in vaccinated vs control (compare mean log&#8321;&#8320; TCID&#8325;&#8320;; expect a few logs reduction).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Secondary endpoints: include peak nasal wash titer, total viral shedding (AUC of nasal titers over time), percentage body weight loss, clinical score, and lung pathology score.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Statistical tests: Two-sample t-test or Mann-Whitney U for comparing viral titers (if non-normal). Repeated measures ANOVA for weight curves. Fisher&rsquo;s exact for incidence of fever or severe disease. Log-rank test if survival curve used.<br></span></li><li class="c0 li-bullet-0"><span class="c2">The study will be considered successful if vaccinated ferrets have significantly lower virus levels and clinical scores (p&lt;0.05) and no signs of enhanced disease.<br></span></li><li class="c0 li-bullet-0"><span class="c2">We will also ensure that no vaccinated ferret has worse outcomes than any control (to rule out any possibility of ADE or ERD).<br></span></li></ul><p class="c9"><span class="c2">Biosafety: The challenge will be done in appropriate containment (influenza typically BSL-2 for human H1N1/H3N2, BSL-3 for avian H5N1/H7N9). Staff will wear PPE and be vaccinated against flu or on antivirals to prevent their infection. All sample handling follows biohazard protocols.</span></p><p class="c9"><span class="c2">Quality Considerations: Not GLP, but standard operating procedures for challenge studies are followed. The data will be recorded in study notebooks and electronic files. The challenge stock will be titer-verified. A randomization code for group assignment will be maintained.</span></p><p class="c9"><span class="c2">Expected Results: We expect the mRNA vaccine to protect ferrets, evidenced by:</span></p><ul class="c4 lst-kix_1ip3hsr9v1h5-0 start"><li class="c0 li-bullet-0"><span class="c2">Minimal weight loss (perhaps &lt;5%) in Vaccine group vs ~10% in Controls around day 3-4.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Shorter fever duration or none in Vaccine group.<br></span></li><li class="c0 li-bullet-0"><span class="c2">A &gt;100-fold (&ge;2 log&#8321;&#8320;) reduction in peak nasal virus titer .<br></span></li><li class="c0 li-bullet-0"><span class="c2">Lung tissues from vaccinated ferrets might even be virus-free or near limit of detection, whereas controls have high titers.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Histologically, vaccinated ferrets may show only mild peribronchiolar lymphocyte infiltrates (sign of an immune response) but no significant pneumonia, whereas controls have moderate bronchopneumonia.<br></span></li><li class="c0 li-bullet-0"><span class="c2">No evidence of enhanced disease in vaccinated ferrets (such as excessive eosinophils or immune complex deposition). Ferret studies with other vaccines (e.g. whole-inactivated RSV historically) showed enhanced pathology; we expect none here given the anticipated Th1 response and robust neutralizing antibodies.<br></span></li><li class="c0 li-bullet-0"><span class="c2">These outcomes will strongly support that the vaccine is efficacious, justifying progression to human trials where the primary endpoint might initially be immune response, but the animal data give confidence in likely protection .<br></span></li></ul><hr><p class="c9 c11"><span class="c2"></span></p><p class="c9"><span class="c2">Each of the above protocols will be finalized and approved by the sponsor and test facility before initiation. Any amendments will be documented. The results from all studies will be compiled into the IND filing, providing a comprehensive non-clinical data package that demonstrates the mRNA influenza vaccine&rsquo;s safety, pharmacokinetics, and efficacy in line with regulatory expectations.</span></p><p class="c9"><span class="c2">Overall, the non-clinical program is designed to ensure a clear justification for proceeding to human trials, with thorough characterization of risks mitigated by appropriate study designs and compliance with GLP where required. The structured approach, from in vitro assays through in vivo efficacy, provides a sound scientific foundation for the IND and subsequent clinical development of the mRNA influenza vaccine.</span></p><p class="c9"><span class="c2">References:</span></p><ol class="c4 lst-kix_kfepckt4l6hg-0 start" start="1"><li class="c0 li-bullet-0"><span class="c2">World Health Organization. WHO Guidelines on Nonclinical Evaluation of Vaccines. WHO Technical Report Series No. 927, 2005: &ldquo;&hellip;the animal species used should develop an immune response to the vaccine antigen. In general, one relevant animal species is sufficient for use in toxicity studies to support initiation of clinical trials.&rdquo;<br></span></li><li class="c0 li-bullet-0"><span class="c2">WHO Guidelines 2013 Update: &ldquo;It is not necessary to conduct the nonclinical safety study in the same animal species used for proof-of-concept or nonclinical pharmacology studies, although it should be immunologically responsive to the vaccine antigen.&rdquo;<br></span></li><li class="c0 li-bullet-0"><span class="c2">WHO Technical Report Series 927: &ldquo;The route of administration should correspond to that intended for use in clinical trials&hellip; The study should also include an assessment of local tolerance.&rdquo;<br></span></li><li class="c0 li-bullet-0"><span class="c2">van der Laan JW, et al. (2023). Evaluation of non-clinical toxicity studies of COVID-19 vaccines. Regul. Toxicol. Pharmacol. &ndash; &ldquo;RDT studies at doses relevant for human treatment showed no adverse effects while remaining observations were expected findings including local reactogenicity, immune response and macroscopic findings at the injection site.&rdquo;<br></span></li><li class="c0 li-bullet-0"><span class="c2">van der Laan JW, et al. (2023): &ldquo;The WHO Guideline on Preclinical Testing of Vaccines recommends the use of a dose leading to maximal antibody responses.&rdquo;<br></span></li><li class="c0 li-bullet-0"><span class="c2">ICH S5(R3) (2020) &amp; WHO 2005: &ldquo;A single species is usually considered to be sufficient to provide important information on potential toxicity of the vaccine and safety of the product during human pregnancy.&rdquo;<br></span></li><li class="c0 li-bullet-0"><span class="c2">FDA Guidance for Industry (2020). Development and Licensure of Vaccines to Prevent COVID-19: &ldquo;The purpose of nonclinical studies&hellip; is to define its immunogenicity and safety characteristics through in vitro and in vivo testing&hellip; Nonclinical studies in animal models help identify potential vaccine-related safety risks and guide the selection of dose, dosing regimen, and route&hellip;&rdquo;<br></span></li><li class="c0 li-bullet-0"><span class="c2">FDA Guidance COVID-19 Vaccines (2020): &ldquo;When needed to support proceeding to FIH clinical trials, nonclinical safety assessments including toxicity and local tolerance studies must be conducted under GLP (21 CFR Part 58). Such studies should be completed and analyzed prior to initiation of FIH clinical trials.&rdquo;<br></span></li><li class="c0 li-bullet-0"><span class="c2">FDA/CBER &ldquo;Regulatory Considerations in the Safety Assessment of Adjuvants&rdquo; (2013): &ldquo;&hellip;CBER encourages sponsors to demonstrate not only the immune response to the antigen, but also that the adjuvant enhances this immune response&hellip; e.g., a head-to-head comparison in animals&hellip; helpful for proof-of-concept and for assessing whether the animal species chosen for toxicology studies is sensitive to the adjuvant effect.&rdquo;<br></span></li><li class="c0 li-bullet-0"><span class="c2">Pateev I, et al. (2023). Biodistribution of RNA Vaccines and their Products: Evidence from Human and Animal Studies: &ldquo;After intramuscular injection of mRNA-LNP, the luminescent signal from luciferase is detected mostly in the muscle (site of injection) and the liver&hellip; robust luciferase expression at the injection site and a weak signal in the liver [in mice]. Further analysis showed presence of the reporter signal in the spleen as well.&rdquo;<br></span></li><li class="c0 li-bullet-0"><span class="c2">Pateev I, et al. (2023): &ldquo;Maximum bioluminescence at the injection site occurs at 6 h and disappears within 4 days regardless of the composition of the LNP.&rdquo;<br></span></li><li class="c0 li-bullet-0"><span class="c2">WuXi AppTec (2022). Preclinical PK Evaluation of mRNA Therapeutics: &ldquo;According to relevant technical guidelines&hellip; vaccines do not usually require PK studies, but several specific vaccines, including mRNA vaccines, need biodistribution studies&hellip; mRNA vaccines encode antigen proteins&hellip; The duration of efficacy is not directly related to the expression level and duration of the antigen&hellip; This also leads to that mRNA vaccines typically do not require PK studies, while PK studies of mRNA therapeutic drugs are [needed].&rdquo;<br></span></li><li class="c0 li-bullet-0"><span class="c2">Peters F, et al. (2022). A universal influenza mRNA vaccine candidate&hellip; in ferrets: &ldquo;After challenge, influenza-experienced ferrets that were boosted with the mRNA vaccine showed reduced disease severity&hellip; whereas mock-boosted influenza-experienced ferrets were not protected.&rdquo;<br></span></li><li class="c0 li-bullet-0"><span class="c2">Peters F, et al. (2022): &ldquo;Ferrets were challenged intratracheally with 10^6 TCID50 H7N9 influenza virus 4 weeks after booster&hellip; At 5 dpi, animals were euthanized and tissue pathology and viral loads were [evaluated].&rdquo;<br></span></li><li class="c0 li-bullet-0"><span class="c2">CIDRAP News (Nov 2023) &ndash; Preclinical findings for mRNA H5N1 vaccine: &ldquo;In mice, the vaccine prompted a robust antibody response, and the antibodies neutralized different clade viruses. Experiments on ferrets showed similar results, and the group&rsquo;s challenge studies in ferrets found the vaccine protected the animals from disease and death.&rdquo;<br></span></li></ol><p class="c9"><span class="c2">Protocol Generation:</span></p><p class="c9 c11"><span class="c2"></span></p><p class="c9"><span class="c2">Prompt: </span></p><p class="c9"><span class="c2">Provide protocols for genotox studies for the above mRNA vaccine as well. Keep the framework, design and narration similar to what was provided in the previous response.</span></p><h3 class="c12" id="h.oshsff6ny6u"><span class="c8">Protocol for Genotoxicity Studies of an mRNA Influenza Vaccine</span></h3><h4 class="c22" id="h.vbl7njk1l426"><span class="c13">Study Title</span></h4><p class="c9"><span class="c2">In Vitro and In Vivo Genotoxicity Studies for an mRNA Influenza Vaccine</span></p><h4 class="c22" id="h.l5jtph9toqmi"><span class="c13">Objective</span></h4><p class="c9"><span class="c2">The objective of this study is to assess the potential genotoxic effects of the mRNA-LNP influenza vaccine. Although mRNA-based vaccines do not integrate into the genome and are expected to degrade rapidly in the cytoplasm, regulatory guidance (e.g., ICH S2(R1)) recommends conducting genotoxicity tests, particularly if any novel excipients (such as lipid nanoparticles) are used. This study will evaluate:</span></p><ul class="c4 lst-kix_129uykneptsf-0 start"><li class="c0 li-bullet-0"><span class="c2">DNA mutagenicity via a bacterial reverse mutation assay (in vitro Ames test).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Chromosomal damage via an in vitro micronucleus test (MN assay) in mammalian cells.<br></span></li><li class="c0 li-bullet-0"><span class="c2">In vivo DNA damage potential via a rat bone marrow micronucleus assay to assess systemic genotoxic potential following intramuscular administration.<br></span></li></ul><p class="c9"><span class="c2">These studies will establish whether the mRNA influenza vaccine or its LNP components have any genotoxic risk, fulfilling regulatory requirements for genotoxicity assessment prior to human clinical trials.</span></p><hr><p class="c9 c11"><span class="c2"></span></p><h2 class="c19" id="h.qofgl0h792pj"><span class="c20">Study 1: Bacterial Reverse Mutation Assay (Ames Test)</span></h2><h3 class="c12" id="h.yvtgitc70pom"><span class="c8">Study Title:</span></h3><p class="c9"><span class="c2">Mutagenicity Assessment of mRNA-LNP Influenza Vaccine Using the Bacterial Reverse Mutation (Ames) Test</span></p><h3 class="c12" id="h.hpfq3jkll8k6"><span class="c8">Objective:</span></h3><p class="c9"><span class="c2">To evaluate the potential mutagenicity of the mRNA-LNP influenza vaccine using a bacterial reverse mutation assay in accordance with OECD Test Guideline 471 and ICH S2(R1). This test detects point mutations in bacterial strains exposed to the test article, with and without metabolic activation.</span></p><h3 class="c12" id="h.r3uxrfg49bfv"><span class="c8">Test Article:</span></h3><ul class="c4 lst-kix_tc68vt7q79wj-0 start"><li class="c0 li-bullet-0"><span class="c2">mRNA-LNP influenza vaccine (fully formulated, containing mRNA encoding hemagglutinin antigen and lipid nanoparticles).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Lot number: [To be assigned].<br></span></li><li class="c0 li-bullet-0"><span class="c2">Storage conditions: &minus;70&deg;C.<br></span></li></ul><h3 class="c12" id="h.52anj3nmr2mf"><span class="c8">Control Articles:</span></h3><ul class="c4 lst-kix_2tnhjv96pndg-0 start"><li class="c0 li-bullet-0"><span class="c2">Negative control: Sterile phosphate-buffered saline (PBS) or vehicle control (if the LNP requires a specific solvent).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Positive controls:<br></span></li></ul><ul class="c4 lst-kix_2tnhjv96pndg-1 start"><li class="c3 li-bullet-0"><span class="c2">Without metabolic activation: Sodium azide (for TA100, TA1535), 4-Nitroquinoline 1-oxide (for TA98, WP2 uvrA).<br></span></li><li class="c3 li-bullet-0"><span class="c2">With metabolic activation (S9 mix): Benzo[a]pyrene (for TA98, WP2 uvrA), 2-Aminoanthracene (for TA100, TA1537, TA1535).<br></span></li></ul><h3 class="c12" id="h.bppjvkbvvyip"><span class="c8">Bacterial Strains Tested:</span></h3><ul class="c4 lst-kix_b01r1xbffk2a-0 start"><li class="c0 li-bullet-0"><span class="c2">Salmonella typhimurium strains: TA98, TA100, TA1535, TA1537.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Escherichia coli WP2 uvrA (for detection of base-pair substitutions).<br></span></li></ul><h3 class="c12" id="h.4fjnfq9t2f3b"><span class="c8">Dosing and Concentrations:</span></h3><ul class="c4 lst-kix_i5y1isusbf26-0 start"><li class="c0 li-bullet-0"><span class="c2">A range-finding study will be conducted first to determine the highest non-cytotoxic concentration of the test article.<br></span></li><li class="c0 li-bullet-0"><span class="c2">At least 5 test concentrations will be used, ranging from 0.1 &micro;g to 500 &micro;g per plate.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Doses will be tested with and without an exogenous metabolic activation system (rat liver S9 fraction).<br></span></li></ul><h3 class="c12" id="h.chpgl75en19j"><span class="c8">Study Design:</span></h3><ul class="c4 lst-kix_s4u6wv582l9n-0 start"><li class="c0 li-bullet-0"><span class="c2">Plate Incorporation Method (for soluble test article).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Pre-incubation Method (if the test article is poorly soluble in the agar medium).<br></span></li></ul><p class="c9"><span class="c2">Each concentration will be tested in triplicate under the following conditions:</span></p><ol class="c4 lst-kix_ytu3x5e7mxt6-0 start" start="1"><li class="c0 li-bullet-0"><span class="c2">Without metabolic activation (direct mutagenicity).<br></span></li><li class="c0 li-bullet-0"><span class="c2">With metabolic activation (S9 mix, to simulate liver metabolism).<br></span></li></ol><h3 class="c12" id="h.g4515c73fp70"><span class="c8">Endpoints and Evaluation:</span></h3><ul class="c4 lst-kix_lkbbmj5sol47-0 start"><li class="c0 li-bullet-0"><span class="c2">Colony counting: Revertant colonies will be counted for each plate.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Statistical analysis: A dose-dependent increase in revertant colonies that is at least twofold higher than the negative control in at least one strain will be considered a positive mutagenic response.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Cytotoxicity assessment: Reduction in bacterial lawn density at high doses will indicate cytotoxicity, necessitating repeat testing at lower doses.<br></span></li></ul><h3 class="c12" id="h.de8xecw3ts3t"><span class="c8">GLP Compliance:</span></h3><ul class="c4 lst-kix_7g6nqlfa7rn7-0 start"><li class="c0 li-bullet-0"><span class="c2">This study will be conducted under GLP regulations as per OECD 471 and ICH S2(R1).<br></span></li><li class="c0 li-bullet-0"><span class="c2">QA monitoring will ensure compliance and documentation.<br></span></li></ul><h3 class="c12" id="h.ejjinloe8j81"><span class="c8">Expected Outcome:</span></h3><p class="c9"><span class="c2">Given that mRNA vaccines do not directly interact with DNA and degrade rapidly in the cytoplasm, no mutagenic activity is expected. Lipid nanoparticles (LNPs) have been tested in similar vaccines (e.g., COVID-19 mRNA vaccines) without showing genotoxicity, supporting the hypothesis that this formulation will also be negative.</span></p><hr><p class="c9 c11"><span class="c2"></span></p><h2 class="c19" id="h.wt2k275l8etu"><span class="c20">Study 2: In Vitro Micronucleus Assay in Mammalian Cells</span></h2><h3 class="c12" id="h.tkk8164allhm"><span class="c8">Study Title:</span></h3><p class="c9"><span class="c2">Chromosomal Aberration Assessment of mRNA-LNP Influenza Vaccine Using the In Vitro Micronucleus Test</span></p><h3 class="c12" id="h.mtz3tb9v4rik"><span class="c8">Objective:</span></h3><p class="c9"><span class="c2">To evaluate the potential clastogenic and aneugenic effects of the mRNA-LNP influenza vaccine on mammalian cells by measuring micronucleus formation. This assay will determine if the test article induces chromosome breaks or aneuploidy (whole chromosome loss or gain).</span></p><h3 class="c12" id="h.twlgwowlfazn"><span class="c8">Test System:</span></h3><ul class="c4 lst-kix_5mkgtms2qzgk-0 start"><li class="c0 li-bullet-0"><span class="c2">Cell line: Human lymphoblastoid TK6 cells (or Chinese hamster ovary (CHO) cells) commonly used in regulatory genotoxicity assays.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Dosing and exposure time:<br></span></li></ul><ul class="c4 lst-kix_5mkgtms2qzgk-1 start"><li class="c3 li-bullet-0"><span class="c2">Short-term exposure: 4 hours with and without S9 metabolic activation.<br></span></li><li class="c3 li-bullet-0"><span class="c2">Long-term exposure: 24 hours without S9 activation.<br></span></li></ul><ul class="c4 lst-kix_5mkgtms2qzgk-0"><li class="c0 li-bullet-0"><span class="c2">Doses tested:<br></span></li></ul><ul class="c4 lst-kix_5mkgtms2qzgk-1 start"><li class="c3 li-bullet-0"><span class="c2">Range-finding cytotoxicity study will determine 5-6 non-cytotoxic concentrations (typically from 1 ng/mL to 500 &micro;g/mL).<br></span></li><li class="c3 li-bullet-0"><span class="c2">Doses will be selected to cover a broad range of exposures while avoiding excessive cytotoxicity (&ge;50% reduction in viability is considered excessive).<br></span></li></ul><h3 class="c12" id="h.wkikbdpkks6j"><span class="c8">Endpoints and Evaluation:</span></h3><ul class="c4 lst-kix_nqcns3qrnch2-0 start"><li class="c0 li-bullet-0"><span class="c2">Micronucleus frequency: Cells with micronuclei will be counted and compared to control groups.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Cytotoxicity assessment: Reduction in cell viability measured via relative survival or cytokinesis-block proliferation index (CBPI).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Statistical analysis: A significant, dose-dependent increase in micronucleus formation (&ge;2-fold above control) is considered genotoxic.<br></span></li></ul><h3 class="c12" id="h.u7yundhbgpdh"><span class="c8">GLP Compliance:</span></h3><ul class="c4 lst-kix_wje7sh1pcpc1-0 start"><li class="c0 li-bullet-0"><span class="c2">This study will be conducted under GLP standards as per OECD 487 and ICH S2(R1) guidelines.<br></span></li></ul><h3 class="c12" id="h.bhzki5q5he3m"><span class="c8">Expected Outcome:</span></h3><p class="c9"><span class="c2">The mRNA-LNP formulation is not expected to induce chromosomal damage, given that similar lipid-based delivery systems have shown no genotoxic effects in regulatory studies for other mRNA vaccines.</span></p><hr><p class="c9 c11"><span class="c2"></span></p><h2 class="c19" id="h.ov2zmg2ipqq8"><span class="c20">Study 3: In Vivo Rat Bone Marrow Micronucleus Assay</span></h2><h3 class="c12" id="h.87tfkw2651np"><span class="c8">Study Title:</span></h3><p class="c9"><span class="c2">Assessment of In Vivo Genotoxicity of mRNA-LNP Influenza Vaccine Using the Rat Bone Marrow Micronucleus Assay</span></p><h3 class="c12" id="h.fjppwa4sr8mb"><span class="c8">Objective:</span></h3><p class="c9"><span class="c2">To evaluate the potential for in vivo chromosomal damage following intramuscular administration of the mRNA-LNP influenza vaccine. This study will determine if the vaccine or its components induce micronuclei in dividing erythrocytes of the bone marrow, an indicator of DNA damage.</span></p><h3 class="c12" id="h.l1g7a9vzebaa"><span class="c8">Test System:</span></h3><ul class="c4 lst-kix_b0v72ufkqzg3-0 start"><li class="c0 li-bullet-0"><span class="c2">Species: Rat (Sprague-Dawley, 8-10 weeks old).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Group Size: 6 males + 6 females per dose group (to detect sex-related differences).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Groups:<br></span></li></ul><ul class="c4 lst-kix_b0v72ufkqzg3-1 start"><li class="c3 li-bullet-0"><span class="c2">Control (PBS)<br></span></li><li class="c3 li-bullet-0"><span class="c2">Low dose (1&times; human dose equivalent)<br></span></li><li class="c3 li-bullet-0"><span class="c2">Mid dose (3&times; human dose equivalent)<br></span></li><li class="c3 li-bullet-0"><span class="c2">High dose (10&times; human dose equivalent)<br></span></li></ul><ul class="c4 lst-kix_b0v72ufkqzg3-0"><li class="c0 li-bullet-0"><span class="c2">Route of Administration: Intramuscular (same as intended clinical use).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Dosing Schedule: Two IM injections (Day 1 and Day 22), with bone marrow collection 48 hours after the second dose (Day 24) to assess peak genotoxic effect.<br></span></li></ul><h3 class="c12" id="h.qmddy91yqh3l"><span class="c8">Endpoints and Evaluation:</span></h3><ul class="c4 lst-kix_haoc7b4kfhm3-0 start"><li class="c0 li-bullet-0"><span class="c2">Micronucleated erythrocytes per 2000 polychromatic erythrocytes (PCEs).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Ratio of polychromatic to normochromatic erythrocytes to detect cytotoxicity.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Statistical analysis: A dose-related increase in micronucleated PCEs (&ge;2&times; control) would be considered genotoxic.<br></span></li></ul><h3 class="c12" id="h.qo31xq6q1xuo"><span class="c8">GLP Compliance:</span></h3><ul class="c4 lst-kix_kruy5m9jmo50-0 start"><li class="c0 li-bullet-0"><span class="c2">This study will follow OECD 474 and ICH S2(R1) guidelines under GLP conditions.<br></span></li></ul><h3 class="c12" id="h.90x1bfrrtbbw"><span class="c8">Expected Outcome:</span></h3><p class="c9"><span class="c2">Previous studies of mRNA vaccines have not demonstrated in vivo genotoxicity. Given the mechanism of action (cytoplasmic translation, no DNA interaction), no significant increase in micronucleus formation is expected.</span></p><hr><p class="c9 c11"><span class="c2"></span></p><h3 class="c12" id="h.be4jh64me6ml"><span class="c8">Conclusion:</span></h3><p class="c9"><span class="c2">These genotoxicity studies will ensure regulatory compliance and confirm the mRNA-LNP influenza vaccine does not pose a genetic toxicity risk. Results will be integrated into the IND submission, strengthening the safety profile for first-in-human trials.</span></p><p class="c9 c11"><span class="c2"></span></p><p class="c9"><span class="c2">EXAMPLE 2: </span></p><p class="c9"><span class="c2">Create a detailed study plan with study designs and protocols for the following:</span></p><p class="c9"><span class="c2">what is the gold standard for the ADME and PK/PD studies for exosomes for animal studies? </span></p><p class="c9"><span class="c2">Response:</span></p><p class="c9"><span class="c2">Below is an example of a detailed study plan outlining the design, protocols, and rationale for conducting ADME (Absorption, Distribution, Metabolism, Excretion) and PK/PD (Pharmacokinetics/Pharmacodynamics) studies of exosomes in animal models. While there is no absolute universal &ldquo;gold standard&rdquo; for exosome ADME and PK/PD (in large part due to evolving methodologies in the field), radiolabeling with isotopes (e.g., 125I, 111In), in combination with highly sensitive imaging (e.g., SPECT, PET, or near-infrared fluorescence imaging) and ex vivo tissue quantification, is often considered the most rigorous approach. Below is a comprehensive plan that incorporates these best practices.</span></p><hr><p class="c9 c11"><span class="c2"></span></p><h2 class="c19" id="h.gularmr7wwkb"><span class="c20">1. Study Objectives</span></h2><ol class="c4 lst-kix_eag5a795mtbr-0 start" start="1"><li class="c0 li-bullet-0"><span class="c2">Characterize the pharmacokinetics (PK) of exosomes following a specific route of administration (e.g., intravenous, intraperitoneal).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Determine the tissue distribution (biodistribution) profile of exosomes over time (Distribution).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Assess clearance pathways (Metabolism and Excretion) and estimate half-life.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Evaluate biological activity of exosome cargo in an in vivo setting (PD end points).<br></span></li></ol><hr><p class="c9 c11"><span class="c2"></span></p><h2 class="c19" id="h.sejs02hds1oo"><span class="c20">2. Choice of Animal Model</span></h2><ul class="c4 lst-kix_5s9vwy7hbq8r-0 start"><li class="c0 li-bullet-0"><span class="c2">Rodent model (mouse or rat): Most common for early PK/PD and biodistribution studies.<br></span></li></ul><ul class="c4 lst-kix_5s9vwy7hbq8r-1 start"><li class="c3 li-bullet-0"><span class="c2">Mice (e.g., BALB/c or C57BL/6) are often used for feasibility and proof-of-concept due to cost-effectiveness.<br></span></li><li class="c3 li-bullet-0"><span class="c2">Rats may be used when larger blood or tissue sample volumes are required.<br></span></li></ul><ul class="c4 lst-kix_5s9vwy7hbq8r-0"><li class="c0 li-bullet-0"><span class="c2">Disease model vs. healthy model: If the exosomes are meant to target a specific disease (e.g., tumor model, inflammatory model), consider using that disease model to assess relevant PD outcomes.<br></span></li></ul><hr><p class="c9 c11"><span class="c2"></span></p><h2 class="c19" id="h.rb6jxu6i2kj"><span class="c20">3. Exosome Preparation and Characterization</span></h2><ol class="c4 lst-kix_zqq05ofg66-0 start" start="1"><li class="c0 li-bullet-0"><span class="c2">Source of Exosomes<br></span></li></ol><ul class="c4 lst-kix_zqq05ofg66-1 start"><li class="c3 li-bullet-0"><span class="c2">Cell line-derived (e.g., MSC, tumor cell line), plasma-derived, or other biological fluids.<br></span></li><li class="c3 li-bullet-0"><span class="c2">Make sure to document cell origin, culture conditions, and any genetic modifications.<br></span></li></ul><ol class="c4 lst-kix_zqq05ofg66-0" start="2"><li class="c0 li-bullet-0"><span class="c2">Isolation and Purification<br></span></li></ol><ul class="c4 lst-kix_zqq05ofg66-1 start"><li class="c3 li-bullet-0"><span class="c2">Use accepted methods such as differential ultracentrifugation, size exclusion chromatography (SEC), or commercial kits.<br></span></li><li class="c3 li-bullet-0"><span class="c2">Optional: Additional purification steps (e.g., density gradient ultracentrifugation) for higher purity.<br></span></li></ul><ol class="c4 lst-kix_zqq05ofg66-0" start="3"><li class="c0 li-bullet-0"><span class="c2">Pre-labeling Characterization<br></span></li></ol><ul class="c4 lst-kix_zqq05ofg66-1 start"><li class="c3 li-bullet-0"><span class="c2">Particle size and concentration: Nanoparticle tracking analysis (NTA) or dynamic light scattering (DLS).<br></span></li><li class="c3 li-bullet-0"><span class="c2">Morphology: Transmission electron microscopy (TEM) or cryo-EM.<br></span></li><li class="c3 li-bullet-0"><span class="c2">Marker expression: Western blot or flow cytometry for known exosome markers (CD63, CD9, CD81).<br></span></li></ul><hr><p class="c9 c11"><span class="c2"></span></p><h2 class="c19" id="h.pwhqnri709f7"><span class="c20">4. Exosome Labeling Strategy (Gold Standard Approach)</span></h2><ul class="c4 lst-kix_90vpss294crz-0 start"><li class="c0 li-bullet-0"><span class="c2">Radiolabeling (e.g., 125I, 111In)<br></span></li></ul><ul class="c4 lst-kix_90vpss294crz-1 start"><li class="c3 li-bullet-0"><span class="c2">Often regarded as the most sensitive and quantitative &ldquo;gold standard&rdquo; for biodistribution and PK.<br></span></li><li class="c3 li-bullet-0"><span class="c2">Protocol typically involves conjugating exosomes to a chelator (e.g., DTPA for 111In) or using the iodination method for 125I labeling.<br></span></li><li class="c3 li-bullet-0"><span class="c2">Ensure minimal alteration of exosome integrity and surface proteins.<br></span></li></ul><ul class="c4 lst-kix_90vpss294crz-0"><li class="c0 li-bullet-0"><span class="c2">Fluorescent Labels (e.g., DiR, DiD, PKH, CFSE)<br></span></li></ul><ul class="c4 lst-kix_90vpss294crz-1 start"><li class="c3 li-bullet-0"><span class="c2">Useful for in vivo imaging (near-infrared fluorescence) and ex vivo tissue section imaging.<br></span></li><li class="c3 li-bullet-0"><span class="c2">Less quantitative than radiolabeling but facilitates non-invasive imaging (optical imaging).<br></span></li><li class="c3 li-bullet-0"><span class="c2">Be aware of potential artifacts (e.g., free dye or micelle formation).<br></span></li></ul><ul class="c4 lst-kix_90vpss294crz-0"><li class="c0 li-bullet-0"><span class="c2">Bioluminescent Labeling<br></span></li></ul><ul class="c4 lst-kix_90vpss294crz-1 start"><li class="c3 li-bullet-0"><span class="c2">Potential approach if exosomes can be loaded or engineered to express luciferase.<br></span></li><li class="c3 li-bullet-0"><span class="c2">Good for real-time tracking but may have lower sensitivity compared to radiolabeling.<br></span></li></ul><p class="c9"><span class="c2">Recommendation: For definitive ADME and PK/PD quantitation, pair a radiolabeling approach for robust biodistribution measurements with optional fluorescent labeling for imaging confirmation.</span></p><hr><p class="c9 c11"><span class="c2"></span></p><h2 class="c19" id="h.xoq45y4glxev"><span class="c20">5. Study Design</span></h2><h3 class="c12" id="h.dp1yckezk6m2"><span class="c8">5.1 Dosing and Administration</span></h3><ol class="c4 lst-kix_awa32z9rxqor-0 start" start="1"><li class="c0 li-bullet-0"><span class="c2">Dose Rationale<br></span></li></ol><ul class="c4 lst-kix_awa32z9rxqor-1 start"><li class="c3 li-bullet-0"><span class="c2">Decide on a dose based on preliminary in vitro studies or pilot in vivo data (e.g., 1 &times; 10^9 to 1 &times; 10^11 exosomes per mouse).<br></span></li><li class="c3 li-bullet-0"><span class="c2">Adjust for the labeling strategy (radioactivity levels or fluorescence intensity).<br></span></li></ul><ol class="c4 lst-kix_awa32z9rxqor-0" start="2"><li class="c0 li-bullet-0"><span class="c2">Route of Administration<br></span></li></ol><ul class="c4 lst-kix_awa32z9rxqor-1 start"><li class="c3 li-bullet-0"><span class="c2">Intravenous (IV): Common for systemic distribution and is the most straightforward route for PK sampling.<br></span></li><li class="c3 li-bullet-0"><span class="c2">Intraperitoneal (IP) or Subcutaneous (SC): If the therapeutic indication requires these routes, account for potential differences in absorption kinetics.<br></span></li></ul><ol class="c4 lst-kix_awa32z9rxqor-0" start="3"><li class="c0 li-bullet-0"><span class="c2">Number of Animals and Groups<br></span></li></ol><ul class="c4 lst-kix_awa32z9rxqor-1 start"><li class="c3 li-bullet-0"><span class="c2">A typical PK study design has n = 3&ndash;5 animals per time point (especially if terminal sampling is needed).<br></span></li><li class="c3 li-bullet-0"><span class="c2">Time points: e.g., 0.25, 0.5, 1, 2, 4, 8, 12, 24 hours, and possibly up to 48&ndash;72 hours or longer, depending on exosome stability and half-life.<br></span></li><li class="c3 li-bullet-0"><span class="c2">Consider at least one control group (placebo or labeled buffer only).<br></span></li></ul><hr><p class="c9 c11"><span class="c2"></span></p><h3 class="c12" id="h.pgxtpl3cic29"><span class="c8">5.2 Sample Collection</span></h3><ol class="c4 lst-kix_8joc3jsoa7de-0 start" start="1"><li class="c0 li-bullet-0"><span class="c2">Blood Sampling<br></span></li></ol><ul class="c4 lst-kix_8joc3jsoa7de-1 start"><li class="c3 li-bullet-0"><span class="c2">For PK, collect blood at predetermined time points.<br></span></li><li class="c3 li-bullet-0"><span class="c2">For mice, limit to ~ &lt;10% total blood volume over the entire study.<br></span></li><li class="c3 li-bullet-0"><span class="c2">Use plasma/serum for radioactivity or fluorescence measurement, exosome marker ELISA, or cargo detection.<br></span></li></ul><ol class="c4 lst-kix_8joc3jsoa7de-0" start="2"><li class="c0 li-bullet-0"><span class="c2">Tissue Harvest<br></span></li></ol><ul class="c4 lst-kix_8joc3jsoa7de-1 start"><li class="c3 li-bullet-0"><span class="c2">Conduct terminal sampling at specific time points (matching blood sampling schedule).<br></span></li><li class="c3 li-bullet-0"><span class="c2">Tissues of interest: liver, kidney, spleen, lung, heart, brain (if blood-brain barrier penetration is relevant), and target tissues relevant to the therapeutic indication.<br></span></li><li class="c3 li-bullet-0"><span class="c2">Weigh each organ/tissue sample immediately.<br></span></li></ul><ol class="c4 lst-kix_8joc3jsoa7de-0" start="3"><li class="c0 li-bullet-0"><span class="c2">Excretion Samples (optional)<br></span></li></ol><ul class="c4 lst-kix_8joc3jsoa7de-1 start"><li class="c3 li-bullet-0"><span class="c2">Collect urine and feces in metabolic cages, especially if metabolism/excretion route is under investigation.<br></span></li></ul><hr><p class="c9 c11"><span class="c2"></span></p><h2 class="c19" id="h.fog1ugm7d4bp"><span class="c20">6. Analytical Methods</span></h2><h3 class="c12" id="h.hex2625elq8m"><span class="c8">6.1 Radiolabel Quantification</span></h3><ul class="c4 lst-kix_8r3w2rcyu87a-0 start"><li class="c0 li-bullet-0"><span class="c2">Gamma Counter (for 125I, 111In, etc.)<br></span></li></ul><ul class="c4 lst-kix_8r3w2rcyu87a-1 start"><li class="c3 li-bullet-0"><span class="c2">Measure radioactivity in plasma and homogenized tissue samples.<br></span></li><li class="c3 li-bullet-0"><span class="c2">Compare counts to a standard curve to convert to activity (&micro;Ci, Bq).<br></span></li><li class="c3 li-bullet-0"><span class="c2">Express results as % injected dose per gram tissue (%ID/g) or per organ.<br></span></li></ul><ul class="c4 lst-kix_8r3w2rcyu87a-0"><li class="c0 li-bullet-0"><span class="c2">HPLC or TLC (if needed)<br></span></li></ul><ul class="c4 lst-kix_8r3w2rcyu87a-1 start"><li class="c3 li-bullet-0"><span class="c2">Confirm stability of radiolabeled exosomes (detect free isotope vs. intact exosome-bound isotope).<br></span></li></ul><h3 class="c12" id="h.7k0bmkqyfj7b"><span class="c8">6.2 Fluorescence Imaging (if used)</span></h3><ul class="c4 lst-kix_8i9khg2xgulr-0 start"><li class="c0 li-bullet-0"><span class="c2">In Vivo Imaging<br></span></li></ul><ul class="c4 lst-kix_8i9khg2xgulr-1 start"><li class="c3 li-bullet-0"><span class="c2">Near-infrared fluorescence imaging (e.g., IVIS) at select time points.<br></span></li><li class="c3 li-bullet-0"><span class="c2">Estimate fluorescence intensity in regions of interest.<br></span></li></ul><ul class="c4 lst-kix_8i9khg2xgulr-0"><li class="c0 li-bullet-0"><span class="c2">Ex Vivo Imaging<br></span></li></ul><ul class="c4 lst-kix_8i9khg2xgulr-1 start"><li class="c3 li-bullet-0"><span class="c2">Harvested organs can be imaged using an IVIS or similar imaging system.<br></span></li><li class="c3 li-bullet-0"><span class="c2">Use tissue homogenates or cryosections for histological imaging (microscopy).<br></span></li></ul><h3 class="c12" id="h.tp5wv11mmz3m"><span class="c8">6.3 Pharmacokinetic Analysis</span></h3><ul class="c4 lst-kix_2fszuplxt2n9-0 start"><li class="c0 li-bullet-0"><span class="c2">Non-Compartmental Analysis (NCA)<br></span></li></ul><ul class="c4 lst-kix_2fszuplxt2n9-1 start"><li class="c3 li-bullet-0"><span class="c2">Calculate half-life (t&frac12;), maximum concentration (Cmax), time to maximum concentration (Tmax), area under the curve (AUC), clearance (CL), and volume of distribution (Vd).<br></span></li></ul><ul class="c4 lst-kix_2fszuplxt2n9-0"><li class="c0 li-bullet-0"><span class="c2">Compartmental Modeling<br></span></li></ul><ul class="c4 lst-kix_2fszuplxt2n9-1 start"><li class="c3 li-bullet-0"><span class="c2">If warranted, fit a one- or two-compartment model.<br></span></li><li class="c3 li-bullet-0"><span class="c2">Useful for comparing across different exosome formulations or routes of administration.<br></span></li></ul><h3 class="c12" id="h.tts1kp5jqdsg"><span class="c8">6.4 Pharmacodynamic Assessments</span></h3><ul class="c4 lst-kix_84yjavmdod6-0 start"><li class="c0 li-bullet-0"><span class="c2">Target Engagement/Biomarker Analysis<br></span></li></ul><ul class="c4 lst-kix_84yjavmdod6-1 start"><li class="c3 li-bullet-0"><span class="c2">If exosomes carry a specific cargo (e.g., siRNA, miRNA, protein), measure gene expression or protein levels in targeted tissues.<br></span></li><li class="c3 li-bullet-0"><span class="c2">PD endpoints can include functional assays (e.g., change in tumor volume, inflammation markers, enzyme activity, etc.).<br></span></li></ul><hr><p class="c9 c11"><span class="c2"></span></p><h2 class="c19" id="h.ooe5bxj189j"><span class="c20">7. Data Interpretation and Reporting</span></h2><ol class="c4 lst-kix_dgmlm9mxaa4-0 start" start="1"><li class="c0 li-bullet-0"><span class="c2">Quantification of Distribution<br></span></li></ol><ul class="c4 lst-kix_dgmlm9mxaa4-1 start"><li class="c3 li-bullet-0"><span class="c2">Express tissue uptake as % of injected dose per gram tissue (%ID/g).<br></span></li><li class="c3 li-bullet-0"><span class="c2">Generate biodistribution profiles over time to identify major organs of accumulation and clearance.<br></span></li></ul><ol class="c4 lst-kix_dgmlm9mxaa4-0" start="2"><li class="c0 li-bullet-0"><span class="c2">PK Parameters<br></span></li></ol><ul class="c4 lst-kix_dgmlm9mxaa4-1 start"><li class="c3 li-bullet-0"><span class="c2">Summarize key PK parameters (half-life, clearance, volume of distribution).<br></span></li><li class="c3 li-bullet-0"><span class="c2">Compare across different exosome preparations, doses, or routes if applicable.<br></span></li></ul><ol class="c4 lst-kix_dgmlm9mxaa4-0" start="3"><li class="c0 li-bullet-0"><span class="c2">PD Correlation<br></span></li></ol><ul class="c4 lst-kix_dgmlm9mxaa4-1 start"><li class="c3 li-bullet-0"><span class="c2">Correlate tissue drug levels (i.e., exosome levels) with pharmacodynamic effects or biomarkers.<br></span></li><li class="c3 li-bullet-0"><span class="c2">Demonstrate whether target engagement is linked to exosome exposure.<br></span></li></ul><ol class="c4 lst-kix_dgmlm9mxaa4-0" start="4"><li class="c0 li-bullet-0"><span class="c2">Safety Observations<br></span></li></ol><ul class="c4 lst-kix_dgmlm9mxaa4-1 start"><li class="c3 li-bullet-0"><span class="c2">Document any adverse effects, changes in body weight, or abnormal clinical signs.<br></span></li><li class="c3 li-bullet-0"><span class="c2">Optionally measure cytokines for immunogenicity assessment.<br></span></li></ul><hr><p class="c9 c11"><span class="c2"></span></p><h2 class="c19" id="h.7s9v2omk78zq"><span class="c20">8. Representative Protocol Outline</span></h2><p class="c9"><span class="c2">Below is a concise example protocol for an IV radiolabeled exosome study in mice:</span></p><ol class="c4 lst-kix_tkg3l6cy3bnc-0 start" start="1"><li class="c0 li-bullet-0"><span class="c2">Exosome Labeling<br></span></li></ol><ul class="c4 lst-kix_tkg3l6cy3bnc-1 start"><li class="c3 li-bullet-0"><span class="c2">Isolate ~1&times;10^12 exosomes from cell culture supernatant via ultracentrifugation/SEC.<br></span></li><li class="c3 li-bullet-0"><span class="c2">Radiolabel with 125I using a standard iodination kit; verify labeling efficiency (&gt;80%).<br></span></li><li class="c3 li-bullet-0"><span class="c2">Optional step: Check for label leakage or dissociation by incubating labeled exosomes in serum (37&deg;C) for 24 hours.<br></span></li></ul><ol class="c4 lst-kix_tkg3l6cy3bnc-0" start="2"><li class="c0 li-bullet-0"><span class="c2">Dosing<br></span></li></ol><ul class="c4 lst-kix_tkg3l6cy3bnc-1 start"><li class="c3 li-bullet-0"><span class="c2">Randomize 30 mice into 6 time-point groups (n=5 each): 0.25, 1, 4, 12, 24, 48 hours.<br></span></li><li class="c3 li-bullet-0"><span class="c2">Inject ~1&times;10^10 exosomes (~10&ndash;20 &micro;Ci per mouse) via the tail vein in a 100 &micro;L volume.<br></span></li></ul><ol class="c4 lst-kix_tkg3l6cy3bnc-0" start="3"><li class="c0 li-bullet-0"><span class="c2">Blood Sampling<br></span></li></ol><ul class="c4 lst-kix_tkg3l6cy3bnc-1 start"><li class="c3 li-bullet-0"><span class="c2">For the 0.25, 1, 4, 12, 24, 48-hour groups, collect ~100&ndash;200 &micro;L blood via submandibular or retro-orbital sampling at the designated time.<br></span></li><li class="c3 li-bullet-0"><span class="c2">Process plasma and store at -80&deg;C until analysis.<br></span></li></ul><ol class="c4 lst-kix_tkg3l6cy3bnc-0" start="4"><li class="c0 li-bullet-0"><span class="c2">Tissue Collection (Terminal)<br></span></li></ol><ul class="c4 lst-kix_tkg3l6cy3bnc-1 start"><li class="c3 li-bullet-0"><span class="c2">At each time point, euthanize mice from that group.<br></span></li><li class="c3 li-bullet-0"><span class="c2">Harvest organs (liver, spleen, kidney, lung, heart, brain, etc.).<br></span></li><li class="c3 li-bullet-0"><span class="c2">Weigh organs, record mass, and measure radioactivity in a gamma counter.<br></span></li></ul><ol class="c4 lst-kix_tkg3l6cy3bnc-0" start="5"><li class="c0 li-bullet-0"><span class="c2">Data Analysis<br></span></li></ol><ul class="c4 lst-kix_tkg3l6cy3bnc-1 start"><li class="c3 li-bullet-0"><span class="c2">Calculate %ID/g for each organ.<br></span></li><li class="c3 li-bullet-0"><span class="c2">Derive PK parameters from plasma radioactivity-time profiles.<br></span></li></ul><ol class="c4 lst-kix_tkg3l6cy3bnc-0" start="6"><li class="c0 li-bullet-0"><span class="c2">Optional PD Measurement<br></span></li></ol><ul class="c4 lst-kix_tkg3l6cy3bnc-1 start"><li class="c3 li-bullet-0"><span class="c2">If exosomes carry a functional cargo (e.g., anti-inflammatory miRNA), measure relevant cytokines in plasma or organ homogenates.<br></span></li></ul><hr><p class="c9 c11"><span class="c2"></span></p><h2 class="c19" id="h.n4gyo2ft87fe"><span class="c20">9. Key Considerations and Recommendations</span></h2><ol class="c4 lst-kix_esoh3w8sgdnn-0 start" start="1"><li class="c0 li-bullet-0"><span class="c2">Integrity of Labeled Exosomes<br></span></li></ol><ul class="c4 lst-kix_esoh3w8sgdnn-1 start"><li class="c3 li-bullet-0"><span class="c2">It is crucial to confirm that labeling does not disrupt exosome surface proteins or cargo.<br></span></li><li class="c3 li-bullet-0"><span class="c2">Perform post-labeling characterization (size, morphology, marker expression).<br></span></li></ul><ol class="c4 lst-kix_esoh3w8sgdnn-0" start="2"><li class="c0 li-bullet-0"><span class="c2">Minimizing Free Label<br></span></li></ol><ul class="c4 lst-kix_esoh3w8sgdnn-1 start"><li class="c3 li-bullet-0"><span class="c2">Thoroughly separate free isotope or dye from labeled exosomes (e.g., size exclusion spin columns).<br></span></li><li class="c3 li-bullet-0"><span class="c2">Confirm labeling stability during pilot in vitro incubations in serum.<br></span></li></ul><ol class="c4 lst-kix_esoh3w8sgdnn-0" start="3"><li class="c0 li-bullet-0"><span class="c2">Route of Administration Impacts ADME<br></span></li></ol><ul class="c4 lst-kix_esoh3w8sgdnn-1 start"><li class="c3 li-bullet-0"><span class="c2">IV injection typically leads to rapid clearance via MPS organs (liver and spleen).<br></span></li><li class="c3 li-bullet-0"><span class="c2">Other routes (e.g., SC or IP) may have delayed or incomplete absorption.<br></span></li></ul><ol class="c4 lst-kix_esoh3w8sgdnn-0" start="4"><li class="c0 li-bullet-0"><span class="c2">Ensuring Adequate Controls<br></span></li></ol><ul class="c4 lst-kix_esoh3w8sgdnn-1 start"><li class="c3 li-bullet-0"><span class="c2">Always include a vehicle-control or label-only group to account for background signal.<br></span></li><li class="c3 li-bullet-0"><span class="c2">Include non-specific exosomes (e.g., from a non-relevant source or heat-inactivated exosomes) to distinguish active targeting.<br></span></li></ul><ol class="c4 lst-kix_esoh3w8sgdnn-0" start="5"><li class="c0 li-bullet-0"><span class="c2">Regulatory and Ethical Compliance<br></span></li></ol><ul class="c4 lst-kix_esoh3w8sgdnn-1 start"><li class="c3 li-bullet-0"><span class="c2">Follow institutional guidelines on the use of radioisotopes and animal welfare.<br></span></li><li class="c3 li-bullet-0"><span class="c2">Prepare an animal use protocol and biosafety measures for radioactive handling.<br></span></li></ul><hr><p class="c9 c11"><span class="c2"></span></p><h2 class="c19" id="h.enbbqpyjnt0l"><span class="c20">Conclusion</span></h2><p class="c9"><span class="c2">While the exact methods may vary depending on specific scientific and regulatory needs, radiolabeling combined with tissue-level quantitation (via gamma counter or SPECT/PET imaging) is generally regarded as the &ldquo;gold standard&rdquo; method for exosome ADME and PK characterization in preclinical models. When paired with fluorescent or bioluminescent imaging, researchers can gain both quantitative and visual insights into how exosomes behave in vivo. By adhering to the outlined protocols&mdash;thorough exosome characterization, careful labeling, strategic study design, and robust analytical methods&mdash;investigators can reliably define the pharmacokinetic profiles, biodistribution, and pharmacodynamic impacts of exosome-based therapies in animal studies.</span></p><p class="c9 c11"><span class="c2"></span></p><p class="c9"><span class="c2">EXAMPLE 3: </span></p><p class="c15"><span class="c2">Prompt:</span></p><p class="c9"><span class="c2">Create a detailed method development plan for particle size measurement of a sterile non pyrogenic lung surfactant suspended in 0.9% saline SFRI 1.it has phospholipids including phosphatydylcholine and desaturated phosphatidylcholine and proteins B and C. Composition is 99%lipids DPCC with1% protein. </span></p><p class="c9"><span class="c2">Include all the parameters we can optimized to get consistent particle size measurement. Problem being that the sample right now in 0.45% saline does not give consistent data and dominationreduces the size of particles. The labs running have about a 1000 and 600 sizes but these are very different. </span></p><p class="c15"><span class="c2">Response:</span></p><p class="c9"><span class="c2">To develop an accurate and reproducible particle size measurement method for the lung surfactant, could you please clarify a few points:</span></p><ol class="c4 lst-kix_3qoh0wbylc0j-0 start" start="1"><li class="c0 li-bullet-0"><span class="c2">What measurement technique(s) do you intend to use (e.g., dynamic light scattering, laser diffraction, nanoparticle tracking analysis, microscopy)?<br></span></li><li class="c0 li-bullet-0"><span class="c2">What particle size range are you primarily interested in measuring (nanometer or micrometer range)?<br></span></li><li class="c0 li-bullet-0"><span class="c2">Are you able to modify the sample preparation process (e.g., sonication, dilution buffer, filtration)?<br></span></li><li class="c0 li-bullet-0"><span class="c2">Is this for method development under GMP conditions or for research purposes only?<br></span></li></ol><p class="c9"><span class="c2">With this information, I can tailor the method development plan to your context more effectively.</span></p><p class="c9"><span class="c2">Prompt: </span></p><p class="c9"><span class="c2">Zetasizer </span></p><p class="c9"><span class="c2">600-1200nm range</span></p><p class="c9"><span class="c2">Yes give justification, we don&rsquo;t want to alter too much since this will be directly injected into patients</span></p><p class="c9"><span class="c2">GMP</span></p><p class="c15"><span class="c2">Response:</span></p><h1 class="c10" id="h.jmhiftfys69r"><span class="c17">Method Development Plan for DLS Particle Size Analysis of Lung Surfactant Suspension</span></h1><h2 class="c19" id="h.g20kvz18nlbw"><span class="c20">Background and Problem Statement</span></h2><p class="c9"><span class="c2">A sterile, non-pyrogenic lung surfactant suspension (99% phospholipids, primarily DPPC, and 1% surfactant proteins SP-B/SP-C in 0.9% saline) is intended for direct intratracheal injection. Accurate particle size characterization by Dynamic Light Scattering (DLS, e.g. Malvern Zetasizer) is critical for quality control and consistency. However, current measurements show significant variability: some laboratories report mean particle sizes around ~1000 nm while others obtain ~600 nm. There is evidence that sample handling and dilution protocols are causing artificial size reduction, likely by altering the native aggregates. The goal is to develop a GMP-compliant, robust method to measure particle size consistently, with minimal perturbation of the sample.</span></p><p class="c9"><span class="c2">Objectives: The method development will focus on:</span></p><ul class="c4 lst-kix_h8tfxvbo4u2a-0 start"><li class="c0 li-bullet-0"><span class="c2">Minimal Sample Alteration: Avoid any preparation steps that could disrupt the surfactant&rsquo;s native aggregation state. The product is used as-is for injection, so the measurement procedure must preserve its original characteristics.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Justified, GMP-Compatible Protocol: Each sample preparation and measurement step will be grounded in scientific rationale and compliant with Good Manufacturing Practice (GMP) expectations for consistency and traceability.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Optimized DLS Parameters: Identify and standardize key parameters (dilution medium, degree of dilution, mixing method, sonication, temperature, DLS measurement angle, count rate, refractive index settings, etc.) to improve inter-lab reproducibility.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Preventing Artifacts: Establish practices to avoid spurious size reductions from osmotic shocks, surfactant destabilization, over-agitation, or instrument-related biases.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Controls and Robustness: Include appropriate controls (instrument calibration standards and procedural controls) and perform robustness testing to ensure the method yields consistent results across different analysts, days, and laboratories.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Standardized SOP: Produce a clear Standard Operating Procedure for sample handling, loading, and measurement, ensuring all labs follow the same protocol. This will include a summary of optimized parameters and acceptance criteria.<br></span></li></ul><h2 class="c19" id="h.us42p84u6i8r"><span class="c20">Step 1: Sample Handling and Preparation (Minimize Alteration)</span></h2><p class="c9"><span class="c2">1.1 Use Original Sample with Minimal Dilution: Whenever possible, measure the suspension in its undiluted or minimally diluted form to reflect the product &ldquo;as injected.&rdquo; DLS requires that the sample not be too turbid, but the priority is to avoid excessive dilution that could disrupt aggregate equilibrium. If dilution is necessary to obtain a readable sample (to avoid multiple scattering), dilute with the same 0.9% saline that the surfactant is formulated in. Matching the diluent to the formulation preserves the ionic strength and surfactant concentration environment, preventing osmotic stress on the lipid/protein structures . Using an identical medium (same pH, salt, and additives) avoids perturbing the surfactant&rsquo;s colloidal stability or causing components to desorb from particle surfaces . Do not use water or other diluents that would change tonicity; a drop in salinity can cause water influx into vesicles or dissociation of lipid-protein complexes, artificially altering size.</span></p><p class="c9"><span class="c2">1.2 Determine the Optimal Dilution Factor: As part of method development, perform a dilution series with isotonic saline to identify the minimal dilution that yields reliable DLS readings without changing the particle size. For example, measure the Z-average size of an undiluted sample, then dilute incrementally (e.g., 1:1, 1:2, 1:5 dilutions) and measure each. If the undiluted sample is too opaque for DLS (low transmission or detector overload), start from a slight dilution. Monitor the mean size as a function of dilution: initially, size might decrease if very high concentration caused multiple scattering errors; but beyond a certain point, further dilution should not significantly change the measured size. Choose a dilution factor in this plateau region. This ensures that the measurement is in the linear range of the DLS (no multiple scattering) while still representing the true aggregate size. For example, if diluting beyond, say, 5x starts to lower the measured size, use a dilution at or below that level. This approach addresses the current discrepancy by standardizing how much dilution is applied across labs. It follows published guidance to test the effect of concentration on DLS results and ensure the chosen sample concentration does not bias the size . All dilution steps should be documented and justified in the method.</span></p><p class="c9"><span class="c2">1.3 Gentle Mixing, No Vigorously Agitation: Do not shake or vortex the surfactant suspension. Vigorous agitation can create foam and shear forces that break apart lipid aggregates. Instead, use gentle inversion or swirling to homogenize the sample prior to sampling . For instance, slowly invert the sealed container 5&ndash;10 times to re-suspend any settled material. This practice is consistent with manufacturer instructions for similar surfactant products, which state to &ldquo;swirl the vial gently (DO NOT SHAKE) to redisperse&rdquo; settled material . Avoid generating bubbles; any foam that does form is inherent to the product and should be allowed to dissipate briefly . Do not sonicate the sample unless absolutely necessary for dispersion &ndash; lung surfactant formulations are designed to be used without sonication (e.g. &ldquo;does not require reconstitution or sonication before use&rdquo; ). Sonication (especially probe sonicators or prolonged bath sonication) can disrupt the multilamellar vesicles into smaller unilamellar vesicles, drastically reducing the particle size. In a GMP context, introducing such a step would alter the product&rsquo;s nature and invalidate the measurement of the original form. Thus, the method will avoid any high-energy input like ultrasonication that could artificially reduce particle size. If a mild dispersion step is needed (for example, if the product forms visible clumps during storage), a short, low-intensity bath sonication (e.g., a few seconds) might be evaluated during development to see if it can improve repeatability without changing the size distribution. This would be carefully studied and, if adopted, well-controlled (e.g., fixed 5-second sonication in a water bath at room temperature) and justified with data. However, the default assumption is no sonication to keep the procedure simple and true to the product&rsquo;s intended use.</span></p><p class="c9"><span class="c2">1.4 No Filtration of the Sample: Do not filter the surfactant suspension before measurement. Filtering can remove larger vesicles or protein aggregates that are part of the product, thus skewing the particle size distribution lower. The product is meant to be delivered with its full complement of particle sizes (up to ~1200 nm), and removing any fraction would undermine the representativeness of the test. Indeed, the product literature for lung surfactants explicitly warns: &ldquo;Do not filter SURVANTA&rdquo; , since filtration could strip out essential components or change concentration (due to adsorption on the filter). In method development, if particulate contaminants (like dust) are a concern, the solution is to improve sample handling in a clean environment (see 1.5) rather than filtering the sample itself. Exception: The diluent (saline) used for any dilution should be filtered (e.g. 0.1 &micro;m or 0.2 &micro;m) or particle-free, so that it does not introduce any dust or foreign particles . But the sample aliquot containing the surfactant should not pass through filters.</span></p><p class="c9"><span class="c2">1.5 Work Cleanly to Prevent Contamination: Even without filtration, we must ensure that extrinsic particles (dust, fibers) don&rsquo;t contaminate the sample and produce spurious DLS readings (dust can produce an apparent large size peak). Perform sample preparation in a clean environment &ndash; ideally under a laminar flow hood or similarly low-dust area . Use clean, particle-free vials and pipette tips. All glassware or syringes that contact the sample should be pre-rinsed with filtered saline to remove any particles . In GMP labs, procedures should require verification of cleanliness (for example, flushing a cuvette with filtered diluent and ensuring the DLS shows no particles in a blank run). Maintaining cleanliness is critical and should be documented (e.g., use of lint-free wipes, gloves, and proper cleaning logs) as part of GMP compliance.</span></p><p class="c9"><span class="c2">1.6 Sample Volume and Replicates: Withdraw the sample using a sterile syringe or pipette (since the product is sterile, maintain aseptic technique during sampling to avoid microbial contamination &ndash; while this is a characterization test, GMP still requires handling samples in a way that doesn&rsquo;t compromise them, especially if retains are kept). Draw enough volume to fill the DLS cuvette properly (typically ~1 mL for a standard cuvette) and to perform replicate measurements. It&rsquo;s advisable to prepare multiple aliquots (e.g., three separate cuvettes) for replicate analysis rather than reusing the exact same aliquot repeatedly, to gauge variability and avoid issues like sedimentation or evaporation affecting a single sample over time. Each aliquot should be prepared in identical fashion (same gentle mix, same dilution if any, etc.). Consistency in sample handling across replicates and labs will be documented in the SOP to reduce inter-operator variability.</span></p><h2 class="c19" id="h.sdmxowne14li"><span class="c20">Step 2: Instrument Setup and Measurement Parameters</span></h2><p class="c9"><span class="c2">2.1 Instrument and Cell Selection: Use a DLS instrument configured for submicron particle sizing, such as Malvern Zetasizer (Nano series or Advance series), which covers ~0.3 nm to 10 &micro;m size range. Ensure the instrument is qualified (IQ/OQ) and calibrated per GMP requirements. The optical configuration should preferably be Non-Invasive Backscatter (NIBS) at ~173&deg; (or similar high-angle backscatter) rather than the traditional 90&deg; angle. Backscatter detection allows analysis of more turbid, concentrated suspensions with less multiple scattering, which is ideal for this lipid suspension . It also generally provides a higher upper size limit and better sensitivity for large particles without needing as much dilution. If the available instrument is a 90&deg; scattering system, it can still be used, but extra care must be taken to sufficiently dilute the sample to avoid attenuation of the beam . (All labs should use the same model or equivalent instrument if possible, to minimize inter-lab differences. If different models are used, part of the method transfer will involve ensuring that results are harmonized between 173&deg; and 90&deg; systems &ndash; in principle they should agree on the true size if measurement conditions are optimized .)</span></p><p class="c9"><span class="c2">Select an appropriate cuvette or cell for measurement. For this size range (~600&ndash;1200 nm, milky suspension), standard disposable polystyrene or glass cuvettes (10 mm path length) are suitable. Disposable polystyrene cuvettes are convenient and avoid cross-contamination (one-time use). If using glass/quartz cuvettes, they must be scrupulously cleaned and particle-free for each run. Document the cuvette type in the method (all labs should use the same type to avoid differences in background scattering). The cuvette cap should be used to prevent dust ingress during measurement. Before loading the sample, inspect the cuvette for scratches or residue. Rinse it with filtered diluent if reusing, and perform a blank DLS run (diluent only) to ensure no spurious particles are detected (this can be part of system suitability).</span></p><p class="c9"><span class="c2">2.2 Temperature Control: Set the DLS measurement temperature to a controlled value and allow the instrument to equilibrate at that temperature. Temperature affects the viscosity of the dispersant and the diffusion rate of particles, thus impacting the calculated size. For consistency, we will perform measurements at 25&deg;C (room temperature) unless there is a specific reason to measure at 37&deg;C (near body temperature). 25&deg;C is convenient for lab work and instrument stability; it is safely below the phase transition temperature of DPPC (~41&deg;C) to avoid any abrupt lipid phase changes during measurement. At 25&deg;C, DPPC is in a gel/solid-ordered phase, but since the formulation includes surfactant proteins and possibly other lipids, it should still reflect the typical aggregate size. We will verify during development if measuring at 37&deg;C (to mimic body conditions) causes any systematic difference in size. If so, we may choose 37&deg;C for final method to reflect in-use conditions. In either case, tightly control the temperature: the instrument&rsquo;s Peltier or water bath system should keep &plusmn;0.1&deg;C stability. The dispersant viscosity and refractive index values in the software must match the chosen temperature (e.g., viscosity of 0.9% saline at 25&deg;C ~0.9 cP, at 37&deg;C slightly lower &mdash; these values can be entered or selected in the software). All labs must use the same temperature setpoint. The SOP should instruct verifying the actual sample temperature (some DLS devices report the temperature or use an external thermometer in a dummy cell) to ensure equilibration before measurement.</span></p><p class="c9"><span class="c2">2.3 Measurement Settings &ndash; Duration, Runs, and Angles: Configure the DLS measurement method with fixed run parameters for consistency:</span></p><ul class="c4 lst-kix_hlp788b4qti1-0 start"><li class="c0 li-bullet-0"><span class="c2">Use multiple acquisitions per sample: e.g., 3 runs of 60 seconds each (or 10 runs of 10 seconds, etc., as appropriate to gather sufficient data). Multiple shorter runs help identify if results are stable or if any transient events (like dust) occurred in one run. We will optimize the total acquisition time during development &ndash; enough to get a good correlation function, but not so long that drift (e.g., settling) occurs. For instance, if 3 x 60s gives a stable result, that might be ideal. This also ties into robustness: the method should specify a certain analysis time; as a robustness test we can check &plusmn;10% of that time to confirm it doesn&rsquo;t affect the result .<br></span></li><li class="c0 li-bullet-0"><span class="c2">The scattering angle for measurement will be fixed (as discussed, preferably backscatter ~173&deg; on appropriate instruments). If an instrument offers multi-angle or MADLS (multi-angle DLS) capability (like Malvern Zetasizer Ultra can do backscatter, side-scatter, forward scatter combined), the method can either stick to the single-angle analysis for simplicity or leverage multi-angle to verify size. In a multi-angle measurement, ensure that the primary reported result is comparable across angles, confirming no angle-dependent artifacts. For simplicity and cross-lab uniformity, we will specify one mode (e.g., standard backscatter DLS) as the official measurement.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Measurement position in cuvette: Many modern DLS instruments automatically adjust the measurement position within the cuvette (especially in backscatter mode) to minimize multiple scattering. Ensure this feature (like Malvern&rsquo;s position optimization) is enabled or consistently set. This ensures each lab&rsquo;s instrument is actually measuring at a similar depth in the cuvette, avoiding differences due to sedimentation at the bottom or meniscus effects at the top.<br></span></li><li class="c0 li-bullet-0"><span class="c2">If applicable, set the attenuator to automatic so the instrument can adjust the laser power or detection attenuation to get a proper count rate. The SOP should note if manual attenuation is needed for any reason (uncommon in modern systems, but if so, provide the attenuator setting that yields the correct count rate as determined during development).<br></span></li></ul><p class="c9"><span class="c2">2.4 Monitor Count Rate and Intensity: When the sample is loaded and the measurement started, observe the scattering count rate (in kilo counts per second, kcps) and the instrument&rsquo;s attenuation setting. A stable count rate within the instrument&rsquo;s recommended range is crucial for reproducibility. For example, Malvern Zetasizer typically performs well when count rates are on the order of 100&ndash;300 kcps (with attenuator adjusting as needed). A wildly fluctuating or extremely high count rate indicates issues (too concentrated, presence of dust spikes, etc.). The method will include criteria: e.g., &ldquo;Proceed with measurement if count rate is stable and below the overload threshold; if the count is saturating (attenuator at minimum and count &gt; e.g. 1,000 kcps), stop and dilute the sample slightly further with saline, mix gently, and reload.&rdquo; Conversely, if the count rate is very low (indicating too dilute or too little sample in the beam), we may need to concentrate the sample or check for issues like the beam not aligned. By specifying a target count rate window, we improve consistency across runs and labs. All labs should record the count rate and attenuator level as part of the raw data (this can be included in the report printout) to document that measurements were within the optimal range.</span></p><p class="c9"><span class="c2">2.5 Data Analysis Settings (Refractive Index, Algorithms): Set the analysis model in the software to appropriate values for the sample:</span></p><ul class="c4 lst-kix_19nen5usjyzs-0 start"><li class="c0 li-bullet-0"><span class="c2">Refractive Index (RI) of particles: Input an RI value representative of the lung surfactant particles. DPPC and similar lipids have RI ~1.47&ndash;1.48 in solid phase ; since the suspension is mostly lipid with some protein, an RI ~1.45&ndash;1.47 is reasonable. We will use, for example, 1.47 (non-absorbing, absorption = 0.001) for the particle material. The dispersant (0.9% saline) RI is ~1.333 (close to water). These values are used by the software for Mie scattering calculations to convert the measured correlation into a size distribution (especially important for particles in the 600&ndash;1200 nm range, which are well outside the Rayleigh regime). Using the correct RI ensures that the intensity-to-volume conversion is accurate and comparable across labs. (All labs must use the same RI values in their analysis protocol.)<br></span></li><li class="c0 li-bullet-0"><span class="c2">Analysis model: Use the standard cumulant analysis to obtain the Z-average (intensity-weighted harmonic mean diameter) and the polydispersity index (PDI). The Z-average size is a robust measure for comparing lot-to-lot and lab-to-lab, and is recommended in standards (e.g., ISO and USP) for DLS results . We will focus on the intensity-weighted mean (Z-average) as the reportable size, as it is a single-value descriptor less sensitive to noise than trying to report multi-modal peaks . The PDI gives a sense of the width of the distribution. Expect that this surfactant suspension may show a relatively high PDI (possibly 0.4&ndash;0.7) since it&rsquo;s polydisperse; however, consistent PDI values across runs indicate stable technique.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Size distribution outputs: The instrument software can also display an intensity distribution and a volume distribution. The method can capture these for information, but comparison between labs should rely on the Z-average and PDI to avoid ambiguity. If needed for deeper analysis, volume distributions can be reported, but these should be generated using the same RI and analysis settings everywhere.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Run repetition and outlier rejection: Define how many replicate measurements (cuvettes) and how many runs per cuvette. For example, measure 3 independent aliquots, each with 3 runs. If one run exhibits an anomaly (e.g., very high PDI or a large second peak indicative of dust), the SOP could allow discarding that run and repeating it, with justification noted (GMP practice typically discourages arbitrarily excluding data; thus the criterion for outlier rejection must be predefined, such as &ldquo;if PDI &gt; 0.7 or correlation function not fitting, then that run is invalid and repeat the run&rdquo;). Ideally, design the measurement so that outliers are rare (by good preparation). In the method development phase, we will determine reasonable acceptance criteria for run quality (e.g., correlation function intercept, PDI cutoff, etc., based on instrument guidance).<br></span></li></ul><p class="c9"><span class="c2">All these instrument parameters (cell type, angle, temperature, run time, analysis settings) will be documented in detail in the final method SOP. Each lab following this setup should be able to obtain comparable raw data quality.</span></p><h2 class="c19" id="h.fiklwethz2uo"><span class="c20">Step 3: Avoiding Artifacts and Ensuring Sample Integrity During Measurement</span></h2><p class="c9"><span class="c2">This step addresses specific potential pitfalls that could lead to artificial size changes during sample preparation or the measurement process, and how to mitigate them:</span></p><p class="c9"><span class="c2">3.1 Prevent Osmotic or Chemical Shifts: As noted, always maintain the sample in an environment equivalent to its original formulation. Never mix the surfactant with pure water or solvents that could cause lipid reorganization. The saline dispersant ensures no osmotic shock &ndash; the lipid vesicles remain the same size. Because pulmonary surfactant lipids have an extremely low critical micelle concentration (~10^&minus;9 M) , diluting with water (lowering ionic strength and potentially raising the monomer chemical activity) might cause some vesicles to release monomers or form transient micelles, altering the size distribution. By using 0.9% NaCl at the same concentration, we keep the chemical activity of lipids and proteins the same, thereby preserving the equilibrium of aggregates . In GMP documentation, we justify that the diluent is the same as the formulation buffer to &ldquo;preserve the existing state of the surface&rdquo; of the particles .</span></p><p class="c9"><span class="c2">3.2 Avoid Surfactant Destabilization: Lung surfactant is a delicate proteo-lipid aggregate. Prolonged exposure to extremes of temperature, or physical stress, can destabilize it:</span></p><ul class="c4 lst-kix_bo4w5l7q0tq9-0 start"><li class="c0 li-bullet-0"><span class="c2">Temperature extremes: Do not let the sample freeze or overheat at any point. The product is usually stored refrigerated (2&ndash;8&deg;C) and then warmed to room temp before use . Follow the same in the lab: allow refrigerated samples to equilibrate to room temp (or the measurement temperature) gently &ndash; e.g., let the vial sit at room temp for ~20 minutes (per product guidelines) . Do not use microwave or hot water bath heating. Also, do not repeatedly warm and cool the sample; any sample removed for testing should not be returned to stock (to avoid any potential degradation &ndash; as per product info, surfactant vials shouldn&rsquo;t be cycled multiple times) . During measurement, ensure the sample stays at the set temperature (the DLS instrument should maintain it; if not, consider a cover to prevent evaporation cooling for longer runs). If measuring at 37&deg;C, be mindful that 37&deg;C is near DPPC&rsquo;s phase transition; slight fluctuations could cause changes in vesicle morphology. Thus, tight control and consistency of temperature is required.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Time and storage: Measure the sample as soon as possible after preparation. Do not let the diluted sample sit for hours before measurement, as slow changes (protein aggregation, microbial growth if not sterile environment, or particle sedimentation) could occur. In method validation, we will evaluate sample holding time stability: e.g., prepare sample and measure immediately vs after 1 hour at room temp to see if size changes. Ideally, the method will specify a maximum allowed time between sample preparation (dilution) and completion of measurement (perhaps no more than 30 minutes, unless stability data shows longer is fine).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Light exposure: While not usually a big factor for DLS in the visible range, it&rsquo;s good practice to protect the sample from direct sunlight or UV, as some proteins or lipids could degrade. The DLS instrument is enclosed, but when handling the sample, keep it in normal lighting conditions.<br></span></li></ul><p class="c9"><span class="c2">3.3 Gentle Sample Loading: When transferring the sample into the cuvette, do it carefully to avoid bubbles. Use a disposable plastic pipette or syringe to dispense into the cuvette tilted at ~45&deg;, allowing the liquid to flow down the side. This prevents bubble entrapment. If any bubbles are visible in the cuvette, gently tap the cuvette or use a clean pipette tip to remove them (for example, sliding a tip along the wall to capture a bubble). Bubbles can scatter light and be misinterpreted as &ldquo;giant&rdquo; particles, skewing the results or correlation function. The SOP will note to inspect the cuvette optically (maybe against a light) for bubbles or particles before inserting it into the instrument.</span></p><p class="c9"><span class="c2">3.4 Sedimentation considerations: Particles ~1 &micro;m can slowly sediment due to gravity if the density difference is significant. DPPC vesicles in saline might be slightly less dense than water (lipid ~0.95 g/mL plus some protein ~1.3 g/mL, net likely ~1.0 g/mL); sedimentation or creaming will be slow. However, during measurement (a few minutes) this is negligible. If a sample sits for a long time, larger aggregates could settle. Therefore, always remix right before measurement (gently invert the prepared cuvette or re-pipette it up and down) to ensure homogeneity. If using multiple cuvettes for replicates, remix the source sample each time before filling each cuvette. Do not assume the suspension stays uniform without agitation. By doing so, we avoid one run getting more of the larger sedimented fraction than another.</span></p><p class="c9"><span class="c2">3.5 Multiple Scattering and Concentration Artifacts: We touched on this in dilution optimization, but to reiterate: an overly concentrated sample can induce multiple scattering, where photons scatter off multiple particles before reaching the detector, distorting the size analysis. This often biases the measured size to smaller values (since multiple scattering accelerates the decay of the correlation function). Our method of minimal necessary dilution addresses this. As a rule of thumb, the sample in the cuvette should appear slightly translucent or opalescent, not completely opaque . The LS Instruments guideline suggests the suspension should be &ldquo;transparent or faintly opaque&rdquo; for reliable DLS . We will ensure the chosen concentration meets this visual criterion. In addition, we can use the instrument&rsquo;s attenuation as a guide: if the attenuator has to go to its maximum (meaning sample is very transparent) we might be too dilute (weak signal), and if it&rsquo;s at minimum (sample very opaque) we might be too concentrated. By fine-tuning to the middle ground, we avoid the extremes of multiple scattering or poor signal. During development, if we suspect multiple scattering still affecting results, one could consider mathematical corrections or using a dual scattering angle method (some advanced DLS instruments do two-angle measurement to correct for it). But ideally, our concentration choice avoids needing such corrections.</span></p><p class="c9"><span class="c2">3.6 Software and Calculation Artifacts: Ensure that the same software version or analysis algorithm is used across labs (differences in algorithm implementation can cause slight differences in PDI or peak finding). Ideally, use the instrument&rsquo;s established software (like Malvern ZS ver. X) with a locked analysis method file. All reported values should come from the same analysis parameters. The GMP documentation should include the software version and even the specific method file name or printout of settings, so each lab can import that and use identical settings.</span></p><p class="c9"><span class="c2">By proactively addressing these artifacts and enforcing consistent handling, we greatly reduce the risk that one lab&rsquo;s preparation technique (e.g., over-sonicating or using water to dilute) could cause a systematically smaller size reading. Each preventative measure above is justified to maintain the sample&rsquo;s original state or the integrity of the measurement, aligning with GMP principles of method consistency and accuracy.</span></p><h2 class="c19" id="h.kk0m63evjguh"><span class="c20">Step 4: Optimization of Key Parameters for Reproducibility</span></h2><p class="c9"><span class="c2">During method development, the following parameters will be systematically optimized. Each is tuned to improve reproducibility and consistency, with changes justified and documented. Below is the plan for each parameter:</span></p><p class="c9"><span class="c2">4.1 Dilution Medium and Factor: Parameter: Diluent composition and degree of dilution. Optimization approach: Prepare the sample in several dilution conditions (no dilution, 1:1, 1:5, 1:10 etc.) using 0.9% saline as diluent. Measure each condition in triplicate. Plot the Z-average size vs. dilution factor. We expect that at very high concentration (nearly no dilution) the apparent size might be unstable or slightly biased due to multiple scattering. At the other extreme, very high dilution might lead to minor size shifts if the sample structure starts to change (though since surfactant is above CMC, it should remain vesicular even when diluted, as long as we&rsquo;re not approaching infinite dilution). We will identify a safe operating concentration that yields maximum size but still good signal. Suppose undiluted gives a Z-average ~950 nm but with high PDI or poor repeatability; 1:2 dilution gives ~900 nm with lower PDI; 1:5 gives ~800 nm with stable PDI; and 1:10 gives ~790 nm, similar to 1:5. We might choose 1:5 as the standard dilution, reasoning that beyond that, size doesn&rsquo;t change significantly (790 vs 800 nm is small change &lt;2%). This ensures any lab dilutes to that same degree. We will verify that using exactly this dilution in a fresh sample yields the same ~800 nm result. GMP justification: selecting a fixed dilution factor (or range) eliminates subjective judgement by analysts and ensures all follow the same practice. The chosen diluent and factor will be documented in the SOP. Furthermore, as part of method validation, we will include this dilution step in reproducibility assessments (different analysts performing the dilution to see if results coincide). The diluent (saline) itself will be controlled (use the same lot if possible, or a specification like USP sterile saline, filtered) across labs.</span></p><p class="c9"><span class="c2">4.2 Mixing/Sonication: Parameter: How the sample is dispersed/mixed (manual inversion vs. sonication time). Optimization approach: Compare sizes from samples prepared under different mixing energies:</span></p><ul class="c4 lst-kix_etn6l1m36dy-0 start"><li class="c0 li-bullet-0"><span class="c2">Simply inverted (gentle mix),<br></span></li><li class="c0 li-bullet-0"><span class="c2">Mild bath sonication for 5 seconds,<br></span></li><li class="c0 li-bullet-0"><span class="c2">Bath sonication for 30 seconds.<br><br> We will check if mild sonication significantly changes the size distribution. It might reduce large aggregates slightly, leading to a lower Z-average (e.g., maybe from 900 nm to 700 nm). If so, that&rsquo;s evidence that sonication breaks the natural structure &ndash; a result we want to avoid. If gentle inversion gives the largest size (which presumably reflects intact aggregates) and sonication reduces it, we conclude no sonication in the method. Alternatively, if the product tends to form a few large clumps and a short sonication only improves the PDI without reducing the main population mean, we might incorporate a very controlled sonication. For example, if without sonication PDI is 0.7 with some runs unstable, and 5-second sonication yields PDI 0.5 and same Z-average, that could justify a quick sonication step. However, given the composition (natural surfactant), we expect any sonication to potentially disrupt lamellar structures. Our likely decision is that the optimal &ldquo;mixing&rdquo; is just gentle swirling, which we&rsquo;ll lock in the SOP. We&rsquo;ll document that excessive agitation was tested and shown to alter results (hence strictly avoided in routine practice). In GMP terms, this means the method is robust against operator inadvertently over-mixing because the SOP explicitly forbids it and instructs the proper technique (with perhaps a training video or demonstration to ensure uniform understanding of &ldquo;gentle&rdquo; mixing).<br></span></li></ul><p class="c9"><span class="c2">4.3 Temperature: Parameter: Measurement temperature setpoint. Optimization approach: Evaluate the DLS results at two temperatures, e.g., 25&deg;C vs 37&deg;C (since these are relevant temperatures for lab and usage). We will ensure the instrument is equilibrated and use identical samples. If the Z-average at 25&deg;C vs 37&deg;C differs significantly, that indicates temperature affects either the physical particle size (maybe slight swelling of vesicles or fluidity changes) or the measurement conditions (viscosity differences are accounted for, so any large change might be physical). For instance, perhaps at 37&deg;C the Z-average drops by 10% (maybe DPPC becomes more fluid so vesicles slightly shrink or pack tighter). If the difference is small (within run variability), we might choose 25&deg;C for convenience. If there is a difference, we might lean toward 37&deg;C as the more physiologically relevant condition and require all labs to measure at 37&deg;C. The key is not to allow each lab to pick arbitrarily; one temperature will be mandated. We&rsquo;ll also test intermediate stability: if measuring at 25&deg;C, what happens if lab ambient is 22&deg;C or 27&deg;C? The instrument should control it, but as a robustness test, maybe measure at 25&deg;C and 30&deg;C to ensure no drastic change (this covers any minor deviations). The final SOP will specify the temperature and require verification of the temperature each time (some labs include a step &ldquo;verify instrument at 25.0 &plusmn;0.2&deg;C before running sample&rdquo;).</span></p><p class="c9"><span class="c2">4.4 Measurement Angle and Optics: Parameter: Scattering angle (if variable) and path length. We largely addressed this by choosing backscatter instruments. However, as a development step, if both 90&deg; and 173&deg; instruments are in play, we will do a cross-comparison study: Measure the same sample on a 90&deg; DLS (requiring more dilution maybe) and on a 173&deg; DLS, to ensure the reported Z-averages converge. According to Malvern, provided the sample is not overly concentrated, the two configurations should give the same size . Our data should confirm that within experimental error. If a discrepancy is found, it may point to an issue (like the 90&deg; measurement still had multiple scattering or the algorithms differ). We will adjust the 90&deg; measurement conditions (further dilution, or use a low volume/high sensitivity cell) until it matches. This exercise will inform the final method if multiple instrument types must be supported. Ideally, though, all labs will use the same configuration (so this step is more of a verification of method transfer if needed). We will also optimize the number of sub-runs and duration (as mentioned, e.g., is 3x60s better than 1x180s? We&rsquo;ll choose the one that gives the lowest variation between runs).</span></p><p class="c9"><span class="c2">4.5 Count Rate (Concentration) Optimization: Although count rate itself is an outcome, we treat it as a parameter by adjusting concentration or instrument settings to hit the target range. During development, we note the count rate at the chosen dilution (from 4.1). If it&rsquo;s too low (leading to high uncertainty in correlation), we might slightly concentrate the sample (if possible) or increase run duration to compensate. If it&rsquo;s too high, we further dilute slightly. Essentially, we fine-tune the sample concentration to produce a count rate that yields a high-quality correlation function without artifacts. We may also explore using the instrument&rsquo;s attenuator manually &ndash; for instance, if using a different cell (like a flow cell with shorter path) we might need to manually set attenuation. The final method will likely simply rely on auto-attenuation, but will specify an acceptable count range (which we find empirically). For example, we determine that at a certain dilution, the count is ~500 kcps with attenuator 7. That gives good data. We ensure that any lab replicating this gets a similar count (they might have to tweak by tiny dilution adjustments if their laser power is slightly different, but this should be minimal if same model).</span></p><p class="c9"><span class="c2">4.6 Refractive Index Sensitivity: We will do a quick check on the effect of assumed particle refractive index on the result. As noted, the Z-average from DLS is actually independent of RI (since it comes from time correlation, not scattering intensity magnitude). However, the derived volume distribution can shift if RI is way off. We might test RI values 1.43 vs 1.50 in the software and see if the reported Z-average or PDI changes; it shouldn&rsquo;t affect Z-average (only how the peak is converted to volume or number). This is just to verify that our choice of 1.47 is suitable and doesn&rsquo;t introduce any bias in reporting. The outcome likely confirms that any RI ~1.45 &plusmn;0.05 yields the same Z-average, so we stick with 1.47 for consistency. This is documented so that labs don&rsquo;t worry if their software library uses 1.45 vs 1.47 (but we prefer to instruct them to manually input our chosen value to be exact).</span></p><p class="c9"><span class="c2">4.7 Polydispersity and Data Interpretation: Recognizing that the sample is polydisperse, we will also decide how to handle that in reporting. If the PDI is consistently high (say 0.5&ndash;0.6), the method might just report that value and consider it a characteristic of the product. However, we should verify that the DLS algorithm is not being pushed to its limits. If PDI &gt; 0.7, results can be unreliable. If we encounter such high PDI, we might consider whether the sample itself is too polydisperse to measure accurately by cumulants. One approach could be to use multi-modal analysis (e.g., regularization algorithms) to see if there are two distinct size populations (perhaps a small one and a large one). If there are, and different labs were focusing on different peaks, that could explain the disparity (one lab might have reported the large peak ~1000 nm, another the overall mean ~600 nm if there was a substantial small fraction). To ensure consistency, the method will explicitly state what value to report: e.g., &ldquo;Report the Z-average diameter (intensity-weighted mean) as the particle size. If multiple peaks are present in the distribution, do not report each peak &ndash; instead rely on the Z-average and PDI, unless an additional reporting metric is justified during development.&rdquo; We will confirm that all labs following the method indeed see similar distributions. If the distribution shape is complex, we might include a reference chromatogram (sizing graph) in the method as an example, so analysts know what to expect.</span></p><p class="c9"><span class="c2">By thoroughly optimizing these parameters, we build a robust method. Each optimization outcome will be captured in a development report with rationale. For example, we will note: &ldquo;Sonication for 30s was found to reduce particle size from 950 nm to 600 nm, so the method avoids sonication to preserve true particle size.&rdquo; Or: &ldquo;Dilution beyond 5x did not further change size, so 5x was selected to ensure low multiple scattering while maintaining sample integrity.&rdquo; These decisions and their justifications will be part of the method documentation, aligning with GMP expectations that method conditions are scientifically sound and not arbitrarily chosen.</span></p><h2 class="c19" id="h.fuy2siuvktzw"><span class="c20">Step 5: Controls, System Suitability, and Documentation in a GMP Environment</span></h2><p class="c9"><span class="c2">To ensure the method performs reliably in a GMP setting, we will establish controls and system suitability criteria, and enforce rigorous documentation practices:</span></p><p class="c9"><span class="c2">5.1 Instrument Calibration and Verification: The DLS instrument should be regularly calibrated or performance-verified with traceable size standards. USP &lt;729&gt; (globule size in lipid emulsions) and the emerging USP &lt;430&gt; chapters reference using polystyrene latex standards in the 100 nm &ndash; 400 nm range for system verification . Although those sizes are smaller than our surfactant particles, they test the instrument&rsquo;s fundamental capability. We will follow a calibration schedule (e.g., measure a 100 nm NIST-traceable latex bead standard weekly and confirm the result within &plusmn;10% or as per USP criteria ). Additionally, we could include a larger particle standard (not officially required, but for confidence) &ndash; for instance, a ~1 &micro;m polystyrene standard or a certified polydisperse standard if available. Entegris (Nicomp) suggests using at least one size standard to verify performance . In our SOP, before measuring the surfactant sample batch, the analyst must ensure the instrument passed its calibration or system suitability test: e.g., &ldquo;measured 100 nm standard on the day of use and obtained 95&ndash;105 nm result, PDI &lt; 0.1&rdquo;. This check is recorded in the log.</span></p><p class="c9"><span class="c2">5.2 Daily System Suitability &ndash; Control Sample: Implement a system suitability test specifically for this method. For example, keep a vial of a well-characterized internal reference sample &ndash; perhaps a previous batch of the lung surfactant (known to have a stable size ~800 nm) or a surrogate sample with similar size (like a polydisperse lipid emulsion). Each day of analysis, measure the control sample under the same conditions. Criteria could be that the Z-average of the control must fall within a control chart range (established during method validation), say 800 &plusmn; 100 nm (since variability for DLS can be ~10% for such large particles ). If the control is out of range or trending, investigate instrument or preparation issues before measuring actual samples. This control acts as a positive control for method performance &ndash; it can catch issues like a laser alignment drift or an operator error in prep.</span></p><p class="c9"><span class="c2">5.3 Duplicates and Trend Analysis: For each batch measurement, perform it in (at least) duplicate as described. The results should be averaged and reported. The individual values should be within a certain % difference (for example, require that two independent preps are within 10% of each other; if not, a third measurement is done and outlier investigation initiated). Over time, track the particle size results of multiple batches to ensure the method produces consistent data (aside from actual product differences). In a GMP lab, this trending can be part of ongoing method performance monitoring.</span></p><p class="c9"><span class="c2">5.4 Robustness Testing: As part of method validation, we will formally test robustness by deliberately varying certain parameters slightly and confirming the method&rsquo;s results are not significantly affected . This includes:</span></p><ul class="c4 lst-kix_8luyzq1gqi6-0 start"><li class="c0 li-bullet-0"><span class="c2">Analyst-to-Analyst variation: Two or more analysts will independently prepare and measure the same sample following the SOP. Their results should agree within the pre-defined variance (e.g., %RSD).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Instrument-to-Instrument (if applicable): If the method may be executed on more than one DLS instrument (either within the same lab or at a different site), perform a cross-instrument study on a homogenous sample set. This will validate that the method transfers well.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Small method parameter changes: e.g., perform one set of measurements at 24&deg;C and one at 26&deg;C if the method states 25&deg;C, to show minimal impact; use one extra inversion or one fewer inversion in mixing to see if size changes; use a sample that sat 15 min vs 30 min to see if any drift. Each of these tests should demonstrate that the method yields consistent results, or if a factor is found to be critical, we tighten control on it.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Ruggedness (inter-lab): This overlaps with Step 6 below, but essentially we plan an inter-laboratory comparison where the same samples are sent to multiple labs to ensure the method is reproducible outside the original development lab.<br></span></li></ul><p class="c9"><span class="c2">Results of these robustness tests will be included in the validation report. They provide assurance that the method &ldquo;as written&rdquo; is tolerant to normal variation and will control the product&rsquo;s quality appropriately .</span></p><p class="c9"><span class="c2">5.5 Documentation Practices: Under GMP, the method must be clearly documented and followed exactly. The final output will be a controlled document (SOP or Testing Method) that includes:</span></p><ul class="c4 lst-kix_e2h0cgtfdf3g-0 start"><li class="c0 li-bullet-0"><span class="c2">Principle and Scope: A description of DLS sizing and what it&rsquo;s intended to measure (particle size of surfactant suspension) .<br></span></li><li class="c0 li-bullet-0"><span class="c2">Equipment: Specification of the DLS instrument model, laser wavelength (e.g., 632.8 nm He-Ne or 660 nm diode), detector type (APD or PMT), and scattering angle used . Also list ancillary equipment like the type of cuvettes, pipettes, bath sonicator (if used), etc. All equipment must be qualified.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Reagents: Include the saline diluent (0.9% NaCl for injection, sterile, lot controlled), any standards used (with source and lot), and reference to their certification .<br></span></li><li class="c0 li-bullet-0"><span class="c2">Procedure: A step-by-step instruction of how to execute the measurement, from sample receipt to reporting . This will mirror the steps described in this plan (with the refined choices from optimization). It must be detailed enough that another trained operator can reproduce the test . We will include specific values: e.g., &ldquo;Dilute X &micro;L of sample with Y &micro;L of saline in a sterile tube to achieve a 1:5 dilution. Cap and invert gently 5 times. Immediately transfer to cuvette&hellip;&rdquo;.<br></span></li><li class="c0 li-bullet-0"><span class="c2">System Suitability: As noted, criteria that must be met before the sample analysis is considered valid (standard readings, control sample range) .<br></span></li><li class="c0 li-bullet-0"><span class="c2">Acceptance Criteria for the sample: If this test is for release or stability, define what size range is acceptable for the product (if known). For example, perhaps an upper limit like &ldquo;Z-average must be between 600 nm and 1200 nm&rdquo; might be a spec (just illustrative; actual spec setting is a separate process).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Recording Results: How results are reported (intensity z-average in nm, and PDI to two decimal places, for instance) . Also specify that raw correlation graphs or intensity distributions should be saved as part of the record for any future analysis.<br></span></li></ul><p class="c9"><span class="c2">Each lab will undergo training on the SOP, and an official version will be controlled in the quality management system. Any deviations must be documented via deviation reports as per GMP.</span></p><p class="c9"><span class="c2">5.6 Electronic Data Management: DLS instruments often have software that can export data. Ensure compliance with data integrity (21 CFR Part 11 if applicable) &ndash; the software should be validated or data exported to a secure system. Each measurement should have an audit trail or at least a record of date/time and operator. The method should specify how data is saved (e.g., PDF report plus raw intensity correlation file) and how long it&rsquo;s retained.</span></p><p class="c9"><span class="c2">By implementing these controls and documentation standards, we satisfy GMP requirements that the analytical method is suitable for its intended use and produces reliable, traceable results. Every measurement will have a clear record and justification, making it audit-friendly and scientifically sound.</span></p><h2 class="c19" id="h.xosw39le91de"><span class="c20">Step 6: Ensuring Consistency Across Laboratories (Method Transfer and Training)</span></h2><p class="c9"><span class="c2">To reconcile the historical inter-lab differences (~1000 nm vs ~600 nm results), a formal method transfer and alignment is necessary:</span></p><p class="c9"><span class="c2">6.1 Inter-Laboratory Method Transfer: Once the method is developed and validated in the primary lab, it will be transferred to other labs that perform the test. This involves providing the SOP and having the receiving lab execute the method on a common sample. We will send a homogeneous sample (same batch of surfactant) to all participating labs. Each lab will follow the SOP and report the results. We expect, with the carefully controlled method, that all labs&rsquo; results will agree within the method&rsquo;s precision limits (e.g., within ~10-15% given DLS variability ). Any significant deviations will be investigated:</span></p><ul class="c4 lst-kix_565pscofcl0-0 start"><li class="c0 li-bullet-0"><span class="c2">Check that each lab truly followed the procedure (sometimes subtle differences, like one lab using a different cuvette or not equilibrating temperature, can creep in).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Ensure each lab&rsquo;s instrument is performing (verification of calibration at each site).<br></span></li><li class="c0 li-bullet-0"><span class="c2">If a lab still consistently differs, we may do a cross-check where that lab&rsquo;s sample is measured back at the primary lab or vice versa to isolate whether it&rsquo;s an instrument issue or an execution issue.<br></span></li></ul><p class="c9"><span class="c2">This cross-lab study will be documented. Only after demonstrating that the method yields consistent results across labs will it be considered fully implemented. This addresses the initial problem: previously, labs may have been using different methods; now a single harmonized method will eliminate that variability source.</span></p><p class="c9"><span class="c2">6.2 Training and Competency: All analysts who will perform the test need to be trained on the new procedure. This training should emphasize the critical steps that affect results (for instance, how to properly mix without shaking, how to identify bubbles, etc.). Because some of these handling steps are somewhat technique-sensitive, a best practice is to have a training video or live demonstration and then have each analyst perform a run under supervision to verify they can do it reproducibly. Training records will be kept per GMP. A checklist could be used during training (did the analyst filter the diluent? handle cuvette properly? etc.). Emphasize that seemingly minor deviations (like using distilled water for dilution or sonicating &ldquo;to be sure it&rsquo;s mixed&rdquo;) are not allowed as they undermine the method &ndash; tie this to the development findings that justified each step.</span></p><p class="c9"><span class="c2">6.3 Ongoing Consistency Monitoring: After implementation, we will continue to compare results across labs periodically. For example, during split sample tests or proficiency testing: the same sample measured at two labs as part of stability or release might be compared. We could also introduce a periodic exchange: every year, send a blind sample to all labs to measure and confirm results align (a sort of proficiency exercise). Any drift or bias observed can trigger a root cause analysis (equipment re-calibration, retraining, etc.).</span></p><p class="c9"><span class="c2">6.4 Documentation Alignment: Ensure each lab&rsquo;s documentation (SOP version) is the same. In GMP, controlled documents should be identical in content, though possibly location-specific formatting. Any change to the method must go through change control and be rolled out to all labs simultaneously. This prevents one site from using an outdated version that might cause divergence.</span></p><p class="c9"><span class="c2">6.5 Addressing Historical Data: It may be worthwhile to explain internally why previous measurements differed: for instance, Lab A&rsquo;s method perhaps diluted with water, causing a ~600 nm reading, whereas Lab B measured neat and got ~1000 nm. With the new method (say it yields ~800 nm), everyone should understand that this is the true representative value under standardized conditions, and historical values were method-dependent. This understanding is important for regulatory filings or specifications &ndash; if specs were based on an old method, they may need adjustment to the new method&rsquo;s values. All labs and quality units should synchronize on this change.</span></p><p class="c9"><span class="c2">By thoroughly executing method transfer and training, we cement the consistency of this particle size test across all testing sites. The outcome will be that any given batch of the surfactant suspension will yield the same reported particle size regardless of which qualified lab measures it, within the expected measurement uncertainty. This fulfills the GMP mandate for reproducibility and control of analytical procedures.</span></p><h2 class="c19" id="h.vej1zm1oy0gx"><span class="c20">Step 7: Standard Operating Procedure (SOP) Summary for Particle Size Analysis</span></h2><p class="c9"><span class="c2">Below is a summarized step-by-step SOP, incorporating the optimized conditions and controls from the development above. This serves as a quick reference outline; the actual SOP would contain more details as discussed:</span></p><ol class="c4 lst-kix_rggdlfkt6mfq-0 start" start="1"><li class="c0 li-bullet-0"><span class="c2">Scope: Measurement of particle size (Z-average diameter and PDI) of lung surfactant suspension by DLS (Malvern Zetasizer), for QC release and stability testing.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Preparation of Diluent: Use 0.9% saline (sterile) for any dilution. Filter the saline through 0.1 &micro;m filter into a clean container. Record lot and filtration.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Sample Handling: Remove the surfactant sample vial from 2&ndash;8&deg;C storage and allow it to reach room temperature (~25&deg;C) for 20 minutes (DO NOT SHAKE) . If any settling observed, gently invert the vial 5-10 times to homogenize. Do not vortex or sonicate. Confirm the suspension looks uniform (milky white).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Sample Dilution: If required (based on method development, e.g., to 1:5), pipette X mL of surfactant into Y mL of filtered saline in a sterile tube to achieve the specified dilution. Cap and gently invert mix 5 times. (If no dilution required, proceed with neat sample).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Cuvette Loading: Transfer the prepared sample to a clean disposable cuvette, filling at least 1 mL (ensure beam path covered). Cap the cuvette. Inspect for bubbles; eliminate any present.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Instrument Setup: Set instrument to 25&deg;C (or specified temp) and allow to equilibrate. Select 173&deg; backscatter angle and automatic attenuation. Input dispersant properties (saline, viscosity ~0.9 cP, RI 1.333) and material RI ~1.47.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Measurement Execution: Insert cuvette and align in instrument. Measure the sample with the established measurement program (e.g., 3 runs &times; 60 sec each, or as specified). Ensure the correlation function is stable and the count rate is within range (monitor during the first few seconds).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Repeat: Remove the cuvette, gently invert once and reinsert for second run (if using same cuvette for multiple runs). Alternatively, measure separate prepared cuvettes sequentially. Obtain at least N = 3 measurements.<br></span></li><li class="c0 li-bullet-0"><span class="c2">System Suitability: Before or during the sample runs, measure the control standard per SOP: e.g., 100 nm latex &ndash; result must be within range; and/or run the internal control sample &ndash; result must be 800 &plusmn; xx nm. Document these results. If they fail, do not report sample results until resolved.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Data Analysis: After acquisition, record the Z-average diameter (intensity mean in nm) and PDI from the cumulant analysis. Also save the intensity distribution graph. If any run shows aberrant high PDI or multimodal behavior inconsistent with others, note it and repeat that run if allowed by criteria.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Result Reporting: Calculate the average Z-average from the 3 measurements (if they are consistent; if one is an outlier per criteria, exclude with justification). Report the mean diameter (nm) and PDI. Include an indication of result type (e.g., &ldquo;Intensity-weighted Z-average diameter&rdquo;).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Cleaning: Discard disposable cuvette (or clean cuvette thoroughly if reusable). Clean any spills of surfactant with appropriate solvent (the material is biological, handle with gloves).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Documentation: Fill out the Particle Size Test form with all details: sample ID, dilution factor, instrument ID, date, operator, run settings, individual results, average, PDI, system suitability results, and any deviations or observations. Sign and date as per GMP data integrity requirements.<br></span></li></ol><p class="c9"><span class="c2">All steps above are to be performed consistently in all labs. Each lab&rsquo;s QA will review the records to ensure compliance.</span></p><h2 class="c19" id="h.jl3augfhadsg"><span class="c20">Summary of Optimized Parameters and Conditions</span></h2><p class="c9"><span class="c2">The following table summarizes the key parameters optimized for this DLS method, the chosen standard conditions, and the rationale for each choice:</span></p><p class="c15 c11"><span class="c2"></span></p><table class="c25"><tr class="c24"><td class="c7" colspan="1" rowspan="1"><p class="c18"><span class="c13">Parameter</span></p></td><td class="c7" colspan="1" rowspan="1"><p class="c18"><span class="c13">Optimized Setting/Procedure</span></p></td><td class="c7" colspan="1" rowspan="1"><p class="c18"><span class="c13">Rationale and GMP Justification</span></p></td></tr><tr class="c24"><td class="c7" colspan="1" rowspan="1"><p class="c9"><span class="c2">Dilution Medium</span></p></td><td class="c7" colspan="1" rowspan="1"><p class="c9"><span class="c2">0.9% Saline (same as formulation) for any needed dilution. No other solvent used.</span></p></td><td class="c7" colspan="1" rowspan="1"><p class="c9"><span class="c2">Preserves native particle state; prevents osmotic shock or changes in surfactant structure . Ensures zeta/size environment remains constant . GMP: Controlled diluent (traceable lot, sterile filtered) to avoid contamination.</span></p></td></tr><tr class="c24"><td class="c7" colspan="1" rowspan="1"><p class="c9"><span class="c2">Dilution Factor</span></p></td><td class="c7" colspan="1" rowspan="1"><p class="c9"><span class="c2">Minimal dilution to reach DLS linear range (e.g., 1:5 dilution if turbidity necessitates). Exact factor specified (no range).</span></p></td><td class="c7" colspan="1" rowspan="1"><p class="c9"><span class="c2">Avoids multiple scattering while keeping sample as concentrated as possible to reflect true aggregate size . Fixed factor improves inter-lab consistency (all analysts prepare the same way). Determined via method development (size plateau observed beyond this dilution).</span></p></td></tr><tr class="c24"><td class="c7" colspan="1" rowspan="1"><p class="c9"><span class="c2">Mixing Method</span></p></td><td class="c7" colspan="1" rowspan="1"><p class="c9"><span class="c2">Gently invert/swirl sample 5&ndash;10 times. No shaking, no vortexing. No routine sonication.</span></p></td><td class="c7" colspan="1" rowspan="1"><p class="c9"><span class="c2">Prevents shear-induced breakup of lipid vesicles and foaming . Maintains original size distribution (sonication was found to artificially reduce size). GMP: Clear instruction to avoid unapproved sample alterations; easier training (gentle technique reproducible).</span></p></td></tr><tr class="c24"><td class="c7" colspan="1" rowspan="1"><p class="c9"><span class="c2">Sonication</span></p></td><td class="c7" colspan="1" rowspan="1"><p class="c9"><span class="c2">None (default). If any dispersion needed, at most a brief, validated low-power sonication (e.g., 5s bath), documented in method.</span></p></td><td class="c7" colspan="1" rowspan="1"><p class="c9"><span class="c2">Sonication can drastically reduce particle size by breaking aggregates &ndash; avoided to preserve product integrity . Only considered if data showed it doesn&rsquo;t alter size (and then strictly controlled). GMP: Avoids introducing a variable that operators might overdo; aligns with product use (no sonication before administration).</span></p></td></tr><tr class="c24"><td class="c7" colspan="1" rowspan="1"><p class="c9"><span class="c2">Temperature</span></p></td><td class="c7" colspan="1" rowspan="1"><p class="c9"><span class="c2">25&deg;C &plusmn;0.1&deg;C (controlled by instrument). (Alternatively 37&deg;C if validated/decided, but one fixed temp in method.)</span></p></td><td class="c7" colspan="1" rowspan="1"><p class="c9"><span class="c2">Provides consistent viscosity and diffusion conditions. 25&deg;C chosen for stability and convenience (below DPPC transition). Ensures all labs use same thermal conditions. GMP: Instrument calibration at that temperature, documented. If 37&deg;C chosen, reflects in-use condition &ndash; but then all labs must maintain that precisely.</span></p></td></tr><tr class="c24"><td class="c7" colspan="1" rowspan="1"><p class="c9"><span class="c2">Cuvette/Cell</span></p></td><td class="c7" colspan="1" rowspan="1"><p class="c9"><span class="c2">Disposable polystyrene cuvettes, 10 mm path, with caps. Use fresh cuvette for each sample.</span></p></td><td class="c7" colspan="1" rowspan="1"><p class="c9"><span class="c2">Eliminates cleaning variability and risk of cross-contamination. Polystyrene is suitable for DLS in visible range. GMP: Single-use avoids carryover; lot of cuvettes can be noted. Consistent optical path for all labs.</span></p></td></tr><tr class="c24"><td class="c7" colspan="1" rowspan="1"><p class="c9"><span class="c2">Scattering Angle</span></p></td><td class="c7" colspan="1" rowspan="1"><p class="c9"><span class="c2">Backscatter (173&deg;) detection (Malvern NIBS) for measurement.</span></p></td><td class="c7" colspan="1" rowspan="1"><p class="c9"><span class="c2">Allows measurement of relatively concentrated, turbid samples with large particles . Reduces need for high dilution, thereby preserving sample state. Also generally extends upper size range detectable. Ensures consistency (if all labs have same configuration).</span></p></td></tr><tr class="c24"><td class="c7" colspan="1" rowspan="1"><p class="c9"><span class="c2">Measurement Duration</span></p></td><td class="c7" colspan="1" rowspan="1"><p class="c9"><span class="c2">e.g., 3 runs &times; 60 s each (adjusted based on optimization). Total ~3 minutes per sample.</span></p></td><td class="c7" colspan="1" rowspan="1"><p class="c9"><span class="c2">Multiple shorter runs enable detection of anomalies and improve averaging. 60 s is sufficient to get stable correlation for these sizes. This was optimized to balance data quality and throughput. GMP: Fixed run time ensures completeness of test; multiple runs demonstrate repeatability.</span></p></td></tr><tr class="c24"><td class="c7" colspan="1" rowspan="1"><p class="c9"><span class="c2">Attenuation / Count Rate</span></p></td><td class="c7" colspan="1" rowspan="1"><p class="c9"><span class="c2">Auto-attenuation enabled; target count rate ~10^2 - 10^5 cps (within instrument&rsquo;s linear range). If count too high, dilute further; if too low, concentrate or longer run.</span></p></td><td class="c7" colspan="1" rowspan="1"><p class="c9"><span class="c2">Maintains measurements in the linear dynamic range of detector (preventing saturation or poor signal). Ensures data quality and comparability &ndash; all labs aim for similar scattering intensity. GMP: Criteria given in SOP (e.g., &ldquo;if count &gt; X, perform Y&rdquo;) to guide operators uniformly.</span></p></td></tr><tr class="c24"><td class="c7" colspan="1" rowspan="1"><p class="c9"><span class="c2">Data Analysis</span></p></td><td class="c7" colspan="1" rowspan="1"><p class="c9"><span class="c2">Cumulant analysis for Z-average and PDI. Report intensity-weighted Z-average (in nm) and PDI. RI (particle) = 1.47, RI (dispersant) = 1.333; Viscosity = 0.9 cP (25&deg;C).</span></p></td><td class="c7" colspan="1" rowspan="1"><p class="c9"><span class="c2">Simplified, robust output focusing on key metrics . Z-average is reproducible and less subjective than multi-peak interpretation. PDI indicates distribution width for QC monitoring. Using correct RI and viscosity ensures accurate size calculation for large particles (Mie model applied). GMP: Standardized reporting format and parameters means each lab processes data identically (method file locked).</span></p></td></tr><tr class="c24"><td class="c7" colspan="1" rowspan="1"><p class="c9"><span class="c2">System Suitability</span></p></td><td class="c7" colspan="1" rowspan="1"><p class="c9"><span class="c2">100 nm PSL standard run before sample (must read 90&ndash;110 nm); Internal control sample (~800 nm lipid aggregate) each day (must be within control limits).</span></p></td><td class="c7" colspan="1" rowspan="1"><p class="c9"><span class="c2">Verifies instrument performance on day of test . The large-size control specifically assures that the DLS can measure in the micron range correctly (since PSL standard is much smaller). Provides confidence in results and traceability. GMP: Documented in each test record; if criteria not met, investigation and remedial action required before proceeding.</span></p></td></tr><tr class="c24"><td class="c7" colspan="1" rowspan="1"><p class="c9"><span class="c2">Replicates</span></p></td><td class="c7" colspan="1" rowspan="1"><p class="c9"><span class="c2">3 independent measurements per sample (from either one prepared cuvette measured thrice or 3 separately prepared cuvettes). Results averaged.</span></p></td><td class="c7" colspan="1" rowspan="1"><p class="c9"><span class="c2">Accounts for any minor handling variability and instrument run variance, improving result reliability. Also provides a measure of precision (%RSD). GMP: Triplicate ensures the test is sound &ndash; a single outlier run can be identified. All replicates and their stats are reported, demonstrating method precision to auditors/regulators.</span></p></td></tr><tr class="c24"><td class="c7" colspan="1" rowspan="1"><p class="c9"><span class="c2">Acceptance Criteria</span></p></td><td class="c7" colspan="1" rowspan="1"><p class="c9"><span class="c2">(Example) Z-average 600&ndash;1200 nm, PDI &lt; 0.7 for routine batch release (criteria to be finalized based on product spec). No unexpected secondary peaks &gt;~1500 nm (indicating out-of-spec large aggregates).</span></p></td><td class="c7" colspan="1" rowspan="1"><p class="c9"><span class="c2">Ensures product meets its historical size range and no excessive aggregation. PDI limit ensures consistency; if PDI too high, result might be unreliable or product might be unstable. GMP: Criteria are derived from product knowledge and method capability; will be in the specification document. Out-of-spec triggers investigation.</span></p></td></tr></table><p class="c9"><span class="c2">Table: Summary of key method parameters optimized for lung surfactant particle size analysis, with justification for each choice in line with method robustness and GMP principles.</span></p><hr><p class="c15 c11"><span class="c2"></span></p><p class="c9"><span class="c2">Conclusion: By following the above development plan, we will establish a DLS-based particle size method that yields consistent and reliable measurements of the lung surfactant suspension&rsquo;s particle size (~600&ndash;1200 nm range) across different laboratories. Each step of sample handling and analysis is justified to minimize alteration of the sample&rsquo;s native state, which addresses the prior issues of result discrepancies. The finalized method will be thoroughly documented, validated, and transferred to all testing sites, complete with controls and training to ensure ongoing consistency. This GMP-compliant approach will support confident monitoring of the surfactant product&rsquo;s quality, knowing that the particle size results truly reflect the product and are reproducible regardless of where or by whom the test is performed.</span></p><p class="c15 c11"><span class="c2"></span></p><p class="c15"><span class="c2">EXAMPLE:</span></p><p class="c15"><span class="c2">Prompt:</span></p><p class="c9"><span class="c2">I have 6 amphiphilic peptides with oligonucleotides and lipids that 5 of are slightly different in mass but pretty much have similar aa composition. How can we develop a method using LCMS to quantify each peptide in a product that has all of them and a method to identify impurities that arise from each of the peptides</span></p><p class="c15"><span class="c2">Response:</span></p><p class="c9"><span class="c2">To help you best, could you please clarify a few details:</span></p><ol class="c4 lst-kix_sbk9fqxb69uy-0 start" start="1"><li class="c0 li-bullet-0"><span class="c2">What type of LC-MS system do you have available (e.g., Q-TOF, triple quadrupole, Orbitrap)?<br></span></li><li class="c0 li-bullet-0"><span class="c2">What kind of quantification are you aiming for: absolute (with standards) or relative quantification?<br></span></li><li class="c0 li-bullet-0"><span class="c2">Are the oligonucleotides and lipids covalently linked to the peptides, or are they non-covalent complexes?<br></span></li><li class="c0 li-bullet-0"><span class="c2">Are you expecting impurities mainly from incomplete synthesis (e.g., missing one amino acid, lipid variants) or degradation products?<br></span></li><li class="c0 li-bullet-0"><span class="c2">Would you prefer a single method to quantify and identify impurities, or two separate methods?<br></span></li></ol><p class="c9"><span class="c2">This will help me suggest a tailored LC-MS method development strategy for your needs.</span></p><p class="c9"><span class="c2">Prompt:</span></p><p class="c9"><span class="c2">1. Qtof</span></p><p class="c9"><span class="c2">2. Absolute</span></p><p class="c9"><span class="c2">3.covalent </span></p><p class="c9"><span class="c2">4. Lipid variants, and degradation etc</span></p><p class="c9"><span class="c2">5. Give me the both options </span></p><h1 class="c10" id="h.o279qqp5igmt"><span class="c17">LC&ndash;MS Method Development for Amphiphilic Peptide&ndash;Oligonucleotide&ndash;Lipid Conjugates</span></h1><p class="c9"><span class="c2">Developing an LC&ndash;MS strategy for amphiphilic peptide conjugates (peptide&ndash;oligonucleotide&ndash;lipid hybrids) requires balancing chromatographic separation, sensitive detection, and structural analysis. Below are two approaches:</span></p><h2 class="c19" id="h.h2l1h2iabntp"><span class="c20">Option 1: Single LC&ndash;MS Method for Quantification and Impurity Identification</span></h2><p class="c9"><span class="c2">This approach uses one comprehensive method to quantify all six peptides and simultaneously identify impurities.</span></p><h3 class="c12" id="h.bnan0j4x0ide"><span class="c8">Chromatographic Conditions (Single Method)</span></h3><ul class="c4 lst-kix_4y6tqwnm2166-0 start"><li class="c0 li-bullet-0"><span class="c2">Column: A high-performance C18 reversed-phase column with high peak capacity. For example, a 2.1 mm &times; 100 mm (or 150 mm) C18 column (1.7&ndash;3 &micro;m particles) &ndash; such as YMC-Pack Pro C18 or similar &ndash; provides strong retention for lipopeptides and is MS-compatible . Maintain column temperature ~30&ndash;40 &deg;C for reproducible retention.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Mobile Phases: Use ion-pairing reversed-phase (IP-RP) to handle the oligonucleotide&rsquo;s negative charges. For instance, Phase A = water with 0.1% triethylamine (TEA) and ~0.5&ndash;1% hexafluoroisopropanol (HFIP); Phase B = methanol (or 50:50 MeOH/ACN) with the same TEA/HFIP content . This TEA&ndash;HFIP system neutralizes phosphate charges and improves retention of highly polar conjugates. Note: TEA/HFIP are volatile and MS-friendly at low concentrations. (Avoid non-volatile ion-pair reagents or inorganic buffers.)<br></span></li><li class="c0 li-bullet-0"><span class="c2">Gradient: Start at low organic (e.g. 5% B) to allow retention of very polar species, then ramp to high organic ( ~80% B) over a 30&ndash;45 min gradient. A shallow gradient improves resolution between the six analytes, which have similar compositions. For example, a linear ramp from 5% to 60% B over 30 min, then to 80% B by 35 min can separate close eluters. Extend to 90&ndash;95% B for a few minutes to elute any strongly bound lipids, then re-equilibrate at initial conditions.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Flow Rate: ~0.2&ndash;0.3 mL/min on a 2.1 mm column (adjust if using a different ID). This provides adequate time for separation and is within optimal linear velocity for typical UHPLC C18 columns.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Rationale: The IP-RP system ensures even these amphiphilic conjugates (with hydrophilic oligo and hydrophobic lipid) interact sufficiently with the column. HFIP (a weak acid) and TEA (a weak base) pair with the oligonucleotide&rsquo;s phosphate groups, improving peak shape and retention. Without ion pairing, these conjugates might elute at void or show broad, tailing peaks due to their highly polar regions. TEA/HFIP concentration: Use the lowest that gives acceptable peak shape &ndash; higher concentrations sharpen peaks but can suppress MS signal . (For example, ~7 mM TEA and ~70 mM HFIP is a reasonable starting point .) Also, 0.1% formic acid alone is not sufficient for such heavily charged analytes, though it is ideal for standard lipopeptides .<br></span></li></ul><h3 class="c12" id="h.9fd2ane5v86k"><span class="c8">MS Acquisition Settings (Single Method)</span></h3><ul class="c4 lst-kix_vams35lnoe4-0 start"><li class="c0 li-bullet-0"><span class="c2">Ionization &amp; Polarity: Electrospray ionization (ESI) in negative-ion mode is recommended for detecting these conjugates in the IP-RP system. The oligonucleotide&rsquo;s multiple negative charges dominate in negative ESI, yielding abundant multiply charged ions for each conjugate . (Positive mode can be tested if the peptides are highly basic, but the TEA in mobile phase and the oligonucleotide&rsquo;s anionic nature favor negative mode detection.)<br></span></li><li class="c0 li-bullet-0"><span class="c2">Mass Range: Use an extended m/z range to capture the multiply charged ions of large conjugates. For example, scan m/z 400&ndash;3000. A Q-TOF in &ldquo;high m/z&rdquo; or extended range mode can accurately measure masses for molecules up to ~3&ndash;13 kDa with only a few Dalton error . Ensure the instrument is calibrated across this range (use a suitable calibration mix or lock-mass reference).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Acquisition Mode: Employ a combination of full-scan MS and MS/MS to achieve both quantification and identification:<br></span></li></ul><ul class="c4 lst-kix_vams35lnoe4-1 start"><li class="c3 li-bullet-0"><span class="c2">For quantification, collect high-resolution MS^1 spectra continuously (TOF resolving power ~30,000 or higher) to integrate the peaks of each peptide.<br></span></li><li class="c3 li-bullet-0"><span class="c2">For structural analysis of impurities, use data-dependent MS/MS (DDA) or data-independent acquisition concurrently. For example, the Q-TOF can alternate between a low-collision-energy full scan and a high-collision-energy all-ions fragmentation scan (as in Waters MSE) to capture fragment data for all eluting compounds. Alternatively, program DDA to trigger MS/MS on any peaks that are not on a &ldquo;exclude list&rdquo; of the six known peptides, so that minor unknown peaks are fragmented. Using a broad isolation window (or no isolation in an all-ions experiment) ensures even unexpected low-level impurities generate fragment spectra.<br></span></li><li class="c3 li-bullet-0"><span class="c2">Collision Energy: Optimize a collision energy ramp for MS/MS (e.g. 20&ndash;40 eV) to fragment these large ions. Expect both peptide bond fragments (b/y ions) and oligonucleotide fragments (a-B, w ions, etc.), as well as loss of lipid moieties. A high-energy setting may be required for cleaving lipid or stable linkages.<br></span></li></ul><ul class="c4 lst-kix_vams35lnoe4-0"><li class="c0 li-bullet-0"><span class="c2">Scan Speed: Ensure a fast acquisition cycle (e.g. ~1&ndash;2 spectra/sec for MS1) to get sufficient data points across each LC peak. Q-TOFs can typically handle this scan speed for the given m/z range. If using DDA, limit the number of concurrent MS/MS (e.g. top 3 peaks) and use dynamic exclusion for ~0.3 min to avoid repeated fragmentation of the main peptides in every scan. This preserves MS time for quantification and for capturing diverse impurities.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Targeted Quant (optional): If sensitivity is a concern, the Q-TOF&rsquo;s quadrupole can be used in SIM mode to transmit only narrow m/z windows for each analyte (improving S/N by reducing noise). However, in a single-method approach, full-scan high-res MS1 typically suffices since it provides selectivity by exact mass extraction.<br></span></li></ul><h3 class="c12" id="h.10a9eb8zngi7"><span class="c8">Sample Preparation Considerations (Single Method)</span></h3><ul class="c4 lst-kix_ls0ws55dysw8-0 start"><li class="c0 li-bullet-0"><span class="c2">Solubility and Adsorption: Amphiphilic peptides can adsorb to vials or precipitate due to their surfactant-like nature. To prevent losses, dissolve samples in a mixture of water and organic solvent. For example, dilute or reconstitute the peptide mixture in ~50% methanol or acetonitrile (with a little acid or ion-pair reagent matching the mobile phase) . Researchers found that adding MeOH increases recovery of lipopeptides by preventing them from sticking to surfaces or forming aggregates . Use low-bind plastic or silanized glass vials to further minimize surface adsorption.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Avoid Non-Volatile Additives: Ensure the sample is free of salts, buffers, or detergents that can cause ion suppression or contaminate the MS. If the peptides were synthesized or stored in buffers (e.g. PBS or with surfactants), perform a clean-up. Solid-phase extraction (SPE) with a cartridge that retains oligonucleotides (e.g. phenylboronate or anion-exchange based &ldquo;oligonucleotide cleanup&rdquo; SPE) can remove salts and yield the analytes in an MS-friendly solvent. Otherwise, a simple desalting via spin column or ethanol precipitation (for oligonucleotides) could be used, but SPE is preferable for quantitative recovery.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Injection Solvent: Ideally, the sample solvent should be similar to initial LC mobile phase (for IP-RP, that means containing TEA/HFIP). For example, dissolve the sample in 90&ndash;95% water with 0.1% TEA + 0.5% HFIP (matching mobile phase A) and ~5&ndash;10% MeOH for solubility. This prevents precipitation of ion-pair reagents and focusses the sample on the column. Keep injection volume moderate (e.g. 5&ndash;20 &micro;L, depending on column capacity) to avoid overloading, especially since these conjugates may be semi&#8208;volatile and high amounts could cause ion source contamination.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Internal Standard Addition: Spike a known amount of an internal standard into the sample before injection. For a single-run method, a stable-isotope labeled analog of one of the peptides is ideal. A heavy-labeled internal standard will co-elute and ionize like the target, helping correct any suppression or drift . If a full isotopic analog of the entire conjugate is unavailable, consider a surrogate: e.g. a similar peptide&ndash;oligo conjugate with a different sequence, or one of the six peptides used as a quantification reference for the others (assuming their response factors are similar). The internal standard should elute close to the analytes to experience similar matrix effects . This practice normalizes the response and improves quantitation accuracy . (Ensure the internal standard is a unique mass not overlapping with any natural analyte or impurity.)<br></span></li></ul><h3 class="c12" id="h.8m99ql39l0mt"><span class="c8">Data Analysis Strategies (Single Method)</span></h3><ul class="c4 lst-kix_vx5oc7tbpucq-0 start"><li class="c0 li-bullet-0"><span class="c2">Absolute Quantification: Perform calibration using standards for each peptide conjugate. Because the Q-TOF is a high-resolution instrument, use extracted ion chromatograms (XIC) for quantification rather than total ion chromatogram. Select a characteristic ion for each peptide (typically the most intense charge state of the intact molecular ion) and extract the chromatogram at that exact m/z (with a narrow mass window, e.g. &plusmn;5&ndash;10 ppm). The peak area from the EIC is then used to quantify the compound . Software like Agilent MassHunter or Waters TargetLynx will let you integrate the EIC peak area; this area can be plotted against the known concentration to create a calibration curve . If an internal standard was added, compute the response ratio (analyte peak area / IS peak area) for more robust quantitation.<br></span></li></ul><ul class="c4 lst-kix_vx5oc7tbpucq-1 start"><li class="c3 li-bullet-0"><span class="c2">Use at least 5&ndash;7 calibrant levels bracketing the expected concentrations, and include quality control samples to validate accuracy. Q-TOF quantification can achieve good linearity, but the dynamic range may be narrower than triple-quadrupole MRM; be mindful of the upper limit where detector saturation or ion suppression occurs. For these amphiphilic analytes, ionization may deviate from linear at high concentration due to their surface-active nature . Keeping analyte concentrations at low micromolar or below in the sample avoids self-suppression.<br></span></li></ul><ul class="c4 lst-kix_vx5oc7tbpucq-0"><li class="c0 li-bullet-0"><span class="c2">Impurity Identification: Utilize the high-resolution MS and MS/MS data to find and characterize impurities:<br></span></li></ul><ul class="c4 lst-kix_vx5oc7tbpucq-1 start"><li class="c3 li-bullet-0"><span class="c2">Peak Detection: First, examine the total ion chromatogram (TIC) or base peak chromatogram for any peaks other than the six main peptides. Because the main components are known, any additional chromatographic peaks (or unusually shaped shoulders on main peaks) likely indicate impurities. Extract ion traces for common modifications to aid discovery &ndash; for example, if you suspect a lipid chain length variant, look for a mass +14 or &ndash;14 Da from the main species (adding or losing a &ndash;CH&#8322;&ndash; unit). Likewise, potential oligonucleotide truncations would show &ndash;&#8710;(nucleoside mass) differences (e.g. &ndash;$305$ Da if a nucleotide is missing).<br></span></li><li class="c3 li-bullet-0"><span class="c2">Mass Determination: For each suspected impurity peak, check its high-res MS spectrum. Deconvolute the multiply charged envelope to determine the molecular weight. Most vendor software (MassHunter Bioconfirm, Waters Bioanalysis tools, etc.) can do maximum entropy deconvolution to report the intact mass of the species. This helps confirm if an impurity corresponds to a specific variant (e.g. +16 Da might indicate an oxidation; &ndash;80 Da could indicate loss of a nucleotide phosphate, etc.). High-resolution allows assignment of elemental composition for small mass shifts, but for large molecules focus on known likely modifications.<br></span></li><li class="c3 li-bullet-0"><span class="c2">Structural Elucidation: Analyze the MS/MS spectra obtained for each impurity. Look for fragment ions that reveal which part of the conjugate is altered. For instance: if a lipid variant is present, the peptide&ndash;oligonucleotide portion fragments (b/y ions from the peptide or a/b/c ions from the oligo) should match the parent structure, while a fragment corresponding to the lipid moiety might differ. If an impurity is a degradation product (e.g. missing a portion of the peptide), the oligonucleotide-specific fragments (perhaps the PO&#8324;-based ions) would still be observed, but peptide fragment series might be truncated. Compare fragment patterns to those of the parent peptide conjugates to pinpoint differences.<br></span></li><li class="c3 li-bullet-0"><span class="c2">Data Processing: Use qualitative LC&ndash;MS software to assist in finding impurities. Many Q-TOF software packages can do component detection on chromatographic data to list molecular features by m/z and retention time. This unsupervised feature extraction can reveal low-level species that might be missed by eye. Once features are found, assign likely structures by comparing the exact mass to the calculated masses of plausible variants (e.g. different lipid length, water loss, adducts, etc.). For confirmation, manually inspect the MS/MS and, if needed, run targeted MS/MS in a follow-up experiment to get higher quality spectra for a specific m/z.<br></span></li></ul><ul class="c4 lst-kix_vx5oc7tbpucq-0"><li class="c0 li-bullet-0"><span class="c2">Quantification of Impurities: If needed (for a purity assay), the same data can be used to estimate impurity levels. For each impurity, integrate the XIC of its intact ion and compare its area to the calibration curve (if a standard for that impurity is available) or to the response of a parent peptide (assuming similar ionization efficiency). In absence of standards, report impurities semi-quantitatively (e.g. as a percentage of the main peak by area or using the internal standard for reference).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Reporting: The single method yields both the concentration of each of the six target peptides and identifies impurities in one run. Keep in mind that ion-pair reagents can reduce MS sensitivity &ndash; if an impurity is very low-level, you may need to inject a larger volume or a concentrated sample to see it. This one-shot method trades some sensitivity for convenience.<br></span></li></ul><h3 class="c12" id="h.8qcna51rbggq"><span class="c8">Special Considerations for Amphiphilic Molecules</span></h3><ul class="c4 lst-kix_fd9jvhtv6teb-0 start"><li class="c0 li-bullet-0"><span class="c2">Ion Suppression and Matrix Effects: Amphiphilic compounds can cause or suffer from ion suppression because they tend to concentrate at droplet surfaces during ESI . In a mixture of six, a highly abundant peptide might suppress a minor one if they co-elute. Mitigate this by achieving some chromatographic separation between analytes and by not overloading the column. If these conjugates will be analyzed in complex matrices (e.g. plasma), a thorough sample cleanup is critical &ndash; endogenous surfactants like phospholipids strongly suppress ESI signals . Protein precipitation or SPE (e.g. using a reversed-phase or ion-exchange cartridge) can remove most phospholipids . Co-elution of any residual phospholipids with the analytes should be avoided (adjust gradient as needed, since phospholipids often elute in 5&ndash;50% ACN region).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Carryover: Because of their lipid tails, these conjugates may stick to LC surfaces. Use a guard column and consider washing the column with a strong solvent (e.g. 90% acetonitrile/10% isopropanol or even a short isopropanol flush) after each run to prevent carryover ghost peaks. Also, periodically clean the MS source (the lipid component can cause residue in the ESI interface over multiple injections).<br></span></li><li class="c0 li-bullet-0"><span class="c2">System Equilibration: When using ion-pairing mobile phase, the column needs thorough equilibration (10&ndash;20 column volumes or more) for consistent retention, as the stationary phase must become saturated with the TEA&ndash;HFIP pair. Inconsistent equilibration can lead to shifting retention times run-to-run.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Safety: HFIP and TEA are volatile and somewhat toxic; handle in a fume hood and ensure the MS exhaust is properly vented. Also, because HFIP is acidic, use PEEK tubing where possible to avoid corrosion of steel parts over time.<br></span></li></ul><h2 class="c19" id="h.pb0yvq3ktlsi"><span class="c20">Option 2: Two Separate LC&ndash;MS Methods (Dedicated Quantification vs. Impurity Profiling)</span></h2><p class="c9"><span class="c2">In this approach, one method is optimized for absolute quantification of the six target peptides, and a second method is optimized for detailed impurity profiling. This allows each method to be streamlined for its specific purpose.</span></p><h3 class="c12" id="h.9be969hc5pr1"><span class="c8">LC&ndash;MS Method A: Optimized for Absolute Quantification</span></h3><p class="c9"><span class="c2">This method prioritizes reproducible quantitation, sensitivity, and throughput for the six conjugates.</span></p><p class="c9"><span class="c2">Chromatographic Conditions (Quant Method):</span></p><ul class="c4 lst-kix_xq8kn6xuala3-0 start"><li class="c0 li-bullet-0"><span class="c2">Column and Mobile Phase: Use a similar reversed-phase C18 column (2.1 mm ID) as above, but without ion-pair reagents to maximize MS sensitivity. Mobile Phase A can be water + 0.1% formic acid (FA), and B can be acetonitrile + 0.1% FA. Formic acid (0.1%) provides a low pH (~2.7) to keep peptides protonated and gives good peak shape, while being fully volatile for MS . (Avoid TFA here &ndash; even 0.1% TFA can suppress ESI signal severely, whereas formic acid is much more MS-friendly .)<br></span></li><li class="c0 li-bullet-0"><span class="c2">Gradient: Start at ~5&ndash;10% B and ramp to ~70% B over 20 minutes (exact gradient may be adjusted based on retention of the targets). Because we have no ion-pair to retain the oligonucleotide portion, the peptides might elute earlier than in the IP method. A shallow gradient in the mid-organic range will help separate the conjugates by their hydrophobicity differences (particularly distinguishing different lipid or peptide lengths). For example, 5% B to 60% B in 15 min, then to 80% B by 20 min, hold 80% B for 2 min. This should elute all six peptides and most hydrophobic variants. Include a high-organic wash (90&ndash;95% B) at the end to flush the column. The overall runtime can be ~25 min including re-equilibration, balancing throughput with resolution.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Flow Rate &amp; Temperature: ~0.3 mL/min on a 2.1 mm column; temperature 40 &deg;C can be used to improve peak shape (higher temperature can reduce viscosity and slightly sharpen peaks, helpful since no ion-pair is used).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Retention Behavior: Monitor early in development if any target elutes near the void volume (which could happen if the oligonucleotide dominates the molecule&rsquo;s character). If a particular conjugate is very polar and elutes too early or with poor shape, you may need to incorporate a minimal ion-pair additive even in the quant method. One compromise is using a very low concentration of HFIP (e.g. 0.1&ndash;0.2%) and an organic base like dimethylamine or small TEA (0.02&ndash;0.05%) &ndash; enough to aid retention, but as low as possible to preserve sensitivity . Often, however, the lipid and peptide moieties impart sufficient hydrophobicity that the conjugates will retain with just formic acid. Each of the six peptides should produce a distinct retention time if their sequences or lipid attachments differ, but expect relatively close elution; adjust the gradient shallowness as needed to resolve any co-eluting pair.<br></span></li></ul><p class="c9"><span class="c2">MS Acquisition Settings (Quant Method):</span></p><ul class="c4 lst-kix_z4fu53gdnrww-0 start"><li class="c0 li-bullet-0"><span class="c2">ESI Polarity: Use positive-ion mode for quantification. At low pH (formic acid), basic sites on the peptide will carry positive charges. The oligonucleotide&rsquo;s phosphates will mostly be protonated or paired with counter-ions, so the molecule can still be detected as positive ions (e.g. as multiple proton adducts, or possibly as complexes with small cations). In positive mode, the peptide-lipid part of the molecule drives ionization, which often yields higher sensitivity on Q-TOF for peptides. Monitor the spectra from a standard &ndash; if sensitivity in positive is insufficient (some highly anionic conjugates ionize poorly in positive), the quant method can also be run in negative mode with formic acid (the formic acid will still help chromatographically, and the negative ions will form from deprotonated phosphates). Choose the polarity that gives the strongest and cleanest response for the six targets.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Scan Mode: MS-only (full scan) mode is sufficient for quantifying known compounds. Set the TOF to scan, for example, m/z 500&ndash;2500 in 1 s. This will capture the multiple charge states of each peptide conjugate. High resolution (~30K) ensures that even if two analytes elute at similar times, their masses can be distinguished. The fast full-scan also means each peak has many points for good quantitation precision. Do not use any in-source fragmentation or unnecessary MS/MS, to keep all signal focused on the intact ions for quantitation.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Targeted Extraction: Because we know the exact masses, the data analysis will involve extracting each compound&rsquo;s signal. Real-time or scheduled targeted MS is optional &ndash; for example, one could program the quadrupole to transmit only a window around each expected m/z (&ldquo;simultaneous SIM&rdquo;) to reduce noise. However, this is usually not needed; instead, just acquire full-scan and use software to extract the ions post-run.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Dynamic Range: Use the Q-TOF&rsquo;s extended dynamic range mode if available. This might lower resolving power slightly, but prevents detector saturation for high-abundance species, ensuring linear response. Verify that the most intense peptide does not overload the detector &ndash; if it does, either dilute the sample or use a smaller injection volume for the quant method. The separate impurity method can then be used to look for low-level species with a larger injection.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Calibration/Lock Mass: To maintain mass accuracy (critical for correct extraction of each peptide&rsquo;s signal, especially if some co-elute), use continuous calibration if possible. For example, Waters Q-TOFs allow lock-mass infusion (like leucine-enkephalin at m/z 556.2771) &ndash; this keeps mass error &lt;5 ppm throughout the run. High mass accuracy ensures that each peptide&rsquo;s extracted ion chromatogram is selective to that compound .<br></span></li></ul><p class="c9"><span class="c2">Sample Preparation (Quant Method):</span></p><p class="c9"><span class="c2">Sample prep for quant is similar to the single method described, with emphasis on consistency:</span></p><ul class="c4 lst-kix_4hyg4ehfafxi-0 start"><li class="c0 li-bullet-0"><span class="c2">Prepare each sample, calibration standard, and QC in the same diluent (e.g. 50:50 water:acetonitrile + 0.1% formic acid). A consistent organic content in all vials minimizes variability in recovery and injection. The presence of ~50% organic solvent helps keep these amphiphiles from adsorbing to vial and tube surfaces .<br></span></li><li class="c0 li-bullet-0"><span class="c2">If the samples come from a biological matrix, perform extraction (for quantitation, a reliable SPE method is recommended to get reproducible recovery). For instance, use a mixed-mode SPE that can retain the polar oligonucleotide part (many kits exist for oligonucleotide drugs). Elute in a small volume of a high-organic solvent, then dilute with water + FA to match the initial LC conditions. This not only concentrates the analytes but also removes phospholipids and proteins that cause ion suppression . Ensure the internal standard (heavy analog) is spiked into the sample before extraction to correct for any losses during workup .<br></span></li><li class="c0 li-bullet-0"><span class="c2">Filter or centrifuge samples to remove any particulates (membrane filters rated for very low protein binding are ideal, or use a 0.2 &micro;m PTFE filter for organic-rich solutions). Particulates can cause signal noise and pressure spikes.<br></span></li></ul><p class="c9"><span class="c2">Data Analysis and Quantification (Quant Method):</span></p><ul class="c4 lst-kix_e134675mgdj3-0 start"><li class="c0 li-bullet-0"><span class="c2">Extracted Ion Quantification: After acquiring the data, set up extracted ion chromatograms for each peptide&rsquo;s predominant ion. For example, if peptide 1 (with a given conjugate) mostly shows a 5+ charge state at m/z = 600.1234, integrate that peak. Use a narrow extraction width (e.g. &plusmn;0.02 Da or &plusmn;10 ppm) to exclude any nearby interferences. The area under each peak corresponds to that peptide&rsquo;s abundance .<br></span></li><li class="c0 li-bullet-0"><span class="c2">Calibration Curve: Prepare a calibration curve for each peptide by plotting known concentration vs. area (or area ratio to internal std). If you have a stable isotope internal standard for each analyte, the slope of each curve will automatically account for any analyte-specific ionization differences. If using a single internal standard for all six, you assume similar response factors; in practice there will be some variability, so it&rsquo;s better to calibrate each separately if possible.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Internal Standard Usage: Use the internal standard to correct each sample&rsquo;s result. For instance, if a heavy-labeled version of peptide A was added to all samples, quantify peptide A against its heavy IS (which has a separate m/z), and also use that IS&rsquo;s area to normalize peptides B&ndash;F (assuming equal recovery). The internal standard compensates for injection volume differences and any global matrix suppression . Note: Internal standard will not correct different ionization efficiencies between peptides &ndash; that&rsquo;s why separate calibration for each is ideal. But it will correct run-to-run variance.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Absolute Quantitation: Compute the concentration of each peptide in the mixture from the calibration curve. Report in desired units (e.g. ng/mL or pmol/&micro;L). Because this method is optimized for quant, you should achieve high precision and accuracy. Expect %RSD &lt;10% for replicates if the method is well-tuned. The lower limit of quantification (LLOQ) will depend on signal-to-noise; without ion-pair, the MS sensitivity is maximized, so sub-nanogram on-column levels can often be detected on Q-TOF. (For reference, published HRMS assays for oligonucleotides achieved LLOQs in the low ng/mL range in biofluids using similar conditions .)<br></span></li><li class="c0 li-bullet-0"><span class="c2">Throughput Consideration: If many samples are to be run, this quantification method can potentially be fast (e.g. 15&ndash;20 min per run). To shorten run time, one could use a steeper gradient or a shorter column (e.g. 50 mm length). However, be cautious that resolution might suffer &ndash; verify that even in a faster method, each peptide&rsquo;s mass spectrum is not significantly overlapped by another. It&rsquo;s acceptable if two co-elute as long as their m/z are different enough to deconvolute, but heavy co-elution could risk one suppressing the other. Adjust the method as needed to ensure quantitation remains linear and specific.<br></span></li></ul><h3 class="c12" id="h.r5ah68gtdbk6"><span class="c8">LC&ndash;MS Method B: Optimized for Impurity Profiling and Identification</span></h3><p class="c9"><span class="c2">This method is dedicated to detecting impurities, variants, and degradation products with high sensitivity and detailed structural information.</span></p><p class="c9"><span class="c2">Chromatographic Conditions (Impurity Method):</span></p><ul class="c4 lst-kix_4864lfd9zdwy-0 start"><li class="c0 li-bullet-0"><span class="c2">Column: Use the same C18 column type, but consider a longer column (e.g. 150 mm) or smaller particle size to increase resolution. A longer column combined with a shallow gradient can better separate very similar analogues. If available, a higher peak-capacity column (like a 2D LC setup or a core-shell particle column) can be beneficial, but a single C18 is usually sufficient.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Mobile Phases: Re-introduce ion-pairing for this method to enhance separation of polar impurities. Mobile Phase A = 15 mM HFIP + 5 mM TEA in water (approx. 0.2% HFIP, 0.07% TEA); Phase B = 15 mM HFIP + 5 mM TEA in 50:50 ACN/MeOH. (This is a moderately low concentration IP system &ndash; strong enough to retain highly polar species, but not as high as some methods that use 100 mM HFIP , in order to preserve more MS sensitivity for trace impurities.) Ion-pairing here will help, for example, to separate an impurity that is missing a lipid (more polar) or one with an extra phosphate. These species might otherwise co-elute with main peaks if only FA was used.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Gradient: Use a slow, extended gradient to maximize separation. For instance: 0&ndash;5 min 0% B, then 0% to 50% B from 5&ndash;40 min, then 50% to 80% B from 40&ndash;50 min (total ~50&ndash;60 min run). This long gradient allows even very small differences in hydrophobicity to manifest as distinct retention differences. Impurities such as lipid-chain variants (&plusmn;CH&#8322;) often separate well with a difference of a few percent organic. Oligonucleotide length variants (if any) will be separated primarily by the ion-pair mechanism &ndash; shorter oligos elute earlier due to fewer negative charges pairing with TEA. Note: Keep the column temperature consistent (e.g. 35 &deg;C) but you can also experiment with a slightly lower temperature (25 &deg;C) to improve separation &ndash; lower temp often increases retention and might resolve closely eluting species, at the cost of peak width.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Flow Rate: Slightly lower flow (e.g. 0.2 mL/min on a 2.1 mm column) can improve resolution for the long gradient. This also gives the MS more time to accumulate ions for MS/MS of low-level species.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Injection Volume: For impurity profiling, you may inject a larger volume (since quantitation accuracy is not the focus here). For example, if 10 &micro;L was used in quant method, you might inject 20&ndash;30 &micro;L in the impurity method to load more sample and detect trace components. Be cautious: too large an injection with ion-pair can cause peak broadening or saturation of main peaks (which could then suppress the minor ones). Optimize to ensure the main peaks are still reasonably shaped, but load enough sample that impurities at ~1% level can be seen.<br></span></li></ul><p class="c9"><span class="c2">MS Acquisition Settings (Impurity Method):</span></p><ul class="c4 lst-kix_tj1ix0gi2ui9-0 start"><li class="c0 li-bullet-0"><span class="c2">ESI Polarity: Use negative-ion mode for comprehensive impurity detection. With the TEA/HFIP system, negative mode is far more effective &ndash; the impurities, especially any that are more oligonucleotide-like (e.g. phosphate-containing fragments, or free oligo), will ionize as deprotonated species. Negative mode will highlight any species that carry phosphate groups (all genuine conjugates and their metabolites will). Positive mode could miss an impurity that lacks basic residues or lost a lipid. If certain impurities are neutral loss (like a free lipid) and won&rsquo;t ionize in negative, they might not be seen directly but could be inferred (e.g. a free lipid might elute late and could be seen in a TIC by evaporative light scattering or UV if those detectors were available in-line &ndash; though usually we rely on MS). In practice, negative ESI with high mass accuracy will detect even those lipid-only impurities if they form adducts (e.g. deprotonated fatty acid or formate adducts).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Scan Mode: Full scan MS + targeted MS/MS. Because we want to identify unknowns, turn on data-dependent MS/MS for any peaks that appear. Set the full scan range maybe a bit broader (e.g. m/z 300&ndash;3000) to catch small fragments or any low MW impurities. Use a short MS scan (~0.5 s) followed by MS/MS on, say, the top 3&ndash;5 ions in each scan (each MS/MS 0.1&ndash;0.2 s). Use dynamic exclusion (e.g. exclude an ion for 0.5 min after 1&ndash;2 occurrences) so that once a major peak (like a main peptide) has been fragmented, the instrument moves on to other less abundant peaks in subsequent cycles. Also consider an inclusion list if you have prior knowledge of specific impurity masses (for example, if you expect a certain m/z from a known side product, force the MS to target it).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Alternate approach (DIA): If the sample is complex or impurities are very low, you might use a data-independent acquisition (DIA) approach. For example, perform one injection where the Q-TOF does broadband fragmentation: either an MSE (simultaneous no-CollisionEnergy and high-CE scans) or SWATH-type small window fragmentation across the mass range. This way, even very low-level ions that might not trigger DDA can still produce fragments. The trade-off is more complicated data &ndash; you&rsquo;ll rely on chromatographic coelution and deconvolution to link fragments to precursors. This can be powerful for capturing everything in one run. Modern Q-TOF software can handle DIA by correlating fragment ions to likely precursors by retention time and accurate mass.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Mass Resolution: Use the highest resolution setting for both MS and MS/MS to maximize the chance of formula determination. Impurities might be distinguished by tiny mass differences, so a high resolving power (e.g. 30k&ndash;60k) helps. The small penalty in sensitivity is acceptable here given the focus on identification.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Sensitivity Tips: Since impurity profiling can be limited by sensitivity, tune the MS source for maximum response: use a slightly lower gas flow and lower fragmentor/skimmer voltages to keep large ions intact and get more signal. You can also sum multiple injections &ndash; if sample is limited but you can afford to use more, injecting the same sample twice and averaging the data can enhance S/N for very low peaks (some software allows combining runs).<br></span></li></ul><p class="c9"><span class="c2">Data Analysis (Impurity Method):</span></p><ul class="c4 lst-kix_tibxkvatc3h0-0 start"><li class="c0 li-bullet-0"><span class="c2">Comprehensive Peak Review: Use qualitative analysis software to list all peaks detected. In Agilent MassHunter or Waters UNIFI, for example, you can use &ldquo;Find by Molecular Feature&rdquo; or &ldquo;Component detection&rdquo; which will group together isotopes and charge states and report components by their neutral mass. Review this list for any species that are not the six main peptides (the main six will be obvious by mass). Pay attention to species that might correspond to: truncated peptides, oligonucleotide-only fragments, lipid-only fragments, different adducts (sodium adducts of oligos can appear, etc.), or dimeric associations (sometimes amphiphiles form non-covalent dimers that appear as separate masses).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Identification Strategy: For each impurity, try to match it to a plausible structure:<br></span></li></ul><ul class="c4 lst-kix_tibxkvatc3h0-1 start"><li class="c3 li-bullet-0"><span class="c2">Mass Matching: Subtract the known mass of a main conjugate from the impurity mass to see the difference. If +14.02 Da, likely one extra &ndash;CH&#8322;&ndash; (perhaps a longer lipid chain) ; if &ndash;14.02 Da, a shorter lipid. A difference of &ndash;$$\approx$$ 300 Da could indicate loss of a nucleotide (e.g. loss of a deoxy nucleoside monophosphate); &ndash;$$\approx$$ 16 Da could indicate loss of an &ndash;NH&#8322; (deamination) or gain of an oxygen (oxidation) depending on context. Use high-resolution exact differences to distinguish, for example, +O (16.000) vs +NH&#8322; (15.995) &ndash; high-res can usually separate those by mass.<br></span></li><li class="c3 li-bullet-0"><span class="c2">Fragment Interpretation: Examine the MS/MS spectra for signature fragment ions:<br></span></li></ul><ul class="c4 lst-kix_tibxkvatc3h0-2 start"><li class="c6 li-bullet-0"><span class="c2">Oligonucleotide fragments: Look for nucleotide base ions (for instance, ion of protonated/deprotonated adenine or guanine), or the backbone fragment ions [(base + sugar + phosphate) fragments which in negative mode appear as [a-Bions]. The presence or absence of certain base fragments will tell if an impurity has lost a specific nucleotide. For example, if the main has a sequence and one impurity&rsquo;s MS/MS is missing all fragments that contain the last base, that impurity might be missing that base (truncated).<br></span></li><li class="c6 li-bullet-0"><span class="c2">Peptide fragments: Look for b/y ion series in MS/MS. If an impurity is missing part of the peptide, its b/y ions will indicate a sequence gap. For instance, if the normal peptide sequence yields b3, b4, b5 ions, and an impurity shows b3 and b4 but no b5, it might be missing a residue after the 4th amino acid. Common peptide modifications like oxidation (Met to sulfoxide) can be spotted by a +16 Da on relevant fragment ions.<br></span></li><li class="c6 li-bullet-0"><span class="c2">Lipid fragments: Many lipidated peptides show a neutral loss of the lipid (for instance, a loss of 264 Da might indicate a palmitoyl chain). If an impurity has a different lipid, its MS/MS may show a different neutral loss or a diagnostic fragment. For example, cholesterol conjugates often show a fragment around m/z 369 in positive mode (cholesterol fragment). In negative mode, fatty acids show up as [RCOO]^&ndash; ions. Be attentive to such signals in the spectra.<br></span></li></ul><ul class="c4 lst-kix_tibxkvatc3h0-1"><li class="c3 li-bullet-0"><span class="c2">Software Tools: Consider using peptide mapping or biopolymer analysis software if available. These tools can be fed the sequence of the expected molecule and will highlight differences needed to match an observed mass. Some packages (e.g. BioPharmaLynx, Skyline for small peptides, etc.) allow you to input potential modifications and will search the data for evidence of them. This can streamline identifying known degradation products like deamidation, oxidation, etc., by comparing theoretical masses to observed ones.<br></span></li></ul><ul class="c4 lst-kix_tibxkvatc3h0-0"><li class="c0 li-bullet-0"><span class="c2">Quantification of Impurities: While this method is mainly qualitative, you can estimate impurity levels. If an impurity peak is observed, integrate its area from the XIC (use the appropriate charge state ion). Compare that area to the area of the main component in the same run. If the MS response factors are assumed equal, area percentage gives an approximate abundance percentage. For more accuracy, you could run a purified sample of that impurity (if available) to get its response factor. For reporting purposes in QC, one might say &ldquo;Impurity X is present at ~2% relative to main component Y by LC&ndash;HRMS area.&rdquo; Absolute amounts can be calculated if the quant method (A) determined the main component&rsquo;s concentration &ndash; then 2% area of known concentration gives an absolute concentration for the impurity.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Documentation: Use the high mass accuracy in your favor &ndash; list the measured m/z and the calculated formula or assignment for each impurity. For example, &ldquo;Impurity 1: m/z 615.2345^4&ndash;, deconvoluted mass 2460.9 Da, likely missing one thymidine (&ndash;306 Da from main, consistent with loss of one nucleotide).&rdquo; This level of detail will be useful for troubleshooting synthesis or storage stability.<br></span></li></ul><p class="c9"><span class="c2">Internal Standards (Impurity Method): A traditional internal standard is not critical for identification, but you may still add one for reference. For instance, adding a known amount of a standard compound (even something like a standard oligonucleotide or peptide not found in the sample) can act as a retention time marker and general performance check. It&rsquo;s more common to rely on the main peaks themselves as internal reference points (their retention and intensity can indicate if the run was normal). Do ensure the instrument is calibrated for mass accuracy &ndash; use lock-mass during the run or calibrate immediately before. This way you trust the exact masses when identifying unknowns.</span></p><p class="c9"><span class="c2">Handling Amphiphilic Character in Impurity Method: The presence of TEA/HFIP will mitigate extreme polarity issues, but remember that these molecules can form micelles or aggregates especially when concentrated. In the impurity analysis, we often push the sample loading, which could promote aggregation. To counter this, you can incorporate a short pre-run dwell time where the gradient is held at initial conditions for a few minutes after sample injection &ndash; this gives time for any micelles to break up on the column before elution begins (the strong adsorption of the hydrophobes to the column packing will disrupt aggregates). Additionally, if you suspect any carryover of a highly hydrophobic impurity, include a blank injection (or a high-organic flush injection) between heavy samples to ensure it doesn&rsquo;t contaminate the next run.</span></p><hr><p class="c9 c11"><span class="c2"></span></p><p class="c9"><span class="c2">In summary: The two-method approach allows Method A to be simplified for robust quantitation (using formic acid and positive ESI for sensitivity ), and Method B to use more complex conditions (ion pairing, long gradients, negative ESI) to deeply probe the sample for impurities. By leveraging the high resolution of the Q-TOF and appropriate chromatography, the user can confidently quantify each of the six amphiphilic peptide conjugates and thoroughly identify lipid variants, oligonucleotide truncations, or other degradation products present in the mixture. Both methods should employ internal standards and careful sample handling to account for the peculiarities of amphiphilic molecules, ensuring accurate and reliable results.</span></p><p class="c9"><span class="c2">EXAMPLE :</span></p><p class="c9 c11"><span class="c2"></span></p><p class="c9"><span class="c2">Prompt:</span></p><p class="c9"><span class="c2">Suggest parameters for an IND biodistribution study for a small molecule drug.</span></p><p class="c9"><span class="c2">Here&rsquo;s a regulatory-aligned IND biodistribution study outline for a small molecule drug, optimized based on your uploaded regulatory documents and current FDA expectations:</span></p><hr><p class="c9 c11"><span class="c2"></span></p><h3 class="c12" id="h.cu7ljruo137o"><span class="c8">IND Biodistribution Study &mdash; Small Molecule Drug</span></h3><h4 class="c22" id="h.n9m1u5x974ya"><span class="c13">Study Objectives</span></h4><ul class="c4 lst-kix_auto7t5cf4tl-0 start"><li class="c0 li-bullet-0"><span class="c2">To characterize the tissue distribution and clearance of the small molecule following administration.<br></span></li><li class="c0 li-bullet-0"><span class="c2">To determine systemic exposure, target tissue localization, and accumulation profiles.<br></span></li><li class="c0 li-bullet-0"><span class="c2">To support human dose selection and risk assessment.<br></span></li></ul><h4 class="c22" id="h.l17t1yhwk1n4"><span class="c13">Animal Model</span></h4><ul class="c4 lst-kix_mktumxqlukrw-0 start"><li class="c0 li-bullet-0"><span class="c2">Species: Rat (preferred rodent) and/or Non-rodent (e.g., dog, monkey) &nbsp;.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Sex: Both sexes, unless justified.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Age: Young adults (sexually mature, but not aged).<br></span></li></ul><h4 class="c22" id="h.xmp1v83dozp8"><span class="c13">Group Design</span></h4><p class="c9 c11"><span class="c2"></span></p><table class="c25"><tr class="c24"><td class="c5" colspan="1" rowspan="1"><p class="c18"><span class="c13">Group</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c18"><span class="c13">Treatment</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c18"><span class="c13">No. of Animals</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c18"><span class="c13">Purpose</span></p></td></tr><tr class="c24"><td class="c5" colspan="1" rowspan="1"><p class="c9"><span class="c2">Group 1</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c9"><span class="c2">Low Dose</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c9"><span class="c2">3M + 3F per time point</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c9"><span class="c2">Distribution profile</span></p></td></tr><tr class="c24"><td class="c5" colspan="1" rowspan="1"><p class="c9"><span class="c2">Group 2</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c9"><span class="c2">Mid Dose</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c9"><span class="c2">3M + 3F per time point</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c9"><span class="c2">Distribution profile</span></p></td></tr><tr class="c24"><td class="c5" colspan="1" rowspan="1"><p class="c9"><span class="c2">Group 3</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c9"><span class="c2">High Dose (MTD/NOAEL)</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c9"><span class="c2">3M + 3F per time point</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c9"><span class="c2">Worst-case accumulation</span></p></td></tr><tr class="c24"><td class="c5" colspan="1" rowspan="1"><p class="c9"><span class="c2">Group 4</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c9"><span class="c2">Vehicle Control</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c9"><span class="c2">3M + 3F per time point</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c9"><span class="c2">Background distribution</span></p></td></tr></table><p class="c9"><span class="c2">Note: If your molecule is known to accumulate or have a long half-life, additional recovery groups may be included (e.g., Day 14, Day 28 after last dose).</span></p><h4 class="c22" id="h.jcynvpsigs20"><span class="c13">Dose Levels</span></h4><ul class="c4 lst-kix_2najoyx65arn-0 start"><li class="c0 li-bullet-0"><span class="c2">Low: Clinical intended human equivalent dose (HED).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Mid: 10x HED.<br></span></li><li class="c0 li-bullet-0"><span class="c2">High: Maximum Tolerated Dose (MTD) or based on saturation of absorption.<br></span></li></ul><h4 class="c22" id="h.y3k6t72a1eyr"><span class="c13">Route of Administration</span></h4><ul class="c4 lst-kix_fsz4dbursko6-0 start"><li class="c0 li-bullet-0"><span class="c2">Same as intended clinical route (e.g., oral, IV, subcutaneous).<br></span></li></ul><h4 class="c22" id="h.8n8mfv10nx3j"><span class="c13">Dosing Schedule</span></h4><ul class="c4 lst-kix_9ei06drf1sg0-0 start"><li class="c0 li-bullet-0"><span class="c2">Single Dose for Phase 1 enabling studies.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Multiple Dosing (7&ndash;28 days) if expected clinical regimen involves repeated dosing .<br></span></li></ul><h4 class="c22" id="h.w2bhhppxzk5r"><span class="c13">Sampling Time Points</span></h4><ul class="c4 lst-kix_ij4fg4tarntm-0 start"><li class="c0 li-bullet-0"><span class="c2">Acute Phase: 0.5h, 1h, 4h, 8h, 24h post-dose.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Subacute Phase: 2d, 7d, 14d, 28d (if needed).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Terminal sampling at each timepoint.<br></span></li></ul><h4 class="c22" id="h.nx6cmwtcftb4"><span class="c13">Tissues Collected</span></h4><ul class="c4 lst-kix_blb26b155eyp-0 start"><li class="c0 li-bullet-0"><span class="c2">Blood, liver, kidney, lung, brain, heart, spleen, gonads, gastrointestinal tract, skin, bone, muscle, injection site (if applicable).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Optional: Fat, adrenal gland, pancreas, lymph nodes .<br></span></li></ul><h4 class="c22" id="h.acnxgeq8z3kn"><span class="c13">Analytical Method</span></h4><ul class="c4 lst-kix_lcass77ih41v-0 start"><li class="c0 li-bullet-0"><span class="c2">Bioanalytical Assay: Quantitative LC-MS/MS validated per FDA bioanalytical method validation guidelines.<br></span></li><li class="c0 li-bullet-0"><span class="c2">LLOQ must be sensitive enough to detect the drug at expected tissue concentrations .<br></span></li></ul><h4 class="c22" id="h.p1shhij5x62i"><span class="c13">Endpoints</span></h4><ul class="c4 lst-kix_wk2xuqsimxh-0 start"><li class="c0 li-bullet-0"><span class="c2">Quantification of parent drug and relevant metabolites in plasma and tissues.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Pharmacokinetic parameters: Cmax, Tmax, AUC, half-life (t&frac12;), clearance (CL), and volume of distribution (Vd).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Tissue/plasma ratio.<br></span></li></ul><h4 class="c22" id="h.3ms3eiyfccjy"><span class="c13">Reporting Requirements</span></h4><ul class="c4 lst-kix_2ws5k2m4lsl8-0 start"><li class="c0 li-bullet-0"><span class="c2">Mean and individual animal concentrations.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Representative concentration vs. time graphs.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Tissue retention analysis (if drug persists &gt;14 days).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Include method validation summary in the report .<br></span></li></ul><h4 class="c22" id="h.irqzb6jswpzj"><span class="c13">Compliance</span></h4><ul class="c4 lst-kix_k2lnp2ex00id-0 start"><li class="c0 li-bullet-0"><span class="c2">GLP-compliant study (21 CFR Part 58) strongly recommended for IND support .<br></span></li><li class="c0 li-bullet-0"><span class="c2">Biosafety and animal welfare according to 21 CFR 58.90 and IACUC approval.<br></span></li></ul><hr><p class="c9 c11"><span class="c2"></span></p><p class="c9"><span class="c2">Would you also like me to draft a sample schedule (like a timepoint matrix) and full tissue list for you, ready for direct inclusion into your IND protocol?</span></p><p class="c9"><span class="c2">It&rsquo;ll make your next step even faster!</span></p><p class="c9"><span class="c2">Prompt: YES</span></p><p class="c9"><span class="c2">Response:</span></p><p class="c9"><span class="c2">Awesome! Here&rsquo;s a sample timepoint matrix and a full tissue collection list you can directly insert into your IND protocol.</span></p><hr><p class="c9 c11"><span class="c2"></span></p><h1 class="c10" id="h.4f5so4ivct9f"><span class="c17">Sample Dosing and Sampling Schedule (Matrix)</span></h1><p class="c9 c11"><span class="c2"></span></p><table class="c25"><tr class="c24"><td class="c5" colspan="1" rowspan="1"><p class="c18"><span class="c13">Timepoint Post Dose</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c18"><span class="c13">Group</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c18"><span class="c13">Tissues/Organs Collected</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c18"><span class="c13">No. of Animals (M+F)</span></p></td></tr><tr class="c24"><td class="c5" colspan="1" rowspan="1"><p class="c9"><span class="c2">0.5 hours</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c9"><span class="c2">1, 2, 3, 4</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c9"><span class="c2">Full tissue panel</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c9"><span class="c2">3M + 3F</span></p></td></tr><tr class="c24"><td class="c5" colspan="1" rowspan="1"><p class="c9"><span class="c2">1 hour</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c9"><span class="c2">1, 2, 3, 4</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c9"><span class="c2">Full tissue panel</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c9"><span class="c2">3M + 3F</span></p></td></tr><tr class="c24"><td class="c5" colspan="1" rowspan="1"><p class="c9"><span class="c2">4 hours</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c9"><span class="c2">1, 2, 3, 4</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c9"><span class="c2">Full tissue panel</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c9"><span class="c2">3M + 3F</span></p></td></tr><tr class="c24"><td class="c5" colspan="1" rowspan="1"><p class="c9"><span class="c2">8 hours</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c9"><span class="c2">1, 2, 3, 4</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c9"><span class="c2">Full tissue panel</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c9"><span class="c2">3M + 3F</span></p></td></tr><tr class="c24"><td class="c5" colspan="1" rowspan="1"><p class="c9"><span class="c2">24 hours</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c9"><span class="c2">1, 2, 3, 4</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c9"><span class="c2">Full tissue panel</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c9"><span class="c2">3M + 3F</span></p></td></tr><tr class="c24"><td class="c5" colspan="1" rowspan="1"><p class="c9"><span class="c2">7 days</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c9"><span class="c2">1, 2, 3, 4</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c9"><span class="c2">Full tissue panel</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c9"><span class="c2">3M + 3F</span></p></td></tr><tr class="c24"><td class="c5" colspan="1" rowspan="1"><p class="c9"><span class="c2">14 days (optional)</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c9"><span class="c2">1, 2, 3, 4</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c9"><span class="c2">Full tissue panel</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c9"><span class="c2">3M + 3F</span></p></td></tr><tr class="c24"><td class="c5" colspan="1" rowspan="1"><p class="c9"><span class="c2">28 days (optional recovery)</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c9"><span class="c2">1, 2, 3, 4</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c9"><span class="c2">Full tissue panel</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c9"><span class="c2">3M + 3F</span></p></td></tr></table><ul class="c4 lst-kix_eb0mkv6he1ge-0 start"><li class="c0 li-bullet-0"><span class="c2">Optional Recovery Groups can be included if drug accumulation is expected based on PK data.<br></span></li></ul><hr><p class="c9 c11"><span class="c2"></span></p><h1 class="c10" id="h.vl2ju8d5sy4k"><span class="c17">Tissue Collection List</span></h1><p class="c9"><span class="c2">Primary (mandatory for biodistribution)</span></p><ul class="c4 lst-kix_ihv36zxyu5h9-0 start"><li class="c0 li-bullet-0"><span class="c2">Plasma (blood collected into EDTA or heparin tubes)<br></span></li><li class="c0 li-bullet-0"><span class="c2">Liver (left and right lobes separately)<br></span></li><li class="c0 li-bullet-0"><span class="c2">Kidneys (each separately)<br></span></li><li class="c0 li-bullet-0"><span class="c2">Spleen<br></span></li><li class="c0 li-bullet-0"><span class="c2">Heart<br></span></li><li class="c0 li-bullet-0"><span class="c2">Lung<br></span></li><li class="c0 li-bullet-0"><span class="c2">Brain (whole or dissected by regions if relevant)<br></span></li><li class="c0 li-bullet-0"><span class="c2">Gonads (testes or ovaries)<br></span></li><li class="c0 li-bullet-0"><span class="c2">Gastrointestinal tract (stomach, small intestine, large intestine, separately)<br></span></li><li class="c0 li-bullet-0"><span class="c2">Skin (preferably shaved dorsal skin)<br></span></li><li class="c0 li-bullet-0"><span class="c2">Muscle (thigh or pectoral muscle)<br></span></li><li class="c0 li-bullet-0"><span class="c2">Bone (femur or tibia)<br></span></li><li class="c0 li-bullet-0"><span class="c2">Injection site (if non-oral)<br></span></li></ul><p class="c9"><span class="c2">Secondary (recommended depending on mechanism)</span></p><ul class="c4 lst-kix_p73ghpm47z3w-0 start"><li class="c0 li-bullet-0"><span class="c2">Adrenal glands<br></span></li><li class="c0 li-bullet-0"><span class="c2">Pancreas<br></span></li><li class="c0 li-bullet-0"><span class="c2">Lymph nodes (mesenteric or axillary)<br></span></li><li class="c0 li-bullet-0"><span class="c2">Fat (visceral and/or subcutaneous)<br></span></li><li class="c0 li-bullet-0"><span class="c2">Thymus<br></span></li><li class="c0 li-bullet-0"><span class="c2">Urinary bladder<br></span></li><li class="c0 li-bullet-0"><span class="c2">Bone marrow (if feasible)<br></span></li></ul><p class="c9"><span class="c2">Notes:</span></p><ul class="c4 lst-kix_an05xfuo7mar-0 start"><li class="c0 li-bullet-0"><span class="c2">Always freeze tissues immediately (e.g., snap frozen in liquid nitrogen or dry ice) unless processed fresh.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Consider perfusing animals with saline prior to tissue collection for better background clearance (especially brain).<br></span></li></ul><hr><p class="c9 c11"><span class="c2"></span></p><h1 class="c10" id="h.7iwme76lz8hi"><span class="c17">Example Tissue Processing SOP (Short)</span></h1><ul class="c4 lst-kix_qxeaj9fcw57j-0 start"><li class="c0 li-bullet-0"><span class="c2">Euthanasia: By CO&#8322; asphyxiation followed by exsanguination.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Blood: Collected for plasma separation within 30 minutes.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Solid tissues: Weigh individually, homogenize aliquots for bioanalysis.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Archival: Reserve half for histology (if exploratory pathology needed).<br></span></li></ul><hr><p class="c9 c11"><span class="c2"></span></p><p class="c9"><span class="c2">Would you also like me to build a brief wording you can paste into your &ldquo;Methods&rdquo; and &ldquo;Endpoints&rdquo; sections for your formal IND biodistribution study protocol?</span></p><p class="c9"><span class="c2">(That way you&rsquo;d be ready for submission formatting too!)</span></p><p class="c9"><span class="c2">Want me to go ahead and prepare that?</span></p><p class="c9"><span class="c2">Prompt: YES</span></p><p class="c9"><span class="c2">Response:</span></p><p class="c9"><span class="c2">Perfect &mdash; here&rsquo;s ready-to-use &ldquo;Methods&rdquo; and &ldquo;Endpoints&rdquo; text you can paste into your IND biodistribution study protocol:</span></p><hr><p class="c9 c11"><span class="c2"></span></p><h1 class="c10" id="h.warky092ub4l"><span class="c17">Methods &ndash; Biodistribution Study Design</span></h1><p class="c9"><span class="c2">Study Design</span></p><p class="c9"><span class="c2">This nonclinical biodistribution study will be conducted in accordance with Good Laboratory Practice (GLP) regulations (21 CFR Part 58) to characterize the tissue distribution and clearance of [Small Molecule Name] following single and/or repeat-dose administration. The study will be performed in both male and female rats [and/or second species, e.g., dogs], to assess potential sex-based differences. Animals will be allocated into vehicle control and three dose groups (low, mid, and high) with defined timepoints for terminal tissue collection.</span></p><p class="c9"><span class="c2">Dosing</span></p><p class="c9"><span class="c2">Animals will receive a single [or multiple] dose(s) of [Small Molecule Name] via [Route of Administration, e.g., intravenous (IV), oral gavage (PO), subcutaneous (SC)] at dose levels determined based on projected clinical exposures (low), mid-range (10x clinical), and Maximum Tolerated Dose (MTD) levels. Vehicle control animals will receive an equivalent volume of vehicle solution.</span></p><p class="c9"><span class="c2">Sample Collection</span></p><p class="c9"><span class="c2">At pre-specified timepoints (0.5 h, 1 h, 4 h, 8 h, 24 h, 7 d, and optionally 14 d and 28 d post-dose), animals will be euthanized, and biological samples will be collected. Blood will be collected via cardiac puncture for plasma separation. Tissues will be dissected immediately post-mortem, weighed, and processed for quantitative analysis. All samples will be snap-frozen and stored at -80&deg;C pending analysis.</span></p><p class="c9"><span class="c2">Tissues Collected Include:</span></p><ul class="c4 lst-kix_85agg0xrrcik-0 start"><li class="c0 li-bullet-0"><span class="c2">Plasma<br></span></li><li class="c0 li-bullet-0"><span class="c2">Liver<br></span></li><li class="c0 li-bullet-0"><span class="c2">Kidneys<br></span></li><li class="c0 li-bullet-0"><span class="c2">Heart<br></span></li><li class="c0 li-bullet-0"><span class="c2">Lung<br></span></li><li class="c0 li-bullet-0"><span class="c2">Brain<br></span></li><li class="c0 li-bullet-0"><span class="c2">Spleen<br></span></li><li class="c0 li-bullet-0"><span class="c2">Gonads (testes or ovaries)<br></span></li><li class="c0 li-bullet-0"><span class="c2">Gastrointestinal tract (stomach, small intestine, large intestine)<br></span></li><li class="c0 li-bullet-0"><span class="c2">Skin<br></span></li><li class="c0 li-bullet-0"><span class="c2">Muscle<br></span></li><li class="c0 li-bullet-0"><span class="c2">Bone (femur)<br></span></li><li class="c0 li-bullet-0"><span class="c2">Injection site (if applicable)<br></span></li><li class="c0 li-bullet-0"><span class="c2">[Optional: Adrenal glands, pancreas, lymph nodes, fat, urinary bladder]<br></span></li></ul><p class="c9"><span class="c2">Analytical Methodology</span></p><p class="c9"><span class="c2">Concentrations of [Small Molecule Name] in plasma and tissues will be determined using a validated quantitative LC-MS/MS assay following bioanalytical method validation guidance (FDA, May 2018). Key validation parameters including selectivity, sensitivity, accuracy, precision, recovery, and stability will be confirmed.</span></p><hr><p class="c9 c11"><span class="c2"></span></p><h1 class="c10" id="h.r39ylonvsujl"><span class="c17">Endpoints</span></h1><p class="c9"><span class="c2">Primary Endpoints</span></p><ul class="c4 lst-kix_cb1r7mwbu1gi-0 start"><li class="c0 li-bullet-0"><span class="c2">Quantitative tissue concentrations of [Small Molecule Name] at each timepoint.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Pharmacokinetic parameters including Cmax, Tmax, AUC, and tissue half-life (t&frac12;).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Tissue-to-plasma concentration ratios to evaluate distribution extent and tissue accumulation.<br></span></li></ul><p class="c9"><span class="c2">Secondary Endpoints</span></p><ul class="c4 lst-kix_k38n94udk0el-0 start"><li class="c0 li-bullet-0"><span class="c2">Assessment of persistence or clearance of [Small Molecule Name] over time.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Evaluation of potential accumulation in critical tissues (e.g., brain, gonads, heart, kidney).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Supportive toxicokinetic interpretation if integrated with concurrent toxicity studies.<br></span></li></ul><hr><p class="c9 c11"><span class="c2"></span></p><p class="c9"><span class="c2">Regulatory Compliance</span></p><p class="c9"><span class="c2">This study will be conducted to meet current FDA expectations for nonclinical support of Investigational New Drug (IND) applications, including CMC expectations for biodistribution in early clinical development .</span></p><hr><p class="c9 c11"><span class="c2"></span></p><p class="c9"><span class="c2">Would you also like me to suggest optional exploratory endpoints like &ldquo;histopathology in high-accumulation tissues&rdquo; or plasma metabolite identification, if you want to really impress at pre-IND or IND review meetings?</span></p><p class="c9"><span class="c2">(It&rsquo;s optional, but increasingly recommended!)</span></p><p class="c9"><span class="c2">Would you like me to add that bonus option too?</span></p><p class="c9"><span class="c2">Prompt: YES</span></p><p class="c9"><span class="c2">Response: </span></p><p class="c9"><span class="c2">Excellent! Here&rsquo;s an optional exploratory endpoints section you can add to further strengthen your IND biodistribution study. These bonus analyses are not always required, but the FDA likes seeing them, especially for small molecules with:</span></p><ul class="c4 lst-kix_yxgmy3rkq7bq-0 start"><li class="c0 li-bullet-0"><span class="c2">New MoA<br></span></li><li class="c0 li-bullet-0"><span class="c2">Potential off-target toxicity<br></span></li><li class="c0 li-bullet-0"><span class="c2">Unknown metabolite profiles<br></span></li></ul><p class="c9"><span class="c2">Excellent! Here&rsquo;s an optional exploratory endpoints section you can add to further strengthen your IND biodistribution study. These bonus analyses are not always required, but the FDA likes seeing them, especially for small molecules with:</span></p><ul class="c4 lst-kix_8yxwyie669z2-0 start"><li class="c0 li-bullet-0"><span class="c2">New MoA<br></span></li><li class="c0 li-bullet-0"><span class="c2">Potential off-target toxicity<br></span></li><li class="c0 li-bullet-0"><span class="c2">Unknown metabolite profiles<br></span></li></ul><hr><p class="c9 c11"><span class="c2"></span></p><h1 class="c10" id="h.gmmyy96g7c03"><span class="c17">Optional Exploratory Endpoints (Recommended for IND Enhancement)</span></h1><p class="c9"><span class="c2">Histopathological Evaluation</span></p><ul class="c4 lst-kix_d7lufd20rbm5-0 start"><li class="c0 li-bullet-0"><span class="c2">Formalin-fixed tissues from high-accumulation organs (e.g., liver, kidney, brain, heart, gonads) will be collected and processed for histopathological assessment.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Tissues will be embedded, sectioned, and stained with hematoxylin and eosin (H&amp;E).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Microscopic examination will assess tissue architecture and identify any drug-related morphological changes (e.g., degeneration, necrosis, inflammation).<br></span></li></ul><p class="c9"><span class="c2">Plasma and Tissue Metabolite Profiling</span></p><ul class="c4 lst-kix_upyk6it0f1tq-0 start"><li class="c0 li-bullet-0"><span class="c2">Selected plasma and high-exposure tissue samples (e.g., liver, kidney) will be subjected to metabolite identification studies using LC-MS/MS and/or HRMS (high-resolution mass spectrometry).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Metabolite characterization will support understanding of biotransformation pathways.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Comparative profiling between plasma and tissues will be performed to assess whether unique tissue-specific metabolites exist.<br></span></li></ul><p class="c9"><span class="c2">Tissue Autoradiography (If Radiolabeled Compound Available)</span></p><ul class="c4 lst-kix_iprbjac0k1x3-0 start"><li class="c0 li-bullet-0"><span class="c2">In the case of a radiolabeled [Small Molecule Name] (e.g., C-14 label), quantitative whole-body autoradiography (QWBA) studies may be considered to provide detailed spatial distribution data.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Autoradiography can offer visual confirmation of tissue-specific uptake and retention patterns, especially for CNS or reproductive organs.<br></span></li></ul><hr><p class="c9 c11"><span class="c2"></span></p><h1 class="c10" id="h.x5xcw9uum0rh"><span class="c17">How you could integrate these in a sentence into your IND protocol:</span></h1><p class="c15 c26"><span class="c14">&ldquo;Optional exploratory analyses, including limited histopathological evaluation of selected tissues and preliminary metabolite profiling of plasma and tissue extracts, will be performed to provide additional context for tissue distribution findings and potential off-target effects.&rdquo;</span></p><hr><p class="c9 c11"><span class="c2"></span></p><p class="c9"><span class="c2">Why this matters:</span></p><p class="c9"><span class="c2">Including even a mention of these options &mdash; even if results are exploratory &mdash; signals risk awareness and thoroughness, two key qualities FDA reviewers respect at pre-IND and IND submission stages .</span></p><hr><p class="c9 c11"><span class="c2"></span></p><p class="c9"><span class="c2">Next Bonus (if you want it):</span></p><p class="c9"><span class="c2">I can also give you a one-paragraph justification you could insert into your study rationale section explaining why biodistribution is critical even for small molecules &mdash; very useful if reviewers challenge why you&rsquo;re doing it!</span></p><p class="c9"><span class="c2">Would you like me to quickly prepare that too?</span></p><p class="c9"><span class="c2">(Only a few lines but can make your IND much stronger!)</span></p><p class="c9"><span class="c2">Prompt: yes</span></p><p class="c9"><span class="c2">Response: </span></p><p class="c9"><span class="c2">Awesome, here&rsquo;s a ready-to-insert paragraph you can place under your Study Rationale section of your IND protocol:</span></p><hr><p class="c9 c11"><span class="c2"></span></p><h1 class="c10" id="h.l4eizifxbe7m"><span class="c17">Justification for Biodistribution Assessment in Small Molecule Development</span></h1><p class="c15 c26"><span class="c14">Although biodistribution studies are more commonly emphasized for complex biologics and gene therapies, characterization of tissue distribution and clearance is increasingly recognized as critical for small molecule therapeutics, particularly when novel chemical structures, unique mechanisms of action, or anticipated off-target effects are involved. Understanding the biodistribution of [Small Molecule Name] will support assessment of target engagement, potential tissue accumulation, clearance kinetics, and identification of any high-risk organ systems. This information will aid in human risk assessment, inform clinical monitoring strategies, and contribute to overall safety evaluation in alignment with FDA recommendations for early clinical development &nbsp;.</span></p><hr><p class="c9 c11"><span class="c2"></span></p><p class="c9"><span class="c2">Quick Tip:</span></p><p class="c9"><span class="c2">If you want it even stronger, you can slightly modify it depending on your molecule type:</span></p><ul class="c4 lst-kix_xq7wzd2lr8u5-0 start"><li class="c0 li-bullet-0"><span class="c2">If it&rsquo;s CNS-acting: Mention &ldquo;blood-brain barrier penetration.&rdquo;<br></span></li><li class="c0 li-bullet-0"><span class="c2">If it&rsquo;s immunomodulatory: Mention &ldquo;potential immunologic organ sequestration.&rdquo;<br></span></li></ul><p class="c9"><span class="c2">If you tell me a little more about what kind of small molecule it is (e.g., CNS drug, oncology, anti-inflammatory, metabolic, etc.), I can even tailor this paragraph more specifically if you want!</span></p><p class="c9"><span class="c2">Would you like me to do a quick tailoring?</span></p><p class="c9"><span class="c2">(Just tell me the therapeutic area or MoA &mdash; 1 line is enough!)</span></p><p class="c9"><span class="c2">Prompt: anti-inflammatory </span></p><p class="c9"><span class="c2">Response:</span></p><p class="c9"><span class="c2">Perfect &mdash; since your small molecule is anti-inflammatory, I&rsquo;ll quickly tailor that paragraph to better reflect the concerns regulators usually have for that class (e.g., immune organs, secondary effects on liver, kidney, bone marrow).</span></p><p class="c9"><span class="c2">Here&rsquo;s the optimized version you can paste into your Study Rationale section:</span></p><hr><p class="c9 c11"><span class="c2"></span></p><h1 class="c10" id="h.5nf20smhsdc0"><span class="c17">Justification for Biodistribution Assessment in Small Molecule Anti-Inflammatory Drug Development</span></h1><p class="c15 c26"><span class="c14">Although biodistribution studies are more commonly emphasized for biologics and gene therapies, characterization of tissue distribution and clearance is increasingly critical for small molecule therapeutics, particularly for agents with immunomodulatory or anti-inflammatory mechanisms of action. Understanding the biodistribution of [Small Molecule Name] will support assessment of potential tissue accumulation, systemic immunologic modulation, and identification of any organ systems at risk, such as the liver, spleen, lymphoid tissues, and bone marrow. These data will inform human risk assessment, guide clinical monitoring parameters, and contribute to the overall safety evaluation strategy, consistent with current FDA recommendations for early clinical development &nbsp;.</span></p><hr><p class="c9 c11"><span class="c2"></span></p><p class="c9"><span class="c2">Bonus Tip for Anti-Inflammatories:</span></p><p class="c9"><span class="c2">When analyzing your tissue concentration results, regulators will especially look for:</span></p><ul class="c4 lst-kix_sp4bx5if0jbk-0 start"><li class="c0 li-bullet-0"><span class="c2">High concentrations in spleen, lymph nodes (good or bad depending on your MoA)<br></span></li><li class="c0 li-bullet-0"><span class="c2">Unexpected accumulation in liver or kidney (toxicity risk)<br></span></li><li class="c0 li-bullet-0"><span class="c2">Presence in bone marrow (hematologic toxicity risk)<br></span></li></ul><p class="c9"><span class="c2">We can plan the data tables and graphs to highlight this if you want to be extra polished for your submission.</span></p><hr><p class="c9 c11"><span class="c2"></span></p><p class="c9"><span class="c2">Would you also like a short sample table layout suggestion for your biodistribution results (like how to organize tissue Cmax, Tmax, AUC nicely)?</span></p><p class="c9"><span class="c2">It&rsquo;ll make your final report easier to assemble too!</span></p><p class="c9"><span class="c2">Want me to prep that quickly?</span></p><p class="c9"><span class="c2">Prompt: yes</span></p><p class="c9"><span class="c2">Response:</span></p><p class="c9"><span class="c2">Awesome &mdash; here&rsquo;s a clean, regulatory-style sample table you can use for your biodistribution results in your IND submission or study report.</span></p><hr><p class="c9 c11"><span class="c2"></span></p><h1 class="c10" id="h.qu7do7etuml2"><span class="c17">Sample Table Layout: Biodistribution of [Small Molecule Name] in Rats</span></h1><p class="c9 c11"><span class="c2"></span></p><table class="c25"><tr class="c24"><td class="c1" colspan="1" rowspan="1"><p class="c18"><span class="c13">Tissue</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c18"><span class="c13">Cmax (ng/g)</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c18"><span class="c13">Tmax (h)</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c18"><span class="c13">AUC&#8320;&ndash;last (ng&middot;h/g)</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c18"><span class="c13">t&frac12; (h)</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c18"><span class="c13">Notes (e.g., accumulation, unusual findings)</span></p></td></tr><tr class="c24"><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">Plasma</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">[X]</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">[X]</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">[X]</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">[X]</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">Baseline reference</span></p></td></tr><tr class="c24"><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">Liver</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">[X]</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">[X]</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">[X]</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">[X]</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">High exposure; clearance slower than plasma</span></p></td></tr><tr class="c24"><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">Kidney</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">[X]</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">[X]</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">[X]</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">[X]</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">Moderate accumulation; consistent with excretion</span></p></td></tr><tr class="c24"><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">Spleen</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">[X]</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">[X]</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">[X]</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">[X]</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">Notable retention (monitor in clinic)</span></p></td></tr><tr class="c24"><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">Lung</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">[X]</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">[X]</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">[X]</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">[X]</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">Rapid uptake; rapid clearance</span></p></td></tr><tr class="c24"><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">Brain</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">[X]</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">[X]</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">[X]</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">[X]</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">No significant penetration (optional)</span></p></td></tr><tr class="c24"><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">Heart</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">[X]</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">[X]</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">[X]</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">[X]</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">Low distribution</span></p></td></tr><tr class="c24"><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">Gonads</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">[X]</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">[X]</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">[X]</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">[X]</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">Evaluate for reproductive risk (if applicable)</span></p></td></tr><tr class="c24"><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">Muscle</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">[X]</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">[X]</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">[X]</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">[X]</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">Baseline distribution</span></p></td></tr><tr class="c24"><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">Skin</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">[X]</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">[X]</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">[X]</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">[X]</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">Minimal exposure</span></p></td></tr><tr class="c24"><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">Bone Marrow (if collected)</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">[X]</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">[X]</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">[X]</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">[X]</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">Monitor for hematologic toxicity risk</span></p></td></tr><tr class="c24"><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">GI Tract</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">[X]</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">[X]</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">[X]</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">[X]</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c9"><span class="c2">Transit or direct exposure (especially oral drug)</span></p></td></tr></table><hr><p class="c9 c11"><span class="c2"></span></p><h1 class="c10" id="h.9k618tej37c"><span class="c17">How to Use This Table</span></h1><ul class="c4 lst-kix_eunro7k2bn6e-0 start"><li class="c0 li-bullet-0"><span class="c2">X: Fill with mean values from your LC-MS/MS quantification per tissue at each timepoint.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Notes column: Super useful &mdash; reviewers like quick comments, e.g., &ldquo;high liver exposure, reversible,&rdquo; or &ldquo;no CNS penetration observed.&rdquo;<br></span></li><li class="c0 li-bullet-0"><span class="c2">Highlight spleen, lymph node, bone marrow findings because anti-inflammatory drugs often modulate immune cells there.<br></span></li></ul><hr><p class="c9 c11"><span class="c2"></span></p><h1 class="c10" id="h.hcmiq2jhtxs"><span class="c17">Extra Tip:</span></h1><p class="c9"><span class="c2">You can also summarize Tissue/Plasma Ratios for Cmax and AUC&#8320;&ndash;last separately in another small table, if your molecule shows clear tissue partitioning &mdash; FDA likes that for molecules acting on immune systems.</span></p><hr><p class="c9 c11"><span class="c2"></span></p><p class="c9"><span class="c2">Would you also like me to show you an example of a very short biodistribution summary paragraph (like what you would write in your IND &ldquo;Summary of Results&rdquo; section)?</span></p><p class="c9"><span class="c2">(One strong paragraph that makes it easy for FDA reviewers to see the story immediately.)</span></p><p class="c9"><span class="c2">Want me to prep that next?</span></p><p class="c9"><span class="c2">Prompt: YES</span></p><p class="c9"><span class="c2">Response:</span></p><p class="c9"><span class="c2">Awesome &mdash; here&rsquo;s a concise biodistribution summary paragraph you can paste into your IND in the Nonclinical Summary or Biodistribution Results section:</span></p><hr><p class="c9 c11"><span class="c2"></span></p><h1 class="c10" id="h.4qwegtjy9jg9"><span class="c17">Sample Biodistribution Summary Paragraph</span></h1><p class="c15 c26"><span class="c14">Following administration of [Small Molecule Name], tissue distribution was rapid and consistent with systemic exposure observed in plasma. Highest concentrations were noted in the liver, kidney, and spleen, reflecting anticipated clearance and immune organ uptake for an anti-inflammatory agent. Minimal accumulation was observed in the brain, indicating limited blood-brain barrier penetration. Concentrations in gonads and bone marrow were low and transient. Tissue-to-plasma ratios at peak exposure ranged from [X] to [X], with no unexpected accumulation patterns. These findings support the safety and systemic exposure profile of [Small Molecule Name] and will inform clinical dose selection and monitoring strategies for first-in-human studies.</span></p><hr><p class="c9 c11"><span class="c2"></span></p><h1 class="c10" id="h.jxg9nv4pjyai"><span class="c17">Bonus: Quick Tailoring Tips</span></h1><ul class="c4 lst-kix_arstwhb1c9gy-0 start"><li class="c0 li-bullet-0"><span>If your brain levels are high, just say:<br><br> </span><span class="c14">&ldquo;Moderate CNS penetration was observed, consistent with potential secondary CNS pharmacology.&rdquo;<br></span></li><li class="c0 li-bullet-0"><span>If you see high spleen or lymph node exposure (good for immunomodulation drugs):<br><br> </span><span class="c14">&ldquo;Preferential uptake in immune tissues (e.g., spleen, lymph nodes) was consistent with the anticipated mechanism of action.&rdquo;<br></span></li><li class="c0 li-bullet-0"><span>If bone marrow exposure is notable:<br><br> </span><span class="c14">&ldquo;Low-level exposure to bone marrow observed; monitored for potential hematologic effects.&rdquo;<br></span></li></ul><hr><p class="c9 c11"><span class="c2"></span></p><h1 class="c10" id="h.rtfph021uog"><span class="c17">In Short:</span></h1><p class="c9"><span class="c2">This paragraph will help preemptively answer two key FDA reviewer questions:</span></p><ul class="c4 lst-kix_sr3soljrf7bf-0 start"><li class="c0 li-bullet-0"><span class="c2">&ldquo;Where is the drug going?&rdquo;<br></span></li><li class="c0 li-bullet-0"><span class="c2">&ldquo;Is there any concerning tissue accumulation?&rdquo;<br></span></li></ul><p class="c9"><span class="c2">And because it&rsquo;s written tightly, they won&rsquo;t have to dig through your full report unless they want details &mdash; that makes reviewers&rsquo; lives easier, which they appreciate.</span></p><hr><p class="c9 c11"><span class="c2"></span></p><p class="c9"><span class="c2">Would you also like a bonus visual tip:</span></p><p class="c9"><span class="c2">A suggestion for a simple graph style (like a Tissue Concentration vs. Time curve) you can insert into your report to make it even more IND-polished?</span></p><p class="c9"><span class="c2">(I can quickly sketch that out for you too if you want!)</span></p><p class="c9"><span class="c2">Want me to?</span></p><p class="c9"><span class="c2">Prompt: NO</span></p><p class="c9 c11"><span class="c2"></span></p><p class="c9 c11"><span class="c2"></span></p><p class="c9"><span class="c2">EXAMPLE: </span></p><p class="c9"><span class="c2">Prompt: Design an immunogenicity protocol for a biologic, aligned with GLP and FDA standards.</span></p><p class="c9"><span class="c2">Here&rsquo;s a GLP-compliant immunogenicity protocol for a biologic, aligned with FDA standards, based on your documents and regulatory guidelines :</span></p><hr><p class="c9 c11"><span class="c2"></span></p><h1 class="c10" id="h.74bryrru1og2"><span class="c17">Preclinical Immunogenicity Study Protocol for Biologic XYZ</span></h1><h2 class="c19" id="h.5k0hpcuudoss"><span class="c20">1. Study Title</span></h2><p class="c9"><span class="c2">Assessment of Immunogenicity of Biologic XYZ in [Rodent/Nonhuman Primate] Model</span></p><hr><p class="c9 c11"><span class="c2"></span></p><h2 class="c19" id="h.z4ydubvz2i9y"><span class="c20">2. Study Objectives</span></h2><ul class="c4 lst-kix_9fa76bs6q7z7-0 start"><li class="c0 li-bullet-0"><span class="c2">Evaluate the potential of Biologic XYZ to induce an immune response (ADA &ndash; anti-drug antibodies).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Characterize the nature (binding vs neutralizing) and kinetics (onset, duration) of antibody formation.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Assess potential impacts of immunogenicity on pharmacokinetics, pharmacodynamics, and toxicity profiles .<br></span></li></ul><hr><p class="c9 c11"><span class="c2"></span></p><h2 class="c19" id="h.wqerjeiuruh9"><span class="c20">3. Regulatory Standards</span></h2><ul class="c4 lst-kix_ss89m2334v3w-0 start"><li class="c0 li-bullet-0"><span class="c2">GLP compliance according to 21 CFR Part 58 .<br></span></li><li class="c0 li-bullet-0"><span class="c2">Designed for support of IND submission (21 CFR Part 312, per FDA Guidance ).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Follows FDA and EMA recommendations for immunogenicity assessment .<br></span></li></ul><hr><p class="c9 c11"><span class="c2"></span></p><h2 class="c19" id="h.elbvfyap68ib"><span class="c20">4. Study Design</span></h2><p class="c9 c11"><span class="c2"></span></p><table class="c25"><tr class="c24"><td class="c21" colspan="1" rowspan="1"><p class="c18"><span class="c13">Parameter</span></p></td><td class="c21" colspan="1" rowspan="1"><p class="c18"><span class="c13">Details</span></p></td></tr><tr class="c24"><td class="c21" colspan="1" rowspan="1"><p class="c9"><span class="c2">Species</span></p></td><td class="c21" colspan="1" rowspan="1"><p class="c9"><span class="c2">[Rodents (e.g., CD-1 mice, Wistar rats) / Nonhuman primates (e.g., cynomolgus monkeys)]</span></p></td></tr><tr class="c24"><td class="c21" colspan="1" rowspan="1"><p class="c9"><span class="c2">Group Size</span></p></td><td class="c21" colspan="1" rowspan="1"><p class="c9"><span class="c2">10 animals per group minimum (5 male, 5 female)</span></p></td></tr><tr class="c24"><td class="c21" colspan="1" rowspan="1"><p class="c9"><span class="c2">Dosing Regimen</span></p></td><td class="c21" colspan="1" rowspan="1"><p class="c9"><span class="c2">[Single or multiple dosing, intravenous (IV) or subcutaneous (SC) route]</span></p></td></tr><tr class="c24"><td class="c21" colspan="1" rowspan="1"><p class="c9"><span class="c2">Dose Levels</span></p></td><td class="c21" colspan="1" rowspan="1"><p class="c9"><span class="c2">Low, Mid, High dose + Vehicle control (per NOAEL/tox study data)</span></p></td></tr><tr class="c24"><td class="c21" colspan="1" rowspan="1"><p class="c9"><span class="c2">Sampling Timepoints</span></p></td><td class="c21" colspan="1" rowspan="1"><p class="c9"><span class="c2">Pre-dose, and multiple post-dose points (e.g., Day 7, 14, 28, 56, etc.)</span></p></td></tr><tr class="c24"><td class="c21" colspan="1" rowspan="1"><p class="c9"><span class="c2">Duration</span></p></td><td class="c21" colspan="1" rowspan="1"><p class="c9"><span class="c2">At least 12 weeks, or aligned with toxicology/pharmacokinetic study duration</span></p></td></tr></table><p class="c9"><span class="c2">Here&rsquo;s a GLP-compliant immunogenicity protocol for a biologic, aligned with FDA standards, based on your documents and regulatory guidelines :</span></p><hr><p class="c9 c11"><span class="c2"></span></p><h1 class="c10" id="h.opd289bsx9vh"><span class="c17">Preclinical Immunogenicity Study Protocol for Biologic XYZ</span></h1><h2 class="c19" id="h.5t2u90gxlhe4"><span class="c20">1. Study Title</span></h2><p class="c9"><span class="c2">Assessment of Immunogenicity of Biologic XYZ in [Rodent/Nonhuman Primate] Model</span></p><hr><p class="c9 c11"><span class="c2"></span></p><h2 class="c19" id="h.ppmddciizhxu"><span class="c20">2. Study Objectives</span></h2><ul class="c4 lst-kix_tut3ual308bf-0 start"><li class="c0 li-bullet-0"><span class="c2">Evaluate the potential of Biologic XYZ to induce an immune response (ADA &ndash; anti-drug antibodies).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Characterize the nature (binding vs neutralizing) and kinetics (onset, duration) of antibody formation.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Assess potential impacts of immunogenicity on pharmacokinetics, pharmacodynamics, and toxicity profiles .<br></span></li></ul><hr><p class="c9 c11"><span class="c2"></span></p><h2 class="c19" id="h.ar1msof6q357"><span class="c20">3. Regulatory Standards</span></h2><ul class="c4 lst-kix_skp7x04r24av-0 start"><li class="c0 li-bullet-0"><span class="c2">GLP compliance according to 21 CFR Part 58 .<br></span></li><li class="c0 li-bullet-0"><span class="c2">Designed for support of IND submission (21 CFR Part 312, per FDA Guidance ).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Follows FDA and EMA recommendations for immunogenicity assessment .<br></span></li></ul><hr><p class="c9 c11"><span class="c2"></span></p><h2 class="c19" id="h.ptdad8edm9yc"><span class="c20">4. Study Design</span></h2><table class="c25"><tr class="c24"><td class="c21" colspan="1" rowspan="1"><p class="c18"><span class="c13">Parameter</span></p></td><td class="c21" colspan="1" rowspan="1"><p class="c18"><span class="c13">Details</span></p></td></tr><tr class="c24"><td class="c21" colspan="1" rowspan="1"><p class="c9"><span class="c2">Species</span></p></td><td class="c21" colspan="1" rowspan="1"><p class="c9"><span class="c2">[Rodents (e.g., CD-1 mice, Wistar rats) / Nonhuman primates (e.g., cynomolgus monkeys)]</span></p></td></tr><tr class="c24"><td class="c21" colspan="1" rowspan="1"><p class="c9"><span class="c2">Group Size</span></p></td><td class="c21" colspan="1" rowspan="1"><p class="c9"><span class="c2">10 animals per group minimum (5 male, 5 female)</span></p></td></tr><tr class="c24"><td class="c21" colspan="1" rowspan="1"><p class="c9"><span class="c2">Dosing Regimen</span></p></td><td class="c21" colspan="1" rowspan="1"><p class="c9"><span class="c2">[Single or multiple dosing, intravenous (IV) or subcutaneous (SC) route]</span></p></td></tr><tr class="c24"><td class="c21" colspan="1" rowspan="1"><p class="c9"><span class="c2">Dose Levels</span></p></td><td class="c21" colspan="1" rowspan="1"><p class="c9"><span class="c2">Low, Mid, High dose + Vehicle control (per NOAEL/tox study data)</span></p></td></tr><tr class="c24"><td class="c21" colspan="1" rowspan="1"><p class="c9"><span class="c2">Sampling Timepoints</span></p></td><td class="c21" colspan="1" rowspan="1"><p class="c9"><span class="c2">Pre-dose, and multiple post-dose points (e.g., Day 7, 14, 28, 56, etc.)</span></p></td></tr><tr class="c24"><td class="c21" colspan="1" rowspan="1"><p class="c9"><span class="c2">Duration</span></p></td><td class="c21" colspan="1" rowspan="1"><p class="c9"><span class="c2">At least 12 weeks, or aligned with toxicology/pharmacokinetic study duration</span></p></td></tr></table><hr><p class="c9 c11"><span class="c2"></span></p><h2 class="c19" id="h.9x70hm3q9aeo"><span class="c20">5. Dose Formulation</span></h2><ul class="c4 lst-kix_ehpc0o1sivs1-0 start"><li class="c0 li-bullet-0"><span class="c2">Biologic XYZ manufactured under GMP-like conditions .<br></span></li><li class="c0 li-bullet-0"><span class="c2">Formulation stability, sterility, and concentration verified pre-administration .<br></span></li></ul><hr><p class="c9 c11"><span class="c2"></span></p><h2 class="c19" id="h.2n5f09qp4xs4"><span class="c20">6. Immunogenicity Assays</span></h2><p class="c9"><span class="c2">Sample Type:</span></p><ul class="c4 lst-kix_cwtqd0lozkzu-0 start"><li class="c0 li-bullet-0"><span class="c2">Blood/serum samples collected at specified intervals.<br></span></li></ul><p class="c9"><span class="c2">Assays:</span></p><ul class="c4 lst-kix_x3csall36p7p-0 start"><li class="c0 li-bullet-0"><span class="c2">Screening Assay: ELISA/Bridging immunoassay for detecting binding ADA.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Confirmatory Assay: Competitive inhibition assay.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Titer Determination: Serial dilution of positive samples.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Neutralizing Antibody Assay (NAb): Cell-based bioassay (if ADA detected).<br></span></li></ul><p class="c9"><span class="c2">Validation:</span></p><ul class="c4 lst-kix_umwvyipzf505-0 start"><li class="c0 li-bullet-0"><span class="c2">Methods validated per FDA recommendations on bioanalytical methods .<br></span></li><li class="c0 li-bullet-0"><span class="c2">Sensitivity, specificity, reproducibility, matrix effects assessed.<br></span></li></ul><hr><p class="c9 c11"><span class="c2"></span></p><h2 class="c19" id="h.26gyojqwxq0w"><span class="c20">7. Animal Handling and Sample Collection</span></h2><ul class="c4 lst-kix_1lds0wnd5zeb-0 start"><li class="c0 li-bullet-0"><span class="c2">Performed under GLP, with ethical review board (IACUC) approval.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Serial blood collection using minimal invasive methods.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Proper sedation/anesthesia when needed .<br></span></li><li class="c0 li-bullet-0"><span class="c2">Collection into serum separator tubes, processed promptly and frozen at -80&deg;C.<br></span></li></ul><hr><p class="c9 c11"><span class="c2"></span></p><h2 class="c19" id="h.ap1eugfm9d2j"><span class="c20">8. Data Analysis</span></h2><ul class="c4 lst-kix_q432akg4led-0 start"><li class="c0 li-bullet-0"><span class="c2">ADA Incidence and Prevalence (% ADA positive animals per group).<br></span></li><li class="c0 li-bullet-0"><span class="c2">ADA Titers (geometric mean titer, range).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Kinetics (onset, persistence, and clearance of antibodies).<br></span></li><li class="c0 li-bullet-0"><span class="c2">Correlation Analysis: ADA vs drug exposure, ADA vs clinical observations, ADA vs toxicity findings.<br></span></li></ul><hr><p class="c9 c11"><span class="c2"></span></p><h2 class="c19" id="h.qxjmu69salo"><span class="c20">9. Reporting</span></h2><ul class="c4 lst-kix_lsygr38j1y0-0 start"><li class="c0 li-bullet-0"><span class="c2">Full reporting per FDA guidance for IND support .<br></span></li><li class="c0 li-bullet-0"><span class="c2">Tabulated ADA incidence, titer levels, NAb positivity.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Statistical comparison between dose groups and control.<br></span></li><li class="c0 li-bullet-0"><span class="c2">Integration into toxicology/pharmacokinetics reports if needed.<br></span></li></ul><hr><p class="c9 c11"><span class="c2"></span></p><h2 class="c19" id="h.93n17fujl9lv"><span class="c20">10. Quality Assurance</span></h2><ul class="c4 lst-kix_6vg7rekv2ht5-0 start"><li class="c0 li-bullet-0"><span class="c2">Independent QA unit monitoring protocol compliance (GLP requirement).<br></span></li><li class="c0 li-bullet-0"><span class="c2">All deviations recorded and justified.<br></span></li></ul><hr><p class="c9 c11"><span class="c2"></span></p><h2 class="c19" id="h.flz2hd8yx9xx"><span class="c20">11. Ethical Considerations</span></h2><ul class="c4 lst-kix_8wqgtnyy1nox-0 start"><li class="c0 li-bullet-0"><span class="c2">Study conducted in accordance with the Guide for the Care and Use of Laboratory Animals and 21 CFR Part 58 .<br></span></li><li class="c0 li-bullet-0"><span class="c2">Humane endpoints predefined.<br></span></li></ul><hr><p class="c9 c11"><span class="c2"></span></p><p class="c9"><span class="c2">Would you also like me to generate:</span></p><ul class="c4 lst-kix_pb595s4qvcov-0 start"><li class="c0 li-bullet-0"><span class="c2">a timeline chart (sampling schedule),<br></span></li><li class="c0 li-bullet-0"><span class="c2">a flow diagram (overall study workflow),<br></span></li><li class="c0 li-bullet-0"><span class="c2">or a sample template for ADA assay validation?<br></span></li></ul><p class="c9"><span class="c2">Would be happy to prepare any of those next if you want!</span></p><p class="c9 c11"><span class="c2"></span></p></body></html>